sql/tests: TestRandomSyntaxFunctions failed

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9182780?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9182780?buildTab=artifacts#/) on release-23.1 @ [80c4895c566a7eaa6f16c4098980509dd3795ad7](https://github.com/cockroachdb/cockroach/commits/80c4895c566a7eaa6f16c4098980509dd3795ad7):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 14744262 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:828: 45s of 5m0s: 184539 executions, 114327 successful
    rsg_test.go:828: 50s of 5m0s: 203901 executions, 126321 successful
    rsg_test.go:828: 55s of 5m0s: 224124 executions, 138962 successful
    rsg_test.go:828: 1m0s of 5m0s: 244942 executions, 151887 successful
    rsg_test.go:828: 1m5s of 5m0s: 264489 executions, 164027 successful
    rsg_test.go:828: 1m10s of 5m0s: 286231 executions, 177561 successful
    rsg_test.go:828: 1m15s of 5m0s: 306708 executions, 190169 successful
    rsg_test.go:828: 1m20s of 5m0s: 328477 executions, 203704 successful
    rsg_test.go:828: 1m25s of 5m0s: 350914 executions, 217589 successful
    rsg_test.go:828: 1m30s of 5m0s: 369611 executions, 229264 successful
    rsg_test.go:828: 1m35s of 5m0s: 391949 executions, 243109 successful
    rsg_test.go:828: 1m40s of 5m0s: 411239 executions, 255088 successful
    rsg_test.go:828: 1m45s of 5m0s: 431015 executions, 267258 successful
    rsg_test.go:828: 1m50s of 5m0s: 453401 executions, 281275 successful
    rsg_test.go:828: 1m55s of 5m0s: 470645 executions, 291943 successful
    rsg_test.go:828: 2m0s of 5m0s: 490494 executions, 304159 successful
    rsg_test.go:828: 2m5s of 5m0s: 510413 executions, 316515 successful
    rsg_test.go:828: 2m10s of 5m0s: 532347 executions, 329995 successful
    rsg_test.go:828: 2m15s of 5m0s: 551421 executions, 341800 successful
    rsg_test.go:828: 2m20s of 5m0s: 574984 executions, 356342 successful
    rsg_test.go:828: 2m25s of 5m0s: 596910 executions, 370015 successful
    rsg_test.go:828: 2m30s of 5m0s: 617486 executions, 382749 successful
    rsg_test.go:828: 2m35s of 5m0s: 640619 executions, 397074 successful
    rsg_test.go:828: 2m40s of 5m0s: 664202 executions, 411572 successful
    rsg_test.go:828: 2m45s of 5m0s: 682912 executions, 423226 successful
    rsg_test.go:828: 2m50s of 5m0s: 704889 executions, 436710 successful
    rsg_test.go:828: 2m55s of 5m0s: 726256 executions, 449956 successful
    rsg_test.go:828: 3m0s of 5m0s: 745851 executions, 462135 successful
    rsg_test.go:828: 3m5s of 5m0s: 765690 executions, 474382 successful
    rsg_test.go:828: 3m10s of 5m0s: 788136 executions, 488352 successful
    rsg_test.go:828: 3m15s of 5m0s: 811683 executions, 502956 successful
    rsg_test.go:828: 3m20s of 5m0s: 833731 executions, 516672 successful
    rsg_test.go:828: 3m25s of 5m0s: 854843 executions, 529714 successful
    rsg_test.go:828: 3m30s of 5m0s: 875625 executions, 542602 successful
    rsg_test.go:828: 3m35s of 5m0s: 896712 executions, 555671 successful
    rsg_test.go:828: 3m40s of 5m0s: 920582 executions, 570273 successful
    rsg_test.go:828: 3m45s of 5m0s: 944048 executions, 584776 successful
    rsg_test.go:828: 3m50s of 5m0s: 964197 executions, 597197 successful
    rsg_test.go:828: 3m55s of 5m0s: 988363 executions, 612375 successful
    rsg_test.go:828: 4m0s of 5m0s: 1008301 executions, 624692 successful
    rsg_test.go:828: 4m5s of 5m0s: 1028220 executions, 637050 successful
    rsg_test.go:828: 4m10s of 5m0s: 1049754 executions, 650421 successful
    rsg_test.go:828: 4m15s of 5m0s: 1072363 executions, 664443 successful
    rsg_test.go:828: 4m20s of 5m0s: 1094347 executions, 678049 successful
    rsg_test.go:828: 4m25s of 5m0s: 1116985 executions, 692049 successful
    rsg_test.go:828: 4m30s of 5m0s: 1138611 executions, 705447 successful
    rsg_test.go:828: 4m35s of 5m0s: 1162669 executions, 720398 successful
    rsg_test.go:828: 4m40s of 5m0s: 1183353 executions, 733226 successful
    rsg_test.go:828: 4m45s of 5m0s: 1205810 executions, 747037 successful
    rsg_test.go:828: 4m50s of 5m0s: 1227922 executions, 760707 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
/cc @cockroachdb/sql-sessions
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-25738

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9225824?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9225824?buildTab=artifacts#/) on release-23.1 @ [52e55d2ef172b7cfec14e8a0a954f8864b2be779](https://github.com/cockroachdb/cockroach/commits/52e55d2ef172b7cfec14e8a0a954f8864b2be779):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 12922748 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:828: 4m45s of 5m0s: 1101969 executions, 683528 successful
    rsg_test.go:855: Crash detected: server panic: pq: internal error: interface conversion: tree.FnOverload is func(*eval.Context, tree.Datums) (tree.Datum, error), not func(context.Context, *eval.Context, tree.Datums) (tree.Datum, error)
        SELECT phraseto_tsquery(e'\x14l%*|':::STRING) ;
        
        Stack trace:
    rsg_test.go:855: Crash detected: server panic: pq: internal error: interface conversion: tree.FnOverload is func(*eval.Context, tree.Datums) (tree.Datum, error), not func(context.Context, *eval.Context, tree.Datums) (tree.Datum, error)
        SELECT phraseto_tsquery(e'W$$\n':::STRING) ;
        
        Stack trace:
    rsg_test.go:855: Crash detected: server panic: pq: internal error: interface conversion: tree.FnOverload is func(*eval.Context, tree.Datums) (tree.Datum, error), not func(context.Context, *eval.Context, tree.Datums) (tree.Datum, error)
        SELECT phraseto_tsquery('"':::STRING) ;
        
        Stack trace:
    rsg_test.go:855: Crash detected: server panic: pq: internal error: interface conversion: tree.FnOverload is func(*eval.Context, tree.Datums) (tree.Datum, error), not func(context.Context, *eval.Context, tree.Datums) (tree.Datum, error)
        SELECT phraseto_tsquery(e'\x1e\x01jz':::STRING) ;
        
        Stack trace:
    rsg_test.go:855: Crash detected: server panic: pq: internal error: interface conversion: tree.FnOverload is func(*eval.Context, tree.Datums) (tree.Datum, error), not func(context.Context, *eval.Context, tree.Datums) (tree.Datum, error)
        SELECT phraseto_tsquery('^':::STRING) ;
        
        Stack trace:
    rsg_test.go:855: Crash detected: server panic: pq: internal error: interface conversion: tree.FnOverload is func(*eval.Context, tree.Datums) (tree.Datum, error), not func(context.Context, *eval.Context, tree.Datums) (tree.Datum, error)
        SELECT phraseto_tsquery(e'e\x02W\'':::STRING) ;
        
        Stack trace:
    rsg_test.go:855: Crash detected: server panic: pq: internal error: interface conversion: tree.FnOverload is func(*eval.Context, tree.Datums) (tree.Datum, error), not func(context.Context, *eval.Context, tree.Datums) (tree.Datum, error)
        SELECT phraseto_tsquery(e'*\f$|\x1c':::STRING) ;
        
        Stack trace:
    rsg_test.go:855: Crash detected: server panic: pq: internal error: interface conversion: tree.FnOverload is func(*eval.Context, tree.Datums) (tree.Datum, error), not func(context.Context, *eval.Context, tree.Datums) (tree.Datum, error)
        SELECT phraseto_tsquery(e'cWT|\x06\x12#':::STRING) ;
        
        Stack trace:
    rsg_test.go:828: 4m50s of 5m0s: 1121294 executions, 695496 successful
    rsg_test.go:855: Crash detected: server panic: pq: internal error: interface conversion: tree.FnOverload is func(*eval.Context, tree.Datums) (tree.Datum, error), not func(context.Context, *eval.Context, tree.Datums) (tree.Datum, error)
        SELECT phraseto_tsquery(e'\x0e0wk\x03\x1d/{':::STRING) ;
        
        Stack trace:
    rsg_test.go:855: Crash detected: server panic: pq: internal error: interface conversion: tree.FnOverload is func(*eval.Context, tree.Datums) (tree.Datum, error), not func(context.Context, *eval.Context, tree.Datums) (tree.Datum, error)
        SELECT phraseto_tsquery('F:H-a':::STRING) ;
        
        Stack trace:
    rsg_test.go:855: Crash detected: server panic: pq: internal error: interface conversion: tree.FnOverload is func(*eval.Context, tree.Datums) (tree.Datum, error), not func(context.Context, *eval.Context, tree.Datums) (tree.Datum, error)
        SELECT phraseto_tsquery(e',\x17[N\x0e|Vcc':::STRING) ;
        
        Stack trace:
    rsg_test.go:855: Crash detected: server panic: pq: internal error: interface conversion: tree.FnOverload is func(*eval.Context, tree.Datums) (tree.Datum, error), not func(context.Context, *eval.Context, tree.Datums) (tree.Datum, error)
        SELECT phraseto_tsquery(e'|b\tg':::STRING) ;
        
        Stack trace:
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9244191?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9244191?buildTab=artifacts#/) on release-23.1 @ [f351747ed97862fc037717cadec23f18073fb6be](https://github.com/cockroachdb/cockroach/commits/f351747ed97862fc037717cadec23f18073fb6be):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 13101667 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:828: 45s of 5m0s: 154890 executions, 96052 successful
    rsg_test.go:828: 50s of 5m0s: 173367 executions, 107542 successful
    rsg_test.go:828: 55s of 5m0s: 190273 executions, 118007 successful
    rsg_test.go:828: 1m0s of 5m0s: 210249 executions, 130255 successful
    rsg_test.go:828: 1m5s of 5m0s: 228356 executions, 141490 successful
    rsg_test.go:828: 1m10s of 5m0s: 246297 executions, 152674 successful
    rsg_test.go:828: 1m15s of 5m0s: 264454 executions, 163960 successful
    rsg_test.go:828: 1m20s of 5m0s: 283593 executions, 175849 successful
    rsg_test.go:828: 1m25s of 5m0s: 298900 executions, 185404 successful
    rsg_test.go:828: 1m30s of 5m0s: 318340 executions, 197445 successful
    rsg_test.go:828: 1m35s of 5m0s: 335492 executions, 208107 successful
    rsg_test.go:828: 1m40s of 5m0s: 356934 executions, 221372 successful
    rsg_test.go:828: 1m45s of 5m0s: 374720 executions, 232350 successful
    rsg_test.go:828: 1m50s of 5m0s: 392635 executions, 243399 successful
    rsg_test.go:828: 1m55s of 5m0s: 410133 executions, 254361 successful
    rsg_test.go:828: 2m0s of 5m0s: 427800 executions, 265293 successful
    rsg_test.go:828: 2m5s of 5m0s: 445940 executions, 276596 successful
    rsg_test.go:828: 2m10s of 5m0s: 463288 executions, 287390 successful
    rsg_test.go:828: 2m15s of 5m0s: 479347 executions, 297338 successful
    rsg_test.go:828: 2m20s of 5m0s: 498508 executions, 309274 successful
    rsg_test.go:828: 2m25s of 5m0s: 517864 executions, 321211 successful
    rsg_test.go:828: 2m30s of 5m0s: 536530 executions, 332724 successful
    rsg_test.go:828: 2m35s of 5m0s: 557223 executions, 345625 successful
    rsg_test.go:828: 2m40s of 5m0s: 574076 executions, 356079 successful
    rsg_test.go:828: 2m45s of 5m0s: 590786 executions, 366399 successful
    rsg_test.go:828: 2m50s of 5m0s: 603754 executions, 374395 successful
    rsg_test.go:828: 2m55s of 5m0s: 620571 executions, 384875 successful
    rsg_test.go:828: 3m0s of 5m0s: 637741 executions, 395580 successful
    rsg_test.go:828: 3m5s of 5m0s: 653953 executions, 405616 successful
    rsg_test.go:828: 3m10s of 5m0s: 672287 executions, 417078 successful
    rsg_test.go:828: 3m15s of 5m0s: 691962 executions, 429242 successful
    rsg_test.go:828: 3m20s of 5m0s: 708757 executions, 439764 successful
    rsg_test.go:828: 3m25s of 5m0s: 727153 executions, 451112 successful
    rsg_test.go:828: 3m30s of 5m0s: 746010 executions, 462834 successful
    rsg_test.go:828: 3m35s of 5m0s: 765124 executions, 474661 successful
    rsg_test.go:828: 3m40s of 5m0s: 786301 executions, 487669 successful
    rsg_test.go:828: 3m45s of 5m0s: 802706 executions, 497863 successful
    rsg_test.go:828: 3m50s of 5m0s: 819416 executions, 508168 successful
    rsg_test.go:828: 3m55s of 5m0s: 836625 executions, 518774 successful
    rsg_test.go:828: 4m0s of 5m0s: 853157 executions, 529002 successful
    rsg_test.go:828: 4m5s of 5m0s: 868794 executions, 538762 successful
    rsg_test.go:828: 4m10s of 5m0s: 884533 executions, 548529 successful
    rsg_test.go:828: 4m15s of 5m0s: 900960 executions, 558687 successful
    rsg_test.go:828: 4m20s of 5m0s: 917558 executions, 569077 successful
    rsg_test.go:828: 4m25s of 5m0s: 935041 executions, 579957 successful
    rsg_test.go:828: 4m30s of 5m0s: 954285 executions, 591843 successful
    rsg_test.go:828: 4m35s of 5m0s: 972137 executions, 602799 successful
    rsg_test.go:828: 4m40s of 5m0s: 987727 executions, 612516 successful
    rsg_test.go:828: 4m45s of 5m0s: 1004839 executions, 623037 successful
    rsg_test.go:828: 4m50s of 5m0s: 1023483 executions, 634662 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9262967?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9262967?buildTab=artifacts#/) on release-23.1 @ [186218edb86b5974501b2976cfd0704d4d6d1beb](https://github.com/cockroachdb/cockroach/commits/186218edb86b5974501b2976cfd0704d4d6d1beb):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 13746966 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:828: 45s of 5m0s: 169998 executions, 105336 successful
    rsg_test.go:828: 50s of 5m0s: 189680 executions, 117640 successful
    rsg_test.go:828: 55s of 5m0s: 209489 executions, 130034 successful
    rsg_test.go:828: 1m0s of 5m0s: 230882 executions, 143272 successful
    rsg_test.go:828: 1m5s of 5m0s: 251020 executions, 155705 successful
    rsg_test.go:828: 1m10s of 5m0s: 272728 executions, 169121 successful
    rsg_test.go:828: 1m15s of 5m0s: 293370 executions, 181848 successful
    rsg_test.go:828: 1m20s of 5m0s: 314475 executions, 194926 successful
    rsg_test.go:828: 1m25s of 5m0s: 334045 executions, 207085 successful
    rsg_test.go:828: 1m30s of 5m0s: 353959 executions, 219381 successful
    rsg_test.go:828: 1m35s of 5m0s: 374526 executions, 232182 successful
    rsg_test.go:828: 1m40s of 5m0s: 390957 executions, 242277 successful
    rsg_test.go:828: 1m45s of 5m0s: 411332 executions, 254975 successful
    rsg_test.go:828: 1m50s of 5m0s: 431948 executions, 267764 successful
    rsg_test.go:828: 1m55s of 5m0s: 452863 executions, 280737 successful
    rsg_test.go:828: 2m0s of 5m0s: 473634 executions, 293670 successful
    rsg_test.go:828: 2m5s of 5m0s: 495485 executions, 307253 successful
    rsg_test.go:828: 2m10s of 5m0s: 513817 executions, 318581 successful
    rsg_test.go:828: 2m15s of 5m0s: 535127 executions, 331782 successful
    rsg_test.go:828: 2m20s of 5m0s: 556388 executions, 345008 successful
    rsg_test.go:828: 2m25s of 5m0s: 573051 executions, 355313 successful
    rsg_test.go:828: 2m30s of 5m0s: 593372 executions, 368041 successful
    rsg_test.go:828: 2m35s of 5m0s: 614333 executions, 381006 successful
    rsg_test.go:828: 2m40s of 5m0s: 634976 executions, 393766 successful
    rsg_test.go:828: 2m45s of 5m0s: 655466 executions, 406527 successful
    rsg_test.go:828: 2m50s of 5m0s: 673227 executions, 417574 successful
    rsg_test.go:828: 2m55s of 5m0s: 691068 executions, 428722 successful
    rsg_test.go:828: 3m0s of 5m0s: 710614 executions, 440838 successful
    rsg_test.go:828: 3m5s of 5m0s: 730885 executions, 453340 successful
    rsg_test.go:828: 3m10s of 5m0s: 751852 executions, 466430 successful
    rsg_test.go:828: 3m15s of 5m0s: 774414 executions, 480362 successful
    rsg_test.go:828: 3m20s of 5m0s: 793203 executions, 492055 successful
    rsg_test.go:828: 3m25s of 5m0s: 814142 executions, 505020 successful
    rsg_test.go:828: 3m30s of 5m0s: 836927 executions, 519126 successful
    rsg_test.go:828: 3m35s of 5m0s: 858558 executions, 532559 successful
    rsg_test.go:828: 3m40s of 5m0s: 881253 executions, 546664 successful
    rsg_test.go:828: 3m45s of 5m0s: 899677 executions, 558020 successful
    rsg_test.go:828: 3m50s of 5m0s: 921609 executions, 571600 successful
    rsg_test.go:828: 3m55s of 5m0s: 942734 executions, 584802 successful
    rsg_test.go:828: 4m0s of 5m0s: 964614 executions, 598264 successful
    rsg_test.go:828: 4m5s of 5m0s: 984449 executions, 610561 successful
    rsg_test.go:828: 4m10s of 5m0s: 1006333 executions, 624215 successful
    rsg_test.go:828: 4m15s of 5m0s: 1028514 executions, 638009 successful
    rsg_test.go:828: 4m20s of 5m0s: 1049599 executions, 651102 successful
    rsg_test.go:828: 4m25s of 5m0s: 1071912 executions, 664893 successful
    rsg_test.go:828: 4m30s of 5m0s: 1093575 executions, 678287 successful
    rsg_test.go:828: 4m35s of 5m0s: 1113100 executions, 690436 successful
    rsg_test.go:828: 4m40s of 5m0s: 1132901 executions, 702730 successful
    rsg_test.go:828: 4m45s of 5m0s: 1153128 executions, 715163 successful
    rsg_test.go:828: 4m50s of 5m0s: 1171045 executions, 726311 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9283365?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9283365?buildTab=artifacts#/) on release-23.1 @ [3551dc089ac3efcb3772df2109164dcb01e79d8f](https://github.com/cockroachdb/cockroach/commits/3551dc089ac3efcb3772df2109164dcb01e79d8f):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 14224171 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRandomSyntaxFunctions
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxFunctions778170626
    test_log_scope.go:79: use -show-logs to present logs inline
    rsg_test.go:828: 5s of 5m0s: 20537 executions, 12696 successful
    rsg_test.go:828: 10s of 5m0s: 38321 executions, 23685 successful
    rsg_test.go:828: 15s of 5m0s: 60313 executions, 37344 successful
    rsg_test.go:828: 20s of 5m0s: 79296 executions, 49137 successful
    rsg_test.go:828: 25s of 5m0s: 97434 executions, 60301 successful
    rsg_test.go:828: 30s of 5m0s: 118644 executions, 73443 successful
    rsg_test.go:828: 35s of 5m0s: 140019 executions, 86690 successful
    rsg_test.go:828: 40s of 5m0s: 159839 executions, 99056 successful
    rsg_test.go:828: 45s of 5m0s: 180117 executions, 111623 successful
    rsg_test.go:828: 50s of 5m0s: 200904 executions, 124559 successful
    rsg_test.go:828: 55s of 5m0s: 222889 executions, 138282 successful
    rsg_test.go:828: 1m0s of 5m0s: 243107 executions, 150836 successful
    rsg_test.go:828: 1m5s of 5m0s: 264578 executions, 164112 successful
    rsg_test.go:828: 1m10s of 5m0s: 282741 executions, 175406 successful
    rsg_test.go:828: 1m15s of 5m0s: 303107 executions, 187960 successful
    rsg_test.go:828: 1m20s of 5m0s: 323130 executions, 200353 successful
    rsg_test.go:828: 1m25s of 5m0s: 346186 executions, 214686 successful
    rsg_test.go:828: 1m30s of 5m0s: 366911 executions, 227596 successful
    rsg_test.go:828: 1m35s of 5m0s: 384288 executions, 238372 successful
    rsg_test.go:828: 1m40s of 5m0s: 407918 executions, 252847 successful
    rsg_test.go:828: 1m45s of 5m0s: 431975 executions, 267810 successful
    rsg_test.go:828: 1m50s of 5m0s: 454381 executions, 281721 successful
    rsg_test.go:828: 1m55s of 5m0s: 475234 executions, 294612 successful
    rsg_test.go:828: 2m0s of 5m0s: 494221 executions, 306336 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9329931?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9329931?buildTab=artifacts#/) on release-23.1 @ [aec78f33d45a8376a0ecec885688bae60dbfb85c](https://github.com/cockroachdb/cockroach/commits/aec78f33d45a8376a0ecec885688bae60dbfb85c):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 14633906 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:828: 45s of 5m0s: 169800 executions, 105473 successful
    rsg_test.go:828: 50s of 5m0s: 190513 executions, 118278 successful
    rsg_test.go:828: 55s of 5m0s: 210486 executions, 130548 successful
    rsg_test.go:828: 1m0s of 5m0s: 233381 executions, 144769 successful
    rsg_test.go:828: 1m5s of 5m0s: 251962 executions, 156400 successful
    rsg_test.go:828: 1m10s of 5m0s: 271153 executions, 168288 successful
    rsg_test.go:828: 1m15s of 5m0s: 289851 executions, 179836 successful
    rsg_test.go:828: 1m20s of 5m0s: 307780 executions, 191084 successful
    rsg_test.go:828: 1m25s of 5m0s: 324932 executions, 201706 successful
    rsg_test.go:828: 1m30s of 5m0s: 342258 executions, 212442 successful
    rsg_test.go:828: 1m35s of 5m0s: 364317 executions, 226004 successful
    rsg_test.go:828: 1m40s of 5m0s: 382800 executions, 237528 successful
    rsg_test.go:828: 1m45s of 5m0s: 401920 executions, 249283 successful
    rsg_test.go:828: 1m50s of 5m0s: 422870 executions, 262345 successful
    rsg_test.go:828: 1m55s of 5m0s: 440942 executions, 273461 successful
    rsg_test.go:828: 2m0s of 5m0s: 460187 executions, 285401 successful
    rsg_test.go:828: 2m5s of 5m0s: 481068 executions, 298399 successful
    rsg_test.go:828: 2m10s of 5m0s: 499976 executions, 310204 successful
    rsg_test.go:828: 2m15s of 5m0s: 519712 executions, 322381 successful
    rsg_test.go:828: 2m20s of 5m0s: 537844 executions, 333713 successful
    rsg_test.go:828: 2m25s of 5m0s: 557301 executions, 345779 successful
    rsg_test.go:828: 2m30s of 5m0s: 577820 executions, 358402 successful
    rsg_test.go:828: 2m35s of 5m0s: 597694 executions, 370706 successful
    rsg_test.go:828: 2m40s of 5m0s: 616949 executions, 382652 successful
    rsg_test.go:828: 2m45s of 5m0s: 636863 executions, 395004 successful
    rsg_test.go:828: 2m50s of 5m0s: 657782 executions, 407880 successful
    rsg_test.go:828: 2m55s of 5m0s: 676670 executions, 419618 successful
    rsg_test.go:828: 3m0s of 5m0s: 697764 executions, 432723 successful
    rsg_test.go:828: 3m5s of 5m0s: 716282 executions, 444219 successful
    rsg_test.go:828: 3m10s of 5m0s: 735601 executions, 456217 successful
    rsg_test.go:828: 3m15s of 5m0s: 754460 executions, 467870 successful
    rsg_test.go:828: 3m20s of 5m0s: 773521 executions, 479722 successful
    rsg_test.go:828: 3m25s of 5m0s: 790837 executions, 490466 successful
    rsg_test.go:828: 3m30s of 5m0s: 811917 executions, 503465 successful
    rsg_test.go:828: 3m35s of 5m0s: 827984 executions, 513484 successful
    rsg_test.go:828: 3m40s of 5m0s: 847508 executions, 525575 successful
    rsg_test.go:828: 3m45s of 5m0s: 868651 executions, 538649 successful
    rsg_test.go:828: 3m50s of 5m0s: 887456 executions, 550369 successful
    rsg_test.go:828: 3m55s of 5m0s: 907781 executions, 562825 successful
    rsg_test.go:828: 4m0s of 5m0s: 927900 executions, 575328 successful
    rsg_test.go:828: 4m5s of 5m0s: 947164 executions, 587258 successful
    rsg_test.go:828: 4m10s of 5m0s: 968253 executions, 600324 successful
    rsg_test.go:828: 4m15s of 5m0s: 985977 executions, 611237 successful
    rsg_test.go:828: 4m20s of 5m0s: 1008614 executions, 625374 successful
    rsg_test.go:828: 4m25s of 5m0s: 1027903 executions, 637324 successful
    rsg_test.go:828: 4m30s of 5m0s: 1047083 executions, 649125 successful
    rsg_test.go:828: 4m35s of 5m0s: 1067357 executions, 661578 successful
    rsg_test.go:828: 4m40s of 5m0s: 1086179 executions, 673270 successful
    rsg_test.go:828: 4m45s of 5m0s: 1102513 executions, 683397 successful
    rsg_test.go:828: 4m50s of 5m0s: 1118504 executions, 693139 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9353476?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9353476?buildTab=artifacts#/) on release-23.1 @ [7e72aae900c3ff4b44f1643c2d7ba55fbb2cbe23](https://github.com/cockroachdb/cockroach/commits/7e72aae900c3ff4b44f1643c2d7ba55fbb2cbe23):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 14067578 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:828: 45s of 5m0s: 186650 executions, 115779 successful
    rsg_test.go:828: 50s of 5m0s: 207395 executions, 128530 successful
    rsg_test.go:828: 55s of 5m0s: 226905 executions, 140696 successful
    rsg_test.go:828: 1m0s of 5m0s: 249354 executions, 154514 successful
    rsg_test.go:828: 1m5s of 5m0s: 266671 executions, 165196 successful
    rsg_test.go:828: 1m10s of 5m0s: 285770 executions, 176990 successful
    rsg_test.go:828: 1m15s of 5m0s: 304412 executions, 188528 successful
    rsg_test.go:828: 1m20s of 5m0s: 323010 executions, 200213 successful
    rsg_test.go:828: 1m25s of 5m0s: 342381 executions, 212243 successful
    rsg_test.go:828: 1m30s of 5m0s: 361167 executions, 223850 successful
    rsg_test.go:828: 1m35s of 5m0s: 381326 executions, 236289 successful
    rsg_test.go:828: 1m40s of 5m0s: 399512 executions, 247616 successful
    rsg_test.go:828: 1m45s of 5m0s: 419678 executions, 260101 successful
    rsg_test.go:828: 1m50s of 5m0s: 438219 executions, 271722 successful
    rsg_test.go:828: 1m55s of 5m0s: 458228 executions, 283993 successful
    rsg_test.go:828: 2m0s of 5m0s: 472318 executions, 292730 successful
    rsg_test.go:828: 2m5s of 5m0s: 493348 executions, 305777 successful
    rsg_test.go:828: 2m10s of 5m0s: 513175 executions, 318170 successful
    rsg_test.go:828: 2m15s of 5m0s: 532668 executions, 330235 successful
    rsg_test.go:828: 2m20s of 5m0s: 554847 executions, 343993 successful
    rsg_test.go:828: 2m25s of 5m0s: 575040 executions, 356515 successful
    rsg_test.go:828: 2m30s of 5m0s: 596075 executions, 369622 successful
    rsg_test.go:828: 2m35s of 5m0s: 615413 executions, 381630 successful
    rsg_test.go:828: 2m40s of 5m0s: 636954 executions, 394954 successful
    rsg_test.go:828: 2m45s of 5m0s: 653483 executions, 405281 successful
    rsg_test.go:828: 2m50s of 5m0s: 672729 executions, 417178 successful
    rsg_test.go:828: 2m55s of 5m0s: 693537 executions, 430103 successful
    rsg_test.go:828: 3m0s of 5m0s: 715821 executions, 443827 successful
    rsg_test.go:828: 3m5s of 5m0s: 735503 executions, 456097 successful
    rsg_test.go:828: 3m10s of 5m0s: 756526 executions, 469116 successful
    rsg_test.go:828: 3m15s of 5m0s: 779090 executions, 483043 successful
    rsg_test.go:828: 3m20s of 5m0s: 797487 executions, 494453 successful
    rsg_test.go:828: 3m25s of 5m0s: 814376 executions, 504914 successful
    rsg_test.go:828: 3m30s of 5m0s: 833022 executions, 516393 successful
    rsg_test.go:828: 3m35s of 5m0s: 853665 executions, 529201 successful
    rsg_test.go:828: 3m40s of 5m0s: 872047 executions, 540651 successful
    rsg_test.go:828: 3m45s of 5m0s: 892198 executions, 553130 successful
    rsg_test.go:828: 3m50s of 5m0s: 910802 executions, 564758 successful
    rsg_test.go:828: 3m55s of 5m0s: 930731 executions, 577129 successful
    rsg_test.go:828: 4m0s of 5m0s: 951245 executions, 589951 successful
    rsg_test.go:828: 4m5s of 5m0s: 969821 executions, 601438 successful
    rsg_test.go:828: 4m10s of 5m0s: 990461 executions, 614175 successful
    rsg_test.go:828: 4m15s of 5m0s: 1011658 executions, 627292 successful
    rsg_test.go:828: 4m20s of 5m0s: 1030658 executions, 639048 successful
    rsg_test.go:828: 4m25s of 5m0s: 1049762 executions, 650875 successful
    rsg_test.go:828: 4m30s of 5m0s: 1067814 executions, 662137 successful
    rsg_test.go:828: 4m35s of 5m0s: 1088339 executions, 674803 successful
    rsg_test.go:828: 4m40s of 5m0s: 1109215 executions, 687798 successful
    rsg_test.go:828: 4m45s of 5m0s: 1132592 executions, 702271 successful
    rsg_test.go:828: 4m50s of 5m0s: 1151209 executions, 713801 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9373223?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9373223?buildTab=artifacts#/) on release-23.1 @ [5d5abe704b0551556c35d51aa673d3162f493aa8](https://github.com/cockroachdb/cockroach/commits/5d5abe704b0551556c35d51aa673d3162f493aa8):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 10387569 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:828: 45s of 5m0s: 145262 executions, 90073 successful
    rsg_test.go:828: 50s of 5m0s: 163139 executions, 101149 successful
    rsg_test.go:828: 55s of 5m0s: 178131 executions, 110437 successful
    rsg_test.go:828: 1m0s of 5m0s: 192021 executions, 119055 successful
    rsg_test.go:828: 1m5s of 5m0s: 205020 executions, 127165 successful
    rsg_test.go:828: 1m10s of 5m0s: 219830 executions, 136234 successful
    rsg_test.go:828: 1m15s of 5m0s: 234548 executions, 145430 successful
    rsg_test.go:828: 1m20s of 5m0s: 251959 executions, 156209 successful
    rsg_test.go:828: 1m25s of 5m0s: 268782 executions, 166500 successful
    rsg_test.go:828: 1m30s of 5m0s: 286005 executions, 177124 successful
    rsg_test.go:828: 1m35s of 5m0s: 299248 executions, 185360 successful
    rsg_test.go:828: 1m40s of 5m0s: 313633 executions, 194412 successful
    rsg_test.go:828: 1m45s of 5m0s: 327986 executions, 203254 successful
    rsg_test.go:828: 1m50s of 5m0s: 341746 executions, 211782 successful
    rsg_test.go:828: 1m55s of 5m0s: 357022 executions, 221219 successful
    rsg_test.go:828: 2m0s of 5m0s: 373148 executions, 231279 successful
    rsg_test.go:828: 2m5s of 5m0s: 386789 executions, 239715 successful
    rsg_test.go:828: 2m10s of 5m0s: 403764 executions, 250213 successful
    rsg_test.go:828: 2m15s of 5m0s: 420410 executions, 260443 successful
    rsg_test.go:828: 2m20s of 5m0s: 437427 executions, 271015 successful
    rsg_test.go:828: 2m25s of 5m0s: 452852 executions, 280516 successful
    rsg_test.go:828: 2m30s of 5m0s: 468030 executions, 289958 successful
    rsg_test.go:828: 2m35s of 5m0s: 484149 executions, 299979 successful
    rsg_test.go:828: 2m40s of 5m0s: 498132 executions, 308621 successful
    rsg_test.go:828: 2m45s of 5m0s: 513060 executions, 317942 successful
    rsg_test.go:828: 2m50s of 5m0s: 527686 executions, 326867 successful
    rsg_test.go:828: 2m55s of 5m0s: 541799 executions, 335696 successful
    rsg_test.go:828: 3m0s of 5m0s: 558675 executions, 346058 successful
    rsg_test.go:828: 3m5s of 5m0s: 574671 executions, 356001 successful
    rsg_test.go:828: 3m10s of 5m0s: 590811 executions, 365956 successful
    rsg_test.go:828: 3m15s of 5m0s: 608258 executions, 376748 successful
    rsg_test.go:828: 3m20s of 5m0s: 622217 executions, 385353 successful
    rsg_test.go:828: 3m25s of 5m0s: 637688 executions, 394975 successful
    rsg_test.go:828: 3m30s of 5m0s: 652243 executions, 403976 successful
    rsg_test.go:828: 3m35s of 5m0s: 666282 executions, 412712 successful
    rsg_test.go:828: 3m40s of 5m0s: 683192 executions, 423223 successful
    rsg_test.go:828: 3m45s of 5m0s: 700429 executions, 433935 successful
    rsg_test.go:828: 3m50s of 5m0s: 716191 executions, 443708 successful
    rsg_test.go:828: 3m55s of 5m0s: 731184 executions, 453002 successful
    rsg_test.go:828: 4m0s of 5m0s: 745106 executions, 461715 successful
    rsg_test.go:828: 4m5s of 5m0s: 761031 executions, 471598 successful
    rsg_test.go:828: 4m10s of 5m0s: 776891 executions, 481301 successful
    rsg_test.go:828: 4m15s of 5m0s: 793005 executions, 491329 successful
    rsg_test.go:828: 4m20s of 5m0s: 810704 executions, 502329 successful
    rsg_test.go:828: 4m25s of 5m0s: 826238 executions, 512000 successful
    rsg_test.go:828: 4m30s of 5m0s: 840828 executions, 520976 successful
    rsg_test.go:828: 4m35s of 5m0s: 857643 executions, 531367 successful
    rsg_test.go:828: 4m40s of 5m0s: 874065 executions, 541578 successful
    rsg_test.go:828: 4m45s of 5m0s: 890278 executions, 551501 successful
    rsg_test.go:828: 4m50s of 5m0s: 907625 executions, 562273 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9393410?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9393410?buildTab=artifacts#/) on release-23.1 @ [63b683eab3e0ebfff812bb54fdac936c101d9383](https://github.com/cockroachdb/cockroach/commits/63b683eab3e0ebfff812bb54fdac936c101d9383):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 14634800 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:828: 45s of 5m0s: 182470 executions, 113072 successful
    rsg_test.go:828: 50s of 5m0s: 203399 executions, 126062 successful
    rsg_test.go:828: 55s of 5m0s: 225971 executions, 140123 successful
    rsg_test.go:828: 1m0s of 5m0s: 245436 executions, 152191 successful
    rsg_test.go:828: 1m5s of 5m0s: 265704 executions, 164626 successful
    rsg_test.go:828: 1m10s of 5m0s: 286205 executions, 177333 successful
    rsg_test.go:828: 1m15s of 5m0s: 306708 executions, 189979 successful
    rsg_test.go:828: 1m20s of 5m0s: 328186 executions, 203240 successful
    rsg_test.go:828: 1m25s of 5m0s: 348485 executions, 215738 successful
    rsg_test.go:828: 1m30s of 5m0s: 369701 executions, 228868 successful
    rsg_test.go:828: 1m35s of 5m0s: 390961 executions, 242081 successful
    rsg_test.go:828: 1m40s of 5m0s: 413367 executions, 256045 successful
    rsg_test.go:828: 1m45s of 5m0s: 437547 executions, 270936 successful
    rsg_test.go:828: 1m50s of 5m0s: 458544 executions, 283885 successful
    rsg_test.go:828: 1m55s of 5m0s: 478311 executions, 296181 successful
    rsg_test.go:828: 2m0s of 5m0s: 499623 executions, 309425 successful
    rsg_test.go:828: 2m5s of 5m0s: 522431 executions, 323514 successful
    rsg_test.go:828: 2m10s of 5m0s: 538106 executions, 333240 successful
    rsg_test.go:828: 2m15s of 5m0s: 556774 executions, 344731 successful
    rsg_test.go:828: 2m20s of 5m0s: 575995 executions, 356654 successful
    rsg_test.go:828: 2m25s of 5m0s: 595077 executions, 368433 successful
    rsg_test.go:828: 2m30s of 5m0s: 615086 executions, 380929 successful
    rsg_test.go:828: 2m35s of 5m0s: 637079 executions, 394487 successful
    rsg_test.go:828: 2m40s of 5m0s: 655394 executions, 405861 successful
    rsg_test.go:828: 2m45s of 5m0s: 676309 executions, 418874 successful
    rsg_test.go:828: 2m50s of 5m0s: 699271 executions, 433194 successful
    rsg_test.go:828: 2m55s of 5m0s: 720155 executions, 446115 successful
    rsg_test.go:828: 3m0s of 5m0s: 742423 executions, 459958 successful
    rsg_test.go:828: 3m5s of 5m0s: 765604 executions, 474347 successful
    rsg_test.go:828: 3m10s of 5m0s: 788003 executions, 488306 successful
    rsg_test.go:828: 3m15s of 5m0s: 807126 executions, 500194 successful
    rsg_test.go:828: 3m20s of 5m0s: 829220 executions, 513893 successful
    rsg_test.go:828: 3m25s of 5m0s: 849100 executions, 526319 successful
    rsg_test.go:828: 3m30s of 5m0s: 870434 executions, 539480 successful
    rsg_test.go:828: 3m35s of 5m0s: 891092 executions, 552318 successful
    rsg_test.go:828: 3m40s of 5m0s: 912396 executions, 565576 successful
    rsg_test.go:828: 3m45s of 5m0s: 933257 executions, 578543 successful
    rsg_test.go:828: 3m50s of 5m0s: 954243 executions, 591530 successful
    rsg_test.go:828: 3m55s of 5m0s: 975888 executions, 605082 successful
    rsg_test.go:828: 4m0s of 5m0s: 999068 executions, 619401 successful
    rsg_test.go:828: 4m5s of 5m0s: 1019191 executions, 631844 successful
    rsg_test.go:828: 4m10s of 5m0s: 1043209 executions, 646753 successful
    rsg_test.go:828: 4m15s of 5m0s: 1067971 executions, 662101 successful
    rsg_test.go:828: 4m20s of 5m0s: 1089375 executions, 675352 successful
    rsg_test.go:828: 4m25s of 5m0s: 1110058 executions, 688185 successful
    rsg_test.go:828: 4m30s of 5m0s: 1132673 executions, 702064 successful
    rsg_test.go:828: 4m35s of 5m0s: 1153665 executions, 715100 successful
    rsg_test.go:828: 4m40s of 5m0s: 1174356 executions, 727936 successful
    rsg_test.go:828: 4m45s of 5m0s: 1196325 executions, 741617 successful
    rsg_test.go:828: 4m50s of 5m0s: 1216478 executions, 754072 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9417487?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9417487?buildTab=artifacts#/) on release-23.1 @ [35b724947ac464e6af45ce5193e34035fcacb1ef](https://github.com/cockroachdb/cockroach/commits/35b724947ac464e6af45ce5193e34035fcacb1ef):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 15330553 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:828: 45s of 5m0s: 186730 executions, 115962 successful
    rsg_test.go:828: 50s of 5m0s: 204669 executions, 127002 successful
    rsg_test.go:828: 55s of 5m0s: 225987 executions, 140283 successful
    rsg_test.go:828: 1m0s of 5m0s: 246745 executions, 153107 successful
    rsg_test.go:828: 1m5s of 5m0s: 269495 executions, 167344 successful
    rsg_test.go:828: 1m10s of 5m0s: 288685 executions, 179260 successful
    rsg_test.go:828: 1m15s of 5m0s: 307826 executions, 191102 successful
    rsg_test.go:828: 1m20s of 5m0s: 328231 executions, 203659 successful
    rsg_test.go:828: 1m25s of 5m0s: 348924 executions, 216448 successful
    rsg_test.go:828: 1m30s of 5m0s: 369877 executions, 229385 successful
    rsg_test.go:828: 1m35s of 5m0s: 390228 executions, 242093 successful
    rsg_test.go:828: 1m40s of 5m0s: 409874 executions, 254180 successful
    rsg_test.go:828: 1m45s of 5m0s: 432370 executions, 268207 successful
    rsg_test.go:828: 1m50s of 5m0s: 451842 executions, 280221 successful
    rsg_test.go:828: 1m55s of 5m0s: 472084 executions, 292668 successful
    rsg_test.go:828: 2m0s of 5m0s: 490768 executions, 304234 successful
    rsg_test.go:828: 2m5s of 5m0s: 511278 executions, 317056 successful
    rsg_test.go:828: 2m10s of 5m0s: 534161 executions, 331271 successful
    rsg_test.go:828: 2m15s of 5m0s: 556323 executions, 344919 successful
    rsg_test.go:828: 2m20s of 5m0s: 576143 executions, 357198 successful
    rsg_test.go:828: 2m25s of 5m0s: 598956 executions, 371331 successful
    rsg_test.go:828: 2m30s of 5m0s: 616918 executions, 382476 successful
    rsg_test.go:828: 2m35s of 5m0s: 637217 executions, 394971 successful
    rsg_test.go:828: 2m40s of 5m0s: 657054 executions, 407352 successful
    rsg_test.go:828: 2m45s of 5m0s: 678514 executions, 420683 successful
    rsg_test.go:828: 2m50s of 5m0s: 699322 executions, 433597 successful
    rsg_test.go:828: 2m55s of 5m0s: 722759 executions, 448097 successful
    rsg_test.go:828: 3m0s of 5m0s: 742973 executions, 460633 successful
    rsg_test.go:828: 3m5s of 5m0s: 765944 executions, 474790 successful
    rsg_test.go:828: 3m10s of 5m0s: 786536 executions, 487541 successful
    rsg_test.go:828: 3m15s of 5m0s: 811578 executions, 503063 successful
    rsg_test.go:828: 3m20s of 5m0s: 831918 executions, 515717 successful
    rsg_test.go:828: 3m25s of 5m0s: 853128 executions, 528830 successful
    rsg_test.go:828: 3m30s of 5m0s: 872796 executions, 540915 successful
    rsg_test.go:828: 3m35s of 5m0s: 895156 executions, 554803 successful
    rsg_test.go:828: 3m40s of 5m0s: 917481 executions, 568592 successful
    rsg_test.go:828: 3m45s of 5m0s: 938413 executions, 581583 successful
    rsg_test.go:828: 3m50s of 5m0s: 960231 executions, 595169 successful
    rsg_test.go:828: 3m55s of 5m0s: 982316 executions, 608751 successful
    rsg_test.go:828: 4m0s of 5m0s: 1002739 executions, 621490 successful
    rsg_test.go:828: 4m5s of 5m0s: 1020011 executions, 632227 successful
    rsg_test.go:828: 4m10s of 5m0s: 1040195 executions, 644683 successful
    rsg_test.go:828: 4m15s of 5m0s: 1059747 executions, 656772 successful
    rsg_test.go:828: 4m20s of 5m0s: 1082143 executions, 670676 successful
    rsg_test.go:828: 4m25s of 5m0s: 1104264 executions, 684496 successful
    rsg_test.go:828: 4m30s of 5m0s: 1124344 executions, 696817 successful
    rsg_test.go:828: 4m35s of 5m0s: 1143343 executions, 708627 successful
    rsg_test.go:828: 4m40s of 5m0s: 1165104 executions, 722146 successful
    rsg_test.go:828: 4m45s of 5m0s: 1186493 executions, 735378 successful
    rsg_test.go:828: 4m50s of 5m0s: 1209207 executions, 749482 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9441332?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9441332?buildTab=artifacts#/) on release-23.1 @ [e847e4358c004d4a8977ae26749768ad1d417fba](https://github.com/cockroachdb/cockroach/commits/e847e4358c004d4a8977ae26749768ad1d417fba):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 14288694 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:828: 45s of 5m0s: 165750 executions, 102717 successful
    rsg_test.go:828: 50s of 5m0s: 183519 executions, 113660 successful
    rsg_test.go:828: 55s of 5m0s: 204170 executions, 126523 successful
    rsg_test.go:828: 1m0s of 5m0s: 220374 executions, 136519 successful
    rsg_test.go:828: 1m5s of 5m0s: 238836 executions, 148059 successful
    rsg_test.go:828: 1m10s of 5m0s: 258706 executions, 160338 successful
    rsg_test.go:828: 1m15s of 5m0s: 275867 executions, 170904 successful
    rsg_test.go:828: 1m20s of 5m0s: 295828 executions, 183354 successful
    rsg_test.go:828: 1m25s of 5m0s: 314089 executions, 194534 successful
    rsg_test.go:828: 1m30s of 5m0s: 331695 executions, 205520 successful
    rsg_test.go:828: 1m35s of 5m0s: 349518 executions, 216507 successful
    rsg_test.go:828: 1m40s of 5m0s: 370152 executions, 229260 successful
    rsg_test.go:828: 1m45s of 5m0s: 388319 executions, 240588 successful
    rsg_test.go:828: 1m50s of 5m0s: 405712 executions, 251355 successful
    rsg_test.go:828: 1m55s of 5m0s: 425777 executions, 263657 successful
    rsg_test.go:828: 2m0s of 5m0s: 446736 executions, 276642 successful
    rsg_test.go:828: 2m5s of 5m0s: 466326 executions, 288738 successful
    rsg_test.go:828: 2m10s of 5m0s: 485599 executions, 300739 successful
    rsg_test.go:828: 2m15s of 5m0s: 505549 executions, 313158 successful
    rsg_test.go:828: 2m20s of 5m0s: 525719 executions, 325548 successful
    rsg_test.go:828: 2m25s of 5m0s: 543896 executions, 336841 successful
    rsg_test.go:828: 2m30s of 5m0s: 564240 executions, 349361 successful
    rsg_test.go:828: 2m35s of 5m0s: 584251 executions, 361805 successful
    rsg_test.go:828: 2m40s of 5m0s: 604592 executions, 374360 successful
    rsg_test.go:828: 2m45s of 5m0s: 626012 executions, 387628 successful
    rsg_test.go:828: 2m50s of 5m0s: 644795 executions, 399226 successful
    rsg_test.go:828: 2m55s of 5m0s: 664678 executions, 411603 successful
    rsg_test.go:828: 3m0s of 5m0s: 684170 executions, 423671 successful
    rsg_test.go:828: 3m5s of 5m0s: 702753 executions, 435050 successful
    rsg_test.go:828: 3m10s of 5m0s: 722991 executions, 447640 successful
    rsg_test.go:828: 3m15s of 5m0s: 744171 executions, 460794 successful
    rsg_test.go:828: 3m20s of 5m0s: 761014 executions, 471310 successful
    rsg_test.go:828: 3m25s of 5m0s: 782742 executions, 484722 successful
    rsg_test.go:828: 3m30s of 5m0s: 803184 executions, 497455 successful
    rsg_test.go:828: 3m35s of 5m0s: 821864 executions, 508968 successful
    rsg_test.go:828: 3m40s of 5m0s: 839817 executions, 520064 successful
    rsg_test.go:828: 3m45s of 5m0s: 860534 executions, 532882 successful
    rsg_test.go:828: 3m50s of 5m0s: 880009 executions, 544907 successful
    rsg_test.go:828: 3m55s of 5m0s: 900762 executions, 557806 successful
    rsg_test.go:828: 4m0s of 5m0s: 919713 executions, 569506 successful
    rsg_test.go:828: 4m5s of 5m0s: 939990 executions, 582051 successful
    rsg_test.go:828: 4m10s of 5m0s: 958763 executions, 593722 successful
    rsg_test.go:828: 4m15s of 5m0s: 980264 executions, 607010 successful
    rsg_test.go:828: 4m20s of 5m0s: 1001007 executions, 619960 successful
    rsg_test.go:828: 4m25s of 5m0s: 1019261 executions, 631218 successful
    rsg_test.go:828: 4m30s of 5m0s: 1037427 executions, 642576 successful
    rsg_test.go:828: 4m35s of 5m0s: 1056448 executions, 654403 successful
    rsg_test.go:828: 4m40s of 5m0s: 1075861 executions, 666390 successful
    rsg_test.go:828: 4m45s of 5m0s: 1094095 executions, 677648 successful
    rsg_test.go:828: 4m50s of 5m0s: 1116130 executions, 691383 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9461810?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9461810?buildTab=artifacts#/) on release-23.1 @ [b432e8c20339de5cfa7c811a9ee6f5dc98d15a1e](https://github.com/cockroachdb/cockroach/commits/b432e8c20339de5cfa7c811a9ee6f5dc98d15a1e):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 14608934 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRandomSyntaxFunctions
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxFunctions119133282
    test_log_scope.go:79: use -show-logs to present logs inline
    rsg_test.go:828: 5s of 5m0s: 18384 executions, 11359 successful
    rsg_test.go:828: 10s of 5m0s: 36593 executions, 22721 successful
    rsg_test.go:828: 15s of 5m0s: 57199 executions, 35450 successful
    rsg_test.go:828: 20s of 5m0s: 77640 executions, 48119 successful
    rsg_test.go:828: 25s of 5m0s: 96092 executions, 59574 successful
    rsg_test.go:828: 30s of 5m0s: 116328 executions, 72093 successful
    rsg_test.go:828: 35s of 5m0s: 137006 executions, 85029 successful
    rsg_test.go:828: 40s of 5m0s: 154143 executions, 95660 successful
    rsg_test.go:828: 45s of 5m0s: 169666 executions, 105232 successful
    rsg_test.go:828: 50s of 5m0s: 186078 executions, 115426 successful
    rsg_test.go:828: 55s of 5m0s: 206009 executions, 127845 successful
    rsg_test.go:828: 1m0s of 5m0s: 226237 executions, 140293 successful
    rsg_test.go:828: 1m5s of 5m0s: 247309 executions, 153378 successful
    rsg_test.go:828: 1m10s of 5m0s: 267344 executions, 165841 successful
    rsg_test.go:828: 1m15s of 5m0s: 287211 executions, 178026 successful
    rsg_test.go:828: 1m20s of 5m0s: 307197 executions, 190538 successful
    rsg_test.go:828: 1m25s of 5m0s: 330675 executions, 205222 successful
    rsg_test.go:828: 1m30s of 5m0s: 351032 executions, 217774 successful
    rsg_test.go:828: 1m35s of 5m0s: 370093 executions, 229515 successful
    rsg_test.go:828: 1m40s of 5m0s: 390262 executions, 242071 successful
    rsg_test.go:828: 1m45s of 5m0s: 411994 executions, 255395 successful
    rsg_test.go:828: 1m50s of 5m0s: 433683 executions, 268850 successful
    rsg_test.go:828: 1m55s of 5m0s: 452332 executions, 280375 successful
    rsg_test.go:828: 2m0s of 5m0s: 469481 executions, 290996 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9482245?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9482245?buildTab=artifacts#/) on release-23.1 @ [af5e6abf7efbb1361252839703e3ef8e067c9410](https://github.com/cockroachdb/cockroach/commits/af5e6abf7efbb1361252839703e3ef8e067c9410):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 13131021 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:828: 45s of 5m0s: 163335 executions, 101216 successful
    rsg_test.go:828: 50s of 5m0s: 183936 executions, 113933 successful
    rsg_test.go:828: 55s of 5m0s: 203871 executions, 126251 successful
    rsg_test.go:828: 1m0s of 5m0s: 219052 executions, 135676 successful
    rsg_test.go:828: 1m5s of 5m0s: 237307 executions, 146931 successful
    rsg_test.go:828: 1m10s of 5m0s: 254789 executions, 157842 successful
    rsg_test.go:828: 1m15s of 5m0s: 269467 executions, 167023 successful
    rsg_test.go:828: 1m20s of 5m0s: 286519 executions, 177579 successful
    rsg_test.go:828: 1m25s of 5m0s: 304071 executions, 188468 successful
    rsg_test.go:828: 1m30s of 5m0s: 322629 executions, 199950 successful
    rsg_test.go:828: 1m35s of 5m0s: 341121 executions, 211378 successful
    rsg_test.go:828: 1m40s of 5m0s: 357662 executions, 221650 successful
    rsg_test.go:828: 1m45s of 5m0s: 376166 executions, 233069 successful
    rsg_test.go:828: 1m50s of 5m0s: 393558 executions, 243872 successful
    rsg_test.go:828: 1m55s of 5m0s: 409675 executions, 253984 successful
    rsg_test.go:828: 2m0s of 5m0s: 431594 executions, 267490 successful
    rsg_test.go:828: 2m5s of 5m0s: 448865 executions, 278239 successful
    rsg_test.go:828: 2m10s of 5m0s: 466260 executions, 288889 successful
    rsg_test.go:828: 2m15s of 5m0s: 486129 executions, 301353 successful
    rsg_test.go:828: 2m20s of 5m0s: 503527 executions, 312191 successful
    rsg_test.go:828: 2m25s of 5m0s: 519459 executions, 321939 successful
    rsg_test.go:828: 2m30s of 5m0s: 539007 executions, 334120 successful
    rsg_test.go:828: 2m35s of 5m0s: 559361 executions, 346737 successful
    rsg_test.go:828: 2m40s of 5m0s: 577399 executions, 357860 successful
    rsg_test.go:828: 2m45s of 5m0s: 595887 executions, 369307 successful
    rsg_test.go:828: 2m50s of 5m0s: 614044 executions, 380557 successful
    rsg_test.go:828: 2m55s of 5m0s: 629689 executions, 390248 successful
    rsg_test.go:828: 3m0s of 5m0s: 646243 executions, 400432 successful
    rsg_test.go:828: 3m5s of 5m0s: 662805 executions, 410813 successful
    rsg_test.go:828: 3m10s of 5m0s: 680899 executions, 421878 successful
    rsg_test.go:828: 3m15s of 5m0s: 699846 executions, 433651 successful
    rsg_test.go:828: 3m20s of 5m0s: 715893 executions, 443684 successful
    rsg_test.go:828: 3m25s of 5m0s: 736718 executions, 456662 successful
    rsg_test.go:828: 3m30s of 5m0s: 755711 executions, 468490 successful
    rsg_test.go:828: 3m35s of 5m0s: 774810 executions, 480309 successful
    rsg_test.go:828: 3m40s of 5m0s: 794861 executions, 492767 successful
    rsg_test.go:828: 3m45s of 5m0s: 814808 executions, 505145 successful
    rsg_test.go:828: 3m50s of 5m0s: 833317 executions, 516524 successful
    rsg_test.go:828: 3m55s of 5m0s: 851976 executions, 528070 successful
    rsg_test.go:828: 4m0s of 5m0s: 871562 executions, 540298 successful
    rsg_test.go:828: 4m5s of 5m0s: 888891 executions, 551025 successful
    rsg_test.go:828: 4m10s of 5m0s: 908836 executions, 563384 successful
    rsg_test.go:828: 4m15s of 5m0s: 926030 executions, 574000 successful
    rsg_test.go:828: 4m20s of 5m0s: 943420 executions, 584768 successful
    rsg_test.go:828: 4m25s of 5m0s: 962485 executions, 596624 successful
    rsg_test.go:828: 4m30s of 5m0s: 977101 executions, 605612 successful
    rsg_test.go:828: 4m35s of 5m0s: 995187 executions, 616835 successful
    rsg_test.go:828: 4m40s of 5m0s: 1011879 executions, 627107 successful
    rsg_test.go:828: 4m45s of 5m0s: 1028309 executions, 637310 successful
    rsg_test.go:828: 4m50s of 5m0s: 1046748 executions, 648868 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9507207?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9507207?buildTab=artifacts#/) on release-23.1 @ [de239a7438f44d382c9aefceb65d9c39911dabd2](https://github.com/cockroachdb/cockroach/commits/de239a7438f44d382c9aefceb65d9c39911dabd2):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 14761221 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:828: 45s of 5m0s: 183477 executions, 113716 successful
    rsg_test.go:828: 50s of 5m0s: 203302 executions, 126014 successful
    rsg_test.go:828: 55s of 5m0s: 221838 executions, 137459 successful
    rsg_test.go:828: 1m0s of 5m0s: 237333 executions, 147087 successful
    rsg_test.go:828: 1m5s of 5m0s: 253354 executions, 156951 successful
    rsg_test.go:828: 1m10s of 5m0s: 275195 executions, 170532 successful
    rsg_test.go:828: 1m15s of 5m0s: 296389 executions, 183643 successful
    rsg_test.go:828: 1m20s of 5m0s: 318870 executions, 197552 successful
    rsg_test.go:828: 1m25s of 5m0s: 341893 executions, 211838 successful
    rsg_test.go:828: 1m30s of 5m0s: 358382 executions, 222050 successful
    rsg_test.go:828: 1m35s of 5m0s: 379969 executions, 235439 successful
    rsg_test.go:828: 1m40s of 5m0s: 397443 executions, 246291 successful
    rsg_test.go:828: 1m45s of 5m0s: 418490 executions, 259417 successful
    rsg_test.go:828: 1m50s of 5m0s: 434763 executions, 269520 successful
    rsg_test.go:828: 1m55s of 5m0s: 455847 executions, 282655 successful
    rsg_test.go:828: 2m0s of 5m0s: 476149 executions, 295215 successful
    rsg_test.go:828: 2m5s of 5m0s: 495262 executions, 307025 successful
    rsg_test.go:828: 2m10s of 5m0s: 515196 executions, 319348 successful
    rsg_test.go:828: 2m15s of 5m0s: 533393 executions, 330616 successful
    rsg_test.go:828: 2m20s of 5m0s: 554644 executions, 343654 successful
    rsg_test.go:828: 2m25s of 5m0s: 574683 executions, 356192 successful
    rsg_test.go:828: 2m30s of 5m0s: 595145 executions, 368832 successful
    rsg_test.go:828: 2m35s of 5m0s: 611812 executions, 379155 successful
    rsg_test.go:828: 2m40s of 5m0s: 631437 executions, 391317 successful
    rsg_test.go:828: 2m45s of 5m0s: 651823 executions, 403938 successful
    rsg_test.go:828: 2m50s of 5m0s: 670926 executions, 415845 successful
    rsg_test.go:828: 2m55s of 5m0s: 691446 executions, 428540 successful
    rsg_test.go:828: 3m0s of 5m0s: 713044 executions, 441902 successful
    rsg_test.go:828: 3m5s of 5m0s: 733414 executions, 454441 successful
    rsg_test.go:828: 3m10s of 5m0s: 752520 executions, 466201 successful
    rsg_test.go:828: 3m15s of 5m0s: 774726 executions, 479840 successful
    rsg_test.go:828: 3m20s of 5m0s: 796413 executions, 493340 successful
    rsg_test.go:828: 3m25s of 5m0s: 816479 executions, 505759 successful
    rsg_test.go:828: 3m30s of 5m0s: 838206 executions, 519183 successful
    rsg_test.go:828: 3m35s of 5m0s: 856808 executions, 530804 successful
    rsg_test.go:828: 3m40s of 5m0s: 879107 executions, 544527 successful
    rsg_test.go:828: 3m45s of 5m0s: 899421 executions, 557131 successful
    rsg_test.go:828: 3m50s of 5m0s: 919436 executions, 569516 successful
    rsg_test.go:828: 3m55s of 5m0s: 938144 executions, 581144 successful
    rsg_test.go:828: 4m0s of 5m0s: 957584 executions, 593194 successful
    rsg_test.go:828: 4m5s of 5m0s: 978942 executions, 606350 successful
    rsg_test.go:828: 4m10s of 5m0s: 996953 executions, 617556 successful
    rsg_test.go:828: 4m15s of 5m0s: 1017104 executions, 629954 successful
    rsg_test.go:828: 4m20s of 5m0s: 1037453 executions, 642596 successful
    rsg_test.go:828: 4m25s of 5m0s: 1057893 executions, 655217 successful
    rsg_test.go:828: 4m30s of 5m0s: 1078465 executions, 667999 successful
    rsg_test.go:828: 4m35s of 5m0s: 1098519 executions, 680395 successful
    rsg_test.go:828: 4m40s of 5m0s: 1119455 executions, 693333 successful
    rsg_test.go:828: 4m45s of 5m0s: 1140659 executions, 706411 successful
    rsg_test.go:828: 4m50s of 5m0s: 1161574 executions, 719324 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9516098?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9516098?buildTab=artifacts#/) on release-23.1 @ [2f96695f75b07c872ec5f146acc1fa198135768f](https://github.com/cockroachdb/cockroach/commits/2f96695f75b07c872ec5f146acc1fa198135768f):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 14596195 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:828: 45s of 5m0s: 186628 executions, 115585 successful
    rsg_test.go:828: 50s of 5m0s: 204660 executions, 126714 successful
    rsg_test.go:828: 55s of 5m0s: 226001 executions, 139939 successful
    rsg_test.go:828: 1m0s of 5m0s: 242066 executions, 149935 successful
    rsg_test.go:828: 1m5s of 5m0s: 262802 executions, 162692 successful
    rsg_test.go:828: 1m10s of 5m0s: 286890 executions, 177682 successful
    rsg_test.go:828: 1m15s of 5m0s: 306410 executions, 189721 successful
    rsg_test.go:828: 1m20s of 5m0s: 325665 executions, 201679 successful
    rsg_test.go:828: 1m25s of 5m0s: 346323 executions, 214478 successful
    rsg_test.go:828: 1m30s of 5m0s: 364478 executions, 225672 successful
    rsg_test.go:828: 1m35s of 5m0s: 387448 executions, 239887 successful
    rsg_test.go:828: 1m40s of 5m0s: 408652 executions, 252949 successful
    rsg_test.go:828: 1m45s of 5m0s: 428698 executions, 265429 successful
    rsg_test.go:828: 1m50s of 5m0s: 449082 executions, 277912 successful
    rsg_test.go:828: 1m55s of 5m0s: 470504 executions, 291137 successful
    rsg_test.go:828: 2m0s of 5m0s: 489914 executions, 303302 successful
    rsg_test.go:828: 2m5s of 5m0s: 509470 executions, 315467 successful
    rsg_test.go:828: 2m10s of 5m0s: 528788 executions, 327474 successful
    rsg_test.go:828: 2m15s of 5m0s: 549496 executions, 340336 successful
    rsg_test.go:828: 2m20s of 5m0s: 569007 executions, 352351 successful
    rsg_test.go:828: 2m25s of 5m0s: 590505 executions, 365575 successful
    rsg_test.go:828: 2m30s of 5m0s: 608275 executions, 376561 successful
    rsg_test.go:828: 2m35s of 5m0s: 626299 executions, 387795 successful
    rsg_test.go:828: 2m40s of 5m0s: 646107 executions, 400004 successful
    rsg_test.go:828: 2m45s of 5m0s: 665690 executions, 412134 successful
    rsg_test.go:828: 2m50s of 5m0s: 684030 executions, 423593 successful
    rsg_test.go:828: 2m55s of 5m0s: 705079 executions, 436613 successful
    rsg_test.go:828: 3m0s of 5m0s: 726421 executions, 449804 successful
    rsg_test.go:828: 3m5s of 5m0s: 744957 executions, 461348 successful
    rsg_test.go:828: 3m10s of 5m0s: 766946 executions, 474959 successful
    rsg_test.go:828: 3m15s of 5m0s: 784445 executions, 485790 successful
    rsg_test.go:828: 3m20s of 5m0s: 803635 executions, 497718 successful
    rsg_test.go:828: 3m25s of 5m0s: 825795 executions, 511492 successful
    rsg_test.go:828: 3m30s of 5m0s: 847337 executions, 524788 successful
    rsg_test.go:828: 3m35s of 5m0s: 868456 executions, 537902 successful
    rsg_test.go:828: 3m40s of 5m0s: 888456 executions, 550178 successful
    rsg_test.go:828: 3m45s of 5m0s: 909722 executions, 563386 successful
    rsg_test.go:828: 3m50s of 5m0s: 929931 executions, 575942 successful
    rsg_test.go:828: 3m55s of 5m0s: 952698 executions, 590051 successful
    rsg_test.go:828: 4m0s of 5m0s: 971152 executions, 601377 successful
    rsg_test.go:828: 4m5s of 5m0s: 993132 executions, 615023 successful
    rsg_test.go:828: 4m10s of 5m0s: 1013287 executions, 627440 successful
    rsg_test.go:828: 4m15s of 5m0s: 1031473 executions, 638807 successful
    rsg_test.go:828: 4m20s of 5m0s: 1052493 executions, 651799 successful
    rsg_test.go:828: 4m25s of 5m0s: 1073034 executions, 664399 successful
    rsg_test.go:828: 4m30s of 5m0s: 1092984 executions, 676806 successful
    rsg_test.go:828: 4m35s of 5m0s: 1114877 executions, 690290 successful
    rsg_test.go:828: 4m40s of 5m0s: 1137030 executions, 704082 successful
    rsg_test.go:828: 4m45s of 5m0s: 1155417 executions, 715365 successful
    rsg_test.go:828: 4m50s of 5m0s: 1175534 executions, 727841 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9538119?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9538119?buildTab=artifacts#/) on release-23.1 @ [a9eb6d220fc0957d4f6f8261d447991e9d03b4ad](https://github.com/cockroachdb/cockroach/commits/a9eb6d220fc0957d4f6f8261d447991e9d03b4ad):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 15905970 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:828: 45s of 5m0s: 172596 executions, 106959 successful
    rsg_test.go:828: 50s of 5m0s: 189223 executions, 117229 successful
    rsg_test.go:828: 55s of 5m0s: 207035 executions, 128292 successful
    rsg_test.go:828: 1m0s of 5m0s: 229049 executions, 141949 successful
    rsg_test.go:828: 1m5s of 5m0s: 247163 executions, 153120 successful
    rsg_test.go:828: 1m10s of 5m0s: 267959 executions, 166003 successful
    rsg_test.go:828: 1m15s of 5m0s: 287429 executions, 178087 successful
    rsg_test.go:828: 1m20s of 5m0s: 306015 executions, 189628 successful
    rsg_test.go:828: 1m25s of 5m0s: 326428 executions, 202280 successful
    rsg_test.go:828: 1m30s of 5m0s: 349083 executions, 216298 successful
    rsg_test.go:828: 1m35s of 5m0s: 369028 executions, 228718 successful
    rsg_test.go:828: 1m40s of 5m0s: 388339 executions, 240645 successful
    rsg_test.go:828: 1m45s of 5m0s: 408632 executions, 253049 successful
    rsg_test.go:828: 1m50s of 5m0s: 427774 executions, 264992 successful
    rsg_test.go:828: 1m55s of 5m0s: 448067 executions, 277594 successful
    rsg_test.go:828: 2m0s of 5m0s: 468945 executions, 290530 successful
    rsg_test.go:828: 2m5s of 5m0s: 488197 executions, 302404 successful
    rsg_test.go:828: 2m10s of 5m0s: 507984 executions, 314557 successful
    rsg_test.go:828: 2m15s of 5m0s: 527693 executions, 326726 successful
    rsg_test.go:828: 2m20s of 5m0s: 548925 executions, 339934 successful
    rsg_test.go:828: 2m25s of 5m0s: 568331 executions, 352033 successful
    rsg_test.go:828: 2m30s of 5m0s: 588223 executions, 364401 successful
    rsg_test.go:828: 2m35s of 5m0s: 606505 executions, 375705 successful
    rsg_test.go:828: 2m40s of 5m0s: 625594 executions, 387625 successful
    rsg_test.go:828: 2m45s of 5m0s: 645760 executions, 400129 successful
    rsg_test.go:828: 2m50s of 5m0s: 665937 executions, 412560 successful
    rsg_test.go:828: 2m55s of 5m0s: 685451 executions, 424597 successful
    rsg_test.go:828: 3m0s of 5m0s: 705304 executions, 436999 successful
    rsg_test.go:828: 3m5s of 5m0s: 724929 executions, 449076 successful
    rsg_test.go:828: 3m10s of 5m0s: 744095 executions, 460934 successful
    rsg_test.go:828: 3m15s of 5m0s: 761824 executions, 471933 successful
    rsg_test.go:828: 3m20s of 5m0s: 781945 executions, 484478 successful
    rsg_test.go:828: 3m25s of 5m0s: 802290 executions, 496985 successful
    rsg_test.go:828: 3m30s of 5m0s: 823628 executions, 510284 successful
    rsg_test.go:828: 3m35s of 5m0s: 846824 executions, 524757 successful
    rsg_test.go:828: 3m40s of 5m0s: 867709 executions, 537691 successful
    rsg_test.go:828: 3m45s of 5m0s: 888414 executions, 550539 successful
    rsg_test.go:828: 3m50s of 5m0s: 909987 executions, 563898 successful
    rsg_test.go:828: 3m55s of 5m0s: 933314 executions, 578464 successful
    rsg_test.go:828: 4m0s of 5m0s: 953108 executions, 590780 successful
    rsg_test.go:828: 4m5s of 5m0s: 972655 executions, 602831 successful
    rsg_test.go:828: 4m10s of 5m0s: 992579 executions, 615237 successful
    rsg_test.go:828: 4m15s of 5m0s: 1014885 executions, 629084 successful
    rsg_test.go:828: 4m20s of 5m0s: 1033207 executions, 640518 successful
    rsg_test.go:828: 4m25s of 5m0s: 1055372 executions, 654136 successful
    rsg_test.go:828: 4m30s of 5m0s: 1075402 executions, 666614 successful
    rsg_test.go:828: 4m35s of 5m0s: 1095989 executions, 679322 successful
    rsg_test.go:828: 4m40s of 5m0s: 1116186 executions, 691832 successful
    rsg_test.go:828: 4m45s of 5m0s: 1136643 executions, 704397 successful
    rsg_test.go:828: 4m50s of 5m0s: 1158255 executions, 717830 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9554159?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9554159?buildTab=artifacts#/) on release-23.1 @ [c8a4703f9853a442ac676b44d074b43eb387f60c](https://github.com/cockroachdb/cockroach/commits/c8a4703f9853a442ac676b44d074b43eb387f60c):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 15116313 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:828: 45s of 5m0s: 177977 executions, 110190 successful
    rsg_test.go:828: 50s of 5m0s: 197379 executions, 122271 successful
    rsg_test.go:828: 55s of 5m0s: 218521 executions, 135338 successful
    rsg_test.go:828: 1m0s of 5m0s: 240563 executions, 149137 successful
    rsg_test.go:828: 1m5s of 5m0s: 261794 executions, 162283 successful
    rsg_test.go:828: 1m10s of 5m0s: 282106 executions, 174907 successful
    rsg_test.go:828: 1m15s of 5m0s: 304659 executions, 188901 successful
    rsg_test.go:828: 1m20s of 5m0s: 322542 executions, 199986 successful
    rsg_test.go:828: 1m25s of 5m0s: 343477 executions, 212962 successful
    rsg_test.go:828: 1m30s of 5m0s: 363962 executions, 225628 successful
    rsg_test.go:828: 1m35s of 5m0s: 384657 executions, 238507 successful
    rsg_test.go:828: 1m40s of 5m0s: 403330 executions, 250014 successful
    rsg_test.go:828: 1m45s of 5m0s: 421559 executions, 261193 successful
    rsg_test.go:828: 1m50s of 5m0s: 441904 executions, 273884 successful
    rsg_test.go:828: 1m55s of 5m0s: 464063 executions, 287518 successful
    rsg_test.go:828: 2m0s of 5m0s: 485214 executions, 300637 successful
    rsg_test.go:828: 2m5s of 5m0s: 503604 executions, 312093 successful
    rsg_test.go:828: 2m10s of 5m0s: 520133 executions, 322181 successful
    rsg_test.go:828: 2m15s of 5m0s: 539758 executions, 334382 successful
    rsg_test.go:828: 2m20s of 5m0s: 560803 executions, 347365 successful
    rsg_test.go:828: 2m25s of 5m0s: 581843 executions, 360376 successful
    rsg_test.go:828: 2m30s of 5m0s: 602619 executions, 373268 successful
    rsg_test.go:828: 2m35s of 5m0s: 621909 executions, 385279 successful
    rsg_test.go:828: 2m40s of 5m0s: 643649 executions, 398907 successful
    rsg_test.go:828: 2m45s of 5m0s: 663540 executions, 411098 successful
    rsg_test.go:828: 2m50s of 5m0s: 682387 executions, 422843 successful
    rsg_test.go:828: 2m55s of 5m0s: 703386 executions, 435887 successful
    rsg_test.go:828: 3m0s of 5m0s: 724483 executions, 449007 successful
    rsg_test.go:828: 3m5s of 5m0s: 745450 executions, 462043 successful
    rsg_test.go:828: 3m10s of 5m0s: 766518 executions, 475143 successful
    rsg_test.go:828: 3m15s of 5m0s: 790813 executions, 490146 successful
    rsg_test.go:828: 3m20s of 5m0s: 812102 executions, 503407 successful
    rsg_test.go:828: 3m25s of 5m0s: 832780 executions, 516278 successful
    rsg_test.go:828: 3m30s of 5m0s: 854947 executions, 529943 successful
    rsg_test.go:828: 3m35s of 5m0s: 873902 executions, 541607 successful
    rsg_test.go:828: 3m40s of 5m0s: 890084 executions, 551543 successful
    rsg_test.go:828: 3m45s of 5m0s: 911803 executions, 565002 successful
    rsg_test.go:828: 3m50s of 5m0s: 931830 executions, 577538 successful
    rsg_test.go:828: 3m55s of 5m0s: 948554 executions, 587875 successful
    rsg_test.go:828: 4m0s of 5m0s: 969196 executions, 600708 successful
    rsg_test.go:828: 4m5s of 5m0s: 990721 executions, 613975 successful
    rsg_test.go:828: 4m10s of 5m0s: 1009136 executions, 625311 successful
    rsg_test.go:828: 4m15s of 5m0s: 1028596 executions, 637448 successful
    rsg_test.go:828: 4m20s of 5m0s: 1049080 executions, 650186 successful
    rsg_test.go:828: 4m25s of 5m0s: 1069315 executions, 662691 successful
    rsg_test.go:828: 4m30s of 5m0s: 1089099 executions, 675055 successful
    rsg_test.go:828: 4m35s of 5m0s: 1107617 executions, 686493 successful
    rsg_test.go:828: 4m40s of 5m0s: 1126950 executions, 698487 successful
    rsg_test.go:828: 4m45s of 5m0s: 1150267 executions, 712957 successful
    rsg_test.go:828: 4m50s of 5m0s: 1169577 executions, 724933 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9578006?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9578006?buildTab=artifacts#/) on release-23.1 @ [cf84c1c11f2f9c1a4a979033fa0af3befb2ccbcf](https://github.com/cockroachdb/cockroach/commits/cf84c1c11f2f9c1a4a979033fa0af3befb2ccbcf):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 12932073 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:828: 45s of 5m0s: 148441 executions, 91974 successful
    rsg_test.go:828: 50s of 5m0s: 166982 executions, 103467 successful
    rsg_test.go:828: 55s of 5m0s: 185055 executions, 114733 successful
    rsg_test.go:828: 1m0s of 5m0s: 203789 executions, 126400 successful
    rsg_test.go:828: 1m5s of 5m0s: 223138 executions, 138348 successful
    rsg_test.go:828: 1m10s of 5m0s: 241023 executions, 149289 successful
    rsg_test.go:828: 1m15s of 5m0s: 260918 executions, 161653 successful
    rsg_test.go:828: 1m20s of 5m0s: 279827 executions, 173381 successful
    rsg_test.go:828: 1m25s of 5m0s: 295900 executions, 183415 successful
    rsg_test.go:828: 1m30s of 5m0s: 313587 executions, 194369 successful
    rsg_test.go:828: 1m35s of 5m0s: 332626 executions, 206211 successful
    rsg_test.go:828: 1m40s of 5m0s: 351836 executions, 218104 successful
    rsg_test.go:828: 1m45s of 5m0s: 370493 executions, 229541 successful
    rsg_test.go:828: 1m50s of 5m0s: 389956 executions, 241660 successful
    rsg_test.go:828: 1m55s of 5m0s: 406816 executions, 252066 successful
    rsg_test.go:828: 2m0s of 5m0s: 423004 executions, 262236 successful
    rsg_test.go:828: 2m5s of 5m0s: 440459 executions, 273007 successful
    rsg_test.go:828: 2m10s of 5m0s: 455746 executions, 282532 successful
    rsg_test.go:828: 2m15s of 5m0s: 475245 executions, 294543 successful
    rsg_test.go:828: 2m20s of 5m0s: 493191 executions, 305729 successful
    rsg_test.go:828: 2m25s of 5m0s: 513596 executions, 318355 successful
    rsg_test.go:828: 2m30s of 5m0s: 531714 executions, 329590 successful
    rsg_test.go:828: 2m35s of 5m0s: 550305 executions, 341035 successful
    rsg_test.go:828: 2m40s of 5m0s: 567524 executions, 351688 successful
    rsg_test.go:828: 2m45s of 5m0s: 588001 executions, 364492 successful
    rsg_test.go:828: 2m50s of 5m0s: 609680 executions, 377860 successful
    rsg_test.go:828: 2m55s of 5m0s: 626550 executions, 388336 successful
    rsg_test.go:828: 3m0s of 5m0s: 647180 executions, 401101 successful
    rsg_test.go:828: 3m5s of 5m0s: 670689 executions, 415683 successful
    rsg_test.go:828: 3m10s of 5m0s: 690990 executions, 428271 successful
    rsg_test.go:828: 3m15s of 5m0s: 714148 executions, 442628 successful
    rsg_test.go:828: 3m20s of 5m0s: 732084 executions, 453819 successful
    rsg_test.go:828: 3m25s of 5m0s: 753319 executions, 466933 successful
    rsg_test.go:828: 3m30s of 5m0s: 774915 executions, 480203 successful
    rsg_test.go:828: 3m35s of 5m0s: 798688 executions, 494937 successful
    rsg_test.go:828: 3m40s of 5m0s: 820419 executions, 508492 successful
    rsg_test.go:828: 3m45s of 5m0s: 840117 executions, 520638 successful
    rsg_test.go:828: 3m50s of 5m0s: 862315 executions, 534538 successful
    rsg_test.go:828: 3m55s of 5m0s: 885190 executions, 548656 successful
    rsg_test.go:828: 4m0s of 5m0s: 904804 executions, 560853 successful
    rsg_test.go:828: 4m5s of 5m0s: 923285 executions, 572203 successful
    rsg_test.go:828: 4m10s of 5m0s: 942953 executions, 584495 successful
    rsg_test.go:828: 4m15s of 5m0s: 964042 executions, 597656 successful
    rsg_test.go:828: 4m20s of 5m0s: 986019 executions, 611192 successful
    rsg_test.go:828: 4m25s of 5m0s: 1006781 executions, 624162 successful
    rsg_test.go:828: 4m30s of 5m0s: 1029732 executions, 638395 successful
    rsg_test.go:828: 4m35s of 5m0s: 1048748 executions, 650180 successful
    rsg_test.go:828: 4m40s of 5m0s: 1072207 executions, 664655 successful
    rsg_test.go:828: 4m45s of 5m0s: 1093621 executions, 677949 successful
    rsg_test.go:828: 4m50s of 5m0s: 1113403 executions, 690229 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9616175?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9616175?buildTab=artifacts#/) on release-23.1 @ [84905b8396e7cd7ef999b264faaafe13110d631a](https://github.com/cockroachdb/cockroach/commits/84905b8396e7cd7ef999b264faaafe13110d631a):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 16192664 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:828: 45s of 5m0s: 178820 executions, 110870 successful
    rsg_test.go:828: 50s of 5m0s: 200969 executions, 124543 successful
    rsg_test.go:828: 55s of 5m0s: 220520 executions, 136648 successful
    rsg_test.go:828: 1m0s of 5m0s: 240202 executions, 148939 successful
    rsg_test.go:828: 1m5s of 5m0s: 260078 executions, 161182 successful
    rsg_test.go:828: 1m10s of 5m0s: 279914 executions, 173515 successful
    rsg_test.go:828: 1m15s of 5m0s: 302881 executions, 187733 successful
    rsg_test.go:828: 1m20s of 5m0s: 324052 executions, 200915 successful
    rsg_test.go:828: 1m25s of 5m0s: 343713 executions, 213125 successful
    rsg_test.go:828: 1m30s of 5m0s: 366511 executions, 227268 successful
    rsg_test.go:828: 1m35s of 5m0s: 388969 executions, 241188 successful
    rsg_test.go:828: 1m40s of 5m0s: 411030 executions, 254942 successful
    rsg_test.go:828: 1m45s of 5m0s: 434637 executions, 269640 successful
    rsg_test.go:828: 1m50s of 5m0s: 457520 executions, 283748 successful
    rsg_test.go:828: 1m55s of 5m0s: 475685 executions, 294997 successful
    rsg_test.go:828: 2m0s of 5m0s: 496963 executions, 308153 successful
    rsg_test.go:828: 2m5s of 5m0s: 518708 executions, 321655 successful
    rsg_test.go:828: 2m10s of 5m0s: 537523 executions, 333330 successful
    rsg_test.go:828: 2m15s of 5m0s: 560407 executions, 347533 successful
    rsg_test.go:828: 2m20s of 5m0s: 580258 executions, 359905 successful
    rsg_test.go:828: 2m25s of 5m0s: 603517 executions, 374281 successful
    rsg_test.go:828: 2m30s of 5m0s: 625994 executions, 388199 successful
    rsg_test.go:828: 2m35s of 5m0s: 648394 executions, 402119 successful
    rsg_test.go:828: 2m40s of 5m0s: 668525 executions, 414654 successful
    rsg_test.go:828: 2m45s of 5m0s: 690226 executions, 427984 successful
    rsg_test.go:828: 2m50s of 5m0s: 710891 executions, 440946 successful
    rsg_test.go:828: 2m55s of 5m0s: 730321 executions, 452956 successful
    rsg_test.go:828: 3m0s of 5m0s: 753262 executions, 467214 successful
    rsg_test.go:828: 3m5s of 5m0s: 776895 executions, 481796 successful
    rsg_test.go:828: 3m10s of 5m0s: 798058 executions, 494901 successful
    rsg_test.go:828: 3m15s of 5m0s: 817924 executions, 507187 successful
    rsg_test.go:828: 3m20s of 5m0s: 840939 executions, 521471 successful
    rsg_test.go:828: 3m25s of 5m0s: 861347 executions, 534177 successful
    rsg_test.go:828: 3m30s of 5m0s: 882868 executions, 547542 successful
    rsg_test.go:828: 3m35s of 5m0s: 905379 executions, 561475 successful
    rsg_test.go:828: 3m40s of 5m0s: 923988 executions, 573045 successful
    rsg_test.go:828: 3m45s of 5m0s: 945937 executions, 586513 successful
    rsg_test.go:828: 3m50s of 5m0s: 970288 executions, 601636 successful
    rsg_test.go:828: 3m55s of 5m0s: 990341 executions, 613946 successful
    rsg_test.go:828: 4m0s of 5m0s: 1013960 executions, 628600 successful
    rsg_test.go:828: 4m5s of 5m0s: 1034149 executions, 641084 successful
    rsg_test.go:828: 4m10s of 5m0s: 1056246 executions, 654831 successful
    rsg_test.go:828: 4m15s of 5m0s: 1075906 executions, 666993 successful
    rsg_test.go:828: 4m20s of 5m0s: 1097006 executions, 680047 successful
    rsg_test.go:828: 4m25s of 5m0s: 1121590 executions, 695316 successful
    rsg_test.go:828: 4m30s of 5m0s: 1145960 executions, 710348 successful
    rsg_test.go:828: 4m35s of 5m0s: 1163556 executions, 721266 successful
    rsg_test.go:828: 4m40s of 5m0s: 1186436 executions, 735433 successful
    rsg_test.go:828: 4m45s of 5m0s: 1207834 executions, 748730 successful
    rsg_test.go:828: 4m50s of 5m0s: 1228820 executions, 761673 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9638540?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9638540?buildTab=artifacts#/) on release-23.1 @ [6cce6c746150307a9ecf3b529dfd633a6985c110](https://github.com/cockroachdb/cockroach/commits/6cce6c746150307a9ecf3b529dfd633a6985c110):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 15806050 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:828: 45s of 5m0s: 195576 executions, 121340 successful
    rsg_test.go:828: 50s of 5m0s: 216655 executions, 134543 successful
    rsg_test.go:828: 55s of 5m0s: 237971 executions, 147752 successful
    rsg_test.go:828: 1m0s of 5m0s: 259295 executions, 160980 successful
    rsg_test.go:828: 1m5s of 5m0s: 279042 executions, 173251 successful
    rsg_test.go:828: 1m10s of 5m0s: 300607 executions, 186507 successful
    rsg_test.go:828: 1m15s of 5m0s: 319927 executions, 198479 successful
    rsg_test.go:828: 1m20s of 5m0s: 339156 executions, 210360 successful
    rsg_test.go:828: 1m25s of 5m0s: 357855 executions, 222032 successful
    rsg_test.go:828: 1m30s of 5m0s: 377067 executions, 233954 successful
    rsg_test.go:828: 1m35s of 5m0s: 399436 executions, 247800 successful
    rsg_test.go:828: 1m40s of 5m0s: 421036 executions, 261230 successful
    rsg_test.go:828: 1m45s of 5m0s: 441471 executions, 273858 successful
    rsg_test.go:828: 1m50s of 5m0s: 465769 executions, 288877 successful
    rsg_test.go:828: 1m55s of 5m0s: 486375 executions, 301757 successful
    rsg_test.go:828: 2m0s of 5m0s: 508527 executions, 315451 successful
    rsg_test.go:828: 2m5s of 5m0s: 529936 executions, 328688 successful
    rsg_test.go:828: 2m10s of 5m0s: 550635 executions, 341505 successful
    rsg_test.go:828: 2m15s of 5m0s: 569712 executions, 353332 successful
    rsg_test.go:828: 2m20s of 5m0s: 588132 executions, 364713 successful
    rsg_test.go:828: 2m25s of 5m0s: 607344 executions, 376620 successful
    rsg_test.go:828: 2m30s of 5m0s: 626783 executions, 388639 successful
    rsg_test.go:828: 2m35s of 5m0s: 650379 executions, 403201 successful
    rsg_test.go:828: 2m40s of 5m0s: 670519 executions, 415776 successful
    rsg_test.go:828: 2m45s of 5m0s: 692514 executions, 429323 successful
    rsg_test.go:828: 2m50s of 5m0s: 715368 executions, 443561 successful
    rsg_test.go:828: 2m55s of 5m0s: 737631 executions, 457491 successful
    rsg_test.go:828: 3m0s of 5m0s: 757497 executions, 469697 successful
    rsg_test.go:828: 3m5s of 5m0s: 779255 executions, 483268 successful
    rsg_test.go:828: 3m10s of 5m0s: 797199 executions, 494403 successful
    rsg_test.go:828: 3m15s of 5m0s: 818975 executions, 507850 successful
    rsg_test.go:828: 3m20s of 5m0s: 839446 executions, 520692 successful
    rsg_test.go:828: 3m25s of 5m0s: 861111 executions, 534048 successful
    rsg_test.go:828: 3m30s of 5m0s: 882344 executions, 547193 successful
    rsg_test.go:828: 3m35s of 5m0s: 901963 executions, 559432 successful
    rsg_test.go:828: 3m40s of 5m0s: 924007 executions, 573133 successful
    rsg_test.go:828: 3m45s of 5m0s: 945155 executions, 586307 successful
    rsg_test.go:828: 3m50s of 5m0s: 966875 executions, 599564 successful
    rsg_test.go:828: 3m55s of 5m0s: 988918 executions, 613331 successful
    rsg_test.go:828: 4m0s of 5m0s: 1011112 executions, 627040 successful
    rsg_test.go:828: 4m5s of 5m0s: 1027470 executions, 637217 successful
    rsg_test.go:828: 4m10s of 5m0s: 1048471 executions, 650254 successful
    rsg_test.go:828: 4m15s of 5m0s: 1068881 executions, 662781 successful
    rsg_test.go:828: 4m20s of 5m0s: 1089250 executions, 675400 successful
    rsg_test.go:828: 4m25s of 5m0s: 1113701 executions, 690532 successful
    rsg_test.go:828: 4m30s of 5m0s: 1137073 executions, 705016 successful
    rsg_test.go:828: 4m35s of 5m0s: 1158440 executions, 718204 successful
    rsg_test.go:828: 4m40s of 5m0s: 1179069 executions, 731008 successful
    rsg_test.go:828: 4m45s of 5m0s: 1201234 executions, 744661 successful
    rsg_test.go:828: 4m50s of 5m0s: 1222480 executions, 757900 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9680375?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9680375?buildTab=artifacts#/) on release-23.1 @ [2f86475f2e3d713b4c8c976c34dba99acce7c4c7](https://github.com/cockroachdb/cockroach/commits/2f86475f2e3d713b4c8c976c34dba99acce7c4c7):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 13640417 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRandomSyntaxFunctions
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxFunctions2596570674
    test_log_scope.go:79: use -show-logs to present logs inline
    rsg_test.go:828: 5s of 5m0s: 19106 executions, 11802 successful
    rsg_test.go:828: 10s of 5m0s: 40787 executions, 25350 successful
    rsg_test.go:828: 15s of 5m0s: 64115 executions, 39785 successful
    rsg_test.go:828: 20s of 5m0s: 85855 executions, 53291 successful
    rsg_test.go:828: 25s of 5m0s: 107120 executions, 66507 successful
    rsg_test.go:828: 30s of 5m0s: 128383 executions, 79633 successful
    rsg_test.go:828: 35s of 5m0s: 151056 executions, 93678 successful
    rsg_test.go:828: 40s of 5m0s: 170763 executions, 105881 successful
    rsg_test.go:828: 45s of 5m0s: 188294 executions, 116793 successful
    rsg_test.go:828: 50s of 5m0s: 209229 executions, 129708 successful
    rsg_test.go:828: 55s of 5m0s: 232320 executions, 143980 successful
    rsg_test.go:828: 1m0s of 5m0s: 252605 executions, 156502 successful
    rsg_test.go:828: 1m5s of 5m0s: 274513 executions, 170143 successful
    rsg_test.go:828: 1m10s of 5m0s: 295499 executions, 183174 successful
    rsg_test.go:828: 1m15s of 5m0s: 312179 executions, 193478 successful
    rsg_test.go:828: 1m20s of 5m0s: 334765 executions, 207560 successful
    rsg_test.go:828: 1m25s of 5m0s: 357311 executions, 221570 successful
    rsg_test.go:828: 1m30s of 5m0s: 378662 executions, 234816 successful
    rsg_test.go:828: 1m35s of 5m0s: 399083 executions, 247378 successful
    rsg_test.go:828: 1m40s of 5m0s: 418530 executions, 259413 successful
    rsg_test.go:828: 1m45s of 5m0s: 440121 executions, 272879 successful
    rsg_test.go:828: 1m50s of 5m0s: 461183 executions, 285861 successful
    rsg_test.go:828: 1m55s of 5m0s: 485037 executions, 300611 successful
    rsg_test.go:828: 2m0s of 5m0s: 505711 executions, 313516 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9699497?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9699497?buildTab=artifacts#/) on release-23.1 @ [ad6ce866ea3b5c5bb47ba9a0ac19b721a0c98add](https://github.com/cockroachdb/cockroach/commits/ad6ce866ea3b5c5bb47ba9a0ac19b721a0c98add):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 15018462 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRandomSyntaxFunctions
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxFunctions3386607560
    test_log_scope.go:79: use -show-logs to present logs inline
    rsg_test.go:828: 5s of 5m0s: 20997 executions, 13067 successful
    rsg_test.go:828: 10s of 5m0s: 44421 executions, 27637 successful
    rsg_test.go:828: 15s of 5m0s: 63165 executions, 39192 successful
    rsg_test.go:828: 20s of 5m0s: 85243 executions, 52897 successful
    rsg_test.go:828: 25s of 5m0s: 103698 executions, 64502 successful
    rsg_test.go:828: 30s of 5m0s: 125569 executions, 78007 successful
    rsg_test.go:828: 35s of 5m0s: 148329 executions, 92071 successful
    rsg_test.go:828: 40s of 5m0s: 175513 executions, 109021 successful
    rsg_test.go:828: 45s of 5m0s: 198341 executions, 123192 successful
    rsg_test.go:828: 50s of 5m0s: 222198 executions, 138070 successful
    rsg_test.go:828: 55s of 5m0s: 242438 executions, 150537 successful
    rsg_test.go:828: 1m0s of 5m0s: 263052 executions, 163295 successful
    rsg_test.go:828: 1m5s of 5m0s: 284140 executions, 176464 successful
    rsg_test.go:828: 1m10s of 5m0s: 306458 executions, 190312 successful
    rsg_test.go:828: 1m15s of 5m0s: 328014 executions, 203663 successful
    rsg_test.go:828: 1m20s of 5m0s: 353341 executions, 219543 successful
    rsg_test.go:828: 1m25s of 5m0s: 373286 executions, 232017 successful
    rsg_test.go:828: 1m30s of 5m0s: 394928 executions, 245387 successful
    rsg_test.go:828: 1m35s of 5m0s: 416891 executions, 259063 successful
    rsg_test.go:828: 1m40s of 5m0s: 439503 executions, 273122 successful
    rsg_test.go:828: 1m45s of 5m0s: 459936 executions, 285786 successful
    rsg_test.go:828: 1m50s of 5m0s: 481432 executions, 299234 successful
    rsg_test.go:828: 1m55s of 5m0s: 505442 executions, 314165 successful
    rsg_test.go:828: 2m0s of 5m0s: 525665 executions, 326702 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9721258?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9721258?buildTab=artifacts#/) on release-23.1 @ [2c34f145b83f771667baf3f7ef8633e7e1cd2ad6](https://github.com/cockroachdb/cockroach/commits/2c34f145b83f771667baf3f7ef8633e7e1cd2ad6):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 15001700 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:828: 45s of 5m0s: 168857 executions, 105049 successful
    rsg_test.go:828: 50s of 5m0s: 184887 executions, 114978 successful
    rsg_test.go:828: 55s of 5m0s: 201392 executions, 125236 successful
    rsg_test.go:828: 1m0s of 5m0s: 218667 executions, 136050 successful
    rsg_test.go:828: 1m5s of 5m0s: 236918 executions, 147367 successful
    rsg_test.go:828: 1m10s of 5m0s: 255614 executions, 158975 successful
    rsg_test.go:828: 1m15s of 5m0s: 272551 executions, 169434 successful
    rsg_test.go:828: 1m20s of 5m0s: 289460 executions, 180030 successful
    rsg_test.go:828: 1m25s of 5m0s: 306826 executions, 190782 successful
    rsg_test.go:828: 1m30s of 5m0s: 325020 executions, 202109 successful
    rsg_test.go:828: 1m35s of 5m0s: 345523 executions, 214901 successful
    rsg_test.go:828: 1m40s of 5m0s: 363200 executions, 225909 successful
    rsg_test.go:828: 1m45s of 5m0s: 384656 executions, 239224 successful
    rsg_test.go:828: 1m50s of 5m0s: 404386 executions, 251335 successful
    rsg_test.go:828: 1m55s of 5m0s: 423563 executions, 263359 successful
    rsg_test.go:828: 2m0s of 5m0s: 441638 executions, 274648 successful
    rsg_test.go:828: 2m5s of 5m0s: 463190 executions, 288107 successful
    rsg_test.go:828: 2m10s of 5m0s: 480895 executions, 299017 successful
    rsg_test.go:828: 2m15s of 5m0s: 500145 executions, 310905 successful
    rsg_test.go:828: 2m20s of 5m0s: 520522 executions, 323618 successful
    rsg_test.go:828: 2m25s of 5m0s: 538473 executions, 334615 successful
    rsg_test.go:828: 2m30s of 5m0s: 555665 executions, 345262 successful
    rsg_test.go:828: 2m35s of 5m0s: 575982 executions, 358026 successful
    rsg_test.go:828: 2m40s of 5m0s: 595007 executions, 369855 successful
    rsg_test.go:828: 2m45s of 5m0s: 615637 executions, 382603 successful
    rsg_test.go:828: 2m50s of 5m0s: 636755 executions, 395673 successful
    rsg_test.go:828: 2m55s of 5m0s: 657554 executions, 408633 successful
    rsg_test.go:828: 3m0s of 5m0s: 678024 executions, 421364 successful
    rsg_test.go:828: 3m5s of 5m0s: 701441 executions, 435820 successful
    rsg_test.go:828: 3m10s of 5m0s: 723064 executions, 449318 successful
    rsg_test.go:828: 3m15s of 5m0s: 744549 executions, 462504 successful
    rsg_test.go:828: 3m20s of 5m0s: 767833 executions, 476908 successful
    rsg_test.go:828: 3m25s of 5m0s: 787759 executions, 489222 successful
    rsg_test.go:828: 3m30s of 5m0s: 811171 executions, 503838 successful
    rsg_test.go:828: 3m35s of 5m0s: 831144 executions, 516212 successful
    rsg_test.go:828: 3m40s of 5m0s: 854396 executions, 530659 successful
    rsg_test.go:828: 3m45s of 5m0s: 873677 executions, 542614 successful
    rsg_test.go:828: 3m50s of 5m0s: 894638 executions, 555706 successful
    rsg_test.go:828: 3m55s of 5m0s: 914027 executions, 567813 successful
    rsg_test.go:828: 4m0s of 5m0s: 935198 executions, 581004 successful
    rsg_test.go:828: 4m5s of 5m0s: 955213 executions, 593291 successful
    rsg_test.go:828: 4m10s of 5m0s: 976174 executions, 606293 successful
    rsg_test.go:828: 4m15s of 5m0s: 998466 executions, 620293 successful
    rsg_test.go:828: 4m20s of 5m0s: 1018755 executions, 632828 successful
    rsg_test.go:828: 4m25s of 5m0s: 1039558 executions, 645738 successful
    rsg_test.go:828: 4m30s of 5m0s: 1059878 executions, 658338 successful
    rsg_test.go:828: 4m35s of 5m0s: 1082418 executions, 672347 successful
    rsg_test.go:828: 4m40s of 5m0s: 1100072 executions, 683283 successful
    rsg_test.go:828: 4m45s of 5m0s: 1121151 executions, 696377 successful
    rsg_test.go:828: 4m50s of 5m0s: 1142518 executions, 709553 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9749939?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9749939?buildTab=artifacts#/) on release-23.1 @ [e82cf4ec488258875dcf5e42e7d6745b03210edd](https://github.com/cockroachdb/cockroach/commits/e82cf4ec488258875dcf5e42e7d6745b03210edd):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 14881675 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:828: 45s of 5m0s: 177403 executions, 110186 successful
    rsg_test.go:828: 50s of 5m0s: 198391 executions, 123188 successful
    rsg_test.go:828: 55s of 5m0s: 215897 executions, 133984 successful
    rsg_test.go:828: 1m0s of 5m0s: 234900 executions, 145869 successful
    rsg_test.go:828: 1m5s of 5m0s: 255742 executions, 158941 successful
    rsg_test.go:828: 1m10s of 5m0s: 275567 executions, 171280 successful
    rsg_test.go:828: 1m15s of 5m0s: 296045 executions, 183974 successful
    rsg_test.go:828: 1m20s of 5m0s: 317574 executions, 197405 successful
    rsg_test.go:828: 1m25s of 5m0s: 334646 executions, 207967 successful
    rsg_test.go:828: 1m30s of 5m0s: 354169 executions, 220149 successful
    rsg_test.go:828: 1m35s of 5m0s: 375827 executions, 233455 successful
    rsg_test.go:828: 1m40s of 5m0s: 396398 executions, 246332 successful
    rsg_test.go:828: 1m45s of 5m0s: 416108 executions, 258436 successful
    rsg_test.go:828: 1m50s of 5m0s: 438524 executions, 272372 successful
    rsg_test.go:828: 1m55s of 5m0s: 456737 executions, 283586 successful
    rsg_test.go:828: 2m0s of 5m0s: 478527 executions, 297121 successful
    rsg_test.go:828: 2m5s of 5m0s: 500824 executions, 310915 successful
    rsg_test.go:828: 2m10s of 5m0s: 518662 executions, 322045 successful
    rsg_test.go:828: 2m15s of 5m0s: 538819 executions, 334527 successful
    rsg_test.go:828: 2m20s of 5m0s: 560103 executions, 347691 successful
    rsg_test.go:828: 2m25s of 5m0s: 581733 executions, 361255 successful
    rsg_test.go:828: 2m30s of 5m0s: 598241 executions, 371412 successful
    rsg_test.go:828: 2m35s of 5m0s: 616123 executions, 382636 successful
    rsg_test.go:828: 2m40s of 5m0s: 637212 executions, 395790 successful
    rsg_test.go:828: 2m45s of 5m0s: 660836 executions, 410352 successful
    rsg_test.go:828: 2m50s of 5m0s: 682459 executions, 423806 successful
    rsg_test.go:828: 2m55s of 5m0s: 703399 executions, 436759 successful
    rsg_test.go:828: 3m0s of 5m0s: 723229 executions, 449160 successful
    rsg_test.go:828: 3m5s of 5m0s: 742960 executions, 461365 successful
    rsg_test.go:828: 3m10s of 5m0s: 761539 executions, 473003 successful
    rsg_test.go:828: 3m15s of 5m0s: 782637 executions, 486151 successful
    rsg_test.go:828: 3m20s of 5m0s: 801174 executions, 497644 successful
    rsg_test.go:828: 3m25s of 5m0s: 823469 executions, 511498 successful
    rsg_test.go:828: 3m30s of 5m0s: 843913 executions, 524229 successful
    rsg_test.go:828: 3m35s of 5m0s: 863784 executions, 536545 successful
    rsg_test.go:828: 3m40s of 5m0s: 881923 executions, 547949 successful
    rsg_test.go:828: 3m45s of 5m0s: 903057 executions, 561086 successful
    rsg_test.go:828: 3m50s of 5m0s: 924962 executions, 574702 successful
    rsg_test.go:828: 3m55s of 5m0s: 943944 executions, 586509 successful
    rsg_test.go:828: 4m0s of 5m0s: 960866 executions, 596948 successful
    rsg_test.go:828: 4m5s of 5m0s: 979626 executions, 608662 successful
    rsg_test.go:828: 4m10s of 5m0s: 999373 executions, 620950 successful
    rsg_test.go:828: 4m15s of 5m0s: 1020593 executions, 634184 successful
    rsg_test.go:828: 4m20s of 5m0s: 1040825 executions, 646688 successful
    rsg_test.go:828: 4m25s of 5m0s: 1062738 executions, 660281 successful
    rsg_test.go:828: 4m30s of 5m0s: 1081923 executions, 672181 successful
    rsg_test.go:828: 4m35s of 5m0s: 1101166 executions, 684115 successful
    rsg_test.go:828: 4m40s of 5m0s: 1121647 executions, 696881 successful
    rsg_test.go:828: 4m45s of 5m0s: 1141687 executions, 709376 successful
    rsg_test.go:828: 4m50s of 5m0s: 1161023 executions, 721447 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9782033?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9782033?buildTab=artifacts#/) on release-23.1 @ [fa10e980c24442f18e688bea904ece4777ba0ca9](https://github.com/cockroachdb/cockroach/commits/fa10e980c24442f18e688bea904ece4777ba0ca9):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 14741807 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:828: 45s of 5m0s: 178967 executions, 111185 successful
    rsg_test.go:828: 50s of 5m0s: 198557 executions, 123434 successful
    rsg_test.go:828: 55s of 5m0s: 219523 executions, 136409 successful
    rsg_test.go:828: 1m0s of 5m0s: 238653 executions, 148295 successful
    rsg_test.go:828: 1m5s of 5m0s: 259653 executions, 161290 successful
    rsg_test.go:828: 1m10s of 5m0s: 281451 executions, 174944 successful
    rsg_test.go:828: 1m15s of 5m0s: 303886 executions, 188877 successful
    rsg_test.go:828: 1m20s of 5m0s: 321994 executions, 200106 successful
    rsg_test.go:828: 1m25s of 5m0s: 342371 executions, 212749 successful
    rsg_test.go:828: 1m30s of 5m0s: 361522 executions, 224667 successful
    rsg_test.go:828: 1m35s of 5m0s: 383965 executions, 238529 successful
    rsg_test.go:828: 1m40s of 5m0s: 403491 executions, 250667 successful
    rsg_test.go:828: 1m45s of 5m0s: 422390 executions, 262359 successful
    rsg_test.go:828: 1m50s of 5m0s: 441404 executions, 274222 successful
    rsg_test.go:828: 1m55s of 5m0s: 461807 executions, 286840 successful
    rsg_test.go:828: 2m0s of 5m0s: 482177 executions, 299617 successful
    rsg_test.go:828: 2m5s of 5m0s: 499043 executions, 310053 successful
    rsg_test.go:828: 2m10s of 5m0s: 518217 executions, 321990 successful
    rsg_test.go:828: 2m15s of 5m0s: 539107 executions, 334957 successful
    rsg_test.go:828: 2m20s of 5m0s: 559399 executions, 347564 successful
    rsg_test.go:828: 2m25s of 5m0s: 581061 executions, 361080 successful
    rsg_test.go:828: 2m30s of 5m0s: 601423 executions, 373771 successful
    rsg_test.go:828: 2m35s of 5m0s: 624615 executions, 388297 successful
    rsg_test.go:828: 2m40s of 5m0s: 645283 executions, 401123 successful
    rsg_test.go:828: 2m45s of 5m0s: 665391 executions, 413679 successful
    rsg_test.go:828: 2m50s of 5m0s: 687136 executions, 427225 successful
    rsg_test.go:828: 2m55s of 5m0s: 705245 executions, 438429 successful
    rsg_test.go:828: 3m0s of 5m0s: 721868 executions, 448738 successful
    rsg_test.go:828: 3m5s of 5m0s: 744595 executions, 462877 successful
    rsg_test.go:828: 3m10s of 5m0s: 766498 executions, 476401 successful
    rsg_test.go:828: 3m15s of 5m0s: 785509 executions, 488185 successful
    rsg_test.go:828: 3m20s of 5m0s: 805545 executions, 500619 successful
    rsg_test.go:828: 3m25s of 5m0s: 827775 executions, 514429 successful
    rsg_test.go:828: 3m30s of 5m0s: 845198 executions, 525289 successful
    rsg_test.go:828: 3m35s of 5m0s: 868691 executions, 539935 successful
    rsg_test.go:828: 3m40s of 5m0s: 889730 executions, 553140 successful
    rsg_test.go:828: 3m45s of 5m0s: 908919 executions, 565002 successful
    rsg_test.go:828: 3m50s of 5m0s: 928898 executions, 577371 successful
    rsg_test.go:828: 3m55s of 5m0s: 947833 executions, 589099 successful
    rsg_test.go:828: 4m0s of 5m0s: 966380 executions, 600592 successful
    rsg_test.go:828: 4m5s of 5m0s: 987194 executions, 613508 successful
    rsg_test.go:828: 4m10s of 5m0s: 1007292 executions, 625897 successful
    rsg_test.go:828: 4m15s of 5m0s: 1026199 executions, 637633 successful
    rsg_test.go:828: 4m20s of 5m0s: 1046294 executions, 650183 successful
    rsg_test.go:828: 4m25s of 5m0s: 1064945 executions, 661713 successful
    rsg_test.go:828: 4m30s of 5m0s: 1084956 executions, 674198 successful
    rsg_test.go:828: 4m35s of 5m0s: 1105584 executions, 687100 successful
    rsg_test.go:828: 4m40s of 5m0s: 1123521 executions, 698154 successful
    rsg_test.go:828: 4m45s of 5m0s: 1143701 executions, 710783 successful
    rsg_test.go:828: 4m50s of 5m0s: 1165127 executions, 724101 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9803999?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9803999?buildTab=artifacts#/) on release-23.1 @ [d329422358d677325c818eee10d23f35b080c877](https://github.com/cockroachdb/cockroach/commits/d329422358d677325c818eee10d23f35b080c877):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 16262489 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:828: 45s of 5m0s: 182198 executions, 113204 successful
    rsg_test.go:828: 50s of 5m0s: 202873 executions, 126048 successful
    rsg_test.go:828: 55s of 5m0s: 221721 executions, 137717 successful
    rsg_test.go:828: 1m0s of 5m0s: 242520 executions, 150632 successful
    rsg_test.go:828: 1m5s of 5m0s: 262310 executions, 163045 successful
    rsg_test.go:828: 1m10s of 5m0s: 283485 executions, 176161 successful
    rsg_test.go:828: 1m15s of 5m0s: 303491 executions, 188573 successful
    rsg_test.go:828: 1m20s of 5m0s: 325539 executions, 202348 successful
    rsg_test.go:828: 1m25s of 5m0s: 345768 executions, 214855 successful
    rsg_test.go:828: 1m30s of 5m0s: 366199 executions, 227600 successful
    rsg_test.go:828: 1m35s of 5m0s: 383997 executions, 238687 successful
    rsg_test.go:828: 1m40s of 5m0s: 405141 executions, 251775 successful
    rsg_test.go:828: 1m45s of 5m0s: 424202 executions, 263662 successful
    rsg_test.go:828: 1m50s of 5m0s: 441356 executions, 274396 successful
    rsg_test.go:828: 1m55s of 5m0s: 459163 executions, 285528 successful
    rsg_test.go:828: 2m0s of 5m0s: 478429 executions, 297457 successful
    rsg_test.go:828: 2m5s of 5m0s: 496497 executions, 308631 successful
    rsg_test.go:828: 2m10s of 5m0s: 515989 executions, 320667 successful
    rsg_test.go:828: 2m15s of 5m0s: 536536 executions, 333395 successful
    rsg_test.go:828: 2m20s of 5m0s: 554957 executions, 344998 successful
    rsg_test.go:828: 2m25s of 5m0s: 573924 executions, 356764 successful
    rsg_test.go:828: 2m30s of 5m0s: 595522 executions, 370052 successful
    rsg_test.go:828: 2m35s of 5m0s: 616287 executions, 383012 successful
    rsg_test.go:828: 2m40s of 5m0s: 636526 executions, 395411 successful
    rsg_test.go:828: 2m45s of 5m0s: 658040 executions, 408897 successful
    rsg_test.go:828: 2m50s of 5m0s: 678741 executions, 421673 successful
    rsg_test.go:828: 2m55s of 5m0s: 700038 executions, 434965 successful
    rsg_test.go:828: 3m0s of 5m0s: 716356 executions, 445054 successful
    rsg_test.go:828: 3m5s of 5m0s: 736542 executions, 457715 successful
    rsg_test.go:828: 3m10s of 5m0s: 757801 executions, 470858 successful
    rsg_test.go:828: 3m15s of 5m0s: 776854 executions, 482748 successful
    rsg_test.go:828: 3m20s of 5m0s: 798190 executions, 495883 successful
    rsg_test.go:828: 3m25s of 5m0s: 820895 executions, 509999 successful
    rsg_test.go:828: 3m30s of 5m0s: 839241 executions, 521345 successful
    rsg_test.go:828: 3m35s of 5m0s: 862069 executions, 535572 successful
    rsg_test.go:828: 3m40s of 5m0s: 882681 executions, 548422 successful
    rsg_test.go:828: 3m45s of 5m0s: 904219 executions, 561844 successful
    rsg_test.go:828: 3m50s of 5m0s: 926033 executions, 575379 successful
    rsg_test.go:828: 3m55s of 5m0s: 947631 executions, 588705 successful
    rsg_test.go:828: 4m0s of 5m0s: 968848 executions, 601888 successful
    rsg_test.go:828: 4m5s of 5m0s: 989014 executions, 614498 successful
    rsg_test.go:828: 4m10s of 5m0s: 1009218 executions, 627030 successful
    rsg_test.go:828: 4m15s of 5m0s: 1032136 executions, 641342 successful
    rsg_test.go:828: 4m20s of 5m0s: 1051480 executions, 653354 successful
    rsg_test.go:828: 4m25s of 5m0s: 1072888 executions, 666758 successful
    rsg_test.go:828: 4m30s of 5m0s: 1094389 executions, 680207 successful
    rsg_test.go:828: 4m35s of 5m0s: 1117624 executions, 694647 successful
    rsg_test.go:828: 4m40s of 5m0s: 1138659 executions, 707731 successful
    rsg_test.go:828: 4m45s of 5m0s: 1162172 executions, 722337 successful
    rsg_test.go:828: 4m50s of 5m0s: 1183430 executions, 735596 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9825983?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9825983?buildTab=artifacts#/) on release-23.1 @ [6636edf9d8c0ebf9cdeedeeffd563ecb49da5b8a](https://github.com/cockroachdb/cockroach/commits/6636edf9d8c0ebf9cdeedeeffd563ecb49da5b8a):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 13860446 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:828: 45s of 5m0s: 158531 executions, 98637 successful
    rsg_test.go:828: 50s of 5m0s: 177845 executions, 110692 successful
    rsg_test.go:828: 55s of 5m0s: 193760 executions, 120609 successful
    rsg_test.go:828: 1m0s of 5m0s: 211606 executions, 131704 successful
    rsg_test.go:828: 1m5s of 5m0s: 228549 executions, 142288 successful
    rsg_test.go:828: 1m10s of 5m0s: 247705 executions, 154095 successful
    rsg_test.go:828: 1m15s of 5m0s: 263927 executions, 164198 successful
    rsg_test.go:828: 1m20s of 5m0s: 283588 executions, 176488 successful
    rsg_test.go:828: 1m25s of 5m0s: 301468 executions, 187499 successful
    rsg_test.go:828: 1m30s of 5m0s: 321054 executions, 199714 successful
    rsg_test.go:828: 1m35s of 5m0s: 338163 executions, 210296 successful
    rsg_test.go:828: 1m40s of 5m0s: 354249 executions, 220425 successful
    rsg_test.go:828: 1m45s of 5m0s: 372413 executions, 231728 successful
    rsg_test.go:828: 1m50s of 5m0s: 390082 executions, 242734 successful
    rsg_test.go:828: 1m55s of 5m0s: 410366 executions, 255297 successful
    rsg_test.go:828: 2m0s of 5m0s: 432096 executions, 268772 successful
    rsg_test.go:828: 2m5s of 5m0s: 449698 executions, 279790 successful
    rsg_test.go:828: 2m10s of 5m0s: 467027 executions, 290567 successful
    rsg_test.go:828: 2m15s of 5m0s: 485920 executions, 302310 successful
    rsg_test.go:828: 2m20s of 5m0s: 504528 executions, 313786 successful
    rsg_test.go:828: 2m25s of 5m0s: 524266 executions, 326149 successful
    rsg_test.go:828: 2m30s of 5m0s: 545018 executions, 338923 successful
    rsg_test.go:828: 2m35s of 5m0s: 562003 executions, 349449 successful
    rsg_test.go:828: 2m40s of 5m0s: 581672 executions, 361764 successful
    rsg_test.go:828: 2m45s of 5m0s: 602894 executions, 375011 successful
    rsg_test.go:828: 2m50s of 5m0s: 624655 executions, 388489 successful
    rsg_test.go:828: 2m55s of 5m0s: 642850 executions, 399833 successful
    rsg_test.go:828: 3m0s of 5m0s: 663591 executions, 412719 successful
    rsg_test.go:828: 3m5s of 5m0s: 681111 executions, 423654 successful
    rsg_test.go:828: 3m10s of 5m0s: 700383 executions, 435639 successful
    rsg_test.go:828: 3m15s of 5m0s: 717371 executions, 446246 successful
    rsg_test.go:828: 3m20s of 5m0s: 737140 executions, 458445 successful
    rsg_test.go:828: 3m25s of 5m0s: 755943 executions, 470219 successful
    rsg_test.go:828: 3m30s of 5m0s: 775864 executions, 482560 successful
    rsg_test.go:828: 3m35s of 5m0s: 792039 executions, 492632 successful
    rsg_test.go:828: 3m40s of 5m0s: 810307 executions, 503915 successful
    rsg_test.go:828: 3m45s of 5m0s: 828582 executions, 515320 successful
    rsg_test.go:828: 3m50s of 5m0s: 845718 executions, 525916 successful
    rsg_test.go:828: 3m55s of 5m0s: 863882 executions, 537217 successful
    rsg_test.go:828: 4m0s of 5m0s: 884042 executions, 549883 successful
    rsg_test.go:828: 4m5s of 5m0s: 901597 executions, 560770 successful
    rsg_test.go:828: 4m10s of 5m0s: 922593 executions, 573812 successful
    rsg_test.go:828: 4m15s of 5m0s: 942236 executions, 586062 successful
    rsg_test.go:828: 4m20s of 5m0s: 961978 executions, 598422 successful
    rsg_test.go:828: 4m25s of 5m0s: 981541 executions, 610548 successful
    rsg_test.go:828: 4m30s of 5m0s: 1002125 executions, 623400 successful
    rsg_test.go:828: 4m35s of 5m0s: 1021545 executions, 635498 successful
    rsg_test.go:828: 4m40s of 5m0s: 1040829 executions, 647479 successful
    rsg_test.go:828: 4m45s of 5m0s: 1058454 executions, 658318 successful
    rsg_test.go:828: 4m50s of 5m0s: 1079121 executions, 671218 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9848442?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9848442?buildTab=artifacts#/) on release-23.1 @ [7f6458093d42bb7131cef623deb47f6318a233e9](https://github.com/cockroachdb/cockroach/commits/7f6458093d42bb7131cef623deb47f6318a233e9):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 12784918 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:828: 45s of 5m0s: 193096 executions, 119880 successful
    rsg_test.go:828: 50s of 5m0s: 216440 executions, 134479 successful
    rsg_test.go:828: 55s of 5m0s: 237240 executions, 147368 successful
    rsg_test.go:828: 1m0s of 5m0s: 254504 executions, 158070 successful
    rsg_test.go:828: 1m5s of 5m0s: 270295 executions, 167865 successful
    rsg_test.go:828: 1m10s of 5m0s: 290547 executions, 180419 successful
    rsg_test.go:828: 1m15s of 5m0s: 310426 executions, 192861 successful
    rsg_test.go:828: 1m20s of 5m0s: 330733 executions, 205449 successful
    rsg_test.go:828: 1m25s of 5m0s: 350407 executions, 217775 successful
    rsg_test.go:828: 1m30s of 5m0s: 368547 executions, 228940 successful
    rsg_test.go:828: 1m35s of 5m0s: 391752 executions, 243384 successful
    rsg_test.go:828: 1m40s of 5m0s: 414563 executions, 257605 successful
    rsg_test.go:828: 1m45s of 5m0s: 433780 executions, 269526 successful
    rsg_test.go:828: 1m50s of 5m0s: 454994 executions, 282799 successful
    rsg_test.go:828: 1m55s of 5m0s: 473382 executions, 294148 successful
    rsg_test.go:828: 2m0s of 5m0s: 491167 executions, 305191 successful
    rsg_test.go:828: 2m5s of 5m0s: 513542 executions, 319073 successful
    rsg_test.go:828: 2m10s of 5m0s: 532218 executions, 330745 successful
    rsg_test.go:828: 2m15s of 5m0s: 553677 executions, 344070 successful
    rsg_test.go:828: 2m20s of 5m0s: 572288 executions, 355725 successful
    rsg_test.go:828: 2m25s of 5m0s: 593427 executions, 368904 successful
    rsg_test.go:828: 2m30s of 5m0s: 617013 executions, 383559 successful
    rsg_test.go:828: 2m35s of 5m0s: 640539 executions, 398231 successful
    rsg_test.go:828: 2m40s of 5m0s: 663147 executions, 412330 successful
    rsg_test.go:828: 2m45s of 5m0s: 685330 executions, 426190 successful
    rsg_test.go:828: 2m50s of 5m0s: 703617 executions, 437582 successful
    rsg_test.go:828: 2m55s of 5m0s: 726209 executions, 451622 successful
    rsg_test.go:828: 3m0s of 5m0s: 746991 executions, 464553 successful
    rsg_test.go:828: 3m5s of 5m0s: 769066 executions, 478368 successful
    rsg_test.go:828: 3m10s of 5m0s: 795108 executions, 494495 successful
    rsg_test.go:828: 3m15s of 5m0s: 815141 executions, 506962 successful
    rsg_test.go:828: 3m20s of 5m0s: 833359 executions, 518211 successful
    rsg_test.go:828: 3m25s of 5m0s: 854935 executions, 531679 successful
    rsg_test.go:828: 3m30s of 5m0s: 875904 executions, 544671 successful
    rsg_test.go:828: 3m35s of 5m0s: 895861 executions, 556917 successful
    rsg_test.go:828: 3m40s of 5m0s: 917042 executions, 570080 successful
    rsg_test.go:828: 3m45s of 5m0s: 939918 executions, 584296 successful
    rsg_test.go:828: 3m50s of 5m0s: 960292 executions, 597144 successful
    rsg_test.go:828: 3m55s of 5m0s: 980856 executions, 609827 successful
    rsg_test.go:828: 4m0s of 5m0s: 999015 executions, 621054 successful
    rsg_test.go:828: 4m5s of 5m0s: 1017856 executions, 632847 successful
    rsg_test.go:828: 4m10s of 5m0s: 1036026 executions, 644056 successful
    rsg_test.go:828: 4m15s of 5m0s: 1060178 executions, 659014 successful
    rsg_test.go:828: 4m20s of 5m0s: 1084510 executions, 674180 successful
    rsg_test.go:828: 4m25s of 5m0s: 1104902 executions, 686741 successful
    rsg_test.go:828: 4m30s of 5m0s: 1128751 executions, 701646 successful
    rsg_test.go:828: 4m35s of 5m0s: 1147717 executions, 713463 successful
    rsg_test.go:828: 4m40s of 5m0s: 1168765 executions, 726558 successful
    rsg_test.go:828: 4m45s of 5m0s: 1191877 executions, 740897 successful
    rsg_test.go:828: 4m50s of 5m0s: 1213425 executions, 754263 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9868382?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9868382?buildTab=artifacts#/) on release-23.1 @ [f0e37ada00cf99a1fa25236e158861647c2ea713](https://github.com/cockroachdb/cockroach/commits/f0e37ada00cf99a1fa25236e158861647c2ea713):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 8420944 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:828: 45s of 5m0s: 200920 executions, 124783 successful
    rsg_test.go:828: 50s of 5m0s: 218757 executions, 135940 successful
    rsg_test.go:828: 55s of 5m0s: 239682 executions, 148951 successful
    rsg_test.go:828: 1m0s of 5m0s: 259375 executions, 161096 successful
    rsg_test.go:828: 1m5s of 5m0s: 279635 executions, 173756 successful
    rsg_test.go:828: 1m10s of 5m0s: 300740 executions, 186910 successful
    rsg_test.go:828: 1m15s of 5m0s: 317268 executions, 197148 successful
    rsg_test.go:828: 1m20s of 5m0s: 336258 executions, 208916 successful
    rsg_test.go:828: 1m25s of 5m0s: 357319 executions, 222031 successful
    rsg_test.go:828: 1m30s of 5m0s: 377619 executions, 234656 successful
    rsg_test.go:828: 1m35s of 5m0s: 392421 executions, 243913 successful
    rsg_test.go:828: 1m40s of 5m0s: 411828 executions, 255939 successful
    rsg_test.go:828: 1m45s of 5m0s: 432499 executions, 268795 successful
    rsg_test.go:828: 1m50s of 5m0s: 451333 executions, 280463 successful
    rsg_test.go:828: 1m55s of 5m0s: 470152 executions, 292046 successful
    rsg_test.go:828: 2m0s of 5m0s: 490797 executions, 304853 successful
    rsg_test.go:828: 2m5s of 5m0s: 512545 executions, 318395 successful
    rsg_test.go:828: 2m10s of 5m0s: 531935 executions, 330461 successful
    rsg_test.go:828: 2m15s of 5m0s: 551991 executions, 342881 successful
    rsg_test.go:828: 2m20s of 5m0s: 569501 executions, 353708 successful
    rsg_test.go:828: 2m25s of 5m0s: 591887 executions, 367559 successful
    rsg_test.go:828: 2m30s of 5m0s: 611386 executions, 379707 successful
    rsg_test.go:828: 2m35s of 5m0s: 634012 executions, 393782 successful
    rsg_test.go:828: 2m40s of 5m0s: 654617 executions, 406608 successful
    rsg_test.go:828: 2m45s of 5m0s: 677337 executions, 420759 successful
    rsg_test.go:828: 2m50s of 5m0s: 699469 executions, 434489 successful
    rsg_test.go:828: 2m55s of 5m0s: 718462 executions, 446353 successful
    rsg_test.go:828: 3m0s of 5m0s: 737298 executions, 457992 successful
    rsg_test.go:828: 3m5s of 5m0s: 758526 executions, 471227 successful
    rsg_test.go:828: 3m10s of 5m0s: 777879 executions, 483223 successful
    rsg_test.go:828: 3m15s of 5m0s: 797671 executions, 495587 successful
    rsg_test.go:828: 3m20s of 5m0s: 820043 executions, 509466 successful
    rsg_test.go:828: 3m25s of 5m0s: 840697 executions, 522253 successful
    rsg_test.go:828: 3m30s of 5m0s: 861310 executions, 535041 successful
    rsg_test.go:828: 3m35s of 5m0s: 881820 executions, 547894 successful
    rsg_test.go:828: 3m40s of 5m0s: 901867 executions, 560275 successful
    rsg_test.go:828: 3m45s of 5m0s: 922789 executions, 573271 successful
    rsg_test.go:828: 3m50s of 5m0s: 943734 executions, 586230 successful
    rsg_test.go:828: 3m55s of 5m0s: 962881 executions, 598151 successful
    rsg_test.go:828: 4m0s of 5m0s: 983602 executions, 611107 successful
    rsg_test.go:828: 4m5s of 5m0s: 1002419 executions, 622860 successful
    rsg_test.go:828: 4m10s of 5m0s: 1022127 executions, 635025 successful
    rsg_test.go:828: 4m15s of 5m0s: 1041795 executions, 647283 successful
    rsg_test.go:828: 4m20s of 5m0s: 1061923 executions, 659790 successful
    rsg_test.go:828: 4m25s of 5m0s: 1082649 executions, 672710 successful
    rsg_test.go:828: 4m30s of 5m0s: 1104051 executions, 685925 successful
    rsg_test.go:828: 4m35s of 5m0s: 1124390 executions, 698593 successful
    rsg_test.go:828: 4m40s of 5m0s: 1144094 executions, 710847 successful
    rsg_test.go:828: 4m45s of 5m0s: 1167680 executions, 725448 successful
    rsg_test.go:828: 4m50s of 5m0s: 1188060 executions, 738154 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9880118?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9880118?buildTab=artifacts#/) on release-23.1 @ [6944c293f14d5c1c5b28d6ea53329c3e214dd9ea](https://github.com/cockroachdb/cockroach/commits/6944c293f14d5c1c5b28d6ea53329c3e214dd9ea):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 14241424 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:828: 45s of 5m0s: 183245 executions, 113934 successful
    rsg_test.go:828: 50s of 5m0s: 203643 executions, 126523 successful
    rsg_test.go:828: 55s of 5m0s: 225245 executions, 139990 successful
    rsg_test.go:828: 1m0s of 5m0s: 245346 executions, 152502 successful
    rsg_test.go:828: 1m5s of 5m0s: 263733 executions, 163848 successful
    rsg_test.go:828: 1m10s of 5m0s: 284307 executions, 176697 successful
    rsg_test.go:828: 1m15s of 5m0s: 302279 executions, 187903 successful
    rsg_test.go:828: 1m20s of 5m0s: 319364 executions, 198489 successful
    rsg_test.go:828: 1m25s of 5m0s: 339213 executions, 210794 successful
    rsg_test.go:828: 1m30s of 5m0s: 357359 executions, 221955 successful
    rsg_test.go:828: 1m35s of 5m0s: 378979 executions, 235501 successful
    rsg_test.go:828: 1m40s of 5m0s: 398145 executions, 247453 successful
    rsg_test.go:828: 1m45s of 5m0s: 418372 executions, 259996 successful
    rsg_test.go:828: 1m50s of 5m0s: 440937 executions, 273976 successful
    rsg_test.go:828: 1m55s of 5m0s: 461278 executions, 286608 successful
    rsg_test.go:828: 2m0s of 5m0s: 481293 executions, 298978 successful
    rsg_test.go:828: 2m5s of 5m0s: 503087 executions, 312633 successful
    rsg_test.go:828: 2m10s of 5m0s: 525479 executions, 326452 successful
    rsg_test.go:828: 2m15s of 5m0s: 544274 executions, 338168 successful
    rsg_test.go:828: 2m20s of 5m0s: 563086 executions, 349771 successful
    rsg_test.go:828: 2m25s of 5m0s: 584279 executions, 362985 successful
    rsg_test.go:828: 2m30s of 5m0s: 606087 executions, 376595 successful
    rsg_test.go:828: 2m35s of 5m0s: 623641 executions, 387593 successful
    rsg_test.go:828: 2m40s of 5m0s: 643342 executions, 399767 successful
    rsg_test.go:828: 2m45s of 5m0s: 664715 executions, 413067 successful
    rsg_test.go:828: 2m50s of 5m0s: 682848 executions, 424432 successful
    rsg_test.go:828: 2m55s of 5m0s: 704182 executions, 437667 successful
    rsg_test.go:828: 3m0s of 5m0s: 725490 executions, 450880 successful
    rsg_test.go:828: 3m5s of 5m0s: 747226 executions, 464423 successful
    rsg_test.go:828: 3m10s of 5m0s: 765219 executions, 475654 successful
    rsg_test.go:828: 3m15s of 5m0s: 783262 executions, 486828 successful
    rsg_test.go:828: 3m20s of 5m0s: 803447 executions, 499402 successful
    rsg_test.go:828: 3m25s of 5m0s: 822931 executions, 511482 successful
    rsg_test.go:828: 3m30s of 5m0s: 840942 executions, 522585 successful
    rsg_test.go:828: 3m35s of 5m0s: 859535 executions, 534057 successful
    rsg_test.go:828: 3m40s of 5m0s: 882056 executions, 548144 successful
    rsg_test.go:828: 3m45s of 5m0s: 901947 executions, 560476 successful
    rsg_test.go:828: 3m50s of 5m0s: 918691 executions, 570857 successful
    rsg_test.go:828: 3m55s of 5m0s: 941131 executions, 584928 successful
    rsg_test.go:828: 4m0s of 5m0s: 959516 executions, 596312 successful
    rsg_test.go:828: 4m5s of 5m0s: 980618 executions, 609415 successful
    rsg_test.go:828: 4m10s of 5m0s: 997217 executions, 619716 successful
    rsg_test.go:828: 4m15s of 5m0s: 1014492 executions, 630443 successful
    rsg_test.go:828: 4m20s of 5m0s: 1035104 executions, 643321 successful
    rsg_test.go:828: 4m25s of 5m0s: 1055558 executions, 655977 successful
    rsg_test.go:828: 4m30s of 5m0s: 1073602 executions, 667135 successful
    rsg_test.go:828: 4m35s of 5m0s: 1093537 executions, 679524 successful
    rsg_test.go:828: 4m40s of 5m0s: 1114068 executions, 692334 successful
    rsg_test.go:828: 4m45s of 5m0s: 1137048 executions, 706659 successful
    rsg_test.go:828: 4m50s of 5m0s: 1153717 executions, 717034 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9890712?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9890712?buildTab=artifacts#/) on release-23.1 @ [fa8d237846052ae72cc44b3eaebe1f38c89834f8](https://github.com/cockroachdb/cockroach/commits/fa8d237846052ae72cc44b3eaebe1f38c89834f8):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 12743405 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:828: 45s of 5m0s: 162421 executions, 100697 successful
    rsg_test.go:828: 50s of 5m0s: 181676 executions, 112679 successful
    rsg_test.go:828: 55s of 5m0s: 200216 executions, 124166 successful
    rsg_test.go:828: 1m0s of 5m0s: 220163 executions, 136548 successful
    rsg_test.go:828: 1m5s of 5m0s: 240592 executions, 149124 successful
    rsg_test.go:828: 1m10s of 5m0s: 258790 executions, 160495 successful
    rsg_test.go:828: 1m15s of 5m0s: 277702 executions, 172237 successful
    rsg_test.go:828: 1m20s of 5m0s: 295936 executions, 183534 successful
    rsg_test.go:828: 1m25s of 5m0s: 313965 executions, 194736 successful
    rsg_test.go:828: 1m30s of 5m0s: 329156 executions, 204095 successful
    rsg_test.go:828: 1m35s of 5m0s: 347658 executions, 215507 successful
    rsg_test.go:828: 1m40s of 5m0s: 364195 executions, 225801 successful
    rsg_test.go:828: 1m45s of 5m0s: 383293 executions, 237672 successful
    rsg_test.go:828: 1m50s of 5m0s: 401741 executions, 249212 successful
    rsg_test.go:828: 1m55s of 5m0s: 419385 executions, 260167 successful
    rsg_test.go:828: 2m0s of 5m0s: 433915 executions, 269172 successful
    rsg_test.go:828: 2m5s of 5m0s: 454175 executions, 281755 successful
    rsg_test.go:828: 2m10s of 5m0s: 475615 executions, 295137 successful
    rsg_test.go:828: 2m15s of 5m0s: 492988 executions, 305917 successful
    rsg_test.go:828: 2m20s of 5m0s: 511172 executions, 317290 successful
    rsg_test.go:828: 2m25s of 5m0s: 528271 executions, 327887 successful
    rsg_test.go:828: 2m30s of 5m0s: 545672 executions, 338706 successful
    rsg_test.go:828: 2m35s of 5m0s: 566410 executions, 351674 successful
    rsg_test.go:828: 2m40s of 5m0s: 587663 executions, 364823 successful
    rsg_test.go:828: 2m45s of 5m0s: 606765 executions, 376653 successful
    rsg_test.go:828: 2m50s of 5m0s: 624902 executions, 387855 successful
    rsg_test.go:828: 2m55s of 5m0s: 644764 executions, 400274 successful
    rsg_test.go:828: 3m0s of 5m0s: 662327 executions, 411180 successful
    rsg_test.go:828: 3m5s of 5m0s: 683064 executions, 424166 successful
    rsg_test.go:828: 3m10s of 5m0s: 700571 executions, 435078 successful
    rsg_test.go:828: 3m15s of 5m0s: 720578 executions, 447466 successful
    rsg_test.go:828: 3m20s of 5m0s: 737565 executions, 458072 successful
    rsg_test.go:828: 3m25s of 5m0s: 755198 executions, 468977 successful
    rsg_test.go:828: 3m30s of 5m0s: 773739 executions, 480517 successful
    rsg_test.go:828: 3m35s of 5m0s: 790160 executions, 490806 successful
    rsg_test.go:828: 3m40s of 5m0s: 807592 executions, 501545 successful
    rsg_test.go:828: 3m45s of 5m0s: 824789 executions, 512269 successful
    rsg_test.go:828: 3m50s of 5m0s: 843942 executions, 524167 successful
    rsg_test.go:828: 3m55s of 5m0s: 863405 executions, 536183 successful
    rsg_test.go:828: 4m0s of 5m0s: 886642 executions, 550669 successful
    rsg_test.go:828: 4m5s of 5m0s: 904075 executions, 561454 successful
    rsg_test.go:828: 4m10s of 5m0s: 922751 executions, 573125 successful
    rsg_test.go:828: 4m15s of 5m0s: 941543 executions, 584828 successful
    rsg_test.go:828: 4m20s of 5m0s: 958322 executions, 595335 successful
    rsg_test.go:828: 4m25s of 5m0s: 976963 executions, 606870 successful
    rsg_test.go:828: 4m30s of 5m0s: 995501 executions, 618367 successful
    rsg_test.go:828: 4m35s of 5m0s: 1013037 executions, 629313 successful
    rsg_test.go:828: 4m40s of 5m0s: 1034524 executions, 642690 successful
    rsg_test.go:828: 4m45s of 5m0s: 1050173 executions, 652413 successful
    rsg_test.go:828: 4m50s of 5m0s: 1068782 executions, 663931 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9904312?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9904312?buildTab=artifacts#/) on release-23.1 @ [82fddc52747a86c696a1f77a85abc25d05858cc1](https://github.com/cockroachdb/cockroach/commits/82fddc52747a86c696a1f77a85abc25d05858cc1):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 13989673 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:828: 45s of 5m0s: 185541 executions, 115420 successful
    rsg_test.go:828: 50s of 5m0s: 207801 executions, 129281 successful
    rsg_test.go:828: 55s of 5m0s: 228606 executions, 142209 successful
    rsg_test.go:828: 1m0s of 5m0s: 247328 executions, 153930 successful
    rsg_test.go:828: 1m5s of 5m0s: 267121 executions, 166254 successful
    rsg_test.go:828: 1m10s of 5m0s: 287085 executions, 178743 successful
    rsg_test.go:828: 1m15s of 5m0s: 307867 executions, 191609 successful
    rsg_test.go:828: 1m20s of 5m0s: 328568 executions, 204459 successful
    rsg_test.go:828: 1m25s of 5m0s: 347645 executions, 216329 successful
    rsg_test.go:828: 1m30s of 5m0s: 367816 executions, 228855 successful
    rsg_test.go:828: 1m35s of 5m0s: 388081 executions, 241410 successful
    rsg_test.go:828: 1m40s of 5m0s: 408895 executions, 254334 successful
    rsg_test.go:828: 1m45s of 5m0s: 425798 executions, 264807 successful
    rsg_test.go:828: 1m50s of 5m0s: 447162 executions, 278072 successful
    rsg_test.go:828: 1m55s of 5m0s: 467161 executions, 290586 successful
    rsg_test.go:828: 2m0s of 5m0s: 486097 executions, 302352 successful
    rsg_test.go:828: 2m5s of 5m0s: 505082 executions, 314157 successful
    rsg_test.go:828: 2m10s of 5m0s: 527717 executions, 328183 successful
    rsg_test.go:828: 2m15s of 5m0s: 547361 executions, 340488 successful
    rsg_test.go:828: 2m20s of 5m0s: 567193 executions, 352774 successful
    rsg_test.go:828: 2m25s of 5m0s: 589206 executions, 366548 successful
    rsg_test.go:828: 2m30s of 5m0s: 608179 executions, 378310 successful
    rsg_test.go:828: 2m35s of 5m0s: 628537 executions, 390973 successful
    rsg_test.go:828: 2m40s of 5m0s: 648627 executions, 403492 successful
    rsg_test.go:828: 2m45s of 5m0s: 669522 executions, 416462 successful
    rsg_test.go:828: 2m50s of 5m0s: 689750 executions, 428960 successful
    rsg_test.go:828: 2m55s of 5m0s: 710508 executions, 441846 successful
    rsg_test.go:828: 3m0s of 5m0s: 731549 executions, 454946 successful
    rsg_test.go:828: 3m5s of 5m0s: 751898 executions, 467623 successful
    rsg_test.go:828: 3m10s of 5m0s: 774685 executions, 481792 successful
    rsg_test.go:828: 3m15s of 5m0s: 791226 executions, 492018 successful
    rsg_test.go:828: 3m20s of 5m0s: 812003 executions, 504950 successful
    rsg_test.go:828: 3m25s of 5m0s: 834251 executions, 518820 successful
    rsg_test.go:828: 3m30s of 5m0s: 854451 executions, 531298 successful
    rsg_test.go:828: 3m35s of 5m0s: 873456 executions, 543165 successful
    rsg_test.go:828: 3m40s of 5m0s: 894282 executions, 556121 successful
    rsg_test.go:828: 3m45s of 5m0s: 911672 executions, 566995 successful
    rsg_test.go:828: 3m50s of 5m0s: 929842 executions, 578429 successful
    rsg_test.go:828: 3m55s of 5m0s: 946946 executions, 589128 successful
    rsg_test.go:828: 4m0s of 5m0s: 967767 executions, 602183 successful
    rsg_test.go:828: 4m5s of 5m0s: 989289 executions, 615572 successful
    rsg_test.go:828: 4m10s of 5m0s: 1011783 executions, 629572 successful
    rsg_test.go:828: 4m15s of 5m0s: 1030874 executions, 641411 successful
    rsg_test.go:828: 4m20s of 5m0s: 1053240 executions, 655308 successful
    rsg_test.go:828: 4m25s of 5m0s: 1074285 executions, 668465 successful
    rsg_test.go:828: 4m30s of 5m0s: 1094717 executions, 681137 successful
    rsg_test.go:828: 4m35s of 5m0s: 1114135 executions, 693140 successful
    rsg_test.go:828: 4m40s of 5m0s: 1137063 executions, 707358 successful
    rsg_test.go:828: 4m45s of 5m0s: 1159806 executions, 721451 successful
    rsg_test.go:828: 4m50s of 5m0s: 1180682 executions, 734399 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9928279?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9928279?buildTab=artifacts#/) on release-23.1 @ [fbea48d5cd399ab1889ae8ce76b11f521d914686](https://github.com/cockroachdb/cockroach/commits/fbea48d5cd399ab1889ae8ce76b11f521d914686):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 15347610 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:828: 45s of 5m0s: 188900 executions, 117328 successful
    rsg_test.go:828: 50s of 5m0s: 209773 executions, 130266 successful
    rsg_test.go:828: 55s of 5m0s: 232525 executions, 144432 successful
    rsg_test.go:828: 1m0s of 5m0s: 254950 executions, 158456 successful
    rsg_test.go:828: 1m5s of 5m0s: 274846 executions, 170808 successful
    rsg_test.go:828: 1m10s of 5m0s: 295755 executions, 183786 successful
    rsg_test.go:828: 1m15s of 5m0s: 317824 executions, 197584 successful
    rsg_test.go:828: 1m20s of 5m0s: 336367 executions, 209055 successful
    rsg_test.go:828: 1m25s of 5m0s: 357386 executions, 222068 successful
    rsg_test.go:828: 1m30s of 5m0s: 378762 executions, 235371 successful
    rsg_test.go:828: 1m35s of 5m0s: 397879 executions, 247225 successful
    rsg_test.go:828: 1m40s of 5m0s: 420324 executions, 261112 successful
    rsg_test.go:828: 1m45s of 5m0s: 443710 executions, 275633 successful
    rsg_test.go:828: 1m50s of 5m0s: 464664 executions, 288649 successful
    rsg_test.go:828: 1m55s of 5m0s: 487295 executions, 302748 successful
    rsg_test.go:828: 2m0s of 5m0s: 506169 executions, 314508 successful
    rsg_test.go:828: 2m5s of 5m0s: 524950 executions, 326206 successful
    rsg_test.go:828: 2m10s of 5m0s: 545897 executions, 339304 successful
    rsg_test.go:828: 2m15s of 5m0s: 565531 executions, 351492 successful
    rsg_test.go:828: 2m20s of 5m0s: 587586 executions, 365213 successful
    rsg_test.go:828: 2m25s of 5m0s: 611341 executions, 380023 successful
    rsg_test.go:828: 2m30s of 5m0s: 636443 executions, 395614 successful
    rsg_test.go:828: 2m35s of 5m0s: 657430 executions, 408691 successful
    rsg_test.go:828: 2m40s of 5m0s: 679362 executions, 422306 successful
    rsg_test.go:828: 2m45s of 5m0s: 697471 executions, 433499 successful
    rsg_test.go:828: 2m50s of 5m0s: 718216 executions, 446319 successful
    rsg_test.go:828: 2m55s of 5m0s: 740553 executions, 460289 successful
    rsg_test.go:828: 3m0s of 5m0s: 760587 executions, 472651 successful
    rsg_test.go:828: 3m5s of 5m0s: 780536 executions, 485149 successful
    rsg_test.go:828: 3m10s of 5m0s: 800299 executions, 497298 successful
    rsg_test.go:828: 3m15s of 5m0s: 823817 executions, 512012 successful
    rsg_test.go:828: 3m20s of 5m0s: 845035 executions, 525171 successful
    rsg_test.go:828: 3m25s of 5m0s: 869638 executions, 540369 successful
    rsg_test.go:828: 3m30s of 5m0s: 891908 executions, 554210 successful
    rsg_test.go:828: 3m35s of 5m0s: 916225 executions, 569405 successful
    rsg_test.go:828: 3m40s of 5m0s: 938124 executions, 582946 successful
    rsg_test.go:828: 3m45s of 5m0s: 957346 executions, 594895 successful
    rsg_test.go:828: 3m50s of 5m0s: 978412 executions, 608018 successful
    rsg_test.go:828: 3m55s of 5m0s: 1001428 executions, 622313 successful
    rsg_test.go:828: 4m0s of 5m0s: 1020759 executions, 634335 successful
    rsg_test.go:828: 4m5s of 5m0s: 1040313 executions, 646569 successful
    rsg_test.go:828: 4m10s of 5m0s: 1062356 executions, 660192 successful
    rsg_test.go:828: 4m15s of 5m0s: 1081295 executions, 671986 successful
    rsg_test.go:828: 4m20s of 5m0s: 1102213 executions, 685032 successful
    rsg_test.go:828: 4m25s of 5m0s: 1120914 executions, 696724 successful
    rsg_test.go:828: 4m30s of 5m0s: 1143122 executions, 710470 successful
    rsg_test.go:828: 4m35s of 5m0s: 1164636 executions, 723965 successful
    rsg_test.go:828: 4m40s of 5m0s: 1186141 executions, 737367 successful
    rsg_test.go:828: 4m45s of 5m0s: 1205694 executions, 749464 successful
    rsg_test.go:828: 4m50s of 5m0s: 1229325 executions, 764197 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9940976?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9940976?buildTab=artifacts#/) on release-23.1 @ [26713f81a397d13db4f2257eaa0f6bb70d1a7cb3](https://github.com/cockroachdb/cockroach/commits/26713f81a397d13db4f2257eaa0f6bb70d1a7cb3):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 15108680 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:828: 45s of 5m0s: 172954 executions, 107646 successful
    rsg_test.go:828: 50s of 5m0s: 192257 executions, 119606 successful
    rsg_test.go:828: 55s of 5m0s: 209858 executions, 130608 successful
    rsg_test.go:828: 1m0s of 5m0s: 228654 executions, 142239 successful
    rsg_test.go:828: 1m5s of 5m0s: 249681 executions, 155290 successful
    rsg_test.go:828: 1m10s of 5m0s: 266929 executions, 165941 successful
    rsg_test.go:828: 1m15s of 5m0s: 286484 executions, 178092 successful
    rsg_test.go:828: 1m20s of 5m0s: 305495 executions, 189986 successful
    rsg_test.go:828: 1m25s of 5m0s: 323686 executions, 201320 successful
    rsg_test.go:828: 1m30s of 5m0s: 341977 executions, 212589 successful
    rsg_test.go:828: 1m35s of 5m0s: 361888 executions, 225101 successful
    rsg_test.go:828: 1m40s of 5m0s: 383918 executions, 238770 successful
    rsg_test.go:828: 1m45s of 5m0s: 401132 executions, 249487 successful
    rsg_test.go:828: 1m50s of 5m0s: 422555 executions, 262717 successful
    rsg_test.go:828: 1m55s of 5m0s: 444205 executions, 276218 successful
    rsg_test.go:828: 2m0s of 5m0s: 462398 executions, 287458 successful
    rsg_test.go:828: 2m5s of 5m0s: 483249 executions, 300473 successful
    rsg_test.go:828: 2m10s of 5m0s: 504578 executions, 313706 successful
    rsg_test.go:828: 2m15s of 5m0s: 524200 executions, 326004 successful
    rsg_test.go:828: 2m20s of 5m0s: 542642 executions, 337390 successful
    rsg_test.go:828: 2m25s of 5m0s: 563923 executions, 350609 successful
    rsg_test.go:828: 2m30s of 5m0s: 584030 executions, 363010 successful
    rsg_test.go:828: 2m35s of 5m0s: 600171 executions, 373081 successful
    rsg_test.go:828: 2m40s of 5m0s: 619170 executions, 384916 successful
    rsg_test.go:828: 2m45s of 5m0s: 640969 executions, 398538 successful
    rsg_test.go:828: 2m50s of 5m0s: 660695 executions, 410734 successful
    rsg_test.go:828: 2m55s of 5m0s: 680682 executions, 423265 successful
    rsg_test.go:828: 3m0s of 5m0s: 701547 executions, 436231 successful
    rsg_test.go:828: 3m5s of 5m0s: 725307 executions, 451052 successful
    rsg_test.go:828: 3m10s of 5m0s: 746165 executions, 464038 successful
    rsg_test.go:828: 3m15s of 5m0s: 767907 executions, 477419 successful
    rsg_test.go:828: 3m20s of 5m0s: 787310 executions, 489419 successful
    rsg_test.go:828: 3m25s of 5m0s: 807284 executions, 501905 successful
    rsg_test.go:828: 3m30s of 5m0s: 826268 executions, 513719 successful
    rsg_test.go:828: 3m35s of 5m0s: 847337 executions, 526769 successful
    rsg_test.go:828: 3m40s of 5m0s: 867749 executions, 539375 successful
    rsg_test.go:828: 3m45s of 5m0s: 888827 executions, 552487 successful
    rsg_test.go:828: 3m50s of 5m0s: 908692 executions, 564951 successful
    rsg_test.go:828: 3m55s of 5m0s: 929528 executions, 577858 successful
    rsg_test.go:828: 4m0s of 5m0s: 951251 executions, 591293 successful
    rsg_test.go:828: 4m5s of 5m0s: 973391 executions, 605130 successful
    rsg_test.go:828: 4m10s of 5m0s: 995716 executions, 618945 successful
    rsg_test.go:828: 4m15s of 5m0s: 1015974 executions, 631588 successful
    rsg_test.go:828: 4m20s of 5m0s: 1037627 executions, 644908 successful
    rsg_test.go:828: 4m25s of 5m0s: 1057778 executions, 657514 successful
    rsg_test.go:828: 4m30s of 5m0s: 1077559 executions, 669728 successful
    rsg_test.go:828: 4m35s of 5m0s: 1100010 executions, 683795 successful
    rsg_test.go:828: 4m40s of 5m0s: 1122783 executions, 697975 successful
    rsg_test.go:828: 4m45s of 5m0s: 1143907 executions, 711095 successful
    rsg_test.go:828: 4m50s of 5m0s: 1166455 executions, 725093 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9954973?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9954973?buildTab=artifacts#/) on release-23.1 @ [e8cad69249719e193cedcfd30d6a91350210d9d5](https://github.com/cockroachdb/cockroach/commits/e8cad69249719e193cedcfd30d6a91350210d9d5):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 15302107 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:828: 45s of 5m0s: 176515 executions, 109376 successful
    rsg_test.go:828: 50s of 5m0s: 194099 executions, 120342 successful
    rsg_test.go:828: 55s of 5m0s: 212860 executions, 131962 successful
    rsg_test.go:828: 1m0s of 5m0s: 233483 executions, 144805 successful
    rsg_test.go:828: 1m5s of 5m0s: 254898 executions, 158088 successful
    rsg_test.go:828: 1m10s of 5m0s: 277189 executions, 171946 successful
    rsg_test.go:828: 1m15s of 5m0s: 295900 executions, 183569 successful
    rsg_test.go:828: 1m20s of 5m0s: 316949 executions, 196690 successful
    rsg_test.go:828: 1m25s of 5m0s: 333302 executions, 206845 successful
    rsg_test.go:828: 1m30s of 5m0s: 356265 executions, 221158 successful
    rsg_test.go:828: 1m35s of 5m0s: 375308 executions, 233059 successful
    rsg_test.go:828: 1m40s of 5m0s: 397486 executions, 246774 successful
    rsg_test.go:828: 1m45s of 5m0s: 417306 executions, 259204 successful
    rsg_test.go:828: 1m50s of 5m0s: 435494 executions, 270405 successful
    rsg_test.go:828: 1m55s of 5m0s: 456559 executions, 283552 successful
    rsg_test.go:828: 2m0s of 5m0s: 474056 executions, 294373 successful
    rsg_test.go:828: 2m5s of 5m0s: 493101 executions, 306203 successful
    rsg_test.go:828: 2m10s of 5m0s: 512023 executions, 318008 successful
    rsg_test.go:828: 2m15s of 5m0s: 533274 executions, 331179 successful
    rsg_test.go:828: 2m20s of 5m0s: 550380 executions, 341764 successful
    rsg_test.go:828: 2m25s of 5m0s: 570474 executions, 354331 successful
    rsg_test.go:828: 2m30s of 5m0s: 591291 executions, 367281 successful
    rsg_test.go:828: 2m35s of 5m0s: 611049 executions, 379556 successful
    rsg_test.go:828: 2m40s of 5m0s: 632081 executions, 392607 successful
    rsg_test.go:828: 2m45s of 5m0s: 656135 executions, 407676 successful
    rsg_test.go:828: 2m50s of 5m0s: 677310 executions, 420826 successful
    rsg_test.go:828: 2m55s of 5m0s: 698106 executions, 433735 successful
    rsg_test.go:828: 3m0s of 5m0s: 719116 executions, 446796 successful
    rsg_test.go:828: 3m5s of 5m0s: 737776 executions, 458340 successful
    rsg_test.go:828: 3m10s of 5m0s: 759681 executions, 472089 successful
    rsg_test.go:828: 3m15s of 5m0s: 782086 executions, 485950 successful
    rsg_test.go:828: 3m20s of 5m0s: 803913 executions, 499478 successful
    rsg_test.go:828: 3m25s of 5m0s: 825752 executions, 513060 successful
    rsg_test.go:828: 3m30s of 5m0s: 848368 executions, 527087 successful
    rsg_test.go:828: 3m35s of 5m0s: 870577 executions, 540863 successful
    rsg_test.go:828: 3m40s of 5m0s: 892485 executions, 554393 successful
    rsg_test.go:828: 3m45s of 5m0s: 912583 executions, 566919 successful
    rsg_test.go:828: 3m50s of 5m0s: 934708 executions, 580572 successful
    rsg_test.go:828: 3m55s of 5m0s: 957646 executions, 594844 successful
    rsg_test.go:828: 4m0s of 5m0s: 980142 executions, 608978 successful
    rsg_test.go:828: 4m5s of 5m0s: 998722 executions, 620561 successful
    rsg_test.go:828: 4m10s of 5m0s: 1017600 executions, 632337 successful
    rsg_test.go:828: 4m15s of 5m0s: 1036972 executions, 644354 successful
    rsg_test.go:828: 4m20s of 5m0s: 1056647 executions, 656530 successful
    rsg_test.go:828: 4m25s of 5m0s: 1075787 executions, 668384 successful
    rsg_test.go:828: 4m30s of 5m0s: 1096166 executions, 681108 successful
    rsg_test.go:828: 4m35s of 5m0s: 1119145 executions, 695335 successful
    rsg_test.go:828: 4m40s of 5m0s: 1139131 executions, 707771 successful
    rsg_test.go:828: 4m45s of 5m0s: 1161206 executions, 721504 successful
    rsg_test.go:828: 4m50s of 5m0s: 1181781 executions, 734320 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9963575?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9963575?buildTab=artifacts#/) on release-23.1 @ [4226a83871bbce776bc9389fca5cf084b4bb7632](https://github.com/cockroachdb/cockroach/commits/4226a83871bbce776bc9389fca5cf084b4bb7632):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 15359427 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:828: 45s of 5m0s: 187045 executions, 116377 successful
    rsg_test.go:828: 50s of 5m0s: 207917 executions, 129329 successful
    rsg_test.go:828: 55s of 5m0s: 229818 executions, 142986 successful
    rsg_test.go:828: 1m0s of 5m0s: 251066 executions, 156136 successful
    rsg_test.go:828: 1m5s of 5m0s: 272760 executions, 169542 successful
    rsg_test.go:828: 1m10s of 5m0s: 292051 executions, 181529 successful
    rsg_test.go:828: 1m15s of 5m0s: 311881 executions, 193889 successful
    rsg_test.go:828: 1m20s of 5m0s: 332196 executions, 206488 successful
    rsg_test.go:828: 1m25s of 5m0s: 354394 executions, 220330 successful
    rsg_test.go:828: 1m30s of 5m0s: 375798 executions, 233479 successful
    rsg_test.go:828: 1m35s of 5m0s: 397438 executions, 246895 successful
    rsg_test.go:828: 1m40s of 5m0s: 418128 executions, 259738 successful
    rsg_test.go:828: 1m45s of 5m0s: 439864 executions, 273160 successful
    rsg_test.go:828: 1m50s of 5m0s: 463421 executions, 287848 successful
    rsg_test.go:828: 1m55s of 5m0s: 484192 executions, 300763 successful
    rsg_test.go:828: 2m0s of 5m0s: 502055 executions, 311744 successful
    rsg_test.go:828: 2m5s of 5m0s: 525537 executions, 326315 successful
    rsg_test.go:828: 2m10s of 5m0s: 550451 executions, 341786 successful
    rsg_test.go:828: 2m15s of 5m0s: 569101 executions, 353356 successful
    rsg_test.go:828: 2m20s of 5m0s: 591341 executions, 367274 successful
    rsg_test.go:828: 2m25s of 5m0s: 613349 executions, 380876 successful
    rsg_test.go:828: 2m30s of 5m0s: 633782 executions, 393528 successful
    rsg_test.go:828: 2m35s of 5m0s: 657478 executions, 408291 successful
    rsg_test.go:828: 2m40s of 5m0s: 676047 executions, 419731 successful
    rsg_test.go:828: 2m45s of 5m0s: 696256 executions, 432394 successful
    rsg_test.go:828: 2m50s of 5m0s: 715399 executions, 444308 successful
    rsg_test.go:828: 2m55s of 5m0s: 736771 executions, 457640 successful
    rsg_test.go:828: 3m0s of 5m0s: 757868 executions, 470737 successful
    rsg_test.go:828: 3m5s of 5m0s: 778846 executions, 483764 successful
    rsg_test.go:828: 3m10s of 5m0s: 800917 executions, 497416 successful
    rsg_test.go:828: 3m15s of 5m0s: 820735 executions, 509847 successful
    rsg_test.go:828: 3m20s of 5m0s: 841963 executions, 523095 successful
    rsg_test.go:828: 3m25s of 5m0s: 863446 executions, 536400 successful
    rsg_test.go:828: 3m30s of 5m0s: 883595 executions, 549017 successful
    rsg_test.go:828: 3m35s of 5m0s: 903985 executions, 561641 successful
    rsg_test.go:828: 3m40s of 5m0s: 924693 executions, 574432 successful
    rsg_test.go:828: 3m45s of 5m0s: 945662 executions, 587489 successful
    rsg_test.go:828: 3m50s of 5m0s: 964291 executions, 599161 successful
    rsg_test.go:828: 3m55s of 5m0s: 982806 executions, 610588 successful
    rsg_test.go:828: 4m0s of 5m0s: 1001603 executions, 622273 successful
    rsg_test.go:828: 4m5s of 5m0s: 1020984 executions, 634351 successful
    rsg_test.go:828: 4m10s of 5m0s: 1043708 executions, 648415 successful
    rsg_test.go:828: 4m15s of 5m0s: 1062854 executions, 660365 successful
    rsg_test.go:828: 4m20s of 5m0s: 1085213 executions, 674258 successful
    rsg_test.go:828: 4m25s of 5m0s: 1106681 executions, 687596 successful
    rsg_test.go:828: 4m30s of 5m0s: 1128567 executions, 701263 successful
    rsg_test.go:828: 4m35s of 5m0s: 1149434 executions, 714270 successful
    rsg_test.go:828: 4m40s of 5m0s: 1169114 executions, 726524 successful
    rsg_test.go:828: 4m45s of 5m0s: 1189270 executions, 738954 successful
    rsg_test.go:828: 4m50s of 5m0s: 1212215 executions, 753228 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9974830?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9974830?buildTab=artifacts#/) on release-23.1 @ [4226a83871bbce776bc9389fca5cf084b4bb7632](https://github.com/cockroachdb/cockroach/commits/4226a83871bbce776bc9389fca5cf084b4bb7632):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 14025082 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRandomSyntaxFunctions
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxFunctions808940602
    test_log_scope.go:79: use -show-logs to present logs inline
    rsg_test.go:828: 5s of 5m0s: 15779 executions, 9897 successful
    rsg_test.go:828: 10s of 5m0s: 35816 executions, 22288 successful
    rsg_test.go:828: 15s of 5m0s: 54964 executions, 34149 successful
    rsg_test.go:828: 20s of 5m0s: 73139 executions, 45460 successful
    rsg_test.go:828: 25s of 5m0s: 90330 executions, 56148 successful
    rsg_test.go:828: 30s of 5m0s: 108958 executions, 67676 successful
    rsg_test.go:828: 35s of 5m0s: 126789 executions, 78801 successful
    rsg_test.go:828: 40s of 5m0s: 146758 executions, 91194 successful
    rsg_test.go:828: 45s of 5m0s: 163592 executions, 101630 successful
    rsg_test.go:828: 50s of 5m0s: 183082 executions, 113774 successful
    rsg_test.go:828: 55s of 5m0s: 201236 executions, 125120 successful
    rsg_test.go:828: 1m0s of 5m0s: 223578 executions, 138965 successful
    rsg_test.go:828: 1m5s of 5m0s: 244403 executions, 151957 successful
    rsg_test.go:828: 1m10s of 5m0s: 265286 executions, 164951 successful
    rsg_test.go:828: 1m15s of 5m0s: 285770 executions, 177579 successful
    rsg_test.go:828: 1m20s of 5m0s: 307803 executions, 191241 successful
    rsg_test.go:828: 1m25s of 5m0s: 331491 executions, 206030 successful
    rsg_test.go:828: 1m30s of 5m0s: 353187 executions, 219447 successful
    rsg_test.go:828: 1m35s of 5m0s: 377727 executions, 234566 successful
    rsg_test.go:828: 1m40s of 5m0s: 397938 executions, 247191 successful
    rsg_test.go:828: 1m45s of 5m0s: 421929 executions, 262077 successful
    rsg_test.go:828: 1m50s of 5m0s: 443459 executions, 275436 successful
    rsg_test.go:828: 1m55s of 5m0s: 468930 executions, 291345 successful
    rsg_test.go:828: 2m0s of 5m0s: 487896 executions, 303129 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9990847?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/9990847?buildTab=artifacts#/) on release-23.1 @ [5376e479204d6e8243f67c30aea3d031df529afd](https://github.com/cockroachdb/cockroach/commits/5376e479204d6e8243f67c30aea3d031df529afd):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 15085857 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:828: 45s of 5m0s: 166801 executions, 103785 successful
    rsg_test.go:828: 50s of 5m0s: 183921 executions, 114456 successful
    rsg_test.go:828: 55s of 5m0s: 203451 executions, 126546 successful
    rsg_test.go:828: 1m0s of 5m0s: 224754 executions, 139779 successful
    rsg_test.go:828: 1m5s of 5m0s: 243261 executions, 151350 successful
    rsg_test.go:828: 1m10s of 5m0s: 260760 executions, 162221 successful
    rsg_test.go:828: 1m15s of 5m0s: 279415 executions, 173872 successful
    rsg_test.go:828: 1m20s of 5m0s: 295805 executions, 183957 successful
    rsg_test.go:828: 1m25s of 5m0s: 315912 executions, 196486 successful
    rsg_test.go:828: 1m30s of 5m0s: 336122 executions, 208983 successful
    rsg_test.go:828: 1m35s of 5m0s: 356235 executions, 221567 successful
    rsg_test.go:828: 1m40s of 5m0s: 373517 executions, 232295 successful
    rsg_test.go:828: 1m45s of 5m0s: 394309 executions, 245224 successful
    rsg_test.go:828: 1m50s of 5m0s: 411947 executions, 256174 successful
    rsg_test.go:828: 1m55s of 5m0s: 431676 executions, 268433 successful
    rsg_test.go:828: 2m0s of 5m0s: 452590 executions, 281405 successful
    rsg_test.go:828: 2m5s of 5m0s: 473344 executions, 294151 successful
    rsg_test.go:828: 2m10s of 5m0s: 494590 executions, 307485 successful
    rsg_test.go:828: 2m15s of 5m0s: 518338 executions, 322234 successful
    rsg_test.go:828: 2m20s of 5m0s: 540574 executions, 336040 successful
    rsg_test.go:828: 2m25s of 5m0s: 561677 executions, 349186 successful
    rsg_test.go:828: 2m30s of 5m0s: 582264 executions, 361956 successful
    rsg_test.go:828: 2m35s of 5m0s: 604182 executions, 375561 successful
    rsg_test.go:828: 2m40s of 5m0s: 629264 executions, 391227 successful
    rsg_test.go:828: 2m45s of 5m0s: 649778 executions, 403910 successful
    rsg_test.go:828: 2m50s of 5m0s: 669577 executions, 416334 successful
    rsg_test.go:828: 2m55s of 5m0s: 692160 executions, 430277 successful
    rsg_test.go:828: 3m0s of 5m0s: 711428 executions, 442232 successful
    rsg_test.go:828: 3m5s of 5m0s: 733341 executions, 455772 successful
    rsg_test.go:828: 3m10s of 5m0s: 753157 executions, 468076 successful
    rsg_test.go:828: 3m15s of 5m0s: 775745 executions, 482183 successful
    rsg_test.go:828: 3m20s of 5m0s: 798094 executions, 496149 successful
    rsg_test.go:828: 3m25s of 5m0s: 819709 executions, 509587 successful
    rsg_test.go:828: 3m30s of 5m0s: 839707 executions, 522094 successful
    rsg_test.go:828: 3m35s of 5m0s: 859685 executions, 534433 successful
    rsg_test.go:828: 3m40s of 5m0s: 881952 executions, 548442 successful
    rsg_test.go:828: 3m45s of 5m0s: 906260 executions, 563497 successful
    rsg_test.go:828: 3m50s of 5m0s: 927077 executions, 576414 successful
    rsg_test.go:828: 3m55s of 5m0s: 948789 executions, 589980 successful
    rsg_test.go:828: 4m0s of 5m0s: 971220 executions, 603854 successful
    rsg_test.go:828: 4m5s of 5m0s: 994478 executions, 618307 successful
    rsg_test.go:828: 4m10s of 5m0s: 1014530 executions, 630649 successful
    rsg_test.go:828: 4m15s of 5m0s: 1037013 executions, 644686 successful
    rsg_test.go:828: 4m20s of 5m0s: 1059946 executions, 658991 successful
    rsg_test.go:828: 4m25s of 5m0s: 1080520 executions, 671742 successful
    rsg_test.go:828: 4m30s of 5m0s: 1097901 executions, 682575 successful
    rsg_test.go:828: 4m35s of 5m0s: 1121152 executions, 697023 successful
    rsg_test.go:828: 4m40s of 5m0s: 1143484 executions, 710847 successful
    rsg_test.go:828: 4m45s of 5m0s: 1165722 executions, 724607 successful
    rsg_test.go:828: 4m50s of 5m0s: 1185548 executions, 736968 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10025463?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10025463?buildTab=artifacts#/) on release-23.1 @ [6ceddbd9dc6b987add91ea93a665088e7928cb88](https://github.com/cockroachdb/cockroach/commits/6ceddbd9dc6b987add91ea93a665088e7928cb88):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 8439410 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRandomSyntaxFunctions
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxFunctions1535486592
    test_log_scope.go:79: use -show-logs to present logs inline
    rsg_test.go:828: 5s of 5m0s: 18253 executions, 11349 successful
    rsg_test.go:828: 10s of 5m0s: 39527 executions, 24569 successful
    rsg_test.go:828: 15s of 5m0s: 59466 executions, 37024 successful
    rsg_test.go:828: 20s of 5m0s: 78497 executions, 48815 successful
    rsg_test.go:828: 25s of 5m0s: 95172 executions, 59189 successful
    rsg_test.go:828: 30s of 5m0s: 114517 executions, 71263 successful
    rsg_test.go:828: 35s of 5m0s: 133191 executions, 82818 successful
    rsg_test.go:828: 40s of 5m0s: 153499 executions, 95543 successful
    rsg_test.go:828: 45s of 5m0s: 171596 executions, 106674 successful
    rsg_test.go:828: 50s of 5m0s: 189139 executions, 117590 successful
    rsg_test.go:828: 55s of 5m0s: 208971 executions, 130071 successful
    rsg_test.go:828: 1m0s of 5m0s: 231921 executions, 144255 successful
    rsg_test.go:828: 1m5s of 5m0s: 250121 executions, 155512 successful
    rsg_test.go:828: 1m10s of 5m0s: 267850 executions, 166538 successful
    rsg_test.go:828: 1m15s of 5m0s: 287584 executions, 178838 successful
    rsg_test.go:828: 1m20s of 5m0s: 309592 executions, 192486 successful
    rsg_test.go:828: 1m25s of 5m0s: 329498 executions, 204919 successful
    rsg_test.go:828: 1m30s of 5m0s: 351608 executions, 218606 successful
    rsg_test.go:828: 1m35s of 5m0s: 370855 executions, 230523 successful
    rsg_test.go:828: 1m40s of 5m0s: 388734 executions, 241723 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10044677?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10044677?buildTab=artifacts#/) on release-23.1 @ [bc953426b8cc479406c3b54b99154eb15fa4107b](https://github.com/cockroachdb/cockroach/commits/bc953426b8cc479406c3b54b99154eb15fa4107b):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 9298519 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:828: 45s of 5m0s: 163632 executions, 101508 successful
    rsg_test.go:828: 50s of 5m0s: 182504 executions, 113344 successful
    rsg_test.go:828: 55s of 5m0s: 200555 executions, 124508 successful
    rsg_test.go:828: 1m0s of 5m0s: 220026 executions, 136707 successful
    rsg_test.go:828: 1m5s of 5m0s: 237149 executions, 147406 successful
    rsg_test.go:828: 1m10s of 5m0s: 253636 executions, 157638 successful
    rsg_test.go:828: 1m15s of 5m0s: 269522 executions, 167507 successful
    rsg_test.go:828: 1m20s of 5m0s: 289110 executions, 179742 successful
    rsg_test.go:828: 1m25s of 5m0s: 306500 executions, 190598 successful
    rsg_test.go:828: 1m30s of 5m0s: 321854 executions, 200163 successful
    rsg_test.go:828: 1m35s of 5m0s: 339041 executions, 210848 successful
    rsg_test.go:828: 1m40s of 5m0s: 357714 executions, 222353 successful
    rsg_test.go:828: 1m45s of 5m0s: 378193 executions, 235021 successful
    rsg_test.go:828: 1m50s of 5m0s: 399426 executions, 248209 successful
    rsg_test.go:828: 1m55s of 5m0s: 419762 executions, 260865 successful
    rsg_test.go:828: 2m0s of 5m0s: 441405 executions, 274271 successful
    rsg_test.go:828: 2m5s of 5m0s: 457854 executions, 284467 successful
    rsg_test.go:828: 2m10s of 5m0s: 475820 executions, 295568 successful
    rsg_test.go:828: 2m15s of 5m0s: 493998 executions, 306915 successful
    rsg_test.go:828: 2m20s of 5m0s: 510640 executions, 317214 successful
    rsg_test.go:828: 2m25s of 5m0s: 529303 executions, 328786 successful
    rsg_test.go:828: 2m30s of 5m0s: 547974 executions, 340354 successful
    rsg_test.go:828: 2m35s of 5m0s: 567688 executions, 352624 successful
    rsg_test.go:828: 2m40s of 5m0s: 587618 executions, 365107 successful
    rsg_test.go:828: 2m45s of 5m0s: 606398 executions, 376735 successful
    rsg_test.go:828: 2m50s of 5m0s: 625214 executions, 388461 successful
    rsg_test.go:828: 2m55s of 5m0s: 646149 executions, 401418 successful
    rsg_test.go:828: 3m0s of 5m0s: 664405 executions, 412799 successful
    rsg_test.go:828: 3m5s of 5m0s: 684690 executions, 425394 successful
    rsg_test.go:828: 3m10s of 5m0s: 704905 executions, 437963 successful
    rsg_test.go:828: 3m15s of 5m0s: 721900 executions, 448583 successful
    rsg_test.go:828: 3m20s of 5m0s: 740097 executions, 459848 successful
    rsg_test.go:828: 3m25s of 5m0s: 761735 executions, 473441 successful
    rsg_test.go:828: 3m30s of 5m0s: 780531 executions, 485068 successful
    rsg_test.go:828: 3m35s of 5m0s: 796926 executions, 495246 successful
    rsg_test.go:828: 3m40s of 5m0s: 816372 executions, 507429 successful
    rsg_test.go:828: 3m45s of 5m0s: 835684 executions, 519373 successful
    rsg_test.go:828: 3m50s of 5m0s: 852950 executions, 530105 successful
    rsg_test.go:828: 3m55s of 5m0s: 867877 executions, 539422 successful
    rsg_test.go:828: 4m0s of 5m0s: 886438 executions, 551008 successful
    rsg_test.go:828: 4m5s of 5m0s: 905771 executions, 562980 successful
    rsg_test.go:828: 4m10s of 5m0s: 921473 executions, 572798 successful
    rsg_test.go:828: 4m15s of 5m0s: 938411 executions, 583349 successful
    rsg_test.go:828: 4m20s of 5m0s: 955558 executions, 594002 successful
    rsg_test.go:828: 4m25s of 5m0s: 973336 executions, 605144 successful
    rsg_test.go:828: 4m30s of 5m0s: 992954 executions, 617371 successful
    rsg_test.go:828: 4m35s of 5m0s: 1008665 executions, 627199 successful
    rsg_test.go:828: 4m40s of 5m0s: 1027860 executions, 639206 successful
    rsg_test.go:828: 4m45s of 5m0s: 1044070 executions, 649299 successful
    rsg_test.go:828: 4m50s of 5m0s: 1063612 executions, 661483 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10072022?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10072022?buildTab=artifacts#/) on release-23.1 @ [b4533bdbc4b478f0ad311bad80b62bd072cf61cf](https://github.com/cockroachdb/cockroach/commits/b4533bdbc4b478f0ad311bad80b62bd072cf61cf):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 8667578 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRandomSyntaxFunctions
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxFunctions2461444688
    test_log_scope.go:79: use -show-logs to present logs inline
    rsg_test.go:828: 5s of 5m0s: 18361 executions, 11372 successful
    rsg_test.go:828: 10s of 5m0s: 37928 executions, 23590 successful
    rsg_test.go:828: 15s of 5m0s: 56932 executions, 35447 successful
    rsg_test.go:828: 20s of 5m0s: 76492 executions, 47619 successful
    rsg_test.go:828: 25s of 5m0s: 97434 executions, 60641 successful
    rsg_test.go:828: 30s of 5m0s: 116930 executions, 72826 successful
    rsg_test.go:828: 35s of 5m0s: 136311 executions, 84880 successful
    rsg_test.go:828: 40s of 5m0s: 156552 executions, 97393 successful
    rsg_test.go:828: 45s of 5m0s: 177121 executions, 110205 successful
    rsg_test.go:828: 50s of 5m0s: 195788 executions, 121890 successful
    rsg_test.go:828: 55s of 5m0s: 216484 executions, 134739 successful
    rsg_test.go:828: 1m0s of 5m0s: 236614 executions, 147236 successful
    rsg_test.go:828: 1m5s of 5m0s: 258136 executions, 160615 successful
    rsg_test.go:828: 1m10s of 5m0s: 278505 executions, 173191 successful
    rsg_test.go:828: 1m15s of 5m0s: 299120 executions, 186096 successful
    rsg_test.go:828: 1m20s of 5m0s: 320857 executions, 199566 successful
    rsg_test.go:828: 1m25s of 5m0s: 339234 executions, 211067 successful
    rsg_test.go:828: 1m30s of 5m0s: 360653 executions, 224320 successful
    rsg_test.go:828: 1m35s of 5m0s: 380399 executions, 236575 successful
    rsg_test.go:828: 1m40s of 5m0s: 400220 executions, 248950 successful
    rsg_test.go:828: 1m45s of 5m0s: 422870 executions, 262990 successful
    rsg_test.go:828: 1m50s of 5m0s: 444998 executions, 276698 successful
    rsg_test.go:828: 1m55s of 5m0s: 464235 executions, 288681 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10122176?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10122176?buildTab=artifacts#/) on release-23.1 @ [4c24848334fa068ae0eef950c687b42497bb7bd0](https://github.com/cockroachdb/cockroach/commits/4c24848334fa068ae0eef950c687b42497bb7bd0):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 9265710 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRandomSyntaxFunctions
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxFunctions3733435418
    test_log_scope.go:79: use -show-logs to present logs inline
    rsg_test.go:828: 5s of 5m0s: 20262 executions, 12619 successful
    rsg_test.go:828: 10s of 5m0s: 43633 executions, 27220 successful
    rsg_test.go:828: 15s of 5m0s: 65601 executions, 40878 successful
    rsg_test.go:828: 20s of 5m0s: 88640 executions, 55285 successful
    rsg_test.go:828: 25s of 5m0s: 112032 executions, 69736 successful
    rsg_test.go:828: 30s of 5m0s: 130826 executions, 81438 successful
    rsg_test.go:828: 35s of 5m0s: 148762 executions, 92585 successful
    rsg_test.go:828: 40s of 5m0s: 167468 executions, 104217 successful
    rsg_test.go:828: 45s of 5m0s: 187227 executions, 116485 successful
    rsg_test.go:828: 50s of 5m0s: 205123 executions, 127717 successful
    rsg_test.go:828: 55s of 5m0s: 225349 executions, 140284 successful
    rsg_test.go:828: 1m0s of 5m0s: 247649 executions, 154173 successful
    rsg_test.go:828: 1m5s of 5m0s: 267481 executions, 166509 successful
    rsg_test.go:828: 1m10s of 5m0s: 288699 executions, 179677 successful
    rsg_test.go:828: 1m15s of 5m0s: 310207 executions, 193084 successful
    rsg_test.go:828: 1m20s of 5m0s: 329310 executions, 204980 successful
    rsg_test.go:828: 1m25s of 5m0s: 349221 executions, 217309 successful
    rsg_test.go:828: 1m30s of 5m0s: 368929 executions, 229572 successful
    rsg_test.go:828: 1m35s of 5m0s: 388702 executions, 241952 successful
    rsg_test.go:828: 1m40s of 5m0s: 410575 executions, 255563 successful
    rsg_test.go:828: 1m45s of 5m0s: 432700 executions, 269343 successful
    rsg_test.go:828: 1m50s of 5m0s: 454032 executions, 282604 successful
    rsg_test.go:828: 1m55s of 5m0s: 475943 executions, 296210 successful
    rsg_test.go:828: 2m0s of 5m0s: 497244 executions, 309457 successful
    rsg_test.go:828: 2m5s of 5m0s: 519268 executions, 323129 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #103375 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-23.1.0]
- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot todo-deprecate.branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10137699?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10137699?buildTab=artifacts#/) on release-23.1 @ [4d53217fb2d27559257d46261badb3be91428d1a](https://github.com/cockroachdb/cockroach/commits/4d53217fb2d27559257d46261badb3be91428d1a):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 8749819 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRandomSyntaxFunctions
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxFunctions790107806
    test_log_scope.go:79: use -show-logs to present logs inline
    rsg_test.go:828: 5s of 5m0s: 21420 executions, 13433 successful
    rsg_test.go:828: 10s of 5m0s: 39859 executions, 24739 successful
    rsg_test.go:828: 15s of 5m0s: 59683 executions, 37143 successful
    rsg_test.go:828: 20s of 5m0s: 79272 executions, 49267 successful
    rsg_test.go:828: 25s of 5m0s: 98339 executions, 61126 successful
    rsg_test.go:828: 30s of 5m0s: 120192 executions, 74579 successful
    rsg_test.go:828: 35s of 5m0s: 143041 executions, 88868 successful
    rsg_test.go:828: 40s of 5m0s: 160728 executions, 99789 successful
    rsg_test.go:828: 45s of 5m0s: 183100 executions, 113747 successful
    rsg_test.go:828: 50s of 5m0s: 201524 executions, 125205 successful
    rsg_test.go:828: 55s of 5m0s: 221005 executions, 137276 successful
    rsg_test.go:828: 1m0s of 5m0s: 239640 executions, 148909 successful
    rsg_test.go:828: 1m5s of 5m0s: 258935 executions, 160892 successful
    rsg_test.go:828: 1m10s of 5m0s: 279829 executions, 173914 successful
    rsg_test.go:828: 1m15s of 5m0s: 299030 executions, 185874 successful
    rsg_test.go:828: 1m20s of 5m0s: 318226 executions, 197782 successful
    rsg_test.go:828: 1m25s of 5m0s: 338317 executions, 210230 successful
    rsg_test.go:828: 1m30s of 5m0s: 357916 executions, 222554 successful
    rsg_test.go:828: 1m35s of 5m0s: 376296 executions, 233890 successful
    rsg_test.go:828: 1m40s of 5m0s: 397586 executions, 247112 successful
    rsg_test.go:828: 1m45s of 5m0s: 419273 executions, 260610 successful
    rsg_test.go:828: 1m50s of 5m0s: 441572 executions, 274470 successful
    rsg_test.go:828: 1m55s of 5m0s: 461676 executions, 287015 successful
    rsg_test.go:828: 2m0s of 5m0s: 481245 executions, 299147 successful
    rsg_test.go:828: 2m5s of 5m0s: 499432 executions, 310502 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #103375 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-23.1.0]
- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot todo-deprecate.branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10177858?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10177858?buildTab=artifacts#/) on release-23.1 @ [1443aa2b0a4b906ba2b252da53e04097ed75051a](https://github.com/cockroachdb/cockroach/commits/1443aa2b0a4b906ba2b252da53e04097ed75051a):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 8820573 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:828: 45s of 5m0s: 160564 executions, 99891 successful
    rsg_test.go:828: 50s of 5m0s: 178178 executions, 110940 successful
    rsg_test.go:828: 55s of 5m0s: 198733 executions, 123649 successful
    rsg_test.go:828: 1m0s of 5m0s: 216077 executions, 134488 successful
    rsg_test.go:828: 1m5s of 5m0s: 235210 executions, 146413 successful
    rsg_test.go:828: 1m10s of 5m0s: 253650 executions, 157932 successful
    rsg_test.go:828: 1m15s of 5m0s: 271619 executions, 169170 successful
    rsg_test.go:828: 1m20s of 5m0s: 289197 executions, 180121 successful
    rsg_test.go:828: 1m25s of 5m0s: 307947 executions, 191678 successful
    rsg_test.go:828: 1m30s of 5m0s: 323397 executions, 201304 successful
    rsg_test.go:828: 1m35s of 5m0s: 339175 executions, 211163 successful
    rsg_test.go:828: 1m40s of 5m0s: 354470 executions, 220689 successful
    rsg_test.go:828: 1m45s of 5m0s: 371183 executions, 231002 successful
    rsg_test.go:828: 1m50s of 5m0s: 389941 executions, 242721 successful
    rsg_test.go:828: 1m55s of 5m0s: 411573 executions, 256162 successful
    rsg_test.go:828: 2m0s of 5m0s: 429719 executions, 267385 successful
    rsg_test.go:828: 2m5s of 5m0s: 445898 executions, 277553 successful
    rsg_test.go:828: 2m10s of 5m0s: 461991 executions, 287473 successful
    rsg_test.go:828: 2m15s of 5m0s: 478732 executions, 297994 successful
    rsg_test.go:828: 2m20s of 5m0s: 498642 executions, 310367 successful
    rsg_test.go:828: 2m25s of 5m0s: 516499 executions, 321466 successful
    rsg_test.go:828: 2m30s of 5m0s: 534441 executions, 332654 successful
    rsg_test.go:828: 2m35s of 5m0s: 553518 executions, 344538 successful
    rsg_test.go:828: 2m40s of 5m0s: 569275 executions, 354208 successful
    rsg_test.go:828: 2m45s of 5m0s: 588933 executions, 366440 successful
    rsg_test.go:828: 2m50s of 5m0s: 610492 executions, 379920 successful
    rsg_test.go:828: 2m55s of 5m0s: 631010 executions, 392709 successful
    rsg_test.go:828: 3m0s of 5m0s: 649106 executions, 404028 successful
    rsg_test.go:828: 3m5s of 5m0s: 667406 executions, 415339 successful
    rsg_test.go:828: 3m10s of 5m0s: 683231 executions, 425228 successful
    rsg_test.go:828: 3m15s of 5m0s: 699563 executions, 435362 successful
    rsg_test.go:828: 3m20s of 5m0s: 717698 executions, 446662 successful
    rsg_test.go:828: 3m25s of 5m0s: 737225 executions, 458850 successful
    rsg_test.go:828: 3m30s of 5m0s: 759391 executions, 472674 successful
    rsg_test.go:828: 3m35s of 5m0s: 778150 executions, 484354 successful
    rsg_test.go:828: 3m40s of 5m0s: 797599 executions, 496441 successful
    rsg_test.go:828: 3m45s of 5m0s: 817502 executions, 508842 successful
    rsg_test.go:828: 3m50s of 5m0s: 835025 executions, 519597 successful
    rsg_test.go:828: 3m55s of 5m0s: 855170 executions, 532107 successful
    rsg_test.go:828: 4m0s of 5m0s: 874523 executions, 544128 successful
    rsg_test.go:828: 4m5s of 5m0s: 893680 executions, 556033 successful
    rsg_test.go:828: 4m10s of 5m0s: 910523 executions, 566397 successful
    rsg_test.go:828: 4m15s of 5m0s: 929531 executions, 578210 successful
    rsg_test.go:828: 4m20s of 5m0s: 948926 executions, 590265 successful
    rsg_test.go:828: 4m25s of 5m0s: 964553 executions, 600115 successful
    rsg_test.go:828: 4m30s of 5m0s: 982766 executions, 611367 successful
    rsg_test.go:828: 4m35s of 5m0s: 1000025 executions, 622114 successful
    rsg_test.go:828: 4m40s of 5m0s: 1020146 executions, 634674 successful
    rsg_test.go:828: 4m45s of 5m0s: 1038291 executions, 645893 successful
    rsg_test.go:828: 4m50s of 5m0s: 1055620 executions, 656698 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #103375 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-23.1.0]
- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot todo-deprecate.branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10200598?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10200598?buildTab=artifacts#/) on release-23.1 @ [38af000823855ea428bc9b3618cbe74012aeccdc](https://github.com/cockroachdb/cockroach/commits/38af000823855ea428bc9b3618cbe74012aeccdc):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 7901369 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRandomSyntaxFunctions
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxFunctions3909750421
    test_log_scope.go:79: use -show-logs to present logs inline
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #103375 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-23.1.0]
- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot todo-deprecate.branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10253400?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10253400?buildTab=artifacts#/) on release-23.1 @ [ac03080b01fd57dcaf6a3eeaa6686751a6094fa2](https://github.com/cockroachdb/cockroach/commits/ac03080b01fd57dcaf6a3eeaa6686751a6094fa2):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 11894967 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRandomSyntaxFunctions
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxFunctions939751456
    test_log_scope.go:79: use -show-logs to present logs inline
    rsg_test.go:828: 5s of 5m0s: 23404 executions, 14648 successful
    rsg_test.go:828: 10s of 5m0s: 45836 executions, 28484 successful
    rsg_test.go:828: 15s of 5m0s: 68252 executions, 42349 successful
    rsg_test.go:828: 20s of 5m0s: 88237 executions, 54834 successful
    rsg_test.go:828: 25s of 5m0s: 109997 executions, 68321 successful
    rsg_test.go:828: 30s of 5m0s: 131729 executions, 81885 successful
    rsg_test.go:828: 35s of 5m0s: 154387 executions, 95910 successful
    rsg_test.go:828: 40s of 5m0s: 177664 executions, 110439 successful
    rsg_test.go:828: 45s of 5m0s: 201137 executions, 125134 successful
    rsg_test.go:828: 50s of 5m0s: 224439 executions, 139633 successful
    rsg_test.go:828: 55s of 5m0s: 246132 executions, 153085 successful
    rsg_test.go:828: 1m0s of 5m0s: 266400 executions, 165781 successful
    rsg_test.go:828: 1m5s of 5m0s: 289381 executions, 180066 successful
    rsg_test.go:828: 1m10s of 5m0s: 311371 executions, 193701 successful
    rsg_test.go:828: 1m15s of 5m0s: 332611 executions, 206914 successful
    rsg_test.go:828: 1m20s of 5m0s: 354913 executions, 220683 successful
    rsg_test.go:828: 1m25s of 5m0s: 373742 executions, 232500 successful
    rsg_test.go:828: 1m30s of 5m0s: 395156 executions, 245707 successful
    rsg_test.go:828: 1m35s of 5m0s: 415713 executions, 258596 successful
    rsg_test.go:828: 1m40s of 5m0s: 437548 executions, 272089 successful
    rsg_test.go:828: 1m45s of 5m0s: 458189 executions, 284840 successful
    rsg_test.go:828: 1m50s of 5m0s: 478845 executions, 297747 successful
    rsg_test.go:828: 1m55s of 5m0s: 500401 executions, 311057 successful
    rsg_test.go:828: 2m0s of 5m0s: 522921 executions, 325095 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #103375 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-23.1.0]
- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot todo-deprecate.branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10271852?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10271852?buildTab=artifacts#/) on release-23.1 @ [afe0240f6266af0b21fb521bd6120858ae99f442](https://github.com/cockroachdb/cockroach/commits/afe0240f6266af0b21fb521bd6120858ae99f442):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 8957158 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRandomSyntaxFunctions
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxFunctions4046696124
    test_log_scope.go:79: use -show-logs to present logs inline
    rsg_test.go:828: 5s of 5m0s: 23379 executions, 14716 successful
    rsg_test.go:828: 10s of 5m0s: 45124 executions, 28148 successful
    rsg_test.go:828: 15s of 5m0s: 67735 executions, 42270 successful
    rsg_test.go:828: 20s of 5m0s: 87850 executions, 54758 successful
    rsg_test.go:828: 25s of 5m0s: 107905 executions, 67283 successful
    rsg_test.go:828: 30s of 5m0s: 129014 executions, 80331 successful
    rsg_test.go:828: 35s of 5m0s: 148135 executions, 92199 successful
    rsg_test.go:828: 40s of 5m0s: 169851 executions, 105667 successful
    rsg_test.go:828: 45s of 5m0s: 190916 executions, 118827 successful
    rsg_test.go:828: 50s of 5m0s: 211705 executions, 131741 successful
    rsg_test.go:828: 55s of 5m0s: 235139 executions, 146323 successful
    rsg_test.go:828: 1m0s of 5m0s: 258056 executions, 160588 successful
    rsg_test.go:828: 1m5s of 5m0s: 278440 executions, 173250 successful
    rsg_test.go:828: 1m10s of 5m0s: 296727 executions, 184711 successful
    rsg_test.go:828: 1m15s of 5m0s: 319549 executions, 198856 successful
    rsg_test.go:828: 1m20s of 5m0s: 340719 executions, 212102 successful
    rsg_test.go:828: 1m25s of 5m0s: 362604 executions, 225577 successful
    rsg_test.go:828: 1m30s of 5m0s: 383348 executions, 238516 successful
    rsg_test.go:828: 1m35s of 5m0s: 405300 executions, 252105 successful
    rsg_test.go:828: 1m40s of 5m0s: 424417 executions, 264004 successful
    rsg_test.go:828: 1m45s of 5m0s: 448102 executions, 278825 successful
    rsg_test.go:828: 1m50s of 5m0s: 470710 executions, 292855 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #103375 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-23.1.0]
- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot todo-deprecate.branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10281191?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10281191?buildTab=artifacts#/) on release-23.1 @ [76b4c6ece0e687aefbf45da57a526f7d3b1a7f8a](https://github.com/cockroachdb/cockroach/commits/76b4c6ece0e687aefbf45da57a526f7d3b1a7f8a):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 11417905 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRandomSyntaxFunctions
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxFunctions1125956815
    test_log_scope.go:79: use -show-logs to present logs inline
    rsg_test.go:828: 5s of 5m0s: 21027 executions, 13105 successful
    rsg_test.go:828: 10s of 5m0s: 42356 executions, 26372 successful
    rsg_test.go:828: 15s of 5m0s: 64712 executions, 40208 successful
    rsg_test.go:828: 20s of 5m0s: 84489 executions, 52516 successful
    rsg_test.go:828: 25s of 5m0s: 104544 executions, 64880 successful
    rsg_test.go:828: 30s of 5m0s: 125059 executions, 77654 successful
    rsg_test.go:828: 35s of 5m0s: 148376 executions, 92069 successful
    rsg_test.go:828: 40s of 5m0s: 169243 executions, 105017 successful
    rsg_test.go:828: 45s of 5m0s: 190155 executions, 118069 successful
    rsg_test.go:828: 50s of 5m0s: 210970 executions, 131061 successful
    rsg_test.go:828: 55s of 5m0s: 233387 executions, 144961 successful
    rsg_test.go:828: 1m0s of 5m0s: 251677 executions, 156330 successful
    rsg_test.go:828: 1m5s of 5m0s: 272934 executions, 169642 successful
    rsg_test.go:828: 1m10s of 5m0s: 290846 executions, 180756 successful
    rsg_test.go:828: 1m15s of 5m0s: 311105 executions, 193353 successful
    rsg_test.go:828: 1m20s of 5m0s: 331640 executions, 206206 successful
    rsg_test.go:828: 1m25s of 5m0s: 353495 executions, 219785 successful
    rsg_test.go:828: 1m30s of 5m0s: 377181 executions, 234662 successful
    rsg_test.go:828: 1m35s of 5m0s: 399378 executions, 248509 successful
    rsg_test.go:828: 1m40s of 5m0s: 418720 executions, 260451 successful
    rsg_test.go:828: 1m45s of 5m0s: 443189 executions, 275747 successful
    rsg_test.go:828: 1m50s of 5m0s: 464395 executions, 288950 successful
    rsg_test.go:828: 1m55s of 5m0s: 486648 executions, 302822 successful
    rsg_test.go:828: 2m0s of 5m0s: 510487 executions, 317655 successful
    rsg_test.go:828: 2m5s of 5m0s: 533237 executions, 331764 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #103375 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-23.1.0]
- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot todo-deprecate.branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10301373?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10301373?buildTab=artifacts#/) on release-23.1 @ [f462a0e04526d90b004b6865938587a1bec28c21](https://github.com/cockroachdb/cockroach/commits/f462a0e04526d90b004b6865938587a1bec28c21):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 8353560 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRandomSyntaxFunctions
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxFunctions107403164
    test_log_scope.go:79: use -show-logs to present logs inline
    rsg_test.go:828: 5s of 5m0s: 20234 executions, 12556 successful
    rsg_test.go:828: 10s of 5m0s: 41546 executions, 25882 successful
    rsg_test.go:828: 15s of 5m0s: 62856 executions, 39047 successful
    rsg_test.go:828: 20s of 5m0s: 82451 executions, 51294 successful
    rsg_test.go:828: 25s of 5m0s: 103947 executions, 64674 successful
    rsg_test.go:828: 30s of 5m0s: 125987 executions, 78315 successful
    rsg_test.go:828: 35s of 5m0s: 145684 executions, 90596 successful
    rsg_test.go:828: 40s of 5m0s: 164992 executions, 102629 successful
    rsg_test.go:828: 45s of 5m0s: 184696 executions, 114826 successful
    rsg_test.go:828: 50s of 5m0s: 203891 executions, 126714 successful
    rsg_test.go:828: 55s of 5m0s: 224157 executions, 139382 successful
    rsg_test.go:828: 1m0s of 5m0s: 242283 executions, 150538 successful
    rsg_test.go:828: 1m5s of 5m0s: 263649 executions, 163886 successful
    rsg_test.go:828: 1m10s of 5m0s: 284240 executions, 176735 successful
    rsg_test.go:828: 1m15s of 5m0s: 305616 executions, 190115 successful
    rsg_test.go:828: 1m20s of 5m0s: 325824 executions, 202685 successful
    rsg_test.go:828: 1m25s of 5m0s: 345595 executions, 215008 successful
    rsg_test.go:828: 1m30s of 5m0s: 364942 executions, 227122 successful
    rsg_test.go:828: 1m35s of 5m0s: 385890 executions, 240139 successful
    rsg_test.go:828: 1m40s of 5m0s: 403828 executions, 251352 successful
    rsg_test.go:828: 1m45s of 5m0s: 426474 executions, 265407 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #103375 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-23.1.0]
- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot todo-deprecate.branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10356450?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10356450?buildTab=artifacts#/) on release-23.1 @ [e84e2cabef32a59d88041132f059a6768dfe56bb](https://github.com/cockroachdb/cockroach/commits/e84e2cabef32a59d88041132f059a6768dfe56bb):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 11039072 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:828: 45s of 5m0s: 186986 executions, 116462 successful
    rsg_test.go:828: 50s of 5m0s: 208197 executions, 129613 successful
    rsg_test.go:828: 55s of 5m0s: 230462 executions, 143410 successful
    rsg_test.go:828: 1m0s of 5m0s: 250231 executions, 155721 successful
    rsg_test.go:828: 1m5s of 5m0s: 271021 executions, 168613 successful
    rsg_test.go:828: 1m10s of 5m0s: 290124 executions, 180485 successful
    rsg_test.go:828: 1m15s of 5m0s: 309600 executions, 192553 successful
    rsg_test.go:828: 1m20s of 5m0s: 332057 executions, 206481 successful
    rsg_test.go:828: 1m25s of 5m0s: 351752 executions, 218738 successful
    rsg_test.go:828: 1m30s of 5m0s: 371835 executions, 231286 successful
    rsg_test.go:828: 1m35s of 5m0s: 390900 executions, 243163 successful
    rsg_test.go:828: 1m40s of 5m0s: 410812 executions, 255506 successful
    rsg_test.go:828: 1m45s of 5m0s: 429026 executions, 266851 successful
    rsg_test.go:828: 1m50s of 5m0s: 449554 executions, 279494 successful
    rsg_test.go:828: 1m55s of 5m0s: 472511 executions, 293665 successful
    rsg_test.go:828: 2m0s of 5m0s: 493209 executions, 306495 successful
    rsg_test.go:828: 2m5s of 5m0s: 511485 executions, 317916 successful
    rsg_test.go:828: 2m10s of 5m0s: 533216 executions, 331443 successful
    rsg_test.go:828: 2m15s of 5m0s: 552865 executions, 343595 successful
    rsg_test.go:828: 2m20s of 5m0s: 572659 executions, 355929 successful
    rsg_test.go:828: 2m25s of 5m0s: 591844 executions, 367871 successful
    rsg_test.go:828: 2m30s of 5m0s: 612436 executions, 380631 successful
    rsg_test.go:828: 2m35s of 5m0s: 636094 executions, 395221 successful
    rsg_test.go:828: 2m40s of 5m0s: 654351 executions, 406671 successful
    rsg_test.go:828: 2m45s of 5m0s: 675926 executions, 420138 successful
    rsg_test.go:828: 2m50s of 5m0s: 694031 executions, 431357 successful
    rsg_test.go:828: 2m55s of 5m0s: 715079 executions, 444446 successful
    rsg_test.go:828: 3m0s of 5m0s: 736797 executions, 458033 successful
    rsg_test.go:828: 3m5s of 5m0s: 753118 executions, 468134 successful
    rsg_test.go:828: 3m10s of 5m0s: 771996 executions, 479832 successful
    rsg_test.go:828: 3m15s of 5m0s: 792164 executions, 492439 successful
    rsg_test.go:828: 3m20s of 5m0s: 812634 executions, 505109 successful
    rsg_test.go:828: 3m25s of 5m0s: 832129 executions, 517245 successful
    rsg_test.go:828: 3m30s of 5m0s: 852792 executions, 530068 successful
    rsg_test.go:828: 3m35s of 5m0s: 873912 executions, 543136 successful
    rsg_test.go:828: 3m40s of 5m0s: 895381 executions, 556568 successful
    rsg_test.go:828: 3m45s of 5m0s: 917288 executions, 570161 successful
    rsg_test.go:828: 3m50s of 5m0s: 939365 executions, 583973 successful
    rsg_test.go:828: 3m55s of 5m0s: 961198 executions, 597575 successful
    rsg_test.go:828: 4m0s of 5m0s: 979301 executions, 608797 successful
    rsg_test.go:828: 4m5s of 5m0s: 1000062 executions, 621760 successful
    rsg_test.go:828: 4m10s of 5m0s: 1018470 executions, 633307 successful
    rsg_test.go:828: 4m15s of 5m0s: 1035599 executions, 643954 successful
    rsg_test.go:828: 4m20s of 5m0s: 1055745 executions, 656384 successful
    rsg_test.go:828: 4m25s of 5m0s: 1077595 executions, 670112 successful
    rsg_test.go:828: 4m30s of 5m0s: 1096770 executions, 682028 successful
    rsg_test.go:828: 4m35s of 5m0s: 1118201 executions, 695373 successful
    rsg_test.go:828: 4m40s of 5m0s: 1136982 executions, 707110 successful
    rsg_test.go:828: 4m45s of 5m0s: 1156098 executions, 718981 successful
    rsg_test.go:828: 4m50s of 5m0s: 1174854 executions, 730650 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #103375 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-23.1.0]
- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot todo-deprecate.branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10381247?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10381247?buildTab=artifacts#/) on release-23.1 @ [870f9f667dd34bd0c926d38c08193db2672d78ff](https://github.com/cockroachdb/cockroach/commits/870f9f667dd34bd0c926d38c08193db2672d78ff):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 8834074 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRandomSyntaxFunctions
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxFunctions2379100808
    test_log_scope.go:79: use -show-logs to present logs inline
    rsg_test.go:828: 5s of 5m0s: 19653 executions, 12276 successful
    rsg_test.go:828: 10s of 5m0s: 41212 executions, 25609 successful
    rsg_test.go:828: 15s of 5m0s: 62157 executions, 38772 successful
    rsg_test.go:828: 20s of 5m0s: 84889 executions, 52995 successful
    rsg_test.go:828: 25s of 5m0s: 102440 executions, 63828 successful
    rsg_test.go:828: 30s of 5m0s: 122962 executions, 76644 successful
    rsg_test.go:828: 35s of 5m0s: 143999 executions, 89713 successful
    rsg_test.go:828: 40s of 5m0s: 163122 executions, 101601 successful
    rsg_test.go:828: 45s of 5m0s: 184638 executions, 114963 successful
    rsg_test.go:828: 50s of 5m0s: 207644 executions, 129262 successful
    rsg_test.go:828: 55s of 5m0s: 226629 executions, 141097 successful
    rsg_test.go:828: 1m0s of 5m0s: 246798 executions, 153681 successful
    rsg_test.go:828: 1m5s of 5m0s: 268162 executions, 166984 successful
    rsg_test.go:828: 1m10s of 5m0s: 288999 executions, 179964 successful
    rsg_test.go:828: 1m15s of 5m0s: 310632 executions, 193416 successful
    rsg_test.go:828: 1m20s of 5m0s: 332237 executions, 206759 successful
    rsg_test.go:828: 1m25s of 5m0s: 352869 executions, 219528 successful
    rsg_test.go:828: 1m30s of 5m0s: 371804 executions, 231265 successful
    rsg_test.go:828: 1m35s of 5m0s: 392866 executions, 244403 successful
    rsg_test.go:828: 1m40s of 5m0s: 416600 executions, 259026 successful
    rsg_test.go:828: 1m45s of 5m0s: 435279 executions, 270550 successful
    rsg_test.go:828: 1m50s of 5m0s: 454580 executions, 282554 successful
    rsg_test.go:828: 1m55s of 5m0s: 474461 executions, 294927 successful
    rsg_test.go:828: 2m0s of 5m0s: 496052 executions, 308329 successful
    rsg_test.go:828: 2m5s of 5m0s: 516072 executions, 320758 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #103375 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-23.1.0]
- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot todo-deprecate.branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10390392?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10390392?buildTab=artifacts#/) on release-23.1 @ [ab03bdfe5b635c37bad80e01e1fdc37023844b15](https://github.com/cockroachdb/cockroach/commits/ab03bdfe5b635c37bad80e01e1fdc37023844b15):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 8770678 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRandomSyntaxFunctions
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxFunctions2367022515
    test_log_scope.go:79: use -show-logs to present logs inline
    rsg_test.go:828: 5s of 5m0s: 21621 executions, 13450 successful
    rsg_test.go:828: 10s of 5m0s: 42153 executions, 26152 successful
    rsg_test.go:828: 15s of 5m0s: 65357 executions, 40640 successful
    rsg_test.go:828: 20s of 5m0s: 85179 executions, 52927 successful
    rsg_test.go:828: 25s of 5m0s: 105688 executions, 65818 successful
    rsg_test.go:828: 30s of 5m0s: 124281 executions, 77304 successful
    rsg_test.go:828: 35s of 5m0s: 145949 executions, 90808 successful
    rsg_test.go:828: 40s of 5m0s: 169540 executions, 105385 successful
    rsg_test.go:828: 45s of 5m0s: 186608 executions, 115997 successful
    rsg_test.go:828: 50s of 5m0s: 207227 executions, 128918 successful
    rsg_test.go:828: 55s of 5m0s: 225173 executions, 140006 successful
    rsg_test.go:828: 1m0s of 5m0s: 244550 executions, 151971 successful
    rsg_test.go:828: 1m5s of 5m0s: 264989 executions, 164654 successful
    rsg_test.go:828: 1m10s of 5m0s: 283108 executions, 175953 successful
    rsg_test.go:828: 1m15s of 5m0s: 303370 executions, 188486 successful
    rsg_test.go:828: 1m20s of 5m0s: 323231 executions, 200773 successful
    rsg_test.go:828: 1m25s of 5m0s: 345315 executions, 214465 successful
    rsg_test.go:828: 1m30s of 5m0s: 366011 executions, 227437 successful
    rsg_test.go:828: 1m35s of 5m0s: 388264 executions, 241150 successful
    rsg_test.go:828: 1m40s of 5m0s: 409027 executions, 254122 successful
    rsg_test.go:828: 1m45s of 5m0s: 428604 executions, 266278 successful
    rsg_test.go:828: 1m50s of 5m0s: 448652 executions, 278728 successful
    rsg_test.go:828: 1m55s of 5m0s: 468748 executions, 291203 successful
    rsg_test.go:828: 2m0s of 5m0s: 490333 executions, 304644 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #103375 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-23.1.0]
- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot todo-deprecate.branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10428565?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10428565?buildTab=artifacts#/) on release-23.1 @ [1a5dc2aad16ca4185afcda75d923804324dcfeb9](https://github.com/cockroachdb/cockroach/commits/1a5dc2aad16ca4185afcda75d923804324dcfeb9):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 8637177 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRandomSyntaxFunctions
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxFunctions410460372
    test_log_scope.go:79: use -show-logs to present logs inline
    rsg_test.go:827: 5s of 5m0s: 20674 executions, 12895 successful
    rsg_test.go:827: 10s of 5m0s: 40456 executions, 25269 successful
    rsg_test.go:827: 15s of 5m0s: 58060 executions, 36191 successful
    rsg_test.go:827: 20s of 5m0s: 81459 executions, 50661 successful
    rsg_test.go:827: 25s of 5m0s: 102042 executions, 63579 successful
    rsg_test.go:827: 30s of 5m0s: 121755 executions, 75832 successful
    rsg_test.go:827: 35s of 5m0s: 142820 executions, 88926 successful
    rsg_test.go:827: 40s of 5m0s: 165246 executions, 102940 successful
    rsg_test.go:827: 45s of 5m0s: 188810 executions, 117527 successful
    rsg_test.go:827: 50s of 5m0s: 207332 executions, 129082 successful
    rsg_test.go:827: 55s of 5m0s: 225453 executions, 140315 successful
    rsg_test.go:827: 1m0s of 5m0s: 244549 executions, 152215 successful
    rsg_test.go:827: 1m5s of 5m0s: 263732 executions, 164137 successful
    rsg_test.go:827: 1m10s of 5m0s: 282709 executions, 175947 successful
    rsg_test.go:827: 1m15s of 5m0s: 300105 executions, 186728 successful
    rsg_test.go:827: 1m20s of 5m0s: 319222 executions, 198666 successful
    rsg_test.go:827: 1m25s of 5m0s: 336964 executions, 209746 successful
    rsg_test.go:827: 1m30s of 5m0s: 353354 executions, 219885 successful
    rsg_test.go:827: 1m35s of 5m0s: 374279 executions, 232909 successful
    rsg_test.go:827: 1m40s of 5m0s: 392615 executions, 244361 successful
    rsg_test.go:827: 1m45s of 5m0s: 411169 executions, 255841 successful
    rsg_test.go:827: 1m50s of 5m0s: 432370 executions, 268961 successful
    rsg_test.go:827: 1m55s of 5m0s: 450431 executions, 280153 successful
    rsg_test.go:827: 2m0s of 5m0s: 468933 executions, 291681 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot todo-deprecate.branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10464292?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10464292?buildTab=artifacts#/) on release-23.1 @ [2d33fa6b041ba0a7b362358c9b69fcabe005edba](https://github.com/cockroachdb/cockroach/commits/2d33fa6b041ba0a7b362358c9b69fcabe005edba):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 9409869 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:827: 45s of 5m0s: 167075 executions, 103802 successful
    rsg_test.go:827: 50s of 5m0s: 186446 executions, 115815 successful
    rsg_test.go:827: 55s of 5m0s: 206850 executions, 128589 successful
    rsg_test.go:827: 1m0s of 5m0s: 227629 executions, 141388 successful
    rsg_test.go:827: 1m5s of 5m0s: 242138 executions, 150411 successful
    rsg_test.go:827: 1m10s of 5m0s: 261946 executions, 162831 successful
    rsg_test.go:827: 1m15s of 5m0s: 280805 executions, 174643 successful
    rsg_test.go:827: 1m20s of 5m0s: 298965 executions, 185983 successful
    rsg_test.go:827: 1m25s of 5m0s: 317102 executions, 197261 successful
    rsg_test.go:827: 1m30s of 5m0s: 334071 executions, 207824 successful
    rsg_test.go:827: 1m35s of 5m0s: 351089 executions, 218378 successful
    rsg_test.go:827: 1m40s of 5m0s: 369709 executions, 230007 successful
    rsg_test.go:827: 1m45s of 5m0s: 390340 executions, 242695 successful
    rsg_test.go:827: 1m50s of 5m0s: 409312 executions, 254520 successful
    rsg_test.go:827: 1m55s of 5m0s: 428064 executions, 266240 successful
    rsg_test.go:827: 2m0s of 5m0s: 444612 executions, 276612 successful
    rsg_test.go:827: 2m5s of 5m0s: 462987 executions, 287956 successful
    rsg_test.go:827: 2m10s of 5m0s: 478283 executions, 297532 successful
    rsg_test.go:827: 2m15s of 5m0s: 495528 executions, 308217 successful
    rsg_test.go:827: 2m20s of 5m0s: 514659 executions, 320084 successful
    rsg_test.go:827: 2m25s of 5m0s: 535434 executions, 332982 successful
    rsg_test.go:827: 2m30s of 5m0s: 555399 executions, 345491 successful
    rsg_test.go:827: 2m35s of 5m0s: 574812 executions, 357454 successful
    rsg_test.go:827: 2m40s of 5m0s: 593632 executions, 369225 successful
    rsg_test.go:827: 2m45s of 5m0s: 611116 executions, 380009 successful
    rsg_test.go:827: 2m50s of 5m0s: 630062 executions, 391837 successful
    rsg_test.go:827: 2m55s of 5m0s: 647365 executions, 402573 successful
    rsg_test.go:827: 3m0s of 5m0s: 668264 executions, 415577 successful
    rsg_test.go:827: 3m5s of 5m0s: 686646 executions, 427004 successful
    rsg_test.go:827: 3m10s of 5m0s: 705103 executions, 438463 successful
    rsg_test.go:827: 3m15s of 5m0s: 726343 executions, 451738 successful
    rsg_test.go:827: 3m20s of 5m0s: 745029 executions, 463400 successful
    rsg_test.go:827: 3m25s of 5m0s: 764090 executions, 475353 successful
    rsg_test.go:827: 3m30s of 5m0s: 784080 executions, 487749 successful
    rsg_test.go:827: 3m35s of 5m0s: 801212 executions, 498389 successful
    rsg_test.go:827: 3m40s of 5m0s: 818723 executions, 509312 successful
    rsg_test.go:827: 3m45s of 5m0s: 839026 executions, 521978 successful
    rsg_test.go:827: 3m50s of 5m0s: 859325 executions, 534689 successful
    rsg_test.go:827: 3m55s of 5m0s: 878650 executions, 546658 successful
    rsg_test.go:827: 4m0s of 5m0s: 898448 executions, 558934 successful
    rsg_test.go:827: 4m5s of 5m0s: 918771 executions, 571616 successful
    rsg_test.go:827: 4m10s of 5m0s: 938795 executions, 584005 successful
    rsg_test.go:827: 4m15s of 5m0s: 958705 executions, 596403 successful
    rsg_test.go:827: 4m20s of 5m0s: 979137 executions, 609039 successful
    rsg_test.go:827: 4m25s of 5m0s: 998462 executions, 620964 successful
    rsg_test.go:827: 4m30s of 5m0s: 1017618 executions, 632835 successful
    rsg_test.go:827: 4m35s of 5m0s: 1037296 executions, 645124 successful
    rsg_test.go:827: 4m40s of 5m0s: 1055711 executions, 656545 successful
    rsg_test.go:827: 4m45s of 5m0s: 1075256 executions, 668753 successful
    rsg_test.go:827: 4m50s of 5m0s: 1096502 executions, 682029 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot todo-deprecate.branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10504275?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10504275?buildTab=artifacts#/) on release-23.1 @ [ad8d135f50385be9b8460aca82725480c7a3821d](https://github.com/cockroachdb/cockroach/commits/ad8d135f50385be9b8460aca82725480c7a3821d):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 9630399 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:827: 45s of 5m0s: 172043 executions, 106973 successful
    rsg_test.go:827: 50s of 5m0s: 194199 executions, 120754 successful
    rsg_test.go:827: 55s of 5m0s: 215790 executions, 134221 successful
    rsg_test.go:827: 1m0s of 5m0s: 235059 executions, 146182 successful
    rsg_test.go:827: 1m5s of 5m0s: 253983 executions, 158039 successful
    rsg_test.go:827: 1m10s of 5m0s: 274605 executions, 170757 successful
    rsg_test.go:827: 1m15s of 5m0s: 293360 executions, 182428 successful
    rsg_test.go:827: 1m20s of 5m0s: 311644 executions, 193684 successful
    rsg_test.go:827: 1m25s of 5m0s: 329921 executions, 205014 successful
    rsg_test.go:827: 1m30s of 5m0s: 351438 executions, 218412 successful
    rsg_test.go:827: 1m35s of 5m0s: 371644 executions, 230996 successful
    rsg_test.go:827: 1m40s of 5m0s: 392455 executions, 243896 successful
    rsg_test.go:827: 1m45s of 5m0s: 411777 executions, 255928 successful
    rsg_test.go:827: 1m50s of 5m0s: 429192 executions, 266652 successful
    rsg_test.go:827: 1m55s of 5m0s: 451071 executions, 280203 successful
    rsg_test.go:827: 2m0s of 5m0s: 472678 executions, 293711 successful
    rsg_test.go:827: 2m5s of 5m0s: 493461 executions, 306621 successful
    rsg_test.go:827: 2m10s of 5m0s: 513765 executions, 319257 successful
    rsg_test.go:827: 2m15s of 5m0s: 532296 executions, 330927 successful
    rsg_test.go:827: 2m20s of 5m0s: 550776 executions, 342351 successful
    rsg_test.go:827: 2m25s of 5m0s: 570128 executions, 354416 successful
    rsg_test.go:827: 2m30s of 5m0s: 594110 executions, 369407 successful
    rsg_test.go:827: 2m35s of 5m0s: 612436 executions, 380952 successful
    rsg_test.go:827: 2m40s of 5m0s: 631804 executions, 392990 successful
    rsg_test.go:827: 2m45s of 5m0s: 650532 executions, 404633 successful
    rsg_test.go:827: 2m50s of 5m0s: 672060 executions, 418047 successful
    rsg_test.go:827: 2m55s of 5m0s: 693635 executions, 431501 successful
    rsg_test.go:827: 3m0s of 5m0s: 714689 executions, 444696 successful
    rsg_test.go:827: 3m5s of 5m0s: 735150 executions, 457517 successful
    rsg_test.go:827: 3m10s of 5m0s: 753963 executions, 469211 successful
    rsg_test.go:827: 3m15s of 5m0s: 773488 executions, 481371 successful
    rsg_test.go:827: 3m20s of 5m0s: 794010 executions, 494088 successful
    rsg_test.go:827: 3m25s of 5m0s: 816053 executions, 507746 successful
    rsg_test.go:827: 3m30s of 5m0s: 838588 executions, 521660 successful
    rsg_test.go:827: 3m35s of 5m0s: 858312 executions, 533836 successful
    rsg_test.go:827: 3m40s of 5m0s: 878602 executions, 546436 successful
    rsg_test.go:827: 3m45s of 5m0s: 900628 executions, 560111 successful
    rsg_test.go:827: 3m50s of 5m0s: 921638 executions, 573158 successful
    rsg_test.go:827: 3m55s of 5m0s: 939950 executions, 584527 successful
    rsg_test.go:827: 4m0s of 5m0s: 960016 executions, 596966 successful
    rsg_test.go:827: 4m5s of 5m0s: 980397 executions, 609536 successful
    rsg_test.go:827: 4m10s of 5m0s: 998739 executions, 621005 successful
    rsg_test.go:827: 4m15s of 5m0s: 1020209 executions, 634427 successful
    rsg_test.go:827: 4m20s of 5m0s: 1042726 executions, 648379 successful
    rsg_test.go:827: 4m25s of 5m0s: 1062063 executions, 660460 successful
    rsg_test.go:827: 4m30s of 5m0s: 1081891 executions, 672685 successful
    rsg_test.go:827: 4m35s of 5m0s: 1100530 executions, 684263 successful
    rsg_test.go:827: 4m40s of 5m0s: 1120159 executions, 696336 successful
    rsg_test.go:827: 4m45s of 5m0s: 1141819 executions, 709871 successful
    rsg_test.go:827: 4m50s of 5m0s: 1159298 executions, 720670 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot todo-deprecate.branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10522207?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10522207?buildTab=artifacts#/) on release-23.1 @ [407ad56a797df975f5cf5b0784dfe4a061cbf1c3](https://github.com/cockroachdb/cockroach/commits/407ad56a797df975f5cf5b0784dfe4a061cbf1c3):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 10525947 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:827: 45s of 5m0s: 148928 executions, 92511 successful
    rsg_test.go:827: 50s of 5m0s: 166906 executions, 103778 successful
    rsg_test.go:827: 55s of 5m0s: 187339 executions, 116453 successful
    rsg_test.go:827: 1m0s of 5m0s: 206514 executions, 128405 successful
    rsg_test.go:827: 1m5s of 5m0s: 219214 executions, 136277 successful
    rsg_test.go:827: 1m10s of 5m0s: 233584 executions, 145285 successful
    rsg_test.go:827: 1m15s of 5m0s: 248426 executions, 154468 successful
    rsg_test.go:827: 1m20s of 5m0s: 264254 executions, 164402 successful
    rsg_test.go:827: 1m25s of 5m0s: 280723 executions, 174592 successful
    rsg_test.go:827: 1m30s of 5m0s: 299779 executions, 186486 successful
    rsg_test.go:827: 1m35s of 5m0s: 316480 executions, 196848 successful
    rsg_test.go:827: 1m40s of 5m0s: 335748 executions, 208766 successful
    rsg_test.go:827: 1m45s of 5m0s: 349801 executions, 217566 successful
    rsg_test.go:827: 1m50s of 5m0s: 368413 executions, 229129 successful
    rsg_test.go:827: 1m55s of 5m0s: 388355 executions, 241577 successful
    rsg_test.go:827: 2m0s of 5m0s: 407951 executions, 253915 successful
    rsg_test.go:827: 2m5s of 5m0s: 426391 executions, 265352 successful
    rsg_test.go:827: 2m10s of 5m0s: 443614 executions, 276028 successful
    rsg_test.go:827: 2m15s of 5m0s: 461646 executions, 287253 successful
    rsg_test.go:827: 2m20s of 5m0s: 479736 executions, 298443 successful
    rsg_test.go:827: 2m25s of 5m0s: 501886 executions, 312220 successful
    rsg_test.go:827: 2m30s of 5m0s: 522260 executions, 324885 successful
    rsg_test.go:827: 2m35s of 5m0s: 538010 executions, 334794 successful
    rsg_test.go:827: 2m40s of 5m0s: 556431 executions, 346209 successful
    rsg_test.go:827: 2m45s of 5m0s: 578407 executions, 359954 successful
    rsg_test.go:827: 2m50s of 5m0s: 599449 executions, 373041 successful
    rsg_test.go:827: 2m55s of 5m0s: 618459 executions, 384808 successful
    rsg_test.go:827: 3m0s of 5m0s: 639998 executions, 398059 successful
    rsg_test.go:827: 3m5s of 5m0s: 655792 executions, 407963 successful
    rsg_test.go:827: 3m10s of 5m0s: 674871 executions, 419846 successful
    rsg_test.go:827: 3m15s of 5m0s: 696515 executions, 433345 successful
    rsg_test.go:827: 3m20s of 5m0s: 714986 executions, 444866 successful
    rsg_test.go:827: 3m25s of 5m0s: 737148 executions, 458738 successful
    rsg_test.go:827: 3m30s of 5m0s: 756256 executions, 470573 successful
    rsg_test.go:827: 3m35s of 5m0s: 773335 executions, 481244 successful
    rsg_test.go:827: 3m40s of 5m0s: 793735 executions, 493868 successful
    rsg_test.go:827: 3m45s of 5m0s: 814132 executions, 506531 successful
    rsg_test.go:827: 3m50s of 5m0s: 833998 executions, 518855 successful
    rsg_test.go:827: 3m55s of 5m0s: 855335 executions, 532190 successful
    rsg_test.go:827: 4m0s of 5m0s: 874093 executions, 543808 successful
    rsg_test.go:827: 4m5s of 5m0s: 892648 executions, 555357 successful
    rsg_test.go:827: 4m10s of 5m0s: 913050 executions, 568052 successful
    rsg_test.go:827: 4m15s of 5m0s: 932119 executions, 579918 successful
    rsg_test.go:827: 4m20s of 5m0s: 953938 executions, 593378 successful
    rsg_test.go:827: 4m25s of 5m0s: 977437 executions, 608050 successful
    rsg_test.go:827: 4m30s of 5m0s: 996931 executions, 620228 successful
    rsg_test.go:827: 4m35s of 5m0s: 1016318 executions, 632329 successful
    rsg_test.go:827: 4m40s of 5m0s: 1037240 executions, 645301 successful
    rsg_test.go:827: 4m45s of 5m0s: 1057586 executions, 657848 successful
    rsg_test.go:827: 4m50s of 5m0s: 1082280 executions, 673175 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot todo-deprecate.branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10539702?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10539702?buildTab=artifacts#/) on release-23.1 @ [d76049d66d6e3a2cd398d0347a63dd29cc9b928a](https://github.com/cockroachdb/cockroach/commits/d76049d66d6e3a2cd398d0347a63dd29cc9b928a):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 8629785 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRandomSyntaxFunctions
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxFunctions4258620544
    test_log_scope.go:79: use -show-logs to present logs inline
    rsg_test.go:827: 5s of 5m0s: 21449 executions, 13368 successful
    rsg_test.go:827: 10s of 5m0s: 42563 executions, 26443 successful
    rsg_test.go:827: 15s of 5m0s: 62885 executions, 38940 successful
    rsg_test.go:827: 20s of 5m0s: 84873 executions, 52585 successful
    rsg_test.go:827: 25s of 5m0s: 106551 executions, 65974 successful
    rsg_test.go:827: 30s of 5m0s: 126771 executions, 78619 successful
    rsg_test.go:827: 35s of 5m0s: 146591 executions, 90820 successful
    rsg_test.go:827: 40s of 5m0s: 168730 executions, 104603 successful
    rsg_test.go:827: 45s of 5m0s: 189208 executions, 117281 successful
    rsg_test.go:827: 50s of 5m0s: 210105 executions, 130362 successful
    rsg_test.go:827: 55s of 5m0s: 229633 executions, 142439 successful
    rsg_test.go:827: 1m0s of 5m0s: 250525 executions, 155439 successful
    rsg_test.go:827: 1m5s of 5m0s: 267868 executions, 166212 successful
    rsg_test.go:827: 1m10s of 5m0s: 287705 executions, 178589 successful
    rsg_test.go:827: 1m15s of 5m0s: 306336 executions, 190118 successful
    rsg_test.go:827: 1m20s of 5m0s: 324122 executions, 201274 successful
    rsg_test.go:827: 1m25s of 5m0s: 344960 executions, 214233 successful
    rsg_test.go:827: 1m30s of 5m0s: 366819 executions, 227797 successful
    rsg_test.go:827: 1m35s of 5m0s: 386612 executions, 240068 successful
    rsg_test.go:827: 1m40s of 5m0s: 408362 executions, 253636 successful
    rsg_test.go:827: 1m45s of 5m0s: 428801 executions, 266323 successful
    rsg_test.go:827: 1m50s of 5m0s: 449424 executions, 279166 successful
    rsg_test.go:827: 1m55s of 5m0s: 468729 executions, 291114 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot todo-deprecate.branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10556203?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10556203?buildTab=artifacts#/) on release-23.1 @ [cf795114db6331e95f7b97ff411065b22626c3cb](https://github.com/cockroachdb/cockroach/commits/cf795114db6331e95f7b97ff411065b22626c3cb):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 9396909 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:827: 45s of 5m0s: 186991 executions, 116177 successful
    rsg_test.go:827: 50s of 5m0s: 205281 executions, 127627 successful
    rsg_test.go:827: 55s of 5m0s: 224182 executions, 139390 successful
    rsg_test.go:827: 1m0s of 5m0s: 245427 executions, 152565 successful
    rsg_test.go:827: 1m5s of 5m0s: 264688 executions, 164505 successful
    rsg_test.go:827: 1m10s of 5m0s: 286044 executions, 177720 successful
    rsg_test.go:827: 1m15s of 5m0s: 307786 executions, 191242 successful
    rsg_test.go:827: 1m20s of 5m0s: 328546 executions, 204122 successful
    rsg_test.go:827: 1m25s of 5m0s: 344536 executions, 214113 successful
    rsg_test.go:827: 1m30s of 5m0s: 365860 executions, 227492 successful
    rsg_test.go:827: 1m35s of 5m0s: 386443 executions, 240200 successful
    rsg_test.go:827: 1m40s of 5m0s: 409073 executions, 254370 successful
    rsg_test.go:827: 1m45s of 5m0s: 428950 executions, 266871 successful
    rsg_test.go:827: 1m50s of 5m0s: 449986 executions, 280012 successful
    rsg_test.go:827: 1m55s of 5m0s: 471622 executions, 293445 successful
    rsg_test.go:827: 2m0s of 5m0s: 492835 executions, 306535 successful
    rsg_test.go:827: 2m5s of 5m0s: 513624 executions, 319524 successful
    rsg_test.go:827: 2m10s of 5m0s: 533282 executions, 331657 successful
    rsg_test.go:827: 2m15s of 5m0s: 551911 executions, 343275 successful
    rsg_test.go:827: 2m20s of 5m0s: 570772 executions, 355011 successful
    rsg_test.go:827: 2m25s of 5m0s: 591149 executions, 367557 successful
    rsg_test.go:827: 2m30s of 5m0s: 612787 executions, 381081 successful
    rsg_test.go:827: 2m35s of 5m0s: 633531 executions, 393902 successful
    rsg_test.go:827: 2m40s of 5m0s: 654105 executions, 406726 successful
    rsg_test.go:827: 2m45s of 5m0s: 675677 executions, 420145 successful
    rsg_test.go:827: 2m50s of 5m0s: 696635 executions, 433191 successful
    rsg_test.go:827: 2m55s of 5m0s: 716215 executions, 445393 successful
    rsg_test.go:827: 3m0s of 5m0s: 733686 executions, 456277 successful
    rsg_test.go:827: 3m5s of 5m0s: 753871 executions, 468794 successful
    rsg_test.go:827: 3m10s of 5m0s: 772068 executions, 480117 successful
    rsg_test.go:827: 3m15s of 5m0s: 792152 executions, 492504 successful
    rsg_test.go:827: 3m20s of 5m0s: 815497 executions, 507048 successful
    rsg_test.go:827: 3m25s of 5m0s: 835281 executions, 519386 successful
    rsg_test.go:827: 3m30s of 5m0s: 858259 executions, 533743 successful
    rsg_test.go:827: 3m35s of 5m0s: 877691 executions, 545780 successful
    rsg_test.go:827: 3m40s of 5m0s: 894335 executions, 556246 successful
    rsg_test.go:827: 3m45s of 5m0s: 912540 executions, 567485 successful
    rsg_test.go:827: 3m50s of 5m0s: 932936 executions, 580181 successful
    rsg_test.go:827: 3m55s of 5m0s: 952971 executions, 592604 successful
    rsg_test.go:827: 4m0s of 5m0s: 972991 executions, 605162 successful
    rsg_test.go:827: 4m5s of 5m0s: 991715 executions, 616796 successful
    rsg_test.go:827: 4m10s of 5m0s: 1007820 executions, 626745 successful
    rsg_test.go:827: 4m15s of 5m0s: 1029200 executions, 640008 successful
    rsg_test.go:827: 4m20s of 5m0s: 1048780 executions, 652138 successful
    rsg_test.go:827: 4m25s of 5m0s: 1068663 executions, 664541 successful
    rsg_test.go:827: 4m30s of 5m0s: 1085115 executions, 674770 successful
    rsg_test.go:827: 4m35s of 5m0s: 1105281 executions, 687268 successful
    rsg_test.go:827: 4m40s of 5m0s: 1127308 executions, 700998 successful
    rsg_test.go:827: 4m45s of 5m0s: 1147216 executions, 713381 successful
    rsg_test.go:827: 4m50s of 5m0s: 1166697 executions, 725479 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot todo-deprecate.branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10571482?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10571482?buildTab=artifacts#/) on release-23.1 @ [5b0d9c6abb00a2ffac00ad75c3bea30577126f05](https://github.com/cockroachdb/cockroach/commits/5b0d9c6abb00a2ffac00ad75c3bea30577126f05):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 8846330 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:827: 45s of 5m0s: 158356 executions, 98284 successful
    rsg_test.go:827: 50s of 5m0s: 173783 executions, 107900 successful
    rsg_test.go:827: 55s of 5m0s: 191196 executions, 118633 successful
    rsg_test.go:827: 1m0s of 5m0s: 207256 executions, 128673 successful
    rsg_test.go:827: 1m5s of 5m0s: 226535 executions, 140570 successful
    rsg_test.go:827: 1m10s of 5m0s: 242341 executions, 150322 successful
    rsg_test.go:827: 1m15s of 5m0s: 260050 executions, 161251 successful
    rsg_test.go:827: 1m20s of 5m0s: 280970 executions, 174334 successful
    rsg_test.go:827: 1m25s of 5m0s: 298089 executions, 185072 successful
    rsg_test.go:827: 1m30s of 5m0s: 316977 executions, 196718 successful
    rsg_test.go:827: 1m35s of 5m0s: 337347 executions, 209507 successful
    rsg_test.go:827: 1m40s of 5m0s: 355292 executions, 220573 successful
    rsg_test.go:827: 1m45s of 5m0s: 372518 executions, 231295 successful
    rsg_test.go:827: 1m50s of 5m0s: 392059 executions, 243481 successful
    rsg_test.go:827: 1m55s of 5m0s: 410756 executions, 255082 successful
    rsg_test.go:827: 2m0s of 5m0s: 428974 executions, 266458 successful
    rsg_test.go:827: 2m5s of 5m0s: 448540 executions, 278655 successful
    rsg_test.go:827: 2m10s of 5m0s: 466859 executions, 290023 successful
    rsg_test.go:827: 2m15s of 5m0s: 482990 executions, 300128 successful
    rsg_test.go:827: 2m20s of 5m0s: 503079 executions, 312557 successful
    rsg_test.go:827: 2m25s of 5m0s: 522318 executions, 324511 successful
    rsg_test.go:827: 2m30s of 5m0s: 540798 executions, 335974 successful
    rsg_test.go:827: 2m35s of 5m0s: 560419 executions, 348255 successful
    rsg_test.go:827: 2m40s of 5m0s: 575276 executions, 357506 successful
    rsg_test.go:827: 2m45s of 5m0s: 592662 executions, 368319 successful
    rsg_test.go:827: 2m50s of 5m0s: 609289 executions, 378736 successful
    rsg_test.go:827: 2m55s of 5m0s: 626040 executions, 389091 successful
    rsg_test.go:827: 3m0s of 5m0s: 641721 executions, 398852 successful
    rsg_test.go:827: 3m5s of 5m0s: 660544 executions, 410557 successful
    rsg_test.go:827: 3m10s of 5m0s: 679398 executions, 422245 successful
    rsg_test.go:827: 3m15s of 5m0s: 697048 executions, 433344 successful
    rsg_test.go:827: 3m20s of 5m0s: 716552 executions, 445395 successful
    rsg_test.go:827: 3m25s of 5m0s: 732268 executions, 455044 successful
    rsg_test.go:827: 3m30s of 5m0s: 749858 executions, 466007 successful
    rsg_test.go:827: 3m35s of 5m0s: 766274 executions, 476275 successful
    rsg_test.go:827: 3m40s of 5m0s: 784388 executions, 487551 successful
    rsg_test.go:827: 3m45s of 5m0s: 803309 executions, 499314 successful
    rsg_test.go:827: 3m50s of 5m0s: 822494 executions, 511222 successful
    rsg_test.go:827: 3m55s of 5m0s: 840374 executions, 522351 successful
    rsg_test.go:827: 4m0s of 5m0s: 858183 executions, 533464 successful
    rsg_test.go:827: 4m5s of 5m0s: 875354 executions, 544139 successful
    rsg_test.go:827: 4m10s of 5m0s: 895590 executions, 556634 successful
    rsg_test.go:827: 4m15s of 5m0s: 911513 executions, 566580 successful
    rsg_test.go:827: 4m20s of 5m0s: 928872 executions, 577348 successful
    rsg_test.go:827: 4m25s of 5m0s: 947706 executions, 588988 successful
    rsg_test.go:827: 4m30s of 5m0s: 967083 executions, 601061 successful
    rsg_test.go:827: 4m35s of 5m0s: 984461 executions, 611868 successful
    rsg_test.go:827: 4m40s of 5m0s: 1003447 executions, 623684 successful
    rsg_test.go:827: 4m45s of 5m0s: 1023651 executions, 636170 successful
    rsg_test.go:827: 4m50s of 5m0s: 1040795 executions, 646836 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot todo-deprecate.branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10585362?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10585362?buildTab=artifacts#/) on release-23.1 @ [5b0d9c6abb00a2ffac00ad75c3bea30577126f05](https://github.com/cockroachdb/cockroach/commits/5b0d9c6abb00a2ffac00ad75c3bea30577126f05):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 8984699 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRandomSyntaxFunctions
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxFunctions936456123
    test_log_scope.go:79: use -show-logs to present logs inline
    rsg_test.go:827: 5s of 5m0s: 21425 executions, 13438 successful
    rsg_test.go:827: 10s of 5m0s: 43302 executions, 26982 successful
    rsg_test.go:827: 15s of 5m0s: 64499 executions, 40221 successful
    rsg_test.go:827: 20s of 5m0s: 86875 executions, 54213 successful
    rsg_test.go:827: 25s of 5m0s: 104110 executions, 64932 successful
    rsg_test.go:827: 30s of 5m0s: 122075 executions, 76059 successful
    rsg_test.go:827: 35s of 5m0s: 140238 executions, 87414 successful
    rsg_test.go:827: 40s of 5m0s: 159205 executions, 99182 successful
    rsg_test.go:827: 45s of 5m0s: 178580 executions, 111192 successful
    rsg_test.go:827: 50s of 5m0s: 197353 executions, 122814 successful
    rsg_test.go:827: 55s of 5m0s: 220542 executions, 137273 successful
    rsg_test.go:827: 1m0s of 5m0s: 242219 executions, 150683 successful
    rsg_test.go:827: 1m5s of 5m0s: 262749 executions, 163593 successful
    rsg_test.go:827: 1m10s of 5m0s: 281558 executions, 175224 successful
    rsg_test.go:827: 1m15s of 5m0s: 303087 executions, 188560 successful
    rsg_test.go:827: 1m20s of 5m0s: 326036 executions, 202833 successful
    rsg_test.go:827: 1m25s of 5m0s: 347214 executions, 216007 successful
    rsg_test.go:827: 1m30s of 5m0s: 367482 executions, 228602 successful
    rsg_test.go:827: 1m35s of 5m0s: 388539 executions, 241672 successful
    rsg_test.go:827: 1m40s of 5m0s: 408278 executions, 253976 successful
    rsg_test.go:827: 1m45s of 5m0s: 430368 executions, 267693 successful
    rsg_test.go:827: 1m50s of 5m0s: 451752 executions, 281013 successful
    rsg_test.go:827: 1m55s of 5m0s: 469943 executions, 292286 successful
    rsg_test.go:827: 2m0s of 5m0s: 490401 executions, 305035 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot todo-deprecate.branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10594822?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10594822?buildTab=artifacts#/) on release-23.1 @ [199d010284c0d95fad730cf816734dd465b01ae7](https://github.com/cockroachdb/cockroach/commits/199d010284c0d95fad730cf816734dd465b01ae7):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 9104918 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRandomSyntaxFunctions
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxFunctions2299560028
    test_log_scope.go:79: use -show-logs to present logs inline
    rsg_test.go:827: 5s of 5m0s: 21951 executions, 13627 successful
    rsg_test.go:827: 10s of 5m0s: 44520 executions, 27775 successful
    rsg_test.go:827: 15s of 5m0s: 66252 executions, 41116 successful
    rsg_test.go:827: 20s of 5m0s: 85971 executions, 53338 successful
    rsg_test.go:827: 25s of 5m0s: 106146 executions, 65860 successful
    rsg_test.go:827: 30s of 5m0s: 126609 executions, 78588 successful
    rsg_test.go:827: 35s of 5m0s: 145893 executions, 90565 successful
    rsg_test.go:827: 40s of 5m0s: 167536 executions, 103955 successful
    rsg_test.go:827: 45s of 5m0s: 190157 executions, 118110 successful
    rsg_test.go:827: 50s of 5m0s: 210001 executions, 130387 successful
    rsg_test.go:827: 55s of 5m0s: 228825 executions, 142011 successful
    rsg_test.go:827: 1m0s of 5m0s: 252936 executions, 157182 successful
    rsg_test.go:827: 1m5s of 5m0s: 273304 executions, 169852 successful
    rsg_test.go:827: 1m10s of 5m0s: 292484 executions, 181788 successful
    rsg_test.go:827: 1m15s of 5m0s: 313443 executions, 194841 successful
    rsg_test.go:827: 1m20s of 5m0s: 334329 executions, 207891 successful
    rsg_test.go:827: 1m25s of 5m0s: 355908 executions, 221256 successful
    rsg_test.go:827: 1m30s of 5m0s: 379407 executions, 235942 successful
    rsg_test.go:827: 1m35s of 5m0s: 398367 executions, 247641 successful
    rsg_test.go:827: 1m40s of 5m0s: 418508 executions, 260029 successful
    rsg_test.go:827: 1m45s of 5m0s: 440925 executions, 274021 successful
    rsg_test.go:827: 1m50s of 5m0s: 463373 executions, 287966 successful
    rsg_test.go:827: 1m55s of 5m0s: 482726 executions, 299978 successful
    rsg_test.go:827: 2m0s of 5m0s: 502626 executions, 312316 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot todo-deprecate.branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10627983?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10627983?buildTab=artifacts#/) on release-23.1 @ [3532484aa08c3acf6eb5770f1469af09f4ba704d](https://github.com/cockroachdb/cockroach/commits/3532484aa08c3acf6eb5770f1469af09f4ba704d):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 8648499 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:827: 45s of 5m0s: 157665 executions, 97776 successful
    rsg_test.go:827: 50s of 5m0s: 174611 executions, 108378 successful
    rsg_test.go:827: 55s of 5m0s: 193874 executions, 120352 successful
    rsg_test.go:827: 1m0s of 5m0s: 215530 executions, 133766 successful
    rsg_test.go:827: 1m5s of 5m0s: 238532 executions, 148114 successful
    rsg_test.go:827: 1m10s of 5m0s: 259376 executions, 161061 successful
    rsg_test.go:827: 1m15s of 5m0s: 279873 executions, 173914 successful
    rsg_test.go:827: 1m20s of 5m0s: 299633 executions, 186012 successful
    rsg_test.go:827: 1m25s of 5m0s: 319478 executions, 198382 successful
    rsg_test.go:827: 1m30s of 5m0s: 338443 executions, 210198 successful
    rsg_test.go:827: 1m35s of 5m0s: 356956 executions, 221683 successful
    rsg_test.go:827: 1m40s of 5m0s: 372401 executions, 231268 successful
    rsg_test.go:827: 1m45s of 5m0s: 389874 executions, 242123 successful
    rsg_test.go:827: 1m50s of 5m0s: 408261 executions, 253635 successful
    rsg_test.go:827: 1m55s of 5m0s: 427052 executions, 265309 successful
    rsg_test.go:827: 2m0s of 5m0s: 443245 executions, 275279 successful
    rsg_test.go:827: 2m5s of 5m0s: 462210 executions, 287087 successful
    rsg_test.go:827: 2m10s of 5m0s: 485224 executions, 301406 successful
    rsg_test.go:827: 2m15s of 5m0s: 506011 executions, 314343 successful
    rsg_test.go:827: 2m20s of 5m0s: 525234 executions, 326231 successful
    rsg_test.go:827: 2m25s of 5m0s: 544075 executions, 337937 successful
    rsg_test.go:827: 2m30s of 5m0s: 561782 executions, 348838 successful
    rsg_test.go:827: 2m35s of 5m0s: 580834 executions, 360709 successful
    rsg_test.go:827: 2m40s of 5m0s: 600244 executions, 372767 successful
    rsg_test.go:827: 2m45s of 5m0s: 617415 executions, 383413 successful
    rsg_test.go:827: 2m50s of 5m0s: 636879 executions, 395532 successful
    rsg_test.go:827: 2m55s of 5m0s: 655792 executions, 407297 successful
    rsg_test.go:827: 3m0s of 5m0s: 674154 executions, 418811 successful
    rsg_test.go:827: 3m5s of 5m0s: 690152 executions, 428673 successful
    rsg_test.go:827: 3m10s of 5m0s: 706913 executions, 439154 successful
    rsg_test.go:827: 3m15s of 5m0s: 729097 executions, 452877 successful
    rsg_test.go:827: 3m20s of 5m0s: 752594 executions, 467536 successful
    rsg_test.go:827: 3m25s of 5m0s: 770052 executions, 478324 successful
    rsg_test.go:827: 3m30s of 5m0s: 791387 executions, 491546 successful
    rsg_test.go:827: 3m35s of 5m0s: 812536 executions, 504712 successful
    rsg_test.go:827: 3m40s of 5m0s: 829402 executions, 515235 successful
    rsg_test.go:827: 3m45s of 5m0s: 846327 executions, 525822 successful
    rsg_test.go:827: 3m50s of 5m0s: 864657 executions, 537161 successful
    rsg_test.go:827: 3m55s of 5m0s: 882450 executions, 548228 successful
    rsg_test.go:827: 4m0s of 5m0s: 900241 executions, 559260 successful
    rsg_test.go:827: 4m5s of 5m0s: 918239 executions, 570397 successful
    rsg_test.go:827: 4m10s of 5m0s: 936473 executions, 581806 successful
    rsg_test.go:827: 4m15s of 5m0s: 954206 executions, 592774 successful
    rsg_test.go:827: 4m20s of 5m0s: 970331 executions, 602867 successful
    rsg_test.go:827: 4m25s of 5m0s: 990361 executions, 615398 successful
    rsg_test.go:827: 4m30s of 5m0s: 1006891 executions, 625692 successful
    rsg_test.go:827: 4m35s of 5m0s: 1026917 executions, 638121 successful
    rsg_test.go:827: 4m40s of 5m0s: 1048766 executions, 651635 successful
    rsg_test.go:827: 4m45s of 5m0s: 1068387 executions, 663893 successful
    rsg_test.go:827: 4m50s of 5m0s: 1087815 executions, 675928 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot todo-deprecate.branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10644088?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10644088?buildTab=artifacts#/) on release-23.1 @ [624d9dea7f1296af60d16bbb45cc9d1259b3a1be](https://github.com/cockroachdb/cockroach/commits/624d9dea7f1296af60d16bbb45cc9d1259b3a1be):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 9601536 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:827: 45s of 5m0s: 162424 executions, 101102 successful
    rsg_test.go:827: 50s of 5m0s: 179769 executions, 111930 successful
    rsg_test.go:827: 55s of 5m0s: 195677 executions, 121846 successful
    rsg_test.go:827: 1m0s of 5m0s: 214938 executions, 133718 successful
    rsg_test.go:827: 1m5s of 5m0s: 233379 executions, 145185 successful
    rsg_test.go:827: 1m10s of 5m0s: 253775 executions, 157859 successful
    rsg_test.go:827: 1m15s of 5m0s: 269766 executions, 167817 successful
    rsg_test.go:827: 1m20s of 5m0s: 290057 executions, 180452 successful
    rsg_test.go:827: 1m25s of 5m0s: 308160 executions, 191705 successful
    rsg_test.go:827: 1m30s of 5m0s: 326526 executions, 203157 successful
    rsg_test.go:827: 1m35s of 5m0s: 348240 executions, 216616 successful
    rsg_test.go:827: 1m40s of 5m0s: 364383 executions, 226691 successful
    rsg_test.go:827: 1m45s of 5m0s: 381098 executions, 237171 successful
    rsg_test.go:827: 1m50s of 5m0s: 401587 executions, 249931 successful
    rsg_test.go:827: 1m55s of 5m0s: 418445 executions, 260330 successful
    rsg_test.go:827: 2m0s of 5m0s: 438472 executions, 272768 successful
    rsg_test.go:827: 2m5s of 5m0s: 458173 executions, 284965 successful
    rsg_test.go:827: 2m10s of 5m0s: 479466 executions, 298147 successful
    rsg_test.go:827: 2m15s of 5m0s: 496966 executions, 309046 successful
    rsg_test.go:827: 2m20s of 5m0s: 518324 executions, 322401 successful
    rsg_test.go:827: 2m25s of 5m0s: 538967 executions, 335188 successful
    rsg_test.go:827: 2m30s of 5m0s: 559348 executions, 347959 successful
    rsg_test.go:827: 2m35s of 5m0s: 580893 executions, 361145 successful
    rsg_test.go:827: 2m40s of 5m0s: 600564 executions, 373436 successful
    rsg_test.go:827: 2m45s of 5m0s: 621951 executions, 386723 successful
    rsg_test.go:827: 2m50s of 5m0s: 640185 executions, 398045 successful
    rsg_test.go:827: 2m55s of 5m0s: 660757 executions, 410937 successful
    rsg_test.go:827: 3m0s of 5m0s: 681627 executions, 423904 successful
    rsg_test.go:827: 3m5s of 5m0s: 702522 executions, 436813 successful
    rsg_test.go:827: 3m10s of 5m0s: 722293 executions, 449205 successful
    rsg_test.go:827: 3m15s of 5m0s: 744190 executions, 462734 successful
    rsg_test.go:827: 3m20s of 5m0s: 765918 executions, 476283 successful
    rsg_test.go:827: 3m25s of 5m0s: 784916 executions, 488079 successful
    rsg_test.go:827: 3m30s of 5m0s: 804258 executions, 500219 successful
    rsg_test.go:827: 3m35s of 5m0s: 822077 executions, 511231 successful
    rsg_test.go:827: 3m40s of 5m0s: 842864 executions, 524233 successful
    rsg_test.go:827: 3m45s of 5m0s: 862814 executions, 536572 successful
    rsg_test.go:827: 3m50s of 5m0s: 883706 executions, 549555 successful
    rsg_test.go:827: 3m55s of 5m0s: 905156 executions, 562898 successful
    rsg_test.go:827: 4m0s of 5m0s: 925753 executions, 575769 successful
    rsg_test.go:827: 4m5s of 5m0s: 944042 executions, 587175 successful
    rsg_test.go:827: 4m10s of 5m0s: 959687 executions, 596878 successful
    rsg_test.go:827: 4m15s of 5m0s: 978590 executions, 608638 successful
    rsg_test.go:827: 4m20s of 5m0s: 997787 executions, 620572 successful
    rsg_test.go:827: 4m25s of 5m0s: 1017452 executions, 632715 successful
    rsg_test.go:827: 4m30s of 5m0s: 1038099 executions, 645668 successful
    rsg_test.go:827: 4m35s of 5m0s: 1055330 executions, 656372 successful
    rsg_test.go:827: 4m40s of 5m0s: 1076268 executions, 669304 successful
    rsg_test.go:827: 4m45s of 5m0s: 1097840 executions, 682606 successful
    rsg_test.go:827: 4m50s of 5m0s: 1116537 executions, 694263 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot todo-deprecate.branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10657437?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10657437?buildTab=artifacts#/) on release-23.1 @ [dbf6ce8438742a73b9d17391c1ef868a4e7a9569](https://github.com/cockroachdb/cockroach/commits/dbf6ce8438742a73b9d17391c1ef868a4e7a9569):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 6050064 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:827: 40s of 5m0s: 105677 executions, 65698 successful
    rsg_test.go:827: 45s of 5m0s: 116927 executions, 72639 successful
    rsg_test.go:827: 50s of 5m0s: 131832 executions, 81856 successful
    rsg_test.go:827: 55s of 5m0s: 140482 executions, 87292 successful
    rsg_test.go:827: 1m0s of 5m0s: 153335 executions, 95281 successful
    rsg_test.go:827: 1m5s of 5m0s: 164177 executions, 102021 successful
    rsg_test.go:827: 1m10s of 5m0s: 177715 executions, 110442 successful
    rsg_test.go:827: 1m15s of 5m0s: 192121 executions, 119483 successful
    rsg_test.go:827: 1m20s of 5m0s: 204928 executions, 127474 successful
    rsg_test.go:827: 1m25s of 5m0s: 216889 executions, 134952 successful
    rsg_test.go:827: 1m30s of 5m0s: 225862 executions, 140480 successful
    rsg_test.go:827: 1m35s of 5m0s: 235819 executions, 146770 successful
    rsg_test.go:827: 1m40s of 5m0s: 247625 executions, 154116 successful
    rsg_test.go:827: 1m45s of 5m0s: 262970 executions, 163612 successful
    rsg_test.go:827: 1m50s of 5m0s: 274320 executions, 170668 successful
    rsg_test.go:827: 1m55s of 5m0s: 284643 executions, 177076 successful
    rsg_test.go:827: 2m0s of 5m0s: 294139 executions, 182930 successful
    rsg_test.go:827: 2m5s of 5m0s: 304324 executions, 189288 successful
    rsg_test.go:827: 2m10s of 5m0s: 315471 executions, 196202 successful
    rsg_test.go:827: 2m15s of 5m0s: 327349 executions, 203617 successful
    rsg_test.go:827: 2m20s of 5m0s: 340473 executions, 211744 successful
    rsg_test.go:827: 2m25s of 5m0s: 350891 executions, 218342 successful
    rsg_test.go:827: 2m30s of 5m0s: 364076 executions, 226529 successful
    rsg_test.go:827: 2m35s of 5m0s: 376268 executions, 233993 successful
    rsg_test.go:827: 2m40s of 5m0s: 387015 executions, 240811 successful
    rsg_test.go:827: 2m45s of 5m0s: 399314 executions, 248351 successful
    rsg_test.go:827: 2m50s of 5m0s: 411043 executions, 255677 successful
    rsg_test.go:827: 2m55s of 5m0s: 421241 executions, 262042 successful
    rsg_test.go:827: 3m0s of 5m0s: 429422 executions, 267178 successful
    rsg_test.go:827: 3m5s of 5m0s: 438983 executions, 273119 successful
    rsg_test.go:827: 3m10s of 5m0s: 451342 executions, 280726 successful
    rsg_test.go:827: 3m15s of 5m0s: 462234 executions, 287468 successful
    rsg_test.go:827: 3m20s of 5m0s: 472392 executions, 293828 successful
    rsg_test.go:827: 3m25s of 5m0s: 483723 executions, 300900 successful
    rsg_test.go:827: 3m30s of 5m0s: 492638 executions, 306475 successful
    rsg_test.go:827: 3m35s of 5m0s: 502224 executions, 312427 successful
    rsg_test.go:827: 3m40s of 5m0s: 512595 executions, 318795 successful
    rsg_test.go:827: 3m46s of 5m0s: 527761 executions, 328229 successful
    rsg_test.go:827: 3m51s of 5m0s: 543860 executions, 338302 successful
    rsg_test.go:827: 3m56s of 5m0s: 556471 executions, 346025 successful
    rsg_test.go:827: 4m1s of 5m0s: 567413 executions, 352844 successful
    rsg_test.go:827: 4m6s of 5m0s: 579442 executions, 360357 successful
    rsg_test.go:827: 4m11s of 5m0s: 591221 executions, 367645 successful
    rsg_test.go:827: 4m16s of 5m0s: 604611 executions, 376007 successful
    rsg_test.go:827: 4m21s of 5m0s: 618839 executions, 384964 successful
    rsg_test.go:827: 4m26s of 5m0s: 628224 executions, 390836 successful
    rsg_test.go:827: 4m31s of 5m0s: 639210 executions, 397600 successful
    rsg_test.go:827: 4m36s of 5m0s: 650899 executions, 404875 successful
    rsg_test.go:827: 4m41s of 5m0s: 662036 executions, 411724 successful
    rsg_test.go:827: 4m46s of 5m0s: 676153 executions, 420609 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot todo-deprecate.branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10664724?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10664724?buildTab=artifacts#/) on release-23.1 @ [dbf6ce8438742a73b9d17391c1ef868a4e7a9569](https://github.com/cockroachdb/cockroach/commits/dbf6ce8438742a73b9d17391c1ef868a4e7a9569):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 6098816 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:827: 45s of 5m0s: 114481 executions, 71205 successful
    rsg_test.go:827: 50s of 5m0s: 127145 executions, 79075 successful
    rsg_test.go:827: 55s of 5m0s: 138454 executions, 86147 successful
    rsg_test.go:827: 1m0s of 5m0s: 148840 executions, 92509 successful
    rsg_test.go:827: 1m5s of 5m0s: 162499 executions, 100991 successful
    rsg_test.go:827: 1m10s of 5m0s: 177987 executions, 110604 successful
    rsg_test.go:827: 1m15s of 5m0s: 190824 executions, 118572 successful
    rsg_test.go:827: 1m20s of 5m0s: 205720 executions, 127864 successful
    rsg_test.go:827: 1m25s of 5m0s: 219405 executions, 136387 successful
    rsg_test.go:827: 1m30s of 5m0s: 234173 executions, 145613 successful
    rsg_test.go:827: 1m35s of 5m0s: 245551 executions, 152677 successful
    rsg_test.go:827: 1m40s of 5m0s: 256716 executions, 159623 successful
    rsg_test.go:827: 1m45s of 5m0s: 265841 executions, 165202 successful
    rsg_test.go:827: 1m50s of 5m0s: 278614 executions, 173164 successful
    rsg_test.go:827: 1m55s of 5m0s: 292459 executions, 181778 successful
    rsg_test.go:827: 2m0s of 5m0s: 305101 executions, 189691 successful
    rsg_test.go:827: 2m5s of 5m0s: 317660 executions, 197399 successful
    rsg_test.go:827: 2m10s of 5m0s: 329364 executions, 204708 successful
    rsg_test.go:827: 2m15s of 5m0s: 339925 executions, 211205 successful
    rsg_test.go:827: 2m20s of 5m0s: 354625 executions, 220409 successful
    rsg_test.go:827: 2m25s of 5m0s: 368253 executions, 228785 successful
    rsg_test.go:827: 2m30s of 5m0s: 379995 executions, 235986 successful
    rsg_test.go:827: 2m35s of 5m0s: 391067 executions, 242930 successful
    rsg_test.go:827: 2m40s of 5m0s: 402155 executions, 249819 successful
    rsg_test.go:827: 2m45s of 5m0s: 416258 executions, 258524 successful
    rsg_test.go:827: 2m50s of 5m0s: 430890 executions, 267714 successful
    rsg_test.go:827: 2m55s of 5m0s: 443813 executions, 275698 successful
    rsg_test.go:827: 3m0s of 5m0s: 457796 executions, 284402 successful
    rsg_test.go:827: 3m5s of 5m0s: 473240 executions, 294042 successful
    rsg_test.go:827: 3m10s of 5m0s: 486078 executions, 301911 successful
    rsg_test.go:827: 3m15s of 5m0s: 499421 executions, 310183 successful
    rsg_test.go:827: 3m20s of 5m0s: 512156 executions, 318089 successful
    rsg_test.go:827: 3m25s of 5m0s: 525501 executions, 326383 successful
    rsg_test.go:827: 3m30s of 5m0s: 534060 executions, 331683 successful
    rsg_test.go:827: 3m35s of 5m0s: 544489 executions, 338157 successful
    rsg_test.go:827: 3m40s of 5m0s: 556181 executions, 345451 successful
    rsg_test.go:827: 3m45s of 5m0s: 568193 executions, 352899 successful
    rsg_test.go:827: 3m50s of 5m0s: 579610 executions, 360029 successful
    rsg_test.go:827: 3m55s of 5m0s: 593744 executions, 368876 successful
    rsg_test.go:827: 4m0s of 5m0s: 603639 executions, 374999 successful
    rsg_test.go:827: 4m5s of 5m0s: 612926 executions, 380789 successful
    rsg_test.go:827: 4m10s of 5m0s: 623187 executions, 387116 successful
    rsg_test.go:827: 4m15s of 5m0s: 632820 executions, 393145 successful
    rsg_test.go:827: 4m20s of 5m0s: 642630 executions, 399225 successful
    rsg_test.go:827: 4m25s of 5m0s: 656791 executions, 408079 successful
    rsg_test.go:827: 4m30s of 5m0s: 670418 executions, 416605 successful
    rsg_test.go:827: 4m35s of 5m0s: 681003 executions, 423085 successful
    rsg_test.go:827: 4m40s of 5m0s: 691680 executions, 429753 successful
    rsg_test.go:827: 4m45s of 5m0s: 705488 executions, 438196 successful
    rsg_test.go:827: 4m50s of 5m0s: 716589 executions, 445200 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot todo-deprecate.branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10671682?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10671682?buildTab=artifacts#/) on release-23.1 @ [dbf6ce8438742a73b9d17391c1ef868a4e7a9569](https://github.com/cockroachdb/cockroach/commits/dbf6ce8438742a73b9d17391c1ef868a4e7a9569):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 6082429 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:827: 40s of 5m0s: 95146 executions, 59261 successful
    rsg_test.go:827: 45s of 5m0s: 104017 executions, 64791 successful
    rsg_test.go:827: 50s of 5m0s: 116139 executions, 72252 successful
    rsg_test.go:827: 55s of 5m0s: 125814 executions, 78285 successful
    rsg_test.go:827: 1m0s of 5m0s: 135797 executions, 84447 successful
    rsg_test.go:827: 1m5s of 5m0s: 145399 executions, 90442 successful
    rsg_test.go:827: 1m10s of 5m0s: 157507 executions, 97986 successful
    rsg_test.go:827: 1m15s of 5m0s: 170330 executions, 105893 successful
    rsg_test.go:827: 1m20s of 5m0s: 182293 executions, 113353 successful
    rsg_test.go:827: 1m25s of 5m0s: 195633 executions, 121672 successful
    rsg_test.go:827: 1m30s of 5m0s: 206656 executions, 128486 successful
    rsg_test.go:827: 1m35s of 5m0s: 220822 executions, 137369 successful
    rsg_test.go:827: 1m40s of 5m0s: 232137 executions, 144337 successful
    rsg_test.go:827: 1m45s of 5m0s: 241873 executions, 150407 successful
    rsg_test.go:827: 1m50s of 5m0s: 254495 executions, 158150 successful
    rsg_test.go:827: 1m55s of 5m0s: 263229 executions, 163583 successful
    rsg_test.go:827: 2m0s of 5m0s: 275362 executions, 171159 successful
    rsg_test.go:827: 2m5s of 5m0s: 288453 executions, 179348 successful
    rsg_test.go:827: 2m10s of 5m0s: 297727 executions, 185071 successful
    rsg_test.go:827: 2m15s of 5m0s: 310039 executions, 192760 successful
    rsg_test.go:827: 2m20s of 5m0s: 325234 executions, 202152 successful
    rsg_test.go:827: 2m25s of 5m0s: 336978 executions, 209502 successful
    rsg_test.go:827: 2m30s of 5m0s: 351690 executions, 218643 successful
    rsg_test.go:827: 2m35s of 5m0s: 365054 executions, 226959 successful
    rsg_test.go:827: 2m40s of 5m0s: 375259 executions, 233286 successful
    rsg_test.go:827: 2m45s of 5m0s: 387708 executions, 240957 successful
    rsg_test.go:827: 2m50s of 5m0s: 401416 executions, 249525 successful
    rsg_test.go:827: 2m55s of 5m0s: 409796 executions, 254773 successful
    rsg_test.go:827: 3m0s of 5m0s: 423522 executions, 263334 successful
    rsg_test.go:827: 3m5s of 5m0s: 435219 executions, 270620 successful
    rsg_test.go:827: 3m10s of 5m0s: 448407 executions, 278826 successful
    rsg_test.go:827: 3m15s of 5m0s: 461402 executions, 286847 successful
    rsg_test.go:827: 3m20s of 5m0s: 472581 executions, 293727 successful
    rsg_test.go:827: 3m25s of 5m0s: 481291 executions, 299154 successful
    rsg_test.go:827: 3m30s of 5m0s: 494343 executions, 307361 successful
    rsg_test.go:827: 3m35s of 5m0s: 506373 executions, 314784 successful
    rsg_test.go:827: 3m40s of 5m0s: 515195 executions, 320340 successful
    rsg_test.go:827: 3m45s of 5m0s: 526712 executions, 327488 successful
    rsg_test.go:827: 3m50s of 5m0s: 541575 executions, 336599 successful
    rsg_test.go:827: 3m55s of 5m0s: 553650 executions, 344176 successful
    rsg_test.go:827: 4m0s of 5m0s: 563369 executions, 350225 successful
    rsg_test.go:827: 4m5s of 5m0s: 574396 executions, 357056 successful
    rsg_test.go:827: 4m10s of 5m0s: 588166 executions, 365537 successful
    rsg_test.go:827: 4m15s of 5m0s: 602806 executions, 374540 successful
    rsg_test.go:827: 4m20s of 5m0s: 616048 executions, 382730 successful
    rsg_test.go:827: 4m25s of 5m0s: 627309 executions, 389766 successful
    rsg_test.go:827: 4m30s of 5m0s: 639407 executions, 397252 successful
    rsg_test.go:827: 4m35s of 5m0s: 653161 executions, 405843 successful
    rsg_test.go:827: 4m40s of 5m0s: 664804 executions, 413049 successful
    rsg_test.go:827: 4m45s of 5m0s: 675233 executions, 419480 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot todo-deprecate.branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10689064?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10689064?buildTab=artifacts#/) on release-23.1 @ [534c560feecc136ad71975a3d9255fe5ec3b8c49](https://github.com/cockroachdb/cockroach/commits/534c560feecc136ad71975a3d9255fe5ec3b8c49):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 5787534 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:827: 45s of 5m0s: 107851 executions, 66999 successful
    rsg_test.go:827: 50s of 5m0s: 119116 executions, 74052 successful
    rsg_test.go:827: 55s of 5m0s: 131719 executions, 81782 successful
    rsg_test.go:827: 1m0s of 5m0s: 141546 executions, 87840 successful
    rsg_test.go:827: 1m5s of 5m0s: 151603 executions, 94142 successful
    rsg_test.go:827: 1m10s of 5m0s: 162629 executions, 100902 successful
    rsg_test.go:827: 1m15s of 5m0s: 174393 executions, 108265 successful
    rsg_test.go:827: 1m20s of 5m0s: 184576 executions, 114535 successful
    rsg_test.go:827: 1m25s of 5m0s: 195857 executions, 121551 successful
    rsg_test.go:827: 1m30s of 5m0s: 205401 executions, 127448 successful
    rsg_test.go:827: 1m35s of 5m0s: 217637 executions, 134982 successful
    rsg_test.go:827: 1m40s of 5m0s: 230031 executions, 142739 successful
    rsg_test.go:827: 1m45s of 5m0s: 242304 executions, 150282 successful
    rsg_test.go:827: 1m50s of 5m0s: 253968 executions, 157555 successful
    rsg_test.go:827: 1m55s of 5m0s: 264643 executions, 164276 successful
    rsg_test.go:827: 2m0s of 5m0s: 276831 executions, 171756 successful
    rsg_test.go:827: 2m5s of 5m0s: 285647 executions, 177335 successful
    rsg_test.go:827: 2m10s of 5m0s: 295901 executions, 183635 successful
    rsg_test.go:827: 2m15s of 5m0s: 307416 executions, 190769 successful
    rsg_test.go:827: 2m20s of 5m0s: 318906 executions, 197912 successful
    rsg_test.go:827: 2m25s of 5m0s: 331769 executions, 205982 successful
    rsg_test.go:827: 2m30s of 5m0s: 343752 executions, 213333 successful
    rsg_test.go:827: 2m35s of 5m0s: 354686 executions, 220196 successful
    rsg_test.go:827: 2m40s of 5m0s: 366544 executions, 227499 successful
    rsg_test.go:827: 2m46s of 5m0s: 377654 executions, 234471 successful
    rsg_test.go:827: 2m51s of 5m0s: 388786 executions, 241448 successful
    rsg_test.go:827: 2m56s of 5m0s: 399423 executions, 247972 successful
    rsg_test.go:827: 3m1s of 5m0s: 411089 executions, 255229 successful
    rsg_test.go:827: 3m6s of 5m0s: 421732 executions, 261876 successful
    rsg_test.go:827: 3m11s of 5m0s: 432983 executions, 268893 successful
    rsg_test.go:827: 3m16s of 5m0s: 442742 executions, 274917 successful
    rsg_test.go:827: 3m21s of 5m0s: 454255 executions, 282145 successful
    rsg_test.go:827: 3m26s of 5m0s: 463967 executions, 288185 successful
    rsg_test.go:827: 3m31s of 5m0s: 476834 executions, 296217 successful
    rsg_test.go:827: 3m36s of 5m0s: 489678 executions, 304087 successful
    rsg_test.go:827: 3m41s of 5m0s: 503008 executions, 312419 successful
    rsg_test.go:827: 3m46s of 5m0s: 514503 executions, 319502 successful
    rsg_test.go:827: 3m51s of 5m0s: 527408 executions, 327435 successful
    rsg_test.go:827: 3m56s of 5m0s: 539077 executions, 334693 successful
    rsg_test.go:827: 4m1s of 5m0s: 549963 executions, 341475 successful
    rsg_test.go:827: 4m6s of 5m0s: 563071 executions, 349558 successful
    rsg_test.go:827: 4m11s of 5m0s: 576564 executions, 357942 successful
    rsg_test.go:827: 4m16s of 5m0s: 589995 executions, 366299 successful
    rsg_test.go:827: 4m21s of 5m0s: 602694 executions, 374038 successful
    rsg_test.go:827: 4m26s of 5m0s: 614364 executions, 381315 successful
    rsg_test.go:827: 4m31s of 5m0s: 629497 executions, 390684 successful
    rsg_test.go:827: 4m36s of 5m0s: 643117 executions, 399179 successful
    rsg_test.go:827: 4m41s of 5m0s: 656582 executions, 407561 successful
    rsg_test.go:827: 4m46s of 5m0s: 668439 executions, 415030 successful
    rsg_test.go:827: 4m51s of 5m0s: 680885 executions, 422745 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot todo-deprecate.branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10706526?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10706526?buildTab=artifacts#/) on release-23.1 @ [26252d74997efb2da6a8c0c15e2e90afa8d1c25d](https://github.com/cockroachdb/cockroach/commits/26252d74997efb2da6a8c0c15e2e90afa8d1c25d):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 6110552 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:827: 45s of 5m0s: 116008 executions, 72202 successful
    rsg_test.go:827: 50s of 5m0s: 128963 executions, 80292 successful
    rsg_test.go:827: 55s of 5m0s: 140406 executions, 87425 successful
    rsg_test.go:827: 1m0s of 5m0s: 156002 executions, 97028 successful
    rsg_test.go:827: 1m5s of 5m0s: 166296 executions, 103443 successful
    rsg_test.go:827: 1m10s of 5m0s: 179137 executions, 111438 successful
    rsg_test.go:827: 1m15s of 5m0s: 193769 executions, 120674 successful
    rsg_test.go:827: 1m20s of 5m0s: 207850 executions, 129357 successful
    rsg_test.go:827: 1m25s of 5m0s: 218668 executions, 136192 successful
    rsg_test.go:827: 1m30s of 5m0s: 230409 executions, 143455 successful
    rsg_test.go:827: 1m35s of 5m0s: 243712 executions, 151735 successful
    rsg_test.go:827: 1m40s of 5m0s: 253244 executions, 157705 successful
    rsg_test.go:827: 1m45s of 5m0s: 266491 executions, 165932 successful
    rsg_test.go:827: 1m50s of 5m0s: 279335 executions, 173800 successful
    rsg_test.go:827: 1m55s of 5m0s: 292072 executions, 181706 successful
    rsg_test.go:827: 2m0s of 5m0s: 303089 executions, 188656 successful
    rsg_test.go:827: 2m5s of 5m0s: 316249 executions, 196896 successful
    rsg_test.go:827: 2m10s of 5m0s: 326575 executions, 203245 successful
    rsg_test.go:827: 2m15s of 5m0s: 337244 executions, 209902 successful
    rsg_test.go:827: 2m20s of 5m0s: 350908 executions, 218356 successful
    rsg_test.go:827: 2m25s of 5m0s: 363669 executions, 226235 successful
    rsg_test.go:827: 2m30s of 5m0s: 377408 executions, 234796 successful
    rsg_test.go:827: 2m35s of 5m0s: 388076 executions, 241415 successful
    rsg_test.go:827: 2m40s of 5m0s: 400769 executions, 249301 successful
    rsg_test.go:827: 2m45s of 5m0s: 410476 executions, 255255 successful
    rsg_test.go:827: 2m50s of 5m0s: 421223 executions, 261994 successful
    rsg_test.go:827: 2m55s of 5m0s: 433561 executions, 269647 successful
    rsg_test.go:827: 3m0s of 5m0s: 445866 executions, 277340 successful
    rsg_test.go:827: 3m5s of 5m0s: 456466 executions, 283913 successful
    rsg_test.go:827: 3m10s of 5m0s: 468905 executions, 291694 successful
    rsg_test.go:827: 3m15s of 5m0s: 481967 executions, 299783 successful
    rsg_test.go:827: 3m20s of 5m0s: 496808 executions, 308944 successful
    rsg_test.go:827: 3m25s of 5m0s: 509776 executions, 317104 successful
    rsg_test.go:827: 3m30s of 5m0s: 521052 executions, 324111 successful
    rsg_test.go:827: 3m35s of 5m0s: 533644 executions, 331909 successful
    rsg_test.go:827: 3m40s of 5m0s: 546730 executions, 340034 successful
    rsg_test.go:827: 3m45s of 5m0s: 556758 executions, 346293 successful
    rsg_test.go:827: 3m50s of 5m0s: 566467 executions, 352278 successful
    rsg_test.go:827: 3m55s of 5m0s: 579466 executions, 360306 successful
    rsg_test.go:827: 4m0s of 5m0s: 594406 executions, 369561 successful
    rsg_test.go:827: 4m5s of 5m0s: 610429 executions, 379572 successful
    rsg_test.go:827: 4m10s of 5m0s: 622514 executions, 387105 successful
    rsg_test.go:827: 4m15s of 5m0s: 635978 executions, 395388 successful
    rsg_test.go:827: 4m20s of 5m0s: 645548 executions, 401358 successful
    rsg_test.go:827: 4m25s of 5m0s: 656863 executions, 408479 successful
    rsg_test.go:827: 4m30s of 5m0s: 669566 executions, 416300 successful
    rsg_test.go:827: 4m35s of 5m0s: 680726 executions, 423234 successful
    rsg_test.go:827: 4m40s of 5m0s: 693369 executions, 431169 successful
    rsg_test.go:827: 4m45s of 5m0s: 705912 executions, 438842 successful
    rsg_test.go:827: 4m50s of 5m0s: 720970 executions, 448154 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot todo-deprecate.branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10722940?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10722940?buildTab=artifacts#/) on release-23.1 @ [f13c021bb97531ac876c2c1438a9d4bfc80d1c32](https://github.com/cockroachdb/cockroach/commits/f13c021bb97531ac876c2c1438a9d4bfc80d1c32):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 6245297 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:827: 45s of 5m0s: 112664 executions, 70012 successful
    rsg_test.go:827: 50s of 5m0s: 122997 executions, 76504 successful
    rsg_test.go:827: 55s of 5m0s: 129455 executions, 80404 successful
    rsg_test.go:827: 1m0s of 5m0s: 141456 executions, 87857 successful
    rsg_test.go:827: 1m5s of 5m0s: 153926 executions, 95614 successful
    rsg_test.go:827: 1m10s of 5m0s: 166968 executions, 103737 successful
    rsg_test.go:827: 1m15s of 5m0s: 180568 executions, 112136 successful
    rsg_test.go:827: 1m20s of 5m0s: 194427 executions, 120695 successful
    rsg_test.go:827: 1m25s of 5m0s: 201936 executions, 125371 successful
    rsg_test.go:827: 1m30s of 5m0s: 215621 executions, 133947 successful
    rsg_test.go:827: 1m35s of 5m0s: 229388 executions, 142493 successful
    rsg_test.go:827: 1m40s of 5m0s: 241420 executions, 150025 successful
    rsg_test.go:827: 1m45s of 5m0s: 253331 executions, 157436 successful
    rsg_test.go:827: 1m50s of 5m0s: 265976 executions, 165284 successful
    rsg_test.go:827: 1m55s of 5m0s: 277546 executions, 172491 successful
    rsg_test.go:827: 2m0s of 5m0s: 287584 executions, 178803 successful
    rsg_test.go:827: 2m5s of 5m0s: 300364 executions, 186640 successful
    rsg_test.go:827: 2m10s of 5m0s: 313907 executions, 195091 successful
    rsg_test.go:827: 2m15s of 5m0s: 325088 executions, 202048 successful
    rsg_test.go:827: 2m20s of 5m0s: 338293 executions, 210297 successful
    rsg_test.go:827: 2m25s of 5m0s: 348560 executions, 216711 successful
    rsg_test.go:827: 2m30s of 5m0s: 362805 executions, 225430 successful
    rsg_test.go:827: 2m35s of 5m0s: 377547 executions, 234698 successful
    rsg_test.go:827: 2m40s of 5m0s: 391998 executions, 243615 successful
    rsg_test.go:827: 2m45s of 5m0s: 407372 executions, 253169 successful
    rsg_test.go:827: 2m50s of 5m0s: 419885 executions, 261045 successful
    rsg_test.go:827: 2m55s of 5m0s: 433002 executions, 269245 successful
    rsg_test.go:827: 3m0s of 5m0s: 447011 executions, 277827 successful
    rsg_test.go:827: 3m5s of 5m0s: 460022 executions, 286035 successful
    rsg_test.go:827: 3m10s of 5m0s: 473653 executions, 294387 successful
    rsg_test.go:827: 3m15s of 5m0s: 485887 executions, 302013 successful
    rsg_test.go:827: 3m20s of 5m0s: 497110 executions, 308976 successful
    rsg_test.go:827: 3m25s of 5m0s: 508015 executions, 315748 successful
    rsg_test.go:827: 3m30s of 5m0s: 522148 executions, 324492 successful
    rsg_test.go:827: 3m35s of 5m0s: 533912 executions, 331751 successful
    rsg_test.go:827: 3m40s of 5m0s: 545889 executions, 339246 successful
    rsg_test.go:827: 3m45s of 5m0s: 562173 executions, 349245 successful
    rsg_test.go:827: 3m50s of 5m0s: 574742 executions, 357107 successful
    rsg_test.go:827: 3m55s of 5m0s: 586276 executions, 364257 successful
    rsg_test.go:827: 4m0s of 5m0s: 596970 executions, 370783 successful
    rsg_test.go:827: 4m5s of 5m0s: 609778 executions, 378683 successful
    rsg_test.go:827: 4m10s of 5m0s: 622567 executions, 386767 successful
    rsg_test.go:827: 4m15s of 5m0s: 633747 executions, 393678 successful
    rsg_test.go:827: 4m20s of 5m0s: 645502 executions, 401025 successful
    rsg_test.go:827: 4m25s of 5m0s: 658032 executions, 408800 successful
    rsg_test.go:827: 4m30s of 5m0s: 667986 executions, 415009 successful
    rsg_test.go:827: 4m35s of 5m0s: 679402 executions, 422115 successful
    rsg_test.go:827: 4m40s of 5m0s: 693270 executions, 430852 successful
    rsg_test.go:827: 4m45s of 5m0s: 705081 executions, 438186 successful
    rsg_test.go:827: 4m50s of 5m0s: 717397 executions, 445814 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot todo-deprecate.branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10735800?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10735800?buildTab=artifacts#/) on release-23.1 @ [46b5a2cdc03a1bcbf59aab077cd062a8db62cc1b](https://github.com/cockroachdb/cockroach/commits/46b5a2cdc03a1bcbf59aab077cd062a8db62cc1b):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 6028764 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:827: 45s of 5m0s: 107054 executions, 66451 successful
    rsg_test.go:827: 50s of 5m0s: 119672 executions, 74324 successful
    rsg_test.go:827: 55s of 5m0s: 132710 executions, 82471 successful
    rsg_test.go:827: 1m0s of 5m0s: 141910 executions, 88231 successful
    rsg_test.go:827: 1m5s of 5m0s: 155643 executions, 96709 successful
    rsg_test.go:827: 1m10s of 5m0s: 165884 executions, 103046 successful
    rsg_test.go:827: 1m15s of 5m0s: 178881 executions, 111142 successful
    rsg_test.go:827: 1m20s of 5m0s: 190912 executions, 118652 successful
    rsg_test.go:827: 1m25s of 5m0s: 202322 executions, 125722 successful
    rsg_test.go:827: 1m30s of 5m0s: 215727 executions, 134084 successful
    rsg_test.go:827: 1m35s of 5m0s: 226096 executions, 140645 successful
    rsg_test.go:827: 1m40s of 5m0s: 241341 executions, 150102 successful
    rsg_test.go:827: 1m45s of 5m0s: 253631 executions, 157665 successful
    rsg_test.go:827: 1m50s of 5m0s: 268974 executions, 167210 successful
    rsg_test.go:827: 1m55s of 5m0s: 279239 executions, 173574 successful
    rsg_test.go:827: 2m0s of 5m0s: 292156 executions, 181591 successful
    rsg_test.go:827: 2m5s of 5m0s: 300664 executions, 186914 successful
    rsg_test.go:827: 2m10s of 5m0s: 313413 executions, 194798 successful
    rsg_test.go:827: 2m15s of 5m0s: 324757 executions, 201866 successful
    rsg_test.go:827: 2m20s of 5m0s: 335664 executions, 208662 successful
    rsg_test.go:827: 2m25s of 5m0s: 345311 executions, 214621 successful
    rsg_test.go:827: 2m30s of 5m0s: 354238 executions, 220106 successful
    rsg_test.go:827: 2m35s of 5m0s: 365750 executions, 227281 successful
    rsg_test.go:827: 2m40s of 5m0s: 378512 executions, 235206 successful
    rsg_test.go:827: 2m45s of 5m0s: 390250 executions, 242485 successful
    rsg_test.go:827: 2m50s of 5m0s: 401524 executions, 249506 successful
    rsg_test.go:827: 2m55s of 5m0s: 413857 executions, 257219 successful
    rsg_test.go:827: 3m0s of 5m0s: 425230 executions, 264264 successful
    rsg_test.go:827: 3m5s of 5m0s: 436948 executions, 271580 successful
    rsg_test.go:827: 3m10s of 5m0s: 448896 executions, 278944 successful
    rsg_test.go:827: 3m15s of 5m0s: 457833 executions, 284566 successful
    rsg_test.go:827: 3m20s of 5m0s: 469563 executions, 291800 successful
    rsg_test.go:827: 3m25s of 5m0s: 481970 executions, 299456 successful
    rsg_test.go:827: 3m30s of 5m0s: 493449 executions, 306599 successful
    rsg_test.go:827: 3m35s of 5m0s: 506918 executions, 315003 successful
    rsg_test.go:827: 3m40s of 5m0s: 518743 executions, 322372 successful
    rsg_test.go:827: 3m45s of 5m0s: 529439 executions, 328994 successful
    rsg_test.go:827: 3m50s of 5m0s: 539136 executions, 335026 successful
    rsg_test.go:827: 3m55s of 5m0s: 551610 executions, 342883 successful
    rsg_test.go:827: 4m0s of 5m0s: 562650 executions, 349679 successful
    rsg_test.go:827: 4m5s of 5m0s: 575269 executions, 357527 successful
    rsg_test.go:827: 4m10s of 5m0s: 584575 executions, 363331 successful
    rsg_test.go:827: 4m15s of 5m0s: 596441 executions, 370695 successful
    rsg_test.go:827: 4m20s of 5m0s: 610476 executions, 379476 successful
    rsg_test.go:827: 4m25s of 5m0s: 621789 executions, 386487 successful
    rsg_test.go:827: 4m30s of 5m0s: 635397 executions, 394931 successful
    rsg_test.go:827: 4m35s of 5m0s: 647102 executions, 402253 successful
    rsg_test.go:827: 4m40s of 5m0s: 658127 executions, 409049 successful
    rsg_test.go:827: 4m45s of 5m0s: 669681 executions, 416315 successful
    rsg_test.go:827: 4m50s of 5m0s: 681650 executions, 423785 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot todo-deprecate.branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10750320?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10750320?buildTab=artifacts#/) on release-23.1 @ [e12e85479312972b551677203849d29aeb38ad5f](https://github.com/cockroachdb/cockroach/commits/e12e85479312972b551677203849d29aeb38ad5f):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 6077102 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:827: 45s of 5m0s: 103013 executions, 64147 successful
    rsg_test.go:827: 50s of 5m0s: 113582 executions, 70729 successful
    rsg_test.go:827: 55s of 5m0s: 126944 executions, 79058 successful
    rsg_test.go:827: 1m0s of 5m0s: 136522 executions, 84993 successful
    rsg_test.go:827: 1m5s of 5m0s: 146991 executions, 91466 successful
    rsg_test.go:827: 1m10s of 5m0s: 157543 executions, 98047 successful
    rsg_test.go:827: 1m15s of 5m0s: 169222 executions, 105293 successful
    rsg_test.go:827: 1m20s of 5m0s: 180303 executions, 112158 successful
    rsg_test.go:827: 1m25s of 5m0s: 190845 executions, 118623 successful
    rsg_test.go:827: 1m30s of 5m0s: 202516 executions, 125893 successful
    rsg_test.go:827: 1m35s of 5m0s: 215446 executions, 133956 successful
    rsg_test.go:827: 1m40s of 5m0s: 226498 executions, 140917 successful
    rsg_test.go:827: 1m45s of 5m0s: 238842 executions, 148531 successful
    rsg_test.go:827: 1m50s of 5m0s: 248634 executions, 154600 successful
    rsg_test.go:827: 1m55s of 5m0s: 260038 executions, 161760 successful
    rsg_test.go:827: 2m0s of 5m0s: 273402 executions, 170081 successful
    rsg_test.go:827: 2m5s of 5m0s: 284989 executions, 177294 successful
    rsg_test.go:827: 2m10s of 5m0s: 298267 executions, 185558 successful
    rsg_test.go:827: 2m15s of 5m0s: 313210 executions, 194812 successful
    rsg_test.go:827: 2m20s of 5m0s: 322917 executions, 200871 successful
    rsg_test.go:827: 2m25s of 5m0s: 334764 executions, 208260 successful
    rsg_test.go:827: 2m30s of 5m0s: 344473 executions, 214299 successful
    rsg_test.go:827: 2m35s of 5m0s: 354811 executions, 220760 successful
    rsg_test.go:827: 2m40s of 5m0s: 364512 executions, 226742 successful
    rsg_test.go:827: 2m45s of 5m0s: 377033 executions, 234528 successful
    rsg_test.go:827: 2m50s of 5m0s: 391170 executions, 243368 successful
    rsg_test.go:827: 2m55s of 5m0s: 403070 executions, 250735 successful
    rsg_test.go:827: 3m0s of 5m0s: 417579 executions, 259887 successful
    rsg_test.go:827: 3m5s of 5m0s: 428361 executions, 266524 successful
    rsg_test.go:827: 3m10s of 5m0s: 437294 executions, 272032 successful
    rsg_test.go:827: 3m15s of 5m0s: 450221 executions, 280178 successful
    rsg_test.go:827: 3m20s of 5m0s: 462075 executions, 287508 successful
    rsg_test.go:827: 3m25s of 5m0s: 474773 executions, 295444 successful
    rsg_test.go:827: 3m30s of 5m0s: 487148 executions, 303071 successful
    rsg_test.go:827: 3m35s of 5m0s: 499155 executions, 310572 successful
    rsg_test.go:827: 3m40s of 5m0s: 510901 executions, 317841 successful
    rsg_test.go:827: 3m45s of 5m0s: 524722 executions, 326543 successful
    rsg_test.go:827: 3m50s of 5m0s: 540872 executions, 336551 successful
    rsg_test.go:827: 3m55s of 5m0s: 551670 executions, 343349 successful
    rsg_test.go:827: 4m0s of 5m0s: 563248 executions, 350539 successful
    rsg_test.go:827: 4m5s of 5m0s: 574272 executions, 357296 successful
    rsg_test.go:827: 4m10s of 5m0s: 587201 executions, 365346 successful
    rsg_test.go:827: 4m15s of 5m0s: 597654 executions, 371884 successful
    rsg_test.go:827: 4m20s of 5m0s: 609761 executions, 379274 successful
    rsg_test.go:827: 4m25s of 5m0s: 621958 executions, 386865 successful
    rsg_test.go:827: 4m30s of 5m0s: 633466 executions, 393997 successful
    rsg_test.go:827: 4m35s of 5m0s: 643727 executions, 400406 successful
    rsg_test.go:827: 4m41s of 5m0s: 655069 executions, 407414 successful
    rsg_test.go:827: 4m46s of 5m0s: 668627 executions, 415776 successful
    rsg_test.go:827: 4m51s of 5m0s: 682434 executions, 424281 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations todo-deprecate.branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10756617?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10756617?buildTab=artifacts#/) on release-23.1 @ [e12e85479312972b551677203849d29aeb38ad5f](https://github.com/cockroachdb/cockroach/commits/e12e85479312972b551677203849d29aeb38ad5f):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 6157189 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:827: 45s of 5m0s: 108759 executions, 67562 successful
    rsg_test.go:827: 50s of 5m0s: 119155 executions, 74117 successful
    rsg_test.go:827: 55s of 5m0s: 131202 executions, 81442 successful
    rsg_test.go:827: 1m0s of 5m0s: 144235 executions, 89642 successful
    rsg_test.go:827: 1m5s of 5m0s: 152103 executions, 94463 successful
    rsg_test.go:827: 1m10s of 5m0s: 161582 executions, 100416 successful
    rsg_test.go:827: 1m15s of 5m0s: 172341 executions, 107034 successful
    rsg_test.go:827: 1m20s of 5m0s: 185537 executions, 115207 successful
    rsg_test.go:827: 1m25s of 5m0s: 196487 executions, 122009 successful
    rsg_test.go:827: 1m30s of 5m0s: 205979 executions, 127914 successful
    rsg_test.go:827: 1m35s of 5m0s: 214105 executions, 132982 successful
    rsg_test.go:827: 1m40s of 5m0s: 225795 executions, 140276 successful
    rsg_test.go:827: 1m45s of 5m0s: 235154 executions, 146103 successful
    rsg_test.go:827: 1m50s of 5m0s: 245031 executions, 152243 successful
    rsg_test.go:827: 1m55s of 5m0s: 255683 executions, 158798 successful
    rsg_test.go:827: 2m0s of 5m0s: 268905 executions, 166987 successful
    rsg_test.go:827: 2m5s of 5m0s: 279519 executions, 173580 successful
    rsg_test.go:827: 2m10s of 5m0s: 291069 executions, 180786 successful
    rsg_test.go:827: 2m15s of 5m0s: 301642 executions, 187249 successful
    rsg_test.go:827: 2m20s of 5m0s: 312748 executions, 194233 successful
    rsg_test.go:827: 2m25s of 5m0s: 322104 executions, 200051 successful
    rsg_test.go:827: 2m30s of 5m0s: 334772 executions, 207850 successful
    rsg_test.go:827: 2m35s of 5m0s: 347119 executions, 215620 successful
    rsg_test.go:827: 2m40s of 5m0s: 358973 executions, 222944 successful
    rsg_test.go:827: 2m45s of 5m0s: 370655 executions, 230249 successful
    rsg_test.go:827: 2m50s of 5m0s: 382244 executions, 237451 successful
    rsg_test.go:827: 2m55s of 5m0s: 394297 executions, 244996 successful
    rsg_test.go:827: 3m0s of 5m0s: 406405 executions, 252576 successful
    rsg_test.go:827: 3m5s of 5m0s: 419786 executions, 260896 successful
    rsg_test.go:827: 3m10s of 5m0s: 435045 executions, 270433 successful
    rsg_test.go:827: 3m15s of 5m0s: 448864 executions, 278950 successful
    rsg_test.go:827: 3m20s of 5m0s: 460416 executions, 286088 successful
    rsg_test.go:827: 3m25s of 5m0s: 473130 executions, 294107 successful
    rsg_test.go:827: 3m30s of 5m0s: 483469 executions, 300439 successful
    rsg_test.go:827: 3m35s of 5m0s: 496582 executions, 308589 successful
    rsg_test.go:827: 3m40s of 5m0s: 510669 executions, 317353 successful
    rsg_test.go:827: 3m45s of 5m0s: 522279 executions, 324563 successful
    rsg_test.go:827: 3m50s of 5m0s: 533392 executions, 331532 successful
    rsg_test.go:827: 3m55s of 5m0s: 544632 executions, 338569 successful
    rsg_test.go:827: 4m0s of 5m0s: 555124 executions, 345110 successful
    rsg_test.go:827: 4m5s of 5m0s: 569577 executions, 354133 successful
    rsg_test.go:827: 4m10s of 5m0s: 581467 executions, 361579 successful
    rsg_test.go:827: 4m15s of 5m0s: 592940 executions, 368731 successful
    rsg_test.go:827: 4m20s of 5m0s: 605898 executions, 376801 successful
    rsg_test.go:827: 4m25s of 5m0s: 616663 executions, 383582 successful
    rsg_test.go:827: 4m30s of 5m0s: 632128 executions, 393252 successful
    rsg_test.go:827: 4m35s of 5m0s: 646345 executions, 401971 successful
    rsg_test.go:827: 4m40s of 5m0s: 655273 executions, 407591 successful
    rsg_test.go:827: 4m45s of 5m0s: 666043 executions, 414184 successful
    rsg_test.go:827: 4m50s of 5m0s: 680452 executions, 423165 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations todo-deprecate.branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10763050?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10763050?buildTab=artifacts#/) on release-23.1 @ [e12e85479312972b551677203849d29aeb38ad5f](https://github.com/cockroachdb/cockroach/commits/e12e85479312972b551677203849d29aeb38ad5f):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 5923524 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:827: 45s of 5m0s: 100956 executions, 62706 successful
    rsg_test.go:827: 50s of 5m0s: 110137 executions, 68399 successful
    rsg_test.go:827: 55s of 5m0s: 122511 executions, 76114 successful
    rsg_test.go:827: 1m0s of 5m0s: 134536 executions, 83665 successful
    rsg_test.go:827: 1m5s of 5m0s: 143543 executions, 89181 successful
    rsg_test.go:827: 1m10s of 5m0s: 154958 executions, 96265 successful
    rsg_test.go:827: 1m15s of 5m0s: 167638 executions, 104140 successful
    rsg_test.go:827: 1m20s of 5m0s: 177932 executions, 110569 successful
    rsg_test.go:827: 1m25s of 5m0s: 189030 executions, 117466 successful
    rsg_test.go:827: 1m30s of 5m0s: 200183 executions, 124299 successful
    rsg_test.go:827: 1m35s of 5m0s: 212414 executions, 131955 successful
    rsg_test.go:827: 1m40s of 5m0s: 223593 executions, 138898 successful
    rsg_test.go:827: 1m45s of 5m0s: 232849 executions, 144629 successful
    rsg_test.go:827: 1m50s of 5m0s: 245200 executions, 152376 successful
    rsg_test.go:827: 1m55s of 5m0s: 255647 executions, 158813 successful
    rsg_test.go:827: 2m0s of 5m0s: 264636 executions, 164506 successful
    rsg_test.go:827: 2m5s of 5m0s: 275402 executions, 171136 successful
    rsg_test.go:827: 2m10s of 5m0s: 287513 executions, 178783 successful
    rsg_test.go:827: 2m15s of 5m0s: 296115 executions, 184082 successful
    rsg_test.go:827: 2m20s of 5m0s: 304770 executions, 189526 successful
    rsg_test.go:827: 2m25s of 5m0s: 316100 executions, 196520 successful
    rsg_test.go:827: 2m30s of 5m0s: 329555 executions, 204886 successful
    rsg_test.go:827: 2m35s of 5m0s: 342961 executions, 213203 successful
    rsg_test.go:827: 2m40s of 5m0s: 357006 executions, 221878 successful
    rsg_test.go:827: 2m45s of 5m0s: 369138 executions, 229363 successful
    rsg_test.go:827: 2m50s of 5m0s: 383239 executions, 238173 successful
    rsg_test.go:827: 2m55s of 5m0s: 397491 executions, 246993 successful
    rsg_test.go:827: 3m0s of 5m0s: 408817 executions, 254034 successful
    rsg_test.go:827: 3m5s of 5m0s: 416648 executions, 258924 successful
    rsg_test.go:827: 3m10s of 5m0s: 429512 executions, 266978 successful
    rsg_test.go:827: 3m15s of 5m0s: 441615 executions, 274466 successful
    rsg_test.go:827: 3m20s of 5m0s: 450427 executions, 279994 successful
    rsg_test.go:827: 3m25s of 5m0s: 461752 executions, 287056 successful
    rsg_test.go:827: 3m30s of 5m0s: 473531 executions, 294332 successful
    rsg_test.go:827: 3m35s of 5m0s: 484227 executions, 300993 successful
    rsg_test.go:827: 3m40s of 5m0s: 494603 executions, 307509 successful
    rsg_test.go:827: 3m45s of 5m0s: 505775 executions, 314446 successful
    rsg_test.go:827: 3m50s of 5m0s: 515436 executions, 320462 successful
    rsg_test.go:827: 3m55s of 5m0s: 527452 executions, 327870 successful
    rsg_test.go:827: 4m0s of 5m0s: 538922 executions, 335022 successful
    rsg_test.go:827: 4m5s of 5m0s: 552156 executions, 343296 successful
    rsg_test.go:827: 4m10s of 5m0s: 565179 executions, 351495 successful
    rsg_test.go:827: 4m15s of 5m0s: 580421 executions, 360971 successful
    rsg_test.go:827: 4m21s of 5m0s: 590637 executions, 367288 successful
    rsg_test.go:827: 4m26s of 5m0s: 603203 executions, 375175 successful
    rsg_test.go:827: 4m31s of 5m0s: 613920 executions, 381798 successful
    rsg_test.go:827: 4m36s of 5m0s: 628602 executions, 390924 successful
    rsg_test.go:827: 4m41s of 5m0s: 642322 executions, 399370 successful
    rsg_test.go:827: 4m46s of 5m0s: 650203 executions, 404290 successful
    rsg_test.go:827: 4m51s of 5m0s: 662424 executions, 411900 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations todo-deprecate.branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10771366?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10771366?buildTab=artifacts#/) on release-23.1 @ [a26ee7efd223c92cda531152837302f03a14f461](https://github.com/cockroachdb/cockroach/commits/a26ee7efd223c92cda531152837302f03a14f461):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 6008512 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:827: 45s of 5m0s: 106088 executions, 65981 successful
    rsg_test.go:827: 50s of 5m0s: 115871 executions, 71996 successful
    rsg_test.go:827: 55s of 5m0s: 126147 executions, 78412 successful
    rsg_test.go:827: 1m0s of 5m0s: 137851 executions, 85722 successful
    rsg_test.go:827: 1m5s of 5m0s: 148733 executions, 92454 successful
    rsg_test.go:827: 1m10s of 5m0s: 162381 executions, 100920 successful
    rsg_test.go:827: 1m15s of 5m0s: 174670 executions, 108610 successful
    rsg_test.go:827: 1m20s of 5m0s: 187453 executions, 116510 successful
    rsg_test.go:827: 1m25s of 5m0s: 200448 executions, 124577 successful
    rsg_test.go:827: 1m30s of 5m0s: 211837 executions, 131636 successful
    rsg_test.go:827: 1m35s of 5m0s: 220656 executions, 137123 successful
    rsg_test.go:827: 1m40s of 5m0s: 233436 executions, 145037 successful
    rsg_test.go:827: 1m45s of 5m0s: 240625 executions, 149591 successful
    rsg_test.go:827: 1m50s of 5m0s: 251353 executions, 156316 successful
    rsg_test.go:827: 1m55s of 5m0s: 261952 executions, 162867 successful
    rsg_test.go:827: 2m0s of 5m0s: 269884 executions, 167827 successful
    rsg_test.go:827: 2m5s of 5m0s: 280124 executions, 174245 successful
    rsg_test.go:827: 2m10s of 5m0s: 294705 executions, 183286 successful
    rsg_test.go:827: 2m15s of 5m0s: 307855 executions, 191502 successful
    rsg_test.go:827: 2m20s of 5m0s: 316971 executions, 197166 successful
    rsg_test.go:827: 2m25s of 5m0s: 326451 executions, 202994 successful
    rsg_test.go:827: 2m30s of 5m0s: 337920 executions, 210092 successful
    rsg_test.go:827: 2m35s of 5m0s: 349820 executions, 217505 successful
    rsg_test.go:827: 2m40s of 5m0s: 360554 executions, 224190 successful
    rsg_test.go:827: 2m45s of 5m0s: 373612 executions, 232373 successful
    rsg_test.go:827: 2m50s of 5m0s: 383282 executions, 238386 successful
    rsg_test.go:827: 2m55s of 5m0s: 395427 executions, 245780 successful
    rsg_test.go:827: 3m0s of 5m0s: 410084 executions, 255007 successful
    rsg_test.go:827: 3m5s of 5m0s: 424119 executions, 263616 successful
    rsg_test.go:827: 3m10s of 5m0s: 434571 executions, 270174 successful
    rsg_test.go:827: 3m15s of 5m0s: 444459 executions, 276392 successful
    rsg_test.go:827: 3m20s of 5m0s: 456983 executions, 284083 successful
    rsg_test.go:827: 3m25s of 5m0s: 469856 executions, 292055 successful
    rsg_test.go:827: 3m30s of 5m0s: 481043 executions, 299090 successful
    rsg_test.go:827: 3m35s of 5m0s: 491305 executions, 305460 successful
    rsg_test.go:827: 3m40s of 5m0s: 503139 executions, 312831 successful
    rsg_test.go:827: 3m45s of 5m0s: 517198 executions, 321585 successful
    rsg_test.go:827: 3m50s of 5m0s: 530288 executions, 329653 successful
    rsg_test.go:827: 3m55s of 5m0s: 544238 executions, 338340 successful
    rsg_test.go:827: 4m0s of 5m0s: 557618 executions, 346643 successful
    rsg_test.go:827: 4m5s of 5m0s: 569163 executions, 353771 successful
    rsg_test.go:827: 4m10s of 5m0s: 580330 executions, 360747 successful
    rsg_test.go:827: 4m15s of 5m0s: 592660 executions, 368336 successful
    rsg_test.go:827: 4m20s of 5m0s: 603350 executions, 375048 successful
    rsg_test.go:827: 4m25s of 5m0s: 615507 executions, 382490 successful
    rsg_test.go:827: 4m30s of 5m0s: 630503 executions, 391858 successful
    rsg_test.go:827: 4m35s of 5m0s: 644879 executions, 400793 successful
    rsg_test.go:827: 4m40s of 5m0s: 657508 executions, 408675 successful
    rsg_test.go:827: 4m45s of 5m0s: 672639 executions, 418109 successful
    rsg_test.go:827: 4m50s of 5m0s: 684943 executions, 425660 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations todo-deprecate.branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10780242?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10780242?buildTab=artifacts#/) on release-23.1 @ [f1108313b2b51ffab176b6b230bcc15b0259f170](https://github.com/cockroachdb/cockroach/commits/f1108313b2b51ffab176b6b230bcc15b0259f170):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 5742033 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
        github.com/lib/pq.(*conn).recv1Buf(0xc016138000, 0xc03b9095e0?)
        	github.com/lib/pq/external/com_github_lib_pq/conn.go:1059 +0x2e
        github.com/lib/pq.(*conn).recv1(...)
        	github.com/lib/pq/external/com_github_lib_pq/conn.go:1086
        github.com/lib/pq.(*conn).simpleExec(0xc016138000, {0xc024f6e000?, 0x56})
        	github.com/lib/pq/external/com_github_lib_pq/conn.go:664 +0x1ff
        github.com/lib/pq.(*conn).Exec(0xc016138000, {0xc024f6e000, 0x56}, {0xac91e68, 0x0, 0x0})
        	github.com/lib/pq/external/com_github_lib_pq/conn.go:921 +0x229
        github.com/lib/pq.(*conn).ExecContext(0xc03b9095e0?, {0x745a0d8, 0xc002cf40c0}, {0xc024f6e000, 0x56}, {0xac91e68, 0x0, 0x0?})
        	github.com/lib/pq/external/com_github_lib_pq/conn_go18.go:46 +0x109
        database/sql.ctxDriverExec({0x745a0d8?, 0xc002cf40c0?}, {0x7f3f5950d330?, 0xc016138000?}, {0x0?, 0x0?}, {0xc024f6e000?, 0xc010523350?}, {0xac91e68, 0x0, ...})
        	GOROOT/src/database/sql/ctxutil.go:31 +0xe5
        database/sql.(*DB).execDC.func2()
        	GOROOT/src/database/sql/sql.go:1679 +0x175
        database/sql.withLock({0x742a580, 0xc00d44e630}, 0xc02c941dd8)
        	GOROOT/src/database/sql/sql.go:3439 +0x8c
        database/sql.(*DB).execDC(0x1?, {0x745a0d8, 0xc002cf40c0}, 0xc00d44e630, 0x745a180?, {0xc024f6e000, 0x56}, {0x0, 0x0, 0x0})
        	GOROOT/src/database/sql/sql.go:1674 +0x266
        database/sql.(*DB).exec(0xc02762cd20?, {0x745a0d8, 0xc002cf40c0}, {0xc024f6e000, 0x56}, {0x0, 0x0, 0x0}, 0x80?)
        	GOROOT/src/database/sql/sql.go:1659 +0xe8
        database/sql.(*DB).ExecContext(0xc04bedbf80?, {0x745a0d8, 0xc002cf40c0}, {0xc024f6e000, 0x56}, {0x0, 0x0, 0x0})
        	GOROOT/src/database/sql/sql.go:1633 +0xe5
        github.com/cockroachdb/cockroach/pkg/sql/tests_test.(*verifyFormatDB).execWithResettableTimeout.func2()
        	github.com/cockroachdb/cockroach/pkg/sql/tests_test/pkg/sql/tests/rsg_test.go:173 +0x48
        created by github.com/cockroachdb/cockroach/pkg/sql/tests_test.(*verifyFormatDB).execWithResettableTimeout
        	github.com/cockroachdb/cockroach/pkg/sql/tests_test/pkg/sql/tests/rsg_test.go:172 +0x19d
        
    rsg_test.go:252: SELECT 1 executed successfully: probably a slow statement
    rsg_test.go:854: Crash detected: server panic: statement exec timeout
        SELECT crdb_internal.reset_activity_tables() ;
        
        Stack trace:
        timeout: "SELECT crdb_internal.reset_activity_tables() ". currently executing: map[SELECT crdb_internal.gen_rand_ident('':::STRING, 4438343891207023865:::INT8) :1 SELECT crdb_internal.gen_rand_ident(e'F\x04+xf':::STRING, 8521953078799535325:::INT8) :1 SELECT crdb_internal.gen_rand_ident(e'\x06Z':::STRING, 5581778172517789926:::INT8) :1 SELECT crdb_internal.reset_activity_tables() :1 SELECT right('0s':::STRING, 5574776021694051248:::INT8) :1 SELECT round(1.0657106757861896:::FLOAT8) :1 SELECT set_masklen('128.147.83.219/21':::INET, (-2238286470909539544):::INT8) :1 SELECT setval(e'9\x17\x02\x038[':::STRING, (-3976525699025947051):::INT8) :1]
    rsg_test.go:827: 3m30s of 5m0s: 419198 executions, 260759 successful
    rsg_test.go:827: 3m35s of 5m0s: 429566 executions, 267272 successful
    rsg_test.go:827: 3m40s of 5m0s: 437103 executions, 272015 successful
    rsg_test.go:827: 3m45s of 5m0s: 448749 executions, 279232 successful
    rsg_test.go:827: 3m50s of 5m0s: 460348 executions, 286460 successful
    rsg_test.go:827: 3m55s of 5m0s: 471854 executions, 293571 successful
    rsg_test.go:827: 4m0s of 5m0s: 485011 executions, 301773 successful
    rsg_test.go:827: 4m5s of 5m0s: 494482 executions, 307651 successful
    rsg_test.go:827: 4m10s of 5m0s: 502223 executions, 312452 successful
    rsg_test.go:827: 4m15s of 5m0s: 513591 executions, 319543 successful
    rsg_test.go:827: 4m20s of 5m0s: 524250 executions, 326071 successful
    rsg_test.go:827: 4m25s of 5m0s: 534726 executions, 332699 successful
    rsg_test.go:827: 4m30s of 5m0s: 545572 executions, 339391 successful
    rsg_test.go:827: 4m35s of 5m0s: 556134 executions, 346086 successful
    rsg_test.go:827: 4m40s of 5m0s: 565791 executions, 352056 successful
    rsg_test.go:827: 4m45s of 5m0s: 578264 executions, 359815 successful
    rsg_test.go:827: 4m50s of 5m0s: 588163 executions, 366050 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations todo-deprecate.branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10797197?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10797197?buildTab=artifacts#/) on release-23.1 @ [fa2d7f7c9894d701ac4a393f058aa84552957087](https://github.com/cockroachdb/cockroach/commits/fa2d7f7c9894d701ac4a393f058aa84552957087):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 5795987 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
        github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*Server).ServeConn(0xc0041eee70, {0x745a878, 0xc023335d80}, {0x749eea8, 0xc00b9d4700}, {0x0, 0x4, 0x0, {0x5400, 0x15000, ...}, ...})
        	github.com/cockroachdb/cockroach/pkg/sql/pgwire/server.go:844 +0xc56
        github.com/cockroachdb/cockroach/pkg/server.(*systemServerWrapper).serveConn(0xc008b0a968, {0x745a920, 0xc017863440}, {0x749eea8, 0xc00b9d4700}, {0x0, 0x4, 0x0, {0x5400, 0x15000, ...}, ...})
        	github.com/cockroachdb/cockroach/pkg/server/server_controller_sql.go:94 +0x151
        github.com/cockroachdb/cockroach/pkg/server.(*serverController).sqlMux(0xc0041ef130, {0x745a920, 0xc017863440}, {0x749eea8, 0xc00b9d4700}, {0x0, 0x4, 0x0, {0x5400, 0x15000, ...}, ...})
        	github.com/cockroachdb/cockroach/pkg/server/server_controller_sql.go:74 +0x2f9
        github.com/cockroachdb/cockroach/pkg/server.startServeSQL.func1.1({0x745a920?, 0xc017863410?}, {0x74a5af0, 0xc02a3bb4c0})
        	github.com/cockroachdb/cockroach/pkg/server/server_sql.go:1766 +0x24e
        github.com/cockroachdb/cockroach/pkg/util/netutil.(*TCPServer).ServeWith.func1({0x745a920, 0xc017863410})
        	github.com/cockroachdb/cockroach/pkg/util/netutil/net.go:188 +0xc7
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func2()
        	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:470 +0x146
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx
        	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:461 +0x43b
        
        goroutine 5462243 [select]:
        github.com/cockroachdb/cockroach/pkg/internal/client/requestbatcher.(*RequestBatcher).Send(0xc02cc87080, {0x745a8e8, 0xc01f1a6120}, 0x0?, {0x74b3e40, 0xc018f76480})
        	github.com/cockroachdb/cockroach/pkg/internal/client/requestbatcher/batcher.go:264 +0x165
        github.com/cockroachdb/cockroach/pkg/kv/kvserver/intentresolver.(*IntentResolver).gcTxnRecord(0xc004438900, {0x745a8e8, 0xc01f1a6120}, 0xc00474ddd0?, 0xc0257f0a20)
        	github.com/cockroachdb/cockroach/pkg/kv/kvserver/intentresolver/intent_resolver.go:776 +0x3d9
        github.com/cockroachdb/cockroach/pkg/kv/kvserver/intentresolver.(*IntentResolver).cleanupFinishedTxnIntents.func2.1({0x745a8e8?, 0xc01f1a6120?})
        	github.com/cockroachdb/cockroach/pkg/kv/kvserver/intentresolver/intent_resolver.go:819 +0x38
        github.com/cockroachdb/cockroach/pkg/util/contextutil.RunWithTimeout({0x745a920?, 0xc0234e15f0?}, {0x5e8e4cc, 0x12}, 0x4a817c800, 0xc00474df00)
        	github.com/cockroachdb/cockroach/pkg/util/contextutil/context.go:91 +0xed
        github.com/cockroachdb/cockroach/pkg/kv/kvserver/intentresolver.(*IntentResolver).cleanupFinishedTxnIntents.func2({0x745a920, 0xc0234e15f0})
        	github.com/cockroachdb/cockroach/pkg/kv/kvserver/intentresolver/intent_resolver.go:817 +0xac
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func2()
        	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:470 +0x146
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx
        	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:461 +0x43b
        
    rsg_test.go:252: SELECT 1 executed successfully: probably a slow statement
    rsg_test.go:854: Crash detected: server panic: statement exec timeout
        SELECT crdb_internal.reset_activity_tables() ;
        
        Stack trace:
        timeout: "SELECT crdb_internal.reset_activity_tables() ". currently executing: map[SELECT crdb_internal.gen_rand_ident('T6':::STRING, 7941536685674009994:::INT8) :1 SELECT crdb_internal.gen_rand_ident(e'<6\x1c=':::STRING, 5573923149740257983:::INT8) :1 SELECT crdb_internal.reset_activity_tables() :3 SELECT st_mpolyfromwkb('\x':::BYTES, 3921958088354061987:::INT8) :1 SELECT st_mpolyfromwkb('\x6a0417392a':::BYTES) :1 SELECT st_multilinefromtext(e'\x18':::STRING) :1]
    rsg_test.go:827: 3m50s of 5m0s: 459853 executions, 286359 successful
    rsg_test.go:827: 3m55s of 5m0s: 468786 executions, 291919 successful
    rsg_test.go:827: 4m0s of 5m0s: 477957 executions, 297544 successful
    rsg_test.go:827: 4m5s of 5m0s: 488778 executions, 304325 successful
    rsg_test.go:827: 4m10s of 5m0s: 496462 executions, 309083 successful
    rsg_test.go:827: 4m15s of 5m0s: 508132 executions, 316337 successful
    rsg_test.go:827: 4m20s of 5m0s: 521338 executions, 324595 successful
    rsg_test.go:827: 4m25s of 5m0s: 532747 executions, 331652 successful
    rsg_test.go:827: 4m30s of 5m0s: 543129 executions, 337982 successful
    rsg_test.go:827: 4m35s of 5m0s: 554155 executions, 344888 successful
    rsg_test.go:827: 4m40s of 5m0s: 566007 executions, 352179 successful
    rsg_test.go:827: 4m45s of 5m0s: 576219 executions, 358553 successful
    rsg_test.go:827: 4m50s of 5m0s: 585150 executions, 364121 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101413 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations todo-deprecate.branch-release-23.1.0]
- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10813263?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10813263?buildTab=artifacts#/) on release-23.1 @ [3eeb4361863fd8ad291a635676b2adf44b6acb4d](https://github.com/cockroachdb/cockroach/commits/3eeb4361863fd8ad291a635676b2adf44b6acb4d):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 5818118 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
        github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*Server).serveImpl(0xc00c739080, {0x7464818, 0xc03a3fe940}, 0xc010247c00, 0xc052b321e0, {0x0, 0x4, {0x1, {0x5e3c848, 0x3}, ...}, ...}, ...)
        	github.com/cockroachdb/cockroach/pkg/sql/pgwire/server.go:1230 +0x935
        github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*Server).ServeConn(0xc00c739080, {0x7464818, 0xc03a3fe940}, {0x74a9028, 0xc010cc4380}, {0x0, 0x4, 0x0, {0x5400, 0x15000, ...}, ...})
        	github.com/cockroachdb/cockroach/pkg/sql/pgwire/server.go:844 +0xc56
        github.com/cockroachdb/cockroach/pkg/server.(*systemServerWrapper).serveConn(0xc006fc5568, {0x74648c0, 0xc04c49d3e0}, {0x74a9028, 0xc010cc4380}, {0x0, 0x4, 0x0, {0x5400, 0x15000, ...}, ...})
        	github.com/cockroachdb/cockroach/pkg/server/server_controller_sql.go:94 +0x151
        github.com/cockroachdb/cockroach/pkg/server.(*serverController).sqlMux(0xc00c739340, {0x74648c0, 0xc04c49d3e0}, {0x74a9028, 0xc010cc4380}, {0x0, 0x4, 0x0, {0x5400, 0x15000, ...}, ...})
        	github.com/cockroachdb/cockroach/pkg/server/server_controller_sql.go:74 +0x2f9
        github.com/cockroachdb/cockroach/pkg/server.startServeSQL.func1.1({0x74648c0?, 0xc04c49d3b0?}, {0x74afc70, 0xc05542e5e8})
        	github.com/cockroachdb/cockroach/pkg/server/server_sql.go:1766 +0x24e
        github.com/cockroachdb/cockroach/pkg/util/netutil.(*TCPServer).ServeWith.func1({0x74648c0, 0xc04c49d3b0})
        	github.com/cockroachdb/cockroach/pkg/util/netutil/net.go:188 +0xc7
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func2()
        	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:470 +0x146
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx
        	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:461 +0x43b
        
        goroutine 5583642 [select]:
        github.com/cockroachdb/cockroach/pkg/internal/client/requestbatcher.(*RequestBatcher).Send(0xc01b86a580, {0x7464888, 0xc0153130e0}, 0x0?, {0x74be020, 0xc043153ef0})
        	github.com/cockroachdb/cockroach/pkg/internal/client/requestbatcher/batcher.go:264 +0x165
        github.com/cockroachdb/cockroach/pkg/kv/kvserver/intentresolver.(*IntentResolver).gcTxnRecord(0xc01f8fe400, {0x7464888, 0xc0153130e0}, 0xc04d9f0dd0?, 0xc0498c2240)
        	github.com/cockroachdb/cockroach/pkg/kv/kvserver/intentresolver/intent_resolver.go:776 +0x3d9
        github.com/cockroachdb/cockroach/pkg/kv/kvserver/intentresolver.(*IntentResolver).cleanupFinishedTxnIntents.func2.1({0x7464888?, 0xc0153130e0?})
        	github.com/cockroachdb/cockroach/pkg/kv/kvserver/intentresolver/intent_resolver.go:819 +0x38
        github.com/cockroachdb/cockroach/pkg/util/contextutil.RunWithTimeout({0x74648c0?, 0xc0212def60?}, {0x5e93cb6, 0x12}, 0x4a817c800, 0xc04d9f0f00)
        	github.com/cockroachdb/cockroach/pkg/util/contextutil/context.go:91 +0xed
        github.com/cockroachdb/cockroach/pkg/kv/kvserver/intentresolver.(*IntentResolver).cleanupFinishedTxnIntents.func2({0x74648c0, 0xc0212def60})
        	github.com/cockroachdb/cockroach/pkg/kv/kvserver/intentresolver/intent_resolver.go:817 +0xac
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func2()
        	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:470 +0x146
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx
        	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:461 +0x43b
        
    rsg_test.go:252: SELECT 1 executed successfully: probably a slow statement
    rsg_test.go:854: Crash detected: server panic: statement exec timeout
        SELECT crdb_internal.reset_activity_tables() ;
        
        Stack trace:
        timeout: "SELECT crdb_internal.reset_activity_tables() ". currently executing: map[SELECT crdb_internal.gen_rand_ident('*g5':::STRING, 867002317562921860:::INT8) :1 SELECT crdb_internal.gen_rand_ident(e'\x0bAi\x0er\x04-\x7fC':::STRING, 949434383360521159:::INT8) :1 SELECT crdb_internal.reset_activity_tables() :2 SELECT setval(e'k\x05uz-':::STRING, (-4534874705858790859):::INT8) :1 SELECT shobj_description(3513540372:::OID, '"':::STRING) :1 SELECT st_addpoint('01070000C00900000001060000C00700000001030000C001000000040000007845701A89DFED41CC22C8FA8A4CF7C179657CFD5807F5C17E334F876015F541E41E8BFCA26502425844E545B2D2D3C1201353665DB1FF41F2719CC5425800C23A4F77CC9864E5C14038913ED646F7418C8A8FA4F702024298DDD01AE55EE7417845701A89DFED41CC22C8FA8A4CF7C179657CFD5807F5C17E334F876015F54101030000C0010000000B000000540B4A56CE3DE4C1B64737D73F48E7C17CD961A4CF5CF741C03D560F745AD1C1E92C13458E8100C25F703E63AB84FAC11876BE148C31E441B42D48D43B65E94100D29A06EE9DE341849980D23323E0C19C4496E52016ED415CA2E3E41B30E241EC87100270790242C8E8CA50170101C26CA84165D8CBDDC18A68391A513FE8C19853EE6656CD00426F19A34243BEF4C1CFA9431194FFFFC16A2597429E68F8C1CE3B1A4F732102424022ABA88E40A1C11841AE422AF8FC41E47322375A96F1C1ECD3A45CCCE7F84100EC48C42204BF4196DD4107C4ABE3C17CCED83B0131F0418038DE42EE02F64160A3156857E8E841BBE58D0793EEFBC1404D419046B1FC41403EC4BD95D2D541B89E98774EFEF3416C54337CB5F0EEC1A6F0FAFE490AECC113AEA9EC582EFAC1786DC94B853BEE4168A3F72DCDB6DB4190C6975C79BDDC41540B4A56CE3DE4C1B64737D73F48E7C17CD961A4CF5CF741C03D560F745AD1C101030000C00100000004000000D478E7A440D3FD41AAB1DA221CF4EAC148D00650B34FF3C118E8838E80ADF6417B6B0AC9D2C3F0C158A4BC1B066BF64186BC45EBB21FF141C809461EE4B0FA41F4FDDECD9A95E6C178ED217FA552D24119147AD42F66FDC1E0B2F2C7CF43EFC1D478E7A440D3FD41AAB1DA221CF4EAC148D00650B34FF3C118E8838E80ADF64101030000C0010000000C000000120871DDFF1401C2DC5AC0A15AFDDFC1EE6619142A3DF8419477483BAE8FF9411CE55F7D8C08E2C18C3A43256AD6DDC18B201FC2DC3901C22C3C867BA173F44138D4F98ED5B9D841F70030A91D42F5C1DEC7490209BDF841D48D6607A8DAFB41D8286115266A00425020A0B4984DCAC1B8E95A59EC29F6C1BEA05B349FC7EDC180953C3C6DB5DB41F01396686F84E341027622FFDCF1F841840B328BDDEFFB4198C0F2D4950CE141AE3C60B849CCF4416EE2CF4F51B3F6C1ECB909808B56E04140989DA60588E4411CBDAAB9FD34FB417C87BC426CFC0042513CF93B7302F5C1F0E03241338CEEC16660ED6414BEFA413E362A1C1F0DF041F6C449594D4CFFC1CC7501361FEBFAC12C320FEAA774F641AAE99CDB2AF6004258A7DED3A73EE641264FCDEE8A7FF9C1220F55EEF509F3413C0CE2615909FEC130F75C589F47F0410F1B6353A76EF2C180A1EBD984E0EC41B333E2F7ABA8F3C1A8353AC2179BEB41120871DDFF1401C2DC5AC0A15AFDDFC1EE6619142A3DF8419477483BAE8FF94101030000C0010000000400000000C6F81952B6C7412B54E8D0E20602C284980A7C47E9E2C124A0C88C144BEF41CCC5E5ABD768E041445131B09872D7C180AE687D36E09CC11CA6122C339201C2482A4F37B96EF3C10C327365543BF741602070E742B5D3C128F33D547264E84100C6F81952B6C7412B54E8D0E20602C284980A7C47E9E2C124A0C88C144BEF4101030000C00100000007000000AC051E740957F5C170F4C65F2920FEC1E83B39C3991FF4419C0D103F83D9EFC104F0EBC81AC2E4C15761C00E05B8F5C15EFB989D851801C22088B04B1F45FA417E94CD8978C30042A6CC078D9F18E5C180B6A4331FACB1C128400B118800F6415CB0075D7F01EE414093DE2FBD27CA418080A6D2FA729FC13CD3EF5111E4F5416E4261CD1D72F9C14879843D141E0142A0C2E3AF42B3EBC1CC69168BD055F6418C486931779E01C264FC5922B895EE41648F9407B08BFB41206885B4944FE841AC051E740957F5C170F4C65F2920FEC1E83B39C3991FF4419C0D103F83D9EFC101030000C0010000000C000000A20AE112C3B0F1C19064023ACD22D5C100268D89BCB0D2C175CB0F94691300C2E0FD1745D0C4E8C1D2021206323EFDC16E00B589FAC5FEC1D459CDB3B88FE441A0694A2D9A4FD0C1282A91DCE60A00C23A5AC0471EDAF54125EA90BD7B46F4C1283472CFE801D0C19A2D53451FE700C2A021B90E1511FE418883486DF400FD41546203017D09FF419D0459929B14F1C1BCF3FF1E149FEE414841E7BCB8CBFE41364A23CC3D360142FA4D0906E99BF141E39CD1794C4F01C272D27B6B7741F6C16871E5719D4EE241427CE7F437F7F241F8366B9D3806FF419C4B5B1A9F91FA41CC5972D80BACE44158A48F8F23730242689E7090F66FFF41C4C2F58D7DF2DEC100AEBE5F9AE77841EC36620171F3FE41C882702F1DCE0042FE0DBD0DC1770142BA41A7DE2E3602C2D4EA6416B264EB41643862BBCEA3D6C1F8D10B2DDE00F4C133AF46E4B86C00C278C603C6B51BE541FC4D567F396AF6411F7CB451B239FDC1A20AE112C3B0F1C19064023ACD22D5C100268D89BCB0D2C175CB0F94691300C201010000C02B3AF831034100C2366E748673B5F7411419214C7684F6414E258B84E887E2C101060000C00300000001030000C0010000000B000000802C5EFF06B6C841074E56BCCE7700C2D0DE63DF4925ED41D0346D708A4ED341C86AE9775E4ED8416335C61EC348F0C1D81E8980AB9FDAC16448C2FB44FAF9412CA644AC6041FB413AC38E0AA24C02C2E06DE78E5ECDC341B8B5C0BB6333DEC18842C0218512F7415395CAA4A9FAF7C13C1EE837DBCEE6416EC052BD42010042023FDE927E20F741F0CFEE2AB460FE41D45033E73304E941548363F3D0D6E641E42A7E6CC42BE541C44A98A937F1014244682EA0F924EF4110EB4085C90D00C20A0BDC3E8D86E2C15CE43D344740F9411C82C4BDD9E5FA4190DD91B29053E441E67014045D9FFEC18E0B2275378C0142B8F24B45549BD44102B3A5ECDBBBF9C1F00EEED22E72F7C1BC683B4011FCE141F0D4B795CE71EEC1F22DDF6EF73DE5C14422E12B12B4F7C1B0F192A1663CD74148C27074CDD2D94190443CAC7E4CEF41802C5EFF06B6C841074E56BCCE7700C2D0DE63DF4925ED41D0346D708A4ED34101030000C0010000000C000000538F48867A59FBC16856954B2B21F6C16024FE1BB546D7C1EA7A258ECA1302C218D1C2FF7F68E7C1034B988680D4FDC13A5EFF23B154F8C1ECC08B1734B0F8C15061082901DBD8C16B4377E2BEAEF1C180688EFF036592C16ABCC2CF163900C2005CA0E8DC5694C1562ED8E9C539FBC1E0370CF49F86D7415989B42E617A02C2E4BB5DCEB309FC41771254D5F96FF3C126DEF6C1A44CF8418095B909CE5F00C272B08BFEE32C024258572EA841FFF4C180F680CC6440B2C128E76AB6FAA7F5C1B473E34D716B0242788231835941DF416DEC0EF68AE1F3C16AE5DEF825AAEFC120EB56888570E1C1E0A38B1C23F0E84128F0FF6F58DBFD41669D1C8C8654FFC1DAEBC50A8D8FF0C10220CDE23121F241400722CCFC91F8C175563BF21032F2C1300D23070AF0FFC1D0C727B1AC910242D33EC8ECF243F9C184AFC3895040F941E4CB4C5D850FFBC1D4331C817360FD41341C21A7B6D2EEC1542BE9B7C763EAC1538F48867A59FBC16856954B2B21F6C16024FE1BB546D7C1EA7A258ECA1302C201030000C0010000000D0000001DE8B06447C6F6C1217B0E75E1A2F2C1FC641AF8BEE1FD41A01EBE6AC2E4F7C16097C1F7D7CCE6C1905A6E70EA95EFC1306364643BF9DD41E86A9E0FFCEA004230637A5967DDFAC1331EF4128114FDC19D6E446E2C2102C2EE24126A8C61F241AE42B882B1ECE6C1CCEC9D30A22102C2022DF811475FE7C1DB5CD01B91AAFBC1F02135AD124CDD41681E88C5CC19FEC1F45FB114E00CFDC140B9AF605434B841C807FD8CFCBBF3413EAF60681000FBC168CA23655A3AD041CEAB5F8A6FC8F0416C9B9867EEDBE741782A595BF113EBC13CB6C499BA38F8C17C1CFAEC1FD0F841A080B0738675F541827CE6DB4344EDC1300C629BB1C1F6412E4E07D8A65000C2CCC00C9BC265FA412818FAD7F59DD1C13013345A41BBCA4140C7B8375CF4B0C1FECA1915EDE800425C0E396BEF49F441ACE3E236F884DCC1F61F120ED96302C2C060D67D9994FCC1BE52BC22A01D00426E37860D47F5F6C16C8AE9AE359E01420E99D31C1D2400C200CCA19284C9AEC138E10965562301C2C01AB99F30D9A9411DE8B06447C6F6C1217B0E75E1A2F2C1FC641AF8BEE1FD41A01EBE6AC2E4F7C101050000C00300000001020000C003000000D09F62B69908ECC1D0D478148E86F3C15DC16C659D4EF4C1E88FF38B065BFD41C0B92D1107FCF94156FD5E3B834EF141C6EE00612FEDF6C1C0DF3ECF7EFFB0417622371C88D3F8415445FEF2211BFD417484EA119657F441AC835E8DD6BDFC4101020000C003000000AB4DE2B9E53BF7C140CF1FC17F52AEC1AA962F6AB63A01C2801B66534C3FA6C104CC230DFC76FEC1DC769E7E706DFF41CC66DB7B015E0142A69553DA2CB601C2BF48063367CA00C22CFBAFE643BAFD41D4969F6B8097EF415E534EB03682FDC101020000C005000000DE01229D88F7F041C86B757B93B4F9C1D66227A3E060E8C148425DBB5867E2411C5ABE20A64EF841EEDA6EFD8603ECC10826CA13C56AEC41D8511DFC4C2DDFC1C2331590C554F04110D533562671C9C154205362D7D8DFC1805E1CDC61E4A6C102B9CC6C9E68F24100BA77ADD04E894146786F31A74A01C2F186FB52606401C2C45E418C0DABFC41A8904F5CC3D0E14100299F7C01B6DF4110C489E5A9BAEF4101050000C00900000001020000C002000000841AF243DED7E7C14002B5134876FC41D0AB6D0C5188DEC1419727AC6C3FF6C1FCBF1E16A155FFC16CF0F19AC4440042B85D49AC85F4DDC1B73DA7993192F0C101020000C003000000D87D28F786D7D641EAB5167AB879FAC14CE6ECC2DA2DD9C116DF57CAC936F2C100BC9901B2BDDB41CB4AA43BFCE9F8C1CC7C564A6B14FD413D3AB9DAC0F6FCC13C4CE6A57084F541C06B5E6B1D04F3415271254045DFEFC1A4A39F9E68F6FFC101020000C005000000D02070EAEFB7CC416724FA43236EFDC1407D32430A48D041F03CDC14599CC4C100227B89A5C3FD416B384ABA10C601C2502AC3899696014298627420D235DA41FAA501DDCD670242839370A08140F4C1E0B8A99EFDFFFB413ACD9ACFD4B600C29CFC16E91354F9416C5880A19578E1416F17D47824B4F0C1FA586C53F1FFF641624E45E435DBF6C1588163AD31B5D8C1306C25F2DE5FDCC13072F1F14C9CDF4101020000C00600000038C39935F6EBD841C2D292D40E31F1C1F015BDB86855E241D0031B98451CE34188B1A20C119DEF414E8F479F03EBEAC1005C858A63CFD4C1B2C346FC0CF90042A028798CC31BF94172A1CEE4EF07E5C1D4537503DFECF641A4076F123F0DFDC1CA868CDC688F0042E007517C15E3FB414071E8B193DB0142C0F70CD5FAF5D941EDFFB8310ACBFEC1DAFB375AB4DAF541F8EB6BBDE45AFC41AEEF4EF872D8FBC1982A71BCD080E0C140FB00782DD9B2C14CBE2FDAE3EDDFC1902EDD5F5C99DAC101020000C003000000580C820D2D97FC41180D3F87C366D341B0D58CAC9DA2C341601BC22B5256DC41009693ADA8A191416017EA6D84E5F4418300BC0F6C13F5C10CE8EE5AF7C5E2C100468F7A6A729541E4693656407AEC41AAEB9473766AF9C120B21BDAF04ECA4101020000C00300000044F630D0AB7DEF4193FC539EC4A9F3C1E0F058F96C6FCCC1845F8E505DC7F2C12020A6498489B4C150E52FE5A284FC4118A10EAD8B2AD0C15089ED723CE4D8C1EE71F2955BBFEEC188AFA827AB37DC418A44A22629070042C0E4694FFC2CE34101020000C003000000E4F6BF1BA064F8C1A843FBAD1B8FE2C1F6666863CD0AF9C1604AB9229917FD413EDDB5A757F20142804F7505B502BFC178C50F05D8F2E3C11C91BE75E5E3EE418E4D74853637F341D2569C49B36EF841B07907916FB1CCC13082BABDE358F5C101020000C0040000008429D4C78B89F041E7E74EF718B2F1C1605BB3F76DC0B0C1E099D3126EE0BCC13822EA3AF7C3FF41B1500D0948FBF6C1DC444D323469FE4124AA3C8DCD67EAC1EAC97D619794F141A0CC6D60CAEDE741380D597EBD4EFB41F8BA6AE2BB52024224B225126C3FFDC194B42C382E88F041802935C2AC68C7416244DD045289F84101020000C006000000A2352ADB019300C2F877DF35AE3EF7C1F457EC9B06CFE2C1A09E4435C817F2C122B5EF4BB5BDE1C1D446E74EA73AF3C1282CB6EA2059DAC1D214C8054DD0F6C150856FBDC507C2C1528DD379333BF6C129061E3184F701C284AD9868F994FD41205F2F92A7D4CCC12AFAD8C1C5FEEDC1A014F8C42412FA419095578D788BE741B4ACD5F99E0CF24144E9140F7FAF01C2B4DD971D8B62FA41A24C40695893F5C1EC1E4E8DE9CFE74190DC732F883AC14122C1F5DA21FCE2C102EA49F8ABEAF54101020000C00200000050649838D32EED417CF201DD4704E141B8F1E03A12CEF1417C8CE373C605E9C1A28B33841519FFC1A0F56D13DBB9CA415235E2B2C3EAF4C11062FE8A596AE34101010000C0B081554B6D11EC411888D320D358D441302E12436F5AF141D08070A7FF44004201030000C0010000000A00000097EFA13A68D8F8C18A43CD2C9D98FAC118BCE1235DF3FE4160D88A3142C9FDC14024AA661344A64121755461C3B3F5C128C776F35A69E3C106E6AF46E2B2ECC178EA63C841BDF641ABCE38318E70FCC1F0B81C83F1ABD141C0F35184B520DF4116848F930A56FA4150B9FA395834D8C18E9BF46DCC0401C27E92BD2B3746004236C38A667CA4F7C172F9A71F1F76F1419101B727318100C29970B987D7A5F6C1A6E043166E60FFC14EBD69ADAECBF1416EDCE4693AF8FEC160637F72308BF541C19313EBEA5302C200C53D12F761C341C800AB89F0F1EFC143EB011835C7F7C1BC432BDC09EDFCC150B576368AB3C0C178D6651FF891E94181A85C5B8C7E01C25995DB7151B4F8C17050EE3BB85CCAC160C7924F977AD2C1607800394205F6C197EFA13A68D8F8C18A43CD2C9D98FAC118BCE1235DF3FE4160D88A3142C9FDC101030000C00100000008000000FE90A3A05CDFE5C122CBA1239D4FF2C160F857877766B0415A1F6EA759F0F44124E9844D5C00F841977BA677329CFDC1307CD863BA31D8C136859A1FB71BEEC1303B92DB1EBEF94140A0879EFB61FAC19C3D01C3EA35F6415424A5FB1295E241D04173CBC362EF41D0DC090ACE3CDB41EEEDBA06DF63F4C13F61BCE2C353FCC13CE77AB95772EF412CDD55B6470FEA41884D0381D361E6416012FDA90A5CF94178266F4D33BFDB415A60E59CDFF4F341FC3EA6326D18E741967A4FCA597DF0C180A6195DD0A1CE4130E9D7618896CDC11C10E558AD3301421C721D537A58F1C1FE90A3A05CDFE5C122CBA1239D4FF2C160F857877766B0415A1F6EA759F0F441':::GEOMETRY, '01070000C00800000001050000C00200000001020000C00300000098CB4ADAF13ED5C12C82072D556D00421F829038F5CCFCC16805B4495DE1F9418D29E0A8C269F4C13014EC66F70AFC41E40CBE900EC5E841408DF1A4D54DACC1F49402E8E34600C240C8A40ED016CD41309D6E9F5AF1D2C1A061C3EA354BD44101020000C00200000098DBAC167C4ED1416C9C21EB277FE441CC5AF7EB3973F64108088CC3D997FAC18F3DDF5AC6F9F0C118A3B4AB873EEF41ACDA726D9C2C0142CCD938EDB123D5C101050000C00300000001020000C00200000080835CB582E9B2C134A29C9EE0C1EE41D26DBF81B0D5F3C1AE947DDC4780F2C1EB75EDDDEF00F8C10428948B3EE0FA4138D03BB5B868F64118F3E843C574E9C101020000C00600000014BBD98D3DCBF54191D044161DB7F0C18A6E18AE7ED8E3C178329DC605E5D2C138BCDA654462FD41FA72D462E32DEFC11E7E978A9236F0C15678DCA9829A0242280318032A42F941903B70FF92CBEC41A4D01B46B7B9FF4130413CD67BE7CA41442F752DA280FD417AA651D61498014298A5604181C0F4410C7ED81D82E5FD41B73E829B682EF4C108490430EF81FE414E15A7BB7C07FBC122CAC58D4818FA41EF8469A95DB901C2804201C809E7CC417BF83B3B7064F7C17AB10C75B126FDC101020000C003000000D4FB22EB7E40E8C1E41164BB809AF041F274B3367F01E4C14C7EE9A05205F4C1C073385AC7D0BF4180DEA522545B9F41FAE1163F22B6E8C19CE59B3D343BFB4138D89A4A010DFF41F4626C9029420142307A6A7251DEF8418477493FF088EC4101030000C0010000000A00000050DEA8716BD6D3C15618486D4A0EEFC1C082AB4A84A3A7414ABA8C714F7CF541883E558009A6E7419262ED56DF38E2C1D8E62641EE7BD1C117B4F0C52A1502C2D83B69C8E1C1F741763D3D01AD17F4C178886E7FA438F5418CB82CD79DFDDCC16AAD3A1EEFA5014246BE75166ACFFBC1A20CDB9273AAF041BE5B7DBE11CE014280B4D742E708F9417434E957A199E2C1341EAAFDB413EFC1ECA6C50F092EF14198F6A3BC38F8F241FC8D961D19BFD8C178278F566CE1D841D63C4B8DC739E8C19E529935BAC30042CE4EC0C5BB7BF741FE201B7C2152F241B288FDA4B99FF6C128E937C2B0EBE74188CF8C5BF524DF410098218C5B2669418652A92E68D601420C38C7EA775802C264067804C881DBC148A1394DBD4FD3410C98AAEB43ECEDC150DEA8716BD6D3C15618486D4A0EEFC1C082AB4A84A3A7414ABA8C714F7CF54101030000C0010000000A0000007019B0F16D75C5414EBA21BB1FBF00C22C5A038EFC61E4C1E020CCC2596E02423C056D1DA240E2419E6FFE14C741F9C1288DCFC1837FEF416ED35FE3445700429C36E361F62BE141B6525C27DD74EAC1C34C971CE777FEC1F80A71D6B3B6E1C1285F680C200DFE41ABFAE9727206F5C1409F1A3B1003D9C102B59A9E721AF34140E4E359064FEC410099D523908FE5C1C8EC4D529770D7414600CB8F4F9FFFC1C83249512D1CD5C1003E719D18647FC140FA47A7C17F01C272E41B943423F2C15B3ABDA744B900C23CEFEB984AFC0042C9B56870DCF8F4C1B81CC34DDEC7F9C1BA20C7033840F6C1787D7AB32AB0EA4128C502767A0FFF41EABF2E11B5E9F2C1CCC7D3AAD224DFC12C8D390D58D0D7C1CFC461D975CF00C26822052374D0D5C17019B0F16D75C5414EBA21BB1FBF00C22C5A038EFC61E4C1E020CCC2596E024201050000C00300000001020000C004000000EC0905B45C79EAC186356E05E355F7C1C67CFA51D53702C210F8649262DBDBC1BA5DF68BA607E6C1D598A10E1A6EF7C1C051952C8FB3F5C1022CE9A715ADFBC1A4A5B8D9F4EFE6417836C24950B9E8412AF4CDE0D2D7F241BC54848DAAEADFC144F8B1246BECF141CCE7BA8FD671F441049462E52A5602C264C223BA8014EFC101020000C0050000007F77FE021CBD00C2DF7C050D8F0002C2E4F61279668FE34154FE5BF97009F641DE48E234BDDEECC16007BAB47B4BCFC13499897CE2AAE9419E6C1411AED4EEC1C03662B60995ACC180F0A82F60CDA44180086E85645BE741C046121F0E53FBC1802CE1FE36ABB6C17429F0CFE603EF4130C47206B227ED414106C62955D5FDC1C078062C712BE3C1800F5AD19003FC41465BB4277A34E3C1E0031652DA6FE7C101020000C0020000002455EEA48CC4DCC1F0DC66B1F3BEC7C130DFB7C661A2DD41FAE5E48118AAE0C15880531F141CFF41F494F0D98B2702C2E42453EE914500422091E3B31618014201030000C0010000000D0000007BE4DED0FC2EF9C155ED2D0ED92502C2A00870E9CE87C5C104F7644DD8F1FBC160FC915194BFD041E23395223D6FF6C18B51755CA79BFDC1A0FC82779A4FD9412A4694A111EBF141B01567ED0FD101C28017A355CD0A00422841AFB348B3F4416A5CB5440C5CF6418998EAE5065700C20ECA7B2658B8F8C13842A66D9C88D441C88CAEC0B786FF41C02BC42C0FD0FAC1CC16C596EFDBF6418E034500447DF3411C1A24202232F641B6B9AA90BEBEECC130FFC7A63916C3C1FE40938A0DD8E6C1405578CED2F901421C9E88CC4678E5C1980F4E99920DF0C11C4A0F0D55BDF0416AA5F10BD338F141501830C62744FC414209FA446F1602C2400EE41401A6FB41906E6E9D3D1EF04190C96DD6C34BF9416E7F0C0F1C66EAC14026D7ACABE6A6C14A30ED66F195E1C17E36CA0FF6C2F9413783ACFF515601C25802F24FD87DE941F9BC2F3B250EF7C18CC670FEEA33FD417C5D6CEC191CDFC17A4C87D93D5BF241D349ED1E8B6DFEC1C06753B2FEDAE241FC130020BF40E1412CA0ED71C5E3F5417BE4DED0FC2EF9C155ED2D0ED92502C2A00870E9CE87C5C104F7644DD8F1FBC101040000C00700000001010000C082C25575C51CE3C1CED6D225853C01C29A388DD6BD71FFC140C4A3888A1BC74101010000C06C2FDA8780B3F2C16CC50BEFF6A0E0C16000DA726CBAE6C1A2B60E85F515EDC101010000C0AEFAA29431D7F7C10454E1E2EBD0EE419CBE8B1CE25BD5C171CACDCFAE8A02C201010000C0C08686A747ACE541B016BA966462E1C1EA524FFEB008F0C1C0747E63AD4FD2C101010000C0403788B871F9FD41DFBF6154997BFFC1547F36106742E241909B4BA3FF9AF7C101010000C046C5089689B1F2C1E8B2BA146FFAE54140898375896BAAC1A0D8177DEA26CC4101010000C087B7D2D9A26D02C285FCBD6B7C8C02C2CC0C9C801A53FD41FEC54E4DFED2F64101060000C00700000001030000C0010000000700000052EED0547815F0C155BC8F84F2F301C2DC637995E891E641551B25A6809CF2C1A08CEA0FDE36BD4117D5940D933FF7C1A0F2EF5A446AC141D2009A8420F7FDC180BA8A796DE1D641607A76B33393E441B0A2C70A6E89C1C146E09638ABD2F341E448B064E81DF341B6188BDBD6750242E83D314CBD32ECC1AC5C86D0D81BF041B4EB0F3BDB3DEFC1DCE107214A68FB41E4A36A265C3AEF41BC5C25645E65F3C16A7B126EF5DAE0C18CE46429CAD2E241A04F8050AC3CB9C1C3A81EABA453F1C152EED0547815F0C155BC8F84F2F301C2DC637995E891E641551B25A6809CF2C101030000C0010000000D0000001E0864CDB346EAC1FCDD73A514E9EF410076A2428ED6B0C1D4900891A34DE6417C3F8711D02900C24055B6437180DFC1DC67CE2C0BA7E2411093AFC1ABF9CE41205FD6B06EA3F8C1EA263756FA50EFC1C8BC577F560F00C25E136F5D693DF5C180E7C79C1EFDAE41D4BCC7045262EB41F877E770E621004298B47C55DB16E74124F3A207CF78004220EFC584B19EB74180BB91CDDC48974134AD8E6BA0EB0042E62927016F96F941587A0022CC39FA4130F98414F140C2414A713DA42066F84186B3AA6036D0F141F8F62857282DFC415639E8748C1AF941D62EA971E48BEEC1803D49EE6284E641666E9C544DD3F941607D75806326CDC1EC1646D52AF7FD41F879741D088DF041F6F5D22BF31500424751417D3A8BFBC160F8D0784060BAC1CCF7DFC0DC2DED41ECCD368EB20D0042E4DC801CCEF7FCC1447BA2839627E6415089BD610405C541E8D724C2E07AFF41A0A26570739100C21CB9231D307C01420E8D593F59C1E0C1FEB443EEDEF0F8414E903E711F3002C268CDC7C5F056E3C11E0864CDB346EAC1FCDD73A514E9EF410076A2428ED6B0C1D4900891A34DE64101030000C001000000050000008119F8CF359500C2888B029BD551D4C119987FD79F1F02C24CA10C090923E641C0FDF5D19B7A004245F69C010551F3C1EC682636DC02E341387CEEE0BD50014234C978B80A19EC411CD0DB5F2547FF41429DCFE311B0F141B63A7CD8D5D7FAC198B5C667EA50DD41BCDE7FF3ED90F9413411C04BCAC9F741FA38C5A9A9EEF6418119F8CF359500C2888B029BD551D4C119987FD79F1F02C24CA10C090923E64101030000C0010000000C0000001C748E5F8F53F9C17CABD05C7EECEAC13817E84DBDB7D3414C5F85BE730BF341E0C820760F8AF3C1CC1BF2EDCDBCEFC187100E1D61EB01C24E3DD73E125B00429DBCBBDDF3C6F9C160623316312A01C22E1300DA1499E7C1FA54F04943ECEFC1906D547DEBBFDAC141DAFEDDAE00F4C1BAF058EEE1FCF741C6AC504BE397F241FCC7A455DD15DBC1DE12B6AC66A6E1C132CEE1BC444EE2C1C82764424620D941B4DE8B462FDEE4417F7099869B75F5C1E0239B48E4B5FF4153F795A5915202C28EDA66B18FC00142E43BC1AB1ABAF7C12BBD06B983A200C240625527F74CC641609B2D1C8CB0ED4118BEB9EAAB05E9C17A98215D0A3A0042BE1BB9C349C5FBC11069E7524F81D2C12C15651EC7C701425A126875986BFDC100FF7C1B52D69941320D908A9505F6C1443B549E31C3F3412CABF987F83AF041688812F6394EFF417D4329A927F2FDC1300DAD0C97DFDF41F003C947F3B0D5418C37CB73712C02C21C748E5F8F53F9C17CABD05C7EECEAC13817E84DBDB7D3414C5F85BE730BF34101030000C0010000000A0000001CB9E8E39DFFEFC12B1B10A4F832F3C138DC80A15688EE41A0F69FAD07F8CBC16509E5005A9200C2CEA51BD512ADFBC14868E433B5DCFA4163FAF8F307CDF4C1CCF97C3F751DDFC12F23F7488A0AF5C19E44AF2979EBF5414B1A1D2FD251FBC1B2FB56E50F90F9415A0BACACD71702C2C086132B431EAB41279AD23CAA0CF5C196E7B9D82B480242F231BC13D31EE4C14C8BE670D8B500C2C84A4E444774F8C1F4ECE4E1D320FE41A04E0B1C234AC2C148FBF4CEA402F641809BE5ACEC1ADA41F0062B0A4DDBEE41809E7549BBA4BA41E6276F9A0A5EF741B3A6108E3D6AFFC1E0594549E080BF4124691444BF3AF0415486011BA241E941A8D8E1B902F4E541BCE139106378FEC1D6DAE6486366E8C1C82E10FD1094ECC1E265979AA22B02421CB9E8E39DFFEFC12B1B10A4F832F3C138DC80A15688EE41A0F69FAD07F8CBC101030000C0010000000B0000002890D60A03C5EA41C4993C570E5CF2C11A985050C0F6F54158A7C12834DEE8C1A459AE78F874EE41D60C90FAEC8EF1C1DA860A19E632F541383CDCA535F8E641720E801E49B8F34128ACCA51134700C23888737811A4F241CCF2D2B6A2E1FF411EB624744067F741307C9E7F4A91FEC1245FBED6AE77FDC12C354FCC81F1DEC1CEB73AF6EAE7F54110014A484958D3C171A6021470A4F3C19460F93BC2FE0042F45A73E0AA95FE41B0F912850320C741FC4E0AF038BBE241D8DE81B0403700C2803906C58C03F6417C7ECB0D46E3E641846E80E1799ADDC1C0520E7B589EEC4102E9B8360A2CF941BC92CC52518FFE41ECF2C90E9554FE410B400DA2BD8BF2C14066A540655CB341983421B08E65FE41B04D3E7FC0C4C6C1843B05056A2001C2CE3760A5ABD7F3C110EFB9F34F49D84160DEA500E8C8FAC15AE66D9C1E70F3412890D60A03C5EA41C4993C570E5CF2C11A985050C0F6F54158A7C12834DEE8C101030000C0010000000A000000A984353A0E0AF3C1ADF89B449A98FBC1D211868F698D014209EB84407F39F9C134A70D3B05C2E541840687DF40EF00C2CDFA0AE16AB901C2883681B2D418DA4140C8CCF625F5E141D2B93741EDE8EBC1BC2F36444105FAC1CCCC26B35DB0FF411E6628C1925E01424C0657E21241F8C1AE9178C16864F54100D4038CB8E6D841E4462003A56AEF41E043ECCC0F7BFE41506991D481FEED4170CAFD35FF9AE94160A8C825FFA8DAC1B63D4D7F9D58FA412057E03ACC9FC341BCDD0D265A7CF2418EFA993DE694F9C1B4AA36B503FBE64120E95C975511C4C120141A380E82CD41E04A72A62E8DF8C13802B18AA799DE413A63E1994C38F1C1A0EE3373CE17D2C10C1108C0D0D7ECC1009CD9210F2D8D41704BE64C2FC9FA417A1CCF863A9EF241A984353A0E0AF3C1ADF89B449A98FBC1D211868F698D014209EB84407F39F9C1':::GEOMETRY) :1]
    rsg_test.go:827: 4m0s of 5m0s: 475605 executions, 295940 successful
    rsg_test.go:827: 4m5s of 5m0s: 483324 executions, 300702 successful
    rsg_test.go:827: 4m10s of 5m0s: 489361 executions, 304405 successful
    rsg_test.go:827: 4m15s of 5m0s: 498610 executions, 310233 successful
    rsg_test.go:827: 4m20s of 5m0s: 507424 executions, 315604 successful
    rsg_test.go:827: 4m25s of 5m0s: 514046 executions, 319786 successful
    rsg_test.go:827: 4m30s of 5m0s: 523228 executions, 325448 successful
    rsg_test.go:827: 4m35s of 5m0s: 535231 executions, 332882 successful
    rsg_test.go:827: 4m40s of 5m0s: 546621 executions, 339937 successful
    rsg_test.go:827: 4m45s of 5m0s: 555035 executions, 345091 successful
    rsg_test.go:827: 4m50s of 5m0s: 562770 executions, 349912 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10830337?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10830337?buildTab=artifacts#/) on release-23.1 @ [5a53114bf5ae9e5ec672d43acf599f145bf22341](https://github.com/cockroachdb/cockroach/commits/5a53114bf5ae9e5ec672d43acf599f145bf22341):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 5979073 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
        github.com/cockroachdb/cockroach/pkg/sql/sqlliveness/slinstance.(*Instance).heartbeatLoop(0xc0090bba40, {0x7465aa0, 0xc044099ce0})
        	github.com/cockroachdb/cockroach/pkg/sql/sqlliveness/slinstance/slinstance.go:305 +0x57
        github.com/cockroachdb/cockroach/pkg/sql/sqlliveness/slinstance.(*Instance).Start.func1({0x7465aa0?, 0xc044099ce0?})
        	github.com/cockroachdb/cockroach/pkg/sql/sqlliveness/slinstance/slinstance.go:442 +0x74
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func2()
        	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:470 +0x146
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx
        	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:461 +0x43b
        
        goroutine 4349555 [select, 3 minutes]:
        github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile.(*Reconciler).run(0xc01fdcc190, {0x7465aa0, 0xc03ee2d3b0}, 0xc03e6cb580)
        	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile/reconciler.go:104 +0x27b
        github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile.(*Reconciler).StartReconciler.func1({0x7465aa0?, 0xc03ee2d3b0?})
        	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile/reconciler.go:77 +0x34
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func2()
        	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:470 +0x146
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx
        	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:461 +0x43b
        
        goroutine 5642800 [select]:
        github.com/lib/pq.(*conn).watchCancel.func1()
        	github.com/lib/pq/external/com_github_lib_pq/conn_go18.go:106 +0x96
        created by github.com/lib/pq.(*conn).watchCancel
        	github.com/lib/pq/external/com_github_lib_pq/conn_go18.go:105 +0xf5
        
        goroutine 4350700 [select]:
        github.com/cockroachdb/cockroach/pkg/kv/kvprober.(*Prober).Start.func1.1({0x7465aa0, 0xc02804f710})
        	github.com/cockroachdb/cockroach/pkg/kv/kvprober/kvprober.go:308 +0x330
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func2()
        	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:470 +0x146
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx
        	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:461 +0x43b
        
    rsg_test.go:252: SELECT 1 executed successfully: probably a slow statement
    rsg_test.go:854: Crash detected: server panic: statement exec timeout
        SELECT crdb_internal.reset_activity_tables() ;
        
        Stack trace:
        timeout: "SELECT crdb_internal.reset_activity_tables() ". currently executing: map[SELECT crdb_internal.gen_rand_ident(e'-B};``\x13\x18':::STRING, 1310496039799820088:::INT8) :1 SELECT crdb_internal.gen_rand_ident(e'\\y;&(m\nO\x0b':::STRING, 4921841838656431560:::INT8) :1 SELECT crdb_internal.reset_activity_tables() :2 SELECT pg_get_viewdef(3372893840:::OID) :1 SELECT pg_get_viewdef(513047409:::OID, true) :1 SELECT pg_sequence_parameters(4204903245:::OID) :1 SELECT pg_table_is_visible(3267777377:::OID) :1]
    rsg_test.go:827: 4m0s of 5m0s: 517110 executions, 321643 successful
    rsg_test.go:827: 4m5s of 5m0s: 531913 executions, 330833 successful
    rsg_test.go:827: 4m10s of 5m0s: 544581 executions, 338772 successful
    rsg_test.go:827: 4m15s of 5m0s: 555062 executions, 345195 successful
    rsg_test.go:827: 4m20s of 5m0s: 565672 executions, 351869 successful
    rsg_test.go:827: 4m25s of 5m0s: 577365 executions, 359092 successful
    rsg_test.go:827: 4m30s of 5m0s: 589393 executions, 366487 successful
    rsg_test.go:827: 4m35s of 5m0s: 600419 executions, 373442 successful
    rsg_test.go:827: 4m40s of 5m0s: 613114 executions, 381336 successful
    rsg_test.go:827: 4m45s of 5m0s: 625257 executions, 388920 successful
    rsg_test.go:827: 4m50s of 5m0s: 635836 executions, 395486 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10837286?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10837286?buildTab=artifacts#/) on release-23.1 @ [15480b8496e7928845330055d191e331b228eb66](https://github.com/cockroachdb/cockroach/commits/15480b8496e7928845330055d191e331b228eb66):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 6001837 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
        	github.com/cockroachdb/cockroach/pkg/sql/gcjob/gc_job.go:335 +0x185
        github.com/cockroachdb/cockroach/pkg/sql/gcjob.schemaChangeGCResumer.Resume({0xc058e56820}, {0x7465b20, 0xc029775f80}, {0x5b685a0?, 0xc044d546b8})
        	github.com/cockroachdb/cockroach/pkg/sql/gcjob/gc_job.go:314 +0x351
        github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine.func2(0xc058e56820?, 0x0?, {0x7438518?, 0xc044d54698?}, {0x7465b20?, 0xc029775f80?}, {0x5b685a0?, 0xc044d546b8?}, 0xc00b7a1810)
        	github.com/cockroachdb/cockroach/pkg/jobs/registry.go:1628 +0xcc
        github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine(0xc002a0c000, {0x7465b20, 0xc028b5ca20}, {0x5b685a0, 0xc044d546b8}, {0x7438518, 0xc044d54698}, 0xc058e56820, {0xc039cce4f8, 0x7}, ...)
        	github.com/cockroachdb/cockroach/pkg/jobs/registry.go:1629 +0xa3f
        github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).runJob(0xc002a0c000, {0x7465a78, 0xc05a4777c0}, {0x7438518, 0xc044d54698}, 0xc058e56820, {0xc039cce4f8, 0x7}, {0xc03e33f4a0, 0x16})
        	github.com/cockroachdb/cockroach/pkg/jobs/adopt.go:474 +0x637
        github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).resumeJob.func2({0x7465b20?, 0xc028b5c6f0?})
        	github.com/cockroachdb/cockroach/pkg/jobs/adopt.go:388 +0x146
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func2()
        	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:470 +0x146
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx
        	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:461 +0x43b
        
    rsg_test.go:252: SELECT 1 executed successfully: probably a slow statement
    rsg_test.go:854: Crash detected: server panic: statement exec timeout
        SELECT crdb_internal.reset_activity_tables() ;
        
        Stack trace:
        timeout: "SELECT crdb_internal.reset_activity_tables() ". currently executing: map[SELECT crdb_internal.gen_rand_ident(e'Y\x0488b"\'q':::STRING, 3912763194518914044:::INT8) :1 SELECT crdb_internal.gen_rand_ident(e'\'':::STRING, 4770355504165345854:::INT8) :1 SELECT crdb_internal.gen_rand_ident(e't$c\x14M\x02':::STRING, 35470667919136039:::INT8) :1 SELECT crdb_internal.reset_activity_tables() :2 SELECT crdb_internal.revalidate_unique_constraint(e'p\x18}':::STRING, 'a?|@P;$z,':::STRING) :1 SELECT crdb_internal.revalidate_unique_constraints_in_table(e'2g\x10\x10!")':::STRING) :1 SELECT crdb_internal.round_decimal_values(NULL::decimal, (-708407309894081543):::INT8) :1]
    rsg_test.go:827: 2m35s of 5m0s: 333827 executions, 207398 successful
    rsg_test.go:827: 2m40s of 5m0s: 343637 executions, 213486 successful
    rsg_test.go:827: 2m45s of 5m0s: 354028 executions, 219946 successful
    rsg_test.go:827: 2m50s of 5m0s: 367154 executions, 228042 successful
    rsg_test.go:827: 2m55s of 5m0s: 379955 executions, 236066 successful
    rsg_test.go:827: 3m0s of 5m0s: 389417 executions, 241985 successful
    rsg_test.go:827: 3m5s of 5m0s: 400161 executions, 248557 successful
    rsg_test.go:827: 3m10s of 5m0s: 412712 executions, 256433 successful
    rsg_test.go:827: 3m15s of 5m0s: 423938 executions, 263431 successful
    rsg_test.go:827: 3m20s of 5m0s: 435366 executions, 270550 successful
    rsg_test.go:827: 3m25s of 5m0s: 446889 executions, 277701 successful
    rsg_test.go:827: 3m30s of 5m0s: 459789 executions, 285653 successful
    rsg_test.go:827: 3m35s of 5m0s: 472347 executions, 293430 successful
    rsg_test.go:827: 3m40s of 5m0s: 485432 executions, 301669 successful
    rsg_test.go:827: 3m45s of 5m0s: 498312 executions, 309677 successful
    rsg_test.go:827: 3m50s of 5m0s: 511494 executions, 317870 successful
    rsg_test.go:827: 3m55s of 5m0s: 522268 executions, 324556 successful
    rsg_test.go:827: 4m0s of 5m0s: 530225 executions, 329567 successful
    rsg_test.go:827: 4m5s of 5m0s: 541867 executions, 336805 successful
    rsg_test.go:827: 4m10s of 5m0s: 555252 executions, 345121 successful
    rsg_test.go:827: 4m15s of 5m0s: 568459 executions, 353341 successful
    rsg_test.go:827: 4m20s of 5m0s: 578636 executions, 359663 successful
    rsg_test.go:827: 4m25s of 5m0s: 590311 executions, 366941 successful
    rsg_test.go:827: 4m30s of 5m0s: 601082 executions, 373611 successful
    rsg_test.go:827: 4m35s of 5m0s: 612284 executions, 380576 successful
    rsg_test.go:827: 4m40s of 5m0s: 622652 executions, 387060 successful
    rsg_test.go:827: 4m45s of 5m0s: 634286 executions, 394382 successful
    rsg_test.go:827: 4m50s of 5m0s: 645615 executions, 401436 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10843956?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10843956?buildTab=artifacts#/) on release-23.1 @ [f62527e83e458f5f4521497e7bfd5c60b70f1e31](https://github.com/cockroachdb/cockroach/commits/f62527e83e458f5f4521497e7bfd5c60b70f1e31):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 5756327 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func2()
        	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:470 +0x146
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx
        	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:461 +0x43b
        
        goroutine 4130575 [select, 2 minutes]:
        github.com/cockroachdb/cockroach/pkg/util/hlc.(*Clock).StartMonitoringForwardClockJumps.func1()
        	github.com/cockroachdb/cockroach/pkg/util/hlc/pkg/util/hlc/hlc.go:275 +0x15a
        created by github.com/cockroachdb/cockroach/pkg/util/hlc.(*Clock).StartMonitoringForwardClockJumps
        	github.com/cockroachdb/cockroach/pkg/util/hlc/pkg/util/hlc/hlc.go:267 +0x13d
        
    rsg_test.go:252: SELECT 1 executed successfully: probably a slow statement
    rsg_test.go:854: Crash detected: server panic: statement exec timeout
        SELECT crdb_internal.reset_activity_tables() ;
        
        Stack trace:
        timeout: "SELECT crdb_internal.reset_activity_tables() ". currently executing: map[SELECT crdb_internal.gen_rand_ident(e'\x16':::STRING, 3749173936119634548:::INT8) :1 SELECT crdb_internal.gen_rand_ident(e'\x19lwkJ':::STRING, 6815074033943640110:::INT8) :1 SELECT crdb_internal.reset_activity_tables() :2 SELECT crdb_internal.show_create_all_tables('':::STRING) :1 SELECT geography_out('01070000A0E61000000600000001050000A0E610000009000000010200008004000000DEFB3CD068C761C0FB180783A3BC50C0F7F9EF5FF167FCC100E16322C00F34C00837AE285D6A31C00C73ADBB88BEFD416CC6C5BDC27C4940502D01365AD115C060A684EC5FA9E341D110D9EB9CC550C010DD3139E8AD2EC0A0FC809A7425B3C101020000800800000000EEC3EC688165C0BA5DC493AD3055C04FBB59B4ED1901C28C756A2A15A75BC0768116AC19E84DC09FA84D0BB6A001C20038F8083A82524040B4A7A0882F45C024339D74033EFB4112B5F134A1D0614096893EAC0767544070C5C071F51FC5C1A0F150E3B4171540F8E1638EF0BD3A405456C2D792B8DBC1F480C2FF40E046C0C0C5B832FD8A3740A6D00D0F626901C29316F6C1792F64C05063E490147336409B2DF13B1349F1C18F64D6E5830062C030EE2058A7822140BD8F492621BBFAC1010200008003000000281BC07650D254408C6E8555F0C24D40AEF319D4BF79F3C1DCE3BFF5920F4140C4F22CBEB0DC544080BD48F2CEB5014210F2D09C1D912140A0F1798CC4684B40E75BD347615901C2010200008004000000C4DC01A7BDAA5240E2F439AEB10E4BC0FFAB6A6A43E900C2CADD521C1B0A65404B8A0E5974AC55C001C46665199202C2D04436A7E4925C40346E9C31B6E54540CCD9268C52CEFC4178C7285AC93D51400C813C4D944A45401811EEE6BA7AD54101020000800200000068152DBBF9905640D13E621E40974BC013DD828E4A90FDC180EF82D816C3634070C761BA589C2EC06CBA745FD87DF4C1010200008003000000545BDECAC0236640DE2AADBC07864DC0324C83D719C0FEC13882CAFE638B5E40B80F82C568A03940501848B969E6F9C13CB0A922FD955C405C5A9148B2123D405E88EBB471CDF3C1010200008003000000A042543CAB5433C00CE7BF127A524AC0F41A22EEA604E2418039999CC25A224082F612E5AB0054C01C7676978727DAC1A6A1796692A76240A813AED964023E40424B74923A59F641010200008004000000F4A542B419315940744A529A73E239C07056012BA4050042701595A952235E400017DC7307970CC0408109A2EA90F4C1F899A88FD7B3414070354AFE9D5827C02C1CE6144B5CDBC1540F7ADA4F0C594080198B10941B1740246018531EDEF541010200008004000000C00AEDDCA40746C062411C98590342C0503E66716D9BC7C1AAE18B8FB88D4BC08184A03BA4F355C01AD98C9304B4FAC12CB3D8F9D36A4540066FF667CE6647C0C2A2C349165FF841AA89D568AE3864409015A27349A221C0CEA106B5F868F8C101020000A0E610000002000000F0EF3A5756E35640C04F32B9868D15C0408D907AA4FC0142387EC77351493E40B0116EB9C698554070AAE88288A9D54101050000A0E610000006000000010200008003000000495F36FC227C57C09115EF70733355C08AF5AA15ADB3F7412C14E06DDB2549C00B02A2FFEE6647C0BAC457E443C3E1C13C1633212F0056404FA6F48A310B55C0309D4AFAFD53E241010200008004000000FCFA9A19C2635740ECAC951CB2EE48C01CA2A4EBCFF3E7C19C1F8FFA70386440F81CA24085C74A40C4C3C37889F6D5C13E136E2184E657C040DE10C6B2705140DCF7F764D124EC417FC92F31788155C0609479AFBC1D4340F04FFDE4A476EEC1010200008004000000701B35AC316D5040D04FA05454A63A407CB030BCC18F0042E25DD5B9988052405C7CE80D134D37402EBBA0145CBF0142C8D0A3CD4D0A634080BE38D09FDEE93F9883BC37C2A5FF41D624D9FCAF3A65407C49B27AC00F494000DBE9B2F0BE8EC10102000080030000001627E706181B61C088CAF63B952B44401A4DDCED5CA8F341C871319B913351C0BC947215309F464082A96D939815E8C1B8E4F1C07A263CC0481350A7B14F53401E445199510101420102000080020000003ACA6978CC1651C0857B2CB23A0746C0744CE17C15BFE541A0CF784CED2A6440EC06DD854D19454000ED490B3068BBC1010200008002000000FE07ADD0ED7350403B1097D3FBC346C02C840E7D9B0EFFC137B66F75E80C53C0C8762BDB28AB4540FCB9F728789ED8C101040000A0E6100000090000000101000080504CE65D445D53C0307CD64C8CCE424086BDEAC27E8CE3C101010000804DF3C847FB7E5CC0F025B512D2FD22C0F247D55EDC8EF1410101000080C075996FD6FB0D40D8D6276E770F54C0CCE0BA7F6629EBC101010000806C2418200FC64E4073EBC5C7502540C029E84CFAC94600C2010100008034E6E23F2FF55040BEA50FC5E9593AC058BBD6994E35E9C10101000080B47356A5C8B05FC0F84BE1FE20874640105C22623773C5C10101000080FCAEE309CADE5D40F2AB7FB8ED0C4240D22AD4E84583ECC1010100008098B358C00FC553401EF74B29772656C018F23A545531F1410101000080D68B570BF61D66402840C48D955239C076750D3831EDF44101030000A0E610000001000000060000000EB4DC49CF5B61C0CD24A820815A46C0086E5C81B78AE6C137858AE51AF354C084945AF8812E41C004B906A21755D5C1F8D2807A807B5A40886DAFBA01E12FC01805C50187F1D2C11E834498EB7C644000355A6FE86BF2BF482C12622D86E8C1440016C34E4253C00439FD8FAE37464028962E8E8784E7410EB4DC49CF5B61C0CD24A820815A46C0086E5C81B78AE6C101040000A0E6100000090000000101000080A488F9AA062E514067167B50ED9C55C0B09202A962ABC7C10101000080B200F93F923A6640C0CC9358AA1D51C01406E3612D21FCC101010000808A722DE7DB1B53C0C91E18D08A3255C064B6B90D74B4FC410101000080106F19C510E033C04AE992D6571142409C9371FB9AAAFFC10101000080E83B58741D9845C0FA6A7D62156840C0819654261A1601C20101000080180FE0CDCF555B40289EF8B5F038534088DB850D789DDDC10101000080965FA7041F1B6540D28F369FE1FE4EC0A6F22B9320BF014201010000808F4497B336EB59C0395F030E4AD249C03B42177A3FC7F3C101010000806C64FCE2EE3145400039A631CB9AF93F162E2415EF7EFA41':::GEOGRAPHY) :1 SELECT geography_recv('010200004002000000ECA2078CF547E3412C75C2F075B9FE41A079C9CC764FBE414EDD44ED6272F1C1B8392020C2CBF5419554B51E3E8FFEC1':::GEOMETRY) :1 SELECT geography_send('0103000020E61000000100000008000000C40EEE72E68161C0F0A137E78C6F39C040648622AD2C37C0B0FC1504FDA654C0402F36DEB54E5940F6F2CF9F202952C0D03A43B30CF45340E4F796329CC632C0329803340DAC604062DE16A77EE9534070AFF14AC28C32C0604B335DED30014098A9027D907550C00842AC265D0A27C0C40EEE72E68161C0F0A137E78C6F39C0':::GEOGRAPHY) :1]
    rsg_test.go:827: 2m10s of 5m0s: 274607 executions, 170820 successful
    rsg_test.go:827: 2m15s of 5m0s: 286101 executions, 177953 successful
    rsg_test.go:827: 2m20s of 5m0s: 299788 executions, 186411 successful
    rsg_test.go:827: 2m25s of 5m0s: 312220 executions, 194186 successful
    rsg_test.go:827: 2m31s of 5m0s: 324251 executions, 201728 successful
    rsg_test.go:827: 2m36s of 5m0s: 332133 executions, 206585 successful
    rsg_test.go:827: 2m41s of 5m0s: 342499 executions, 213038 successful
    rsg_test.go:827: 2m46s of 5m0s: 352272 executions, 219103 successful
    rsg_test.go:827: 2m51s of 5m0s: 362136 executions, 225274 successful
    rsg_test.go:827: 2m56s of 5m0s: 374974 executions, 233254 successful
    rsg_test.go:827: 3m1s of 5m0s: 386794 executions, 240601 successful
    rsg_test.go:827: 3m6s of 5m0s: 395075 executions, 245833 successful
    rsg_test.go:827: 3m11s of 5m0s: 403641 executions, 251055 successful
    rsg_test.go:827: 3m16s of 5m0s: 414538 executions, 257889 successful
    rsg_test.go:827: 3m21s of 5m0s: 426534 executions, 265324 successful
    rsg_test.go:827: 3m26s of 5m0s: 437586 executions, 272226 successful
    rsg_test.go:827: 3m31s of 5m0s: 450501 executions, 280271 successful
    rsg_test.go:827: 3m36s of 5m0s: 464390 executions, 288959 successful
    rsg_test.go:827: 3m41s of 5m0s: 475605 executions, 295966 successful
    rsg_test.go:827: 3m46s of 5m0s: 484616 executions, 301496 successful
    rsg_test.go:827: 3m51s of 5m0s: 496256 executions, 308683 successful
    rsg_test.go:827: 3m56s of 5m0s: 508220 executions, 316210 successful
    rsg_test.go:827: 4m1s of 5m0s: 516991 executions, 321574 successful
    rsg_test.go:827: 4m6s of 5m0s: 526804 executions, 327705 successful
    rsg_test.go:827: 4m11s of 5m0s: 538039 executions, 334706 successful
    rsg_test.go:827: 4m16s of 5m0s: 551168 executions, 342897 successful
    rsg_test.go:827: 4m21s of 5m0s: 563528 executions, 350715 successful
    rsg_test.go:827: 4m26s of 5m0s: 575091 executions, 357911 successful
    rsg_test.go:827: 4m31s of 5m0s: 586066 executions, 364653 successful
    rsg_test.go:827: 4m36s of 5m0s: 597216 executions, 371574 successful
    rsg_test.go:827: 4m41s of 5m0s: 607987 executions, 378278 successful
    rsg_test.go:827: 4m46s of 5m0s: 619405 executions, 385457 successful
    rsg_test.go:827: 4m51s of 5m0s: 632405 executions, 393548 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10858701?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10858701?buildTab=artifacts#/) on release-23.1 @ [cfc535244ecb0566c894ebf5a8120627787fbd00](https://github.com/cockroachdb/cockroach/commits/cfc535244ecb0566c894ebf5a8120627787fbd00):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 5487453 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx
        	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:461 +0x43b
        
        goroutine 4548931 [select]:
        github.com/cockroachdb/cockroach/pkg/kv/kvclient/rangefeed.(*RangeFeed).processEvents(0xc00d8b31e0, {0x7465b18, 0xc0046b1640}, 0xc0032b5fd0?, 0xc042c78420)
        	github.com/cockroachdb/cockroach/pkg/kv/kvclient/rangefeed/rangefeed.go:360 +0xdf
        github.com/cockroachdb/cockroach/pkg/kv/kvclient/rangefeed.(*RangeFeed).run.func2({0x7465b18?, 0xc0046b1640?})
        	github.com/cockroachdb/cockroach/pkg/kv/kvclient/rangefeed/rangefeed.go:317 +0x38
        github.com/cockroachdb/cockroach/pkg/util/ctxgroup.Group.GoCtx.func1()
        	github.com/cockroachdb/cockroach/pkg/util/ctxgroup/ctxgroup.go:168 +0x25
        golang.org/x/sync/errgroup.(*Group).Go.func1()
        	golang.org/x/sync/errgroup/external/org_golang_x_sync/errgroup/errgroup.go:75 +0x64
        created by golang.org/x/sync/errgroup.(*Group).Go
        	golang.org/x/sync/errgroup/external/org_golang_x_sync/errgroup/errgroup.go:72 +0xa5
        
    rsg_test.go:252: SELECT 1 executed successfully: probably a slow statement
    rsg_test.go:854: Crash detected: server panic: statement exec timeout
        SELECT crdb_internal.reset_activity_tables() ;
        
        Stack trace:
        timeout: "SELECT crdb_internal.reset_activity_tables() ". currently executing: map[SELECT crdb_internal.gen_rand_ident('':::STRING, 3338719414960475795:::INT8) :1 SELECT crdb_internal.gen_rand_ident('':::STRING, 4269377123649405059:::INT8) :1 SELECT crdb_internal.gen_rand_ident('':::STRING, 5750585530507208672:::INT8) :1 SELECT crdb_internal.gen_rand_ident(e'W"+q\x18h\x10':::STRING, 7395215682514559392:::INT8) :1 SELECT crdb_internal.reset_activity_tables() :1 SELECT obj_description(2420044862:::OID, e'XG^\x13':::STRING) :1 SELECT obj_description(811577642:::OID) :1 SELECT octet_length(e'\x02r;.f$H0':::STRING) :1]
    rsg_test.go:827: 2m25s of 5m0s: 267922 executions, 166595 successful
    rsg_test.go:827: 2m30s of 5m0s: 278171 executions, 173059 successful
    rsg_test.go:827: 2m35s of 5m0s: 287156 executions, 178561 successful
    rsg_test.go:827: 2m40s of 5m0s: 297314 executions, 184927 successful
    rsg_test.go:827: 2m45s of 5m0s: 306570 executions, 190654 successful
    rsg_test.go:827: 2m50s of 5m0s: 320380 executions, 199299 successful
    rsg_test.go:827: 2m55s of 5m0s: 330656 executions, 205613 successful
    rsg_test.go:827: 3m0s of 5m0s: 340689 executions, 211879 successful
    rsg_test.go:827: 3m5s of 5m0s: 350559 executions, 218030 successful
    rsg_test.go:827: 3m10s of 5m0s: 358945 executions, 223318 successful
    rsg_test.go:827: 3m15s of 5m0s: 361444 executions, 224770 successful
    rsg_test.go:827: 3m20s of 5m0s: 368457 executions, 229185 successful
    rsg_test.go:827: 3m25s of 5m0s: 379963 executions, 236334 successful
    rsg_test.go:827: 3m30s of 5m0s: 392652 executions, 244153 successful
    rsg_test.go:827: 3m35s of 5m0s: 405512 executions, 252124 successful
    rsg_test.go:827: 3m40s of 5m0s: 413604 executions, 257187 successful
    rsg_test.go:827: 3m45s of 5m0s: 423862 executions, 263664 successful
    rsg_test.go:827: 3m50s of 5m0s: 434577 executions, 270264 successful
    rsg_test.go:827: 3m55s of 5m0s: 447613 executions, 278316 successful
    rsg_test.go:827: 4m0s of 5m0s: 459120 executions, 285449 successful
    rsg_test.go:827: 4m5s of 5m0s: 470095 executions, 292318 successful
    rsg_test.go:827: 4m10s of 5m0s: 480006 executions, 298420 successful
    rsg_test.go:827: 4m15s of 5m0s: 489303 executions, 304260 successful
    rsg_test.go:827: 4m20s of 5m0s: 501856 executions, 312011 successful
    rsg_test.go:827: 4m25s of 5m0s: 514111 executions, 319670 successful
    rsg_test.go:827: 4m30s of 5m0s: 518876 executions, 322591 successful
    rsg_test.go:827: 4m35s of 5m0s: 526906 executions, 327577 successful
    rsg_test.go:827: 4m40s of 5m0s: 533960 executions, 331961 successful
    rsg_test.go:827: 4m45s of 5m0s: 542168 executions, 337038 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #96131 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.1]
- #95718 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10876418?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10876418?buildTab=artifacts#/) on release-23.1 @ [57c94bfe124fd08b948caa257acd3cb1e7cf1667](https://github.com/cockroachdb/cockroach/commits/57c94bfe124fd08b948caa257acd3cb1e7cf1667):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 5709813 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
        	github.com/cockroachdb/cockroach/pkg/kv/kvserver/intentresolver/intent_resolver.go:776 +0x3d9
        github.com/cockroachdb/cockroach/pkg/kv/kvserver/intentresolver.(*IntentResolver).cleanupFinishedTxnIntents.func2.1({0x74653e8?, 0xc02ecc08a0?})
        	github.com/cockroachdb/cockroach/pkg/kv/kvserver/intentresolver/intent_resolver.go:819 +0x38
        github.com/cockroachdb/cockroach/pkg/util/contextutil.RunWithTimeout({0x7465420?, 0xc029329f80?}, {0x5e94636, 0x12}, 0x4a817c800, 0xc040ed7f00)
        	github.com/cockroachdb/cockroach/pkg/util/contextutil/context.go:91 +0xed
        github.com/cockroachdb/cockroach/pkg/kv/kvserver/intentresolver.(*IntentResolver).cleanupFinishedTxnIntents.func2({0x7465420, 0xc029329f80})
        	github.com/cockroachdb/cockroach/pkg/kv/kvserver/intentresolver/intent_resolver.go:817 +0xac
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func2()
        	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:470 +0x146
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx
        	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:461 +0x43b
        
        goroutine 5304792 [sync.Cond.Wait]:
        sync.runtime_notifyListWait(0xc02759ce10, 0x1d)
        	GOROOT/src/runtime/sema.go:517 +0x14c
        sync.(*Cond).Wait(0x3bb810e?)
        	GOROOT/src/sync/cond.go:70 +0x8c
        github.com/cockroachdb/cockroach/pkg/sql.(*StmtBuf).CurCmd(0xc0224ecec8)
        	github.com/cockroachdb/cockroach/pkg/sql/conn_io.go:539 +0xc6
        github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).execCmd(0xc00e8bb800)
        	github.com/cockroachdb/cockroach/pkg/sql/conn_executor.go:2129 +0xe5
        github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).run(0xc00e8bb800, {0x7465420, 0xc04b5bec00}, 0x410d0d9000000000?, 0xc0041efdb8?, 0xc0174eb520?)
        	github.com/cockroachdb/cockroach/pkg/sql/conn_executor.go:2103 +0x216
        github.com/cockroachdb/cockroach/pkg/sql.(*Server).ServeConn(0xc03fcdbea0?, {0x7465420?, 0xc04b5bec00?}, {0x5e3d1c8?}, 0x3?, 0xc0171db630?)
        	github.com/cockroachdb/cockroach/pkg/sql/conn_executor.go:900 +0xe6
        github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*conn).processCommandsAsync.func1()
        	github.com/cockroachdb/cockroach/pkg/sql/pgwire/conn.go:310 +0x585
        created by github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*conn).processCommandsAsync
        	github.com/cockroachdb/cockroach/pkg/sql/pgwire/conn.go:219 +0x22a
        
    rsg_test.go:252: SELECT 1 executed successfully: probably a slow statement
    rsg_test.go:854: Crash detected: server panic: statement exec timeout
        SELECT crdb_internal.reset_activity_tables() ;
        
        Stack trace:
        timeout: "SELECT crdb_internal.reset_activity_tables() ". currently executing: map[SELECT crdb_internal.gen_rand_ident('bqj':::STRING, 1686198670800136531:::INT8) :1 SELECT crdb_internal.gen_rand_ident(e'F\' \x07':::STRING, 4337166334742617397:::INT8) :1 SELECT crdb_internal.gen_rand_ident(e'\x0b\t\x17\x1b\x14m\x11~&':::STRING, 50416829992806109:::INT8) :1 SELECT crdb_internal.reset_activity_tables() :1 SELECT st_voronoilines('0105000080030000000102000080050000009C39D9C52449E94186C7759FA8BCF8C100FC97C3DD92B5C168565CE2EEC3DA41D6EF81352724E7C19E6BD4B0381101C2BC84A859BAE60142409891133A41F1C12CF5F47D1169E5413CB924BA4414F94168E4C63E64CDF8415C6BF593AA3FFFC12CD24FC510F7E641043C1B2DE332FB41B25C6F32722AF74101020000800200000028407146D165D141F6848EDD0C470242227A7FC16C04F24148F329B9BA73F3C11E7198827E78F441A84193584932E541010200008006000000A356B73572F801C2A5DBE9A65F0C01C280E243F2AC480242D69163B4E51CF0418C49D09C15A8EB411C9F2ED8529AF1C16839061949EB00429C03F42374F5FD41EB9BCCEA829701C290A17C69FAFC00427209F85F3BC801421A1CAD60D662FEC1CF10B3068A2AF0C19888E90BB81402421E936D0442E901428858599B9A1ED0410C45E052D7B4F1414C0395DCF33D01C2':::GEOMETRY, (-0.33423553681932516):::FLOAT8, '010500008002000000010200008003000000EC4D8E3B2219EE41FF081803037F01C2CA2E3CD106980042249119F21B53F8415AAFF53BB87DE3C10E47C0F79A0B0042B4B670DB0F680042C0C0721DFB6FC241D630FA4CBD2EF841010200008002000000AAA0D8B41E24FA416682B0868705FCC1C0C6CF828E9BFC419BC11D0AD3CEFFC1D09684D141F0EE41312E88AAF18701C2':::GEOMETRY) :1 SELECT st_voronoipolygons('010200000002000000000000000000F03F000000000000F03F00000000000000400000000000000040':::GEOMETRY, (-2.9157554337942044):::FLOAT8, '010400000000000000':::GEOMETRY) :1 SELECT st_xmin('01010000405ACE1F4C8A78F141E032869894C4FB41105C6FE66999DCC1':::GEOMETRY) :1]
    rsg_test.go:827: 3m45s of 5m0s: 454280 executions, 282625 successful
    rsg_test.go:827: 3m50s of 5m0s: 466268 executions, 290093 successful
    rsg_test.go:827: 3m55s of 5m0s: 478512 executions, 297621 successful
    rsg_test.go:827: 4m0s of 5m0s: 487928 executions, 303473 successful
    rsg_test.go:827: 4m5s of 5m0s: 498091 executions, 309829 successful
    rsg_test.go:827: 4m10s of 5m0s: 511965 executions, 318508 successful
    rsg_test.go:827: 4m15s of 5m0s: 525856 executions, 327069 successful
    rsg_test.go:827: 4m20s of 5m0s: 538719 executions, 335201 successful
    rsg_test.go:827: 4m25s of 5m0s: 551533 executions, 343135 successful
    rsg_test.go:827: 4m30s of 5m0s: 561187 executions, 349154 successful
    rsg_test.go:827: 4m35s of 5m0s: 571978 executions, 355843 successful
    rsg_test.go:827: 4m40s of 5m0s: 584301 executions, 363571 successful
    rsg_test.go:827: 4m45s of 5m0s: 595528 executions, 370529 successful
    rsg_test.go:827: 4m50s of 5m0s: 605140 executions, 376482 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #106658 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10892382?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10892382?buildTab=artifacts#/) on release-23.1 @ [0efd1568d09a5c6a54ae41b0bbf42c682331d24b](https://github.com/cockroachdb/cockroach/commits/0efd1568d09a5c6a54ae41b0bbf42c682331d24b):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 5713167 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
        github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine.func2(0xc0170b3a00?, 0x0?, {0x743eb38?, 0xc0065fa1a8?}, {0x746c140?, 0xc02444b0b0?}, {0x5b688c0?, 0xc0065fa1c8?}, 0xc005d35810)
        	github.com/cockroachdb/cockroach/pkg/jobs/registry.go:1628 +0xcc
        github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine(0xc002ae8000, {0x746c140, 0xc04894ddd0}, {0x5b688c0, 0xc0065fa1c8}, {0x743eb38, 0xc0065fa1a8}, 0xc0170b3a00, {0xc019b06e28, 0x7}, ...)
        	github.com/cockroachdb/cockroach/pkg/jobs/registry.go:1629 +0xa3f
        github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).runJob(0xc002ae8000, {0x746c098, 0xc017833f40}, {0x743eb38, 0xc0065fa1a8}, 0xc0170b3a00, {0xc019b06e28, 0x7}, {0xc01478c648, 0x16})
        	github.com/cockroachdb/cockroach/pkg/jobs/adopt.go:474 +0x637
        github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).resumeJob.func2({0x746c140?, 0xc04894db00?})
        	github.com/cockroachdb/cockroach/pkg/jobs/adopt.go:388 +0x146
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func2()
        	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:470 +0x146
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx
        	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:461 +0x43b
        
        goroutine 5312320 [select]:
        github.com/cockroachdb/cockroach/pkg/sql/gcjob.waitForEmptyPrefix({0x746c140, 0xc01d711d10}, 0x6d1053318c0?, 0xa740148?, 0x0?, 0x0, {0xc00e229168, 0x4, 0x8})
        	github.com/cockroachdb/cockroach/pkg/sql/gcjob/gc_job.go:395 +0x285
        github.com/cockroachdb/cockroach/pkg/sql/gcjob.deleteIndexZoneConfigsAfterGC({0x746c140, 0xc01d711d10}, 0xc01d778680, 0x3d, 0xc021e50dc0)
        	github.com/cockroachdb/cockroach/pkg/sql/gcjob/index_garbage_collection.go:188 +0x127
        github.com/cockroachdb/cockroach/pkg/sql/gcjob.waitForGC({0x746c140?, 0xc01d711d10?}, 0xc01d778601?, 0xc3d6203dbd78001?, 0xc021e50dc0?)
        	github.com/cockroachdb/cockroach/pkg/sql/gcjob/gc_job.go:347 +0x70
        github.com/cockroachdb/cockroach/pkg/sql/gcjob.schemaChangeGCResumer.deleteDataAndWaitForGC({_}, {_, _}, {0xc0017ec000, {0xc02b285b60, 0xc002699960, 0xc02b285b50, 0xc0363aa420}, {{0xc002028cc0, 0xc002028cf0}, ...}, ...}, ...)
        	github.com/cockroachdb/cockroach/pkg/sql/gcjob/gc_job.go:335 +0x185
        github.com/cockroachdb/cockroach/pkg/sql/gcjob.schemaChangeGCResumer.Resume({0xc00e0c6ea0}, {0x746c140, 0xc01d711d10}, {0x5b688c0?, 0xc01e78ee60})
        	github.com/cockroachdb/cockroach/pkg/sql/gcjob/gc_job.go:314 +0x351
        github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine.func2(0xc00e0c6ea0?, 0x0?, {0x743eb38?, 0xc01e78ee40?}, {0x746c140?, 0xc01d711d10?}, {0x5b688c0?, 0xc01e78ee60?}, 0xc0069ad810)
        	github.com/cockroachdb/cockroach/pkg/jobs/registry.go:1628 +0xcc
        github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine(0xc002ae8000, {0x746c140, 0xc01b37ca20}, {0x5b688c0, 0xc01e78ee60}, {0x743eb38, 0xc01e78ee40}, 0xc00e0c6ea0, {0xc01c670988, 0x7}, ...)
        	github.com/cockroachdb/cockroach/pkg/jobs/registry.go:1629 +0xa3f
        github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).runJob(0xc002ae8000, {0x746c098, 0xc02499bc40}, {0x743eb38, 0xc01e78ee40}, 0xc00e0c6ea0, {0xc01c670988, 0x7}, {0xc0056ba9d8, 0x16})
        	github.com/cockroachdb/cockroach/pkg/jobs/adopt.go:474 +0x637
        github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).resumeJob.func2({0x746c140?, 0xc01b37c750?})
        	github.com/cockroachdb/cockroach/pkg/jobs/adopt.go:388 +0x146
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func2()
        	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:470 +0x146
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx
        	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:461 +0x43b
        
    rsg_test.go:252: SELECT 1 executed successfully: probably a slow statement
    rsg_test.go:854: Crash detected: server panic: statement exec timeout
        SELECT crdb_internal.reset_activity_tables() ;
        
        Stack trace:
        timeout: "SELECT crdb_internal.reset_activity_tables() ". currently executing: map[SELECT crdb_internal.gen_rand_ident(e'3L$\x14F\f':::STRING, 7129778276942553269:::INT8) :1 SELECT crdb_internal.gen_rand_ident(e'\x1e':::STRING, 6891771660826796351:::INT8) :1 SELECT crdb_internal.gen_rand_ident(e'\x7f9':::STRING, 4937199070089607026:::INT8) :1 SELECT crdb_internal.reset_activity_tables() :2]
    rsg_test.go:827: 4m20s of 5m0s: 517097 executions, 321640 successful
    rsg_test.go:827: 4m25s of 5m0s: 526838 executions, 327724 successful
    rsg_test.go:827: 4m30s of 5m0s: 535899 executions, 333309 successful
    rsg_test.go:827: 4m35s of 5m0s: 549638 executions, 341872 successful
    rsg_test.go:827: 4m40s of 5m0s: 560232 executions, 348530 successful
    rsg_test.go:827: 4m45s of 5m0s: 572013 executions, 355770 successful
    rsg_test.go:827: 4m50s of 5m0s: 583204 executions, 362747 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #106658 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10906815?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10906815?buildTab=artifacts#/) on release-23.1 @ [2463ccadcd1658130cb79a1e061282a75c596ea4](https://github.com/cockroachdb/cockroach/commits/2463ccadcd1658130cb79a1e061282a75c596ea4):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 5638979 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
        	github.com/cockroachdb/cockroach/pkg/sql/plan_opt.go:576 +0x5ba
        github.com/cockroachdb/cockroach/pkg/sql.(*planner).makeOptimizerPlan(0xc05e313e20, {0x746d520?, 0xc029d70540?})
        	github.com/cockroachdb/cockroach/pkg/sql/plan_opt.go:240 +0x1c5
        github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).makeExecPlan(0xc05e313800, {0x746d520, 0xc029d70540}, 0xc05e313e20)
        	github.com/cockroachdb/cockroach/pkg/sql/conn_executor_exec.go:1978 +0x4e
        github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).dispatchToExecutionEngine(0xc05e313800, {0x746d520, 0xc029d70540}, 0xc05e313e20, {0x7f1f2b556bb0, 0xc05e00d328})
        	github.com/cockroachdb/cockroach/pkg/sql/conn_executor_exec.go:1484 +0x70a
        github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).execStmtInOpenState(0xc05e313800, {0x746d520, 0xc029d70540}, {{0x7499680, 0xc01d650c30}, {0x0, 0x0, 0x0}, {0xc05f1975c3, 0x4e}, ...}, ...)
        	github.com/cockroachdb/cockroach/pkg/sql/conn_executor_exec.go:964 +0x3a51
        github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).execStmt.func1({0x746d520?, 0xc029d703c0?})
        	github.com/cockroachdb/cockroach/pkg/sql/conn_executor_exec.go:142 +0xbc
        github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).execWithProfiling(0x746d520?, {0x746d520?, 0xc029d703c0?}, {0x7499680?, 0xc01d650c30?}, 0x22?, 0x0?)
        	github.com/cockroachdb/cockroach/pkg/sql/conn_executor_exec.go:2986 +0x330
        github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).execStmt(0xc05e313800, {0x746d520, 0xc029d703c0}, {{0x7499680, 0xc01d650c30}, {0x0, 0x0, 0x0}, {0xc05f1975c3, 0x4e}, ...}, ...)
        	github.com/cockroachdb/cockroach/pkg/sql/conn_executor_exec.go:141 +0x4dd
        github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).execCmd.func1(0xc0650cba78, 0xc05e313800, 0xc0650cb5d0, 0xc0650cb6a0, 0xc0650cb6e0, 0xc0650cb6c0, 0xc0650cb6b0, 0xc0650cb6d0)
        	github.com/cockroachdb/cockroach/pkg/sql/conn_executor.go:2181 +0x319
        github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).execCmd(0xc05e313800)
        	github.com/cockroachdb/cockroach/pkg/sql/conn_executor.go:2186 +0x1167
        github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).run(0xc05e313800, {0x746d520, 0xc012609050}, 0x74cfc90?, 0x0?, 0xc03f0dc9e0?)
        	github.com/cockroachdb/cockroach/pkg/sql/conn_executor.go:2103 +0x216
        github.com/cockroachdb/cockroach/pkg/sql.(*Server).ServeConn(0xc01a8e2480?, {0x746d520?, 0xc012609050?}, {0x5e3ea08?}, 0x3?, 0xc05eec3bb0?)
        	github.com/cockroachdb/cockroach/pkg/sql/conn_executor.go:900 +0xe6
        github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*conn).processCommandsAsync.func1()
        	github.com/cockroachdb/cockroach/pkg/sql/pgwire/conn.go:310 +0x585
        created by github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*conn).processCommandsAsync
        	github.com/cockroachdb/cockroach/pkg/sql/pgwire/conn.go:219 +0x22a
        
    rsg_test.go:252: SELECT 1 executed successfully: probably a slow statement
    rsg_test.go:854: Crash detected: server panic: statement exec timeout
        SELECT crdb_internal.reset_activity_tables() ;
        
        Stack trace:
        timeout: "SELECT crdb_internal.reset_activity_tables() ". currently executing: map[SELECT crdb_internal.gen_rand_ident(e'VmWk\x1b\n\x7f\x1e':::STRING, 3063596207420283267:::INT8) :1 SELECT crdb_internal.reset_activity_tables() :2 SELECT has_foreign_data_wrapper_privilege('':::STRING, '':::STRING, '"':::STRING) :1 SELECT has_foreign_data_wrapper_privilege(e'BRx\x1a':::STRING, 4008464815:::OID, e'f\x18t\x17N!-,':::STRING) :1 SELECT has_function_privilege('x6$':::STRING, e'\x06e':::STRING, '':::STRING) :1 SELECT has_function_privilege(1465937481:::OID, ')':::STRING, '':::STRING) :1 SELECT has_function_privilege(e'\U00002603':::STRING, 576511848:::OID, e'G\x15*\x1e``\x0f\x12':::STRING) :1]
    rsg_test.go:827: 3m35s of 5m0s: 389852 executions, 242468 successful
    rsg_test.go:827: 3m40s of 5m0s: 404481 executions, 251589 successful
    rsg_test.go:827: 3m45s of 5m0s: 417536 executions, 259671 successful
    rsg_test.go:827: 3m50s of 5m0s: 427999 executions, 266229 successful
    rsg_test.go:827: 3m55s of 5m0s: 439627 executions, 273407 successful
    rsg_test.go:827: 4m0s of 5m0s: 449553 executions, 279528 successful
    rsg_test.go:827: 4m5s of 5m0s: 460577 executions, 286454 successful
    rsg_test.go:827: 4m10s of 5m0s: 469033 executions, 291607 successful
    rsg_test.go:827: 4m15s of 5m0s: 479833 executions, 298372 successful
    rsg_test.go:827: 4m20s of 5m0s: 491333 executions, 305484 successful
    rsg_test.go:827: 4m25s of 5m0s: 500330 executions, 311004 successful
    rsg_test.go:827: 4m30s of 5m0s: 512353 executions, 318576 successful
    rsg_test.go:827: 4m35s of 5m0s: 524482 executions, 326039 successful
    rsg_test.go:827: 4m40s of 5m0s: 532846 executions, 331290 successful
    rsg_test.go:827: 4m45s of 5m0s: 544866 executions, 338847 successful
    rsg_test.go:827: 4m50s of 5m0s: 553250 executions, 343962 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #106658 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10921294?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10921294?buildTab=artifacts#/) on release-23.1 @ [de4ce5b55d2ff48258b6b981fe3f6d376318861e](https://github.com/cockroachdb/cockroach/commits/de4ce5b55d2ff48258b6b981fe3f6d376318861e):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 5690072 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
        crypto/tls.(*Conn).readFromUntil(0xc0467b4700, {0x7430cc0?, 0xc04c7d1720}, 0x0?)
        	GOROOT/src/crypto/tls/conn.go:809 +0xe5
        crypto/tls.(*Conn).readRecordOrCCS(0xc0467b4700, 0x0)
        	GOROOT/src/crypto/tls/conn.go:616 +0x116
        crypto/tls.(*Conn).readRecord(...)
        	GOROOT/src/crypto/tls/conn.go:582
        crypto/tls.(*Conn).Read(0xc0467b4700, {0xc01d0e4000, 0x1000, 0x48a94c?})
        	GOROOT/src/crypto/tls/conn.go:1315 +0x16f
        bufio.(*Reader).Read(0xc02f7f1560, {0xc01aaec620, 0x5, 0xc025218870?})
        	GOROOT/src/bufio/bufio.go:237 +0x1bb
        io.ReadAtLeast({0x7420300, 0xc02f7f1560}, {0xc01aaec620, 0x5, 0x200}, 0x5)
        	GOROOT/src/io/io.go:332 +0x9a
        io.ReadFull(...)
        	GOROOT/src/io/io.go:351
        github.com/lib/pq.(*conn).recvMessage(0xc01aaec600, 0xc003e2b958)
        	github.com/lib/pq/external/com_github_lib_pq/conn.go:1004 +0xca
        github.com/lib/pq.(*conn).recv1Buf(0xc01aaec600, 0xc025218938?)
        	github.com/lib/pq/external/com_github_lib_pq/conn.go:1059 +0x2e
        github.com/lib/pq.(*conn).recv1(...)
        	github.com/lib/pq/external/com_github_lib_pq/conn.go:1086
        github.com/lib/pq.(*conn).simpleExec(0xc01aaec600, {0xc000bcf580?, 0x78})
        	github.com/lib/pq/external/com_github_lib_pq/conn.go:664 +0x1ff
        github.com/lib/pq.(*conn).Exec(0xc01aaec600, {0xc000bcf580, 0x78}, {0xacab9b8, 0x0, 0x0})
        	github.com/lib/pq/external/com_github_lib_pq/conn.go:921 +0x229
        github.com/lib/pq.(*conn).ExecContext(0x0?, {0x746f178, 0xc0292a2c80}, {0xc000bcf580, 0x78}, {0xacab9b8, 0x0, 0x10100c00ee16f00?})
        	github.com/lib/pq/external/com_github_lib_pq/conn_go18.go:46 +0x109
        database/sql.ctxDriverExec({0x746f178?, 0xc0292a2c80?}, {0x7f92fdf1d7e0?, 0xc01aaec600?}, {0x0?, 0x0?}, {0xc000bcf580?, 0x463547?}, {0xacab9b8, 0x0, ...})
        	GOROOT/src/database/sql/ctxutil.go:31 +0xe5
        database/sql.(*DB).execDC.func2()
        	GOROOT/src/database/sql/sql.go:1679 +0x175
        database/sql.withLock({0x743f620, 0xc034fa8b40}, 0xc025218dd8)
        	GOROOT/src/database/sql/sql.go:3439 +0x8c
        database/sql.(*DB).execDC(0xc01dde4f01?, {0x746f178, 0xc0292a2c80}, 0xc034fa8b40, 0xc00ee16de0?, {0xc000bcf580, 0x78}, {0x0, 0x0, 0x0})
        	GOROOT/src/database/sql/sql.go:1674 +0x266
        database/sql.(*DB).exec(0xc025218f08?, {0x746f178, 0xc0292a2c80}, {0xc000bcf580, 0x78}, {0x0, 0x0, 0x0}, 0x20?)
        	GOROOT/src/database/sql/sql.go:1659 +0xe8
        database/sql.(*DB).ExecContext(0xc002e3e200?, {0x746f178, 0xc0292a2c80}, {0xc000bcf580, 0x78}, {0x0, 0x0, 0x0})
        	GOROOT/src/database/sql/sql.go:1633 +0xe5
        github.com/cockroachdb/cockroach/pkg/sql/tests_test.(*verifyFormatDB).execWithResettableTimeout.func2()
        	github.com/cockroachdb/cockroach/pkg/sql/tests_test/pkg/sql/tests/rsg_test.go:173 +0x48
        created by github.com/cockroachdb/cockroach/pkg/sql/tests_test.(*verifyFormatDB).execWithResettableTimeout
        	github.com/cockroachdb/cockroach/pkg/sql/tests_test/pkg/sql/tests/rsg_test.go:172 +0x19d
        
    rsg_test.go:252: SELECT 1 executed successfully: probably a slow statement
    rsg_test.go:854: Crash detected: server panic: statement exec timeout
        SELECT crdb_internal.reset_activity_tables() ;
        
        Stack trace:
        timeout: "SELECT crdb_internal.reset_activity_tables() ". currently executing: map[SELECT crdb_internal.gen_rand_ident('':::STRING, 591383038923564433:::INT8) :1 SELECT crdb_internal.gen_rand_ident(e'\'':::STRING, 3419834196391605809:::INT8) :1 SELECT crdb_internal.reset_activity_tables() :3 SELECT obj_description(3175644090:::OID, e'\rj\beF$S':::STRING) :1 SELECT overlaps('17:02:01.288681':::TIME, '03:35:52.245633':::TIME, '10:49:27.881722':::TIME, '01:22:49.720313':::TIME) :1 SELECT overlaps('22:02:23.632468':::TIME, '63 years 5 mons 447 days 00:25:46.419246':::INTERVAL, '08:24:26.934294':::TIME, (SELECT NULL)::interval) :1]
    rsg_test.go:827: 4m50s of 5m0s: 562819 executions, 350242 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #106658 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10927813?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10927813?buildTab=artifacts#/) on release-23.1 @ [de4ce5b55d2ff48258b6b981fe3f6d376318861e](https://github.com/cockroachdb/cockroach/commits/de4ce5b55d2ff48258b6b981fe3f6d376318861e):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 5737437 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
        github.com/cockroachdb/cockroach/pkg/sql/gcjob.deleteIndexZoneConfigsAfterGC({0x746f220, 0xc02a7119b0}, 0xc0228dcc00, 0x3c, 0xc01bd5a480)
        	github.com/cockroachdb/cockroach/pkg/sql/gcjob/index_garbage_collection.go:188 +0x127
        github.com/cockroachdb/cockroach/pkg/sql/gcjob.waitForGC({0x746f220?, 0xc02a7119b0?}, 0xc0228dcc01?, 0xc40665702898001?, 0xc01bd5a480?)
        	github.com/cockroachdb/cockroach/pkg/sql/gcjob/gc_job.go:347 +0x70
        github.com/cockroachdb/cockroach/pkg/sql/gcjob.schemaChangeGCResumer.deleteDataAndWaitForGC({_}, {_, _}, {0xc0018b4000, {0xc002f13910, 0xc002c950a0, 0xc002f13900, 0xc002eb7540}, {{0xc04e8de8d0, 0xc04e8de960}, ...}, ...}, ...)
        	github.com/cockroachdb/cockroach/pkg/sql/gcjob/gc_job.go:335 +0x185
        github.com/cockroachdb/cockroach/pkg/sql/gcjob.schemaChangeGCResumer.Resume({0xc017adc340}, {0x746f220, 0xc02a7119b0}, {0x5b6ae20?, 0xc002235238})
        	github.com/cockroachdb/cockroach/pkg/sql/gcjob/gc_job.go:314 +0x351
        github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine.func2(0xc017adc340?, 0x0?, {0x7441c18?, 0xc0022351f8?}, {0x746f220?, 0xc02a7119b0?}, {0x5b6ae20?, 0xc002235238?}, 0xc00453d810)
        	github.com/cockroachdb/cockroach/pkg/jobs/registry.go:1628 +0xcc
        github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine(0xc0087ca000, {0x746f220, 0xc020a6ade0}, {0x5b6ae20, 0xc002235238}, {0x7441c18, 0xc0022351f8}, 0xc017adc340, {0xc01aac20f8, 0x7}, ...)
        	github.com/cockroachdb/cockroach/pkg/jobs/registry.go:1629 +0xa3f
        github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).runJob(0xc0087ca000, {0x746f178, 0xc00377f940}, {0x7441c18, 0xc0022351f8}, 0xc017adc340, {0xc01aac20f8, 0x7}, {0xc0311e60c0, 0x16})
        	github.com/cockroachdb/cockroach/pkg/jobs/adopt.go:474 +0x637
        github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).resumeJob.func2({0x746f220?, 0xc020a6aae0?})
        	github.com/cockroachdb/cockroach/pkg/jobs/adopt.go:388 +0x146
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func2()
        	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:470 +0x146
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx
        	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:461 +0x43b
        
        goroutine 5322388 [select]:
        github.com/cockroachdb/cockroach/pkg/sql/stats.(*Refresher).Start.func1.1({0x746f220, 0xc028a517d0})
        	github.com/cockroachdb/cockroach/pkg/sql/stats/pkg/sql/stats/automatic_stats.go:484 +0x1e9
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func2()
        	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:470 +0x146
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx
        	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:461 +0x43b
        
    rsg_test.go:252: SELECT 1 executed successfully: probably a slow statement
    rsg_test.go:854: Crash detected: server panic: statement exec timeout
        SELECT crdb_internal.reset_activity_tables() ;
        
        Stack trace:
        timeout: "SELECT crdb_internal.reset_activity_tables() ". currently executing: map[SELECT crdb_internal.gen_rand_ident(e'\'':::STRING, 3271127679562406086:::INT8) :1 SELECT crdb_internal.gen_rand_ident(e'\x05^':::STRING, 5634783663219612741:::INT8) :1 SELECT crdb_internal.reset_activity_tables() :1 SELECT to_regclass(e':P\x03\'Z\x01\'q':::STRING) :1 SELECT to_regtype('r``&':::STRING) :1 SELECT trunc((-8.597983574675321052E+34):::DECIMAL, 7758722785234760816:::INT8) :1 SELECT ts_rank(ARRAY[(-2.2371623579003312):::FLOAT8,0.7377575097735339:::FLOAT8,0.08392722550270876:::FLOAT8,(-0.7840047542584038):::FLOAT8,(-1.240452108887752):::FLOAT8,0.6266810485228741:::FLOAT8], '''GdMOfAIp'' ''HL'' ''ZUqP'':257A,360C,449A,780A ''cVwbJI'' ''dA'' ''fzFVyxLDj'':453B,827B ''h'':24B,216C,861C,869B':::TSVECTOR, '''XFwzsfUD'' | ''KwR'' | ''YMheXXuq''':::TSQUERY, (-8521512975715864623):::INT8) :1 SELECT ts_rank(NULL::float[], '''KYIAe'':842C,990C ''TCcHaPhR'':309A,325C,700B,735B,909B ''VHvCWilN'':121C,596A,896A ''bsN'':91A,561C ''ktaMv'':752A ''pmMkRFn'' ''ssWlsROHf'':537B,959A ''y'':326A,925B,992C ''zlX'' ''ztGuPq'':623B,679A':::TSVECTOR, '''I'' <-> ''KNwKL'' <-> !''PPOSTDF'' & ''G''':::TSQUERY) :1]
    rsg_test.go:827: 3m40s of 5m0s: 461531 executions, 287521 successful
    rsg_test.go:827: 3m45s of 5m0s: 470734 executions, 293235 successful
    rsg_test.go:827: 3m50s of 5m0s: 482215 executions, 300368 successful
    rsg_test.go:827: 3m55s of 5m0s: 495685 executions, 308766 successful
    rsg_test.go:827: 4m0s of 5m0s: 509346 executions, 317336 successful
    rsg_test.go:827: 4m5s of 5m0s: 521088 executions, 324620 successful
    rsg_test.go:827: 4m10s of 5m0s: 531522 executions, 331156 successful
    rsg_test.go:827: 4m15s of 5m0s: 543020 executions, 338340 successful
    rsg_test.go:827: 4m20s of 5m0s: 554396 executions, 345385 successful
    rsg_test.go:827: 4m25s of 5m0s: 563882 executions, 351314 successful
    rsg_test.go:827: 4m30s of 5m0s: 571835 executions, 356232 successful
    rsg_test.go:827: 4m35s of 5m0s: 582199 executions, 362644 successful
    rsg_test.go:827: 4m40s of 5m0s: 591917 executions, 368612 successful
    rsg_test.go:827: 4m45s of 5m0s: 597552 executions, 372131 successful
    rsg_test.go:827: 4m50s of 5m0s: 608797 executions, 379132 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #106658 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10934622?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10934622?buildTab=artifacts#/) on release-23.1 @ [de4ce5b55d2ff48258b6b981fe3f6d376318861e](https://github.com/cockroachdb/cockroach/commits/de4ce5b55d2ff48258b6b981fe3f6d376318861e):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 5903690 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
        	github.com/cockroachdb/cockroach/pkg/sql/pgwire/server.go:844 +0xc56
        github.com/cockroachdb/cockroach/pkg/server.(*systemServerWrapper).serveConn(0xc0093cad68, {0x746f220, 0xc020da5e90}, {0x74b3988, 0xc0523d5500}, {0x0, 0x4, 0x0, {0x5400, 0x15000, ...}, ...})
        	github.com/cockroachdb/cockroach/pkg/server/server_controller_sql.go:94 +0x151
        github.com/cockroachdb/cockroach/pkg/server.(*serverController).sqlMux(0xc016fd9a20, {0x746f220, 0xc020da5e90}, {0x74b3988, 0xc0523d5500}, {0x0, 0x4, 0x0, {0x5400, 0x15000, ...}, ...})
        	github.com/cockroachdb/cockroach/pkg/server/server_controller_sql.go:74 +0x2f9
        github.com/cockroachdb/cockroach/pkg/server.startServeSQL.func1.1({0x746f220?, 0xc020da5e60?}, {0x74ba5d0, 0xc00e935998})
        	github.com/cockroachdb/cockroach/pkg/server/server_sql.go:1766 +0x24e
        github.com/cockroachdb/cockroach/pkg/util/netutil.(*TCPServer).ServeWith.func1({0x746f220, 0xc020da5e60})
        	github.com/cockroachdb/cockroach/pkg/util/netutil/net.go:188 +0xc7
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func2()
        	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:470 +0x146
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx
        	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:461 +0x43b
        
        goroutine 5496264 [select]:
        github.com/cockroachdb/cockroach/pkg/sql/gcjob.waitForEmptyPrefix({0x746f220, 0xc04d479a40}, 0x75905333d40?, 0xa744168?, 0x80?, 0x0, {0xc06c903fc0, 0x4, 0x8})
        	github.com/cockroachdb/cockroach/pkg/sql/gcjob/gc_job.go:395 +0x285
        github.com/cockroachdb/cockroach/pkg/sql/gcjob.deleteIndexZoneConfigsAfterGC({0x746f220, 0xc04d479a40}, 0xc05e55cc00, 0x3c, 0xc0307cb500)
        	github.com/cockroachdb/cockroach/pkg/sql/gcjob/index_garbage_collection.go:188 +0x127
        github.com/cockroachdb/cockroach/pkg/sql/gcjob.waitForGC({0x746f220?, 0xc04d479a40?}, 0xc05e55cc01?, 0xc416829a4fc0001?, 0xc0307cb500?)
        	github.com/cockroachdb/cockroach/pkg/sql/gcjob/gc_job.go:347 +0x70
        github.com/cockroachdb/cockroach/pkg/sql/gcjob.schemaChangeGCResumer.deleteDataAndWaitForGC({_}, {_, _}, {0xc002308000, {0xc05aa01f90, 0xc01020e260, 0xc05aa01f80, 0xc003742620}, {{0xc01bf5ddd0, 0xc01bf5de00}, ...}, ...}, ...)
        	github.com/cockroachdb/cockroach/pkg/sql/gcjob/gc_job.go:335 +0x185
        github.com/cockroachdb/cockroach/pkg/sql/gcjob.schemaChangeGCResumer.Resume({0xc04bd67a00}, {0x746f220, 0xc04d479a40}, {0x5b6ae20?, 0xc007771de0})
        	github.com/cockroachdb/cockroach/pkg/sql/gcjob/gc_job.go:314 +0x351
        github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine.func2(0xc04bd67a00?, 0x0?, {0x7441c18?, 0xc007771dc0?}, {0x746f220?, 0xc04d479a40?}, {0x5b6ae20?, 0xc007771de0?}, 0xc00c9d1810)
        	github.com/cockroachdb/cockroach/pkg/jobs/registry.go:1628 +0xcc
        github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine(0xc0135a6a00, {0x746f220, 0xc013712a80}, {0x5b6ae20, 0xc007771de0}, {0x7441c18, 0xc007771dc0}, 0xc04bd67a00, {0xc07a3ea3d8, 0x7}, ...)
        	github.com/cockroachdb/cockroach/pkg/jobs/registry.go:1629 +0xa3f
        github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).runJob(0xc0135a6a00, {0x746f178, 0xc02781f840}, {0x7441c18, 0xc007771dc0}, 0xc04bd67a00, {0xc07a3ea3d8, 0x7}, {0xc06f763e90, 0x16})
        	github.com/cockroachdb/cockroach/pkg/jobs/adopt.go:474 +0x637
        github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).resumeJob.func2({0x746f220?, 0xc0137127b0?})
        	github.com/cockroachdb/cockroach/pkg/jobs/adopt.go:388 +0x146
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func2()
        	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:470 +0x146
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx
        	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:461 +0x43b
        
    rsg_test.go:252: SELECT 1 executed successfully: probably a slow statement
    rsg_test.go:854: Crash detected: server panic: statement exec timeout
        SELECT crdb_internal.reset_activity_tables() ;
        
        Stack trace:
        timeout: "SELECT crdb_internal.reset_activity_tables() ". currently executing: map[SELECT crdb_internal.gen_rand_ident('gy':::STRING, 780988941697935458:::INT8) :1 SELECT crdb_internal.reset_activity_tables() :4 SELECT pg_sequence_parameters(1917592488:::OID) :1 SELECT pg_type_is_visible(3352388585:::OID) :1 SELECT phraseto_tsquery('':::STRING, e'KKe-c\x161I':::STRING) :1]
    rsg_test.go:827: 4m25s of 5m0s: 571972 executions, 356291 successful
    rsg_test.go:827: 4m30s of 5m0s: 580250 executions, 361425 successful
    rsg_test.go:827: 4m35s of 5m0s: 588206 executions, 366399 successful
    rsg_test.go:827: 4m40s of 5m0s: 599220 executions, 373241 successful
    rsg_test.go:827: 4m45s of 5m0s: 611402 executions, 380854 successful
    rsg_test.go:827: 4m50s of 5m0s: 616465 executions, 383963 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #106658 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10950467?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10950467?buildTab=artifacts#/) on release-23.1 @ [ae14f647a4bda2532bc9f2be41f25f6acc113412](https://github.com/cockroachdb/cockroach/commits/ae14f647a4bda2532bc9f2be41f25f6acc113412):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 5742686 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
        	github.com/cockroachdb/cockroach/pkg/sql/execinfra/processorsbase.go:726 +0x73
        github.com/cockroachdb/cockroach/pkg/sql/flowinfra.(*FlowBase).Run(0xc02b581680, {0x7472398?, 0xc029846c80}, 0x0?)
        	github.com/cockroachdb/cockroach/pkg/sql/flowinfra/flow.go:575 +0x21a
        github.com/cockroachdb/cockroach/pkg/sql/colflow.(*vectorizedFlow).Run(0x74e0220?, {0x7472398?, 0xc029846c80?}, 0xe0?)
        	github.com/cockroachdb/cockroach/pkg/sql/colflow/vectorized_flow.go:309 +0x245
        github.com/cockroachdb/cockroach/pkg/sql.(*DistSQLPlanner).Run(0xc030117b80, {0x7472440, 0xc026f13a10}, 0xc0163169a0, 0xc0163166e0, 0xc0150d6300, 0xc00db7c700, 0xc00ae05260, 0xc020a94840)
        	github.com/cockroachdb/cockroach/pkg/sql/distsql_running.go:902 +0xb93
        github.com/cockroachdb/cockroach/pkg/sql.(*DistSQLPlanner).PlanAndRun(0x0?, {0x7472440, 0xc026f13a10}, 0xc016fe9b58?, 0xc0163169a0, 0xc0154de300?, {{0x7474ee8?, 0xc0154de300?}, 0x0?}, 0xc00db7c700, ...)
        	github.com/cockroachdb/cockroach/pkg/sql/distsql_running.go:1913 +0x205
        github.com/cockroachdb/cockroach/pkg/sql.(*DistSQLPlanner).PlanAndRunAll.func3(0xc00ae04e40, 0xc0163169a0?, {0x7472440, 0xc026f13a10}, 0x0?, 0x0?, 0xc016fe9b88?)
        	github.com/cockroachdb/cockroach/pkg/sql/distsql_running.go:1645 +0xcb
        github.com/cockroachdb/cockroach/pkg/sql.(*DistSQLPlanner).PlanAndRunAll(0x0?, {0x7472440?, 0xc026f13a10}, 0xc00ae05260, 0xc0163169a0, 0xc00ae04e40, 0xc00db7c700, 0x0)
        	github.com/cockroachdb/cockroach/pkg/sql/distsql_running.go:1648 +0x245
        github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).execWithDistSQLEngine(0xc00ae04800, {0x7472440?, 0xc026f13a10}, 0xc00ae04e40, 0xc026f13a10?, {0x7f16564dda88?, 0xc012760f28}, 0xc012760f28?, 0xc00c4a4518)
        	github.com/cockroachdb/cockroach/pkg/sql/conn_executor_exec.go:2102 +0x5eb
        github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).dispatchToExecutionEngine(0xc00ae04800, {0x7472398, 0xc029846b00}, 0xc00ae04e40, {0x7f16564dda88, 0xc012760f28})
        	github.com/cockroachdb/cockroach/pkg/sql/conn_executor_exec.go:1656 +0x11b2
        github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).execStmtInOpenState(0xc00ae04800, {0x7472398, 0xc029846b00}, {{0x749e5a0, 0xc01ded4870}, {0x0, 0x0, 0x0}, {0xc013894981, 0x44}, ...}, ...)
        	github.com/cockroachdb/cockroach/pkg/sql/conn_executor_exec.go:964 +0x3a71
        github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).execStmt.func1({0x7472440?, 0xc026f13470?})
        	github.com/cockroachdb/cockroach/pkg/sql/conn_executor_exec.go:142 +0xbc
        github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).execWithProfiling(0x7472440?, {0x7472440?, 0xc026f13470?}, {0x749e5a0?, 0xc01ded4870?}, 0x52?, 0x0?)
        	github.com/cockroachdb/cockroach/pkg/sql/conn_executor_exec.go:2985 +0x330
        github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).execStmt(0xc00ae04800, {0x7472440, 0xc026f13470}, {{0x749e5a0, 0xc01ded4870}, {0x0, 0x0, 0x0}, {0xc013894981, 0x44}, ...}, ...)
        	github.com/cockroachdb/cockroach/pkg/sql/conn_executor_exec.go:141 +0x4dd
        github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).execCmd.func1(0xc016feba78, 0xc00ae04800, 0xc016feb5d0, 0xc016feb6a0, 0xc016feb6e0, 0xc0026a76c0, 0xc0026a76b0, 0xc0026a76d0)
        	github.com/cockroachdb/cockroach/pkg/sql/conn_executor.go:2181 +0x319
        github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).execCmd(0xc00ae04800)
        	github.com/cockroachdb/cockroach/pkg/sql/conn_executor.go:2186 +0x1167
        github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).run(0xc00ae04800, {0x7472440, 0xc011e1f470}, 0x410b3f8800000000?, 0xc025051db8?, 0xc019dbf1a0?)
        	github.com/cockroachdb/cockroach/pkg/sql/conn_executor.go:2103 +0x216
        github.com/cockroachdb/cockroach/pkg/sql.(*Server).ServeConn(0xc032177ea0?, {0x7472440?, 0xc011e1f470?}, {0x5e42ae8?}, 0x3?, 0xc04f2de170?)
        	github.com/cockroachdb/cockroach/pkg/sql/conn_executor.go:900 +0xe6
        github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*conn).processCommandsAsync.func1()
        	github.com/cockroachdb/cockroach/pkg/sql/pgwire/conn.go:310 +0x585
        created by github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*conn).processCommandsAsync
        	github.com/cockroachdb/cockroach/pkg/sql/pgwire/conn.go:219 +0x22a
        
        goroutine 5596303 [select]:
        github.com/lib/pq.(*conn).watchCancel.func1()
        	github.com/lib/pq/external/com_github_lib_pq/conn_go18.go:106 +0x96
        created by github.com/lib/pq.(*conn).watchCancel
        	github.com/lib/pq/external/com_github_lib_pq/conn_go18.go:105 +0xf5
        
    rsg_test.go:252: SELECT 1 executed successfully: probably a slow statement
    rsg_test.go:854: Crash detected: server panic: statement exec timeout
        SELECT crdb_internal.reset_activity_tables() ;
        
        Stack trace:
        timeout: "SELECT crdb_internal.reset_activity_tables() ". currently executing: map[SELECT crdb_internal.gen_rand_ident('':::STRING, 2147483647:::INT8) :1 SELECT crdb_internal.gen_rand_ident('':::STRING, 5073102008966770910:::INT8) :1 SELECT crdb_internal.gen_rand_ident(e'\x1dn0U0";4O':::STRING, 1750533260162767490:::INT8) :1 SELECT crdb_internal.reset_activity_tables() :3 SELECT crdb_internal.scan('\x88891bc5c8':::BYTES, '\xc4':::BYTES) :1 SELECT cume_dist() :1]
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10967201?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10967201?buildTab=artifacts#/) on release-23.1 @ [6ccfa2ca599092e742151ea7017695e99567d6f1](https://github.com/cockroachdb/cockroach/commits/6ccfa2ca599092e742151ea7017695e99567d6f1):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 5688392 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
        	GOROOT/src/crypto/tls/key_schedule.go:63 +0x486
        crypto/tls.(*cipherSuiteTLS13).deriveSecret(0xa56c1e0, {0xc02b6ca820, 0x20, 0x20}, {0x5e7c2b8, 0xc}, {0x7499460?, 0xc012346200?})
        	GOROOT/src/crypto/tls/key_schedule.go:75 +0xd3
        crypto/tls.(*clientHandshakeStateTLS13).establishHandshakeKeys(0xc00ac4d550)
        	GOROOT/src/crypto/tls/handshake_client_tls13.go:382 +0x1fa
        crypto/tls.(*clientHandshakeStateTLS13).handshake(0xc00ac4d550)
        	GOROOT/src/crypto/tls/handshake_client_tls13.go:90 +0x21f
        crypto/tls.(*Conn).clientHandshake(0xc01e982a80, {0x7477218?, 0xc013b3f2c0})
        	GOROOT/src/crypto/tls/handshake_client.go:228 +0x5bb
        crypto/tls.(*Conn).handshakeContext(0xc01e982a80, {0x7477250, 0xc0001b4000})
        	GOROOT/src/crypto/tls/conn.go:1490 +0x32f
        crypto/tls.(*Conn).HandshakeContext(...)
        	GOROOT/src/crypto/tls/conn.go:1433
        crypto/tls.(*Conn).Handshake(...)
        	GOROOT/src/crypto/tls/conn.go:1417
        crypto/tls.(*Conn).Write(0x55fc940?, {0xc021691521, 0x63, 0x1ff})
        	GOROOT/src/crypto/tls/conn.go:1146 +0xf2
        github.com/lib/pq.(*conn).sendStartupPacket(0xc00ac4da10?, 0xc00ac4d988?)
        	github.com/lib/pq/external/com_github_lib_pq/conn.go:965 +0x85
        github.com/lib/pq.(*conn).startup(0xc021691500, 0xc0001b4000?)
        	github.com/lib/pq/external/com_github_lib_pq/conn.go:1170 +0x69e
        github.com/lib/pq.(*Connector).open(0xc0258ab518, {0x7477250, 0xc0001b4000})
        	github.com/lib/pq/external/com_github_lib_pq/conn.go:378 +0x4c5
        github.com/lib/pq.DialOpen({0x74515e0?, 0xc015a4dd40}, {0xc009034000?, 0x7447680?})
        	github.com/lib/pq/external/com_github_lib_pq/conn.go:328 +0x7b
        github.com/lib/pq.Open(...)
        	github.com/lib/pq/external/com_github_lib_pq/conn.go:318
        github.com/lib/pq.Driver.Open({}, {0xc009034000, 0x169})
        	github.com/lib/pq/external/com_github_lib_pq/conn.go:56 +0x85
        database/sql.dsnConnector.Connect(...)
        	GOROOT/src/database/sql/sql.go:759
        database/sql.(*DB).conn(0xc002984c30, {0x7477218, 0xc00623f740}, 0x1)
        	GOROOT/src/database/sql/sql.go:1393 +0x763
        database/sql.(*DB).exec(0xc009056228?, {0x7477218, 0xc00623f740}, {0xc026dcb9e0, 0x53}, {0x0, 0x0, 0x0}, 0xc0?)
        	GOROOT/src/database/sql/sql.go:1655 +0x5d
        database/sql.(*DB).ExecContext(0xc005194b00?, {0x7477218, 0xc00623f740}, {0xc026dcb9e0, 0x53}, {0x0, 0x0, 0x0})
        	GOROOT/src/database/sql/sql.go:1633 +0xe5
        github.com/cockroachdb/cockroach/pkg/sql/tests_test.(*verifyFormatDB).execWithResettableTimeout.func2()
        	github.com/cockroachdb/cockroach/pkg/sql/tests_test/pkg/sql/tests/rsg_test.go:173 +0x48
        created by github.com/cockroachdb/cockroach/pkg/sql/tests_test.(*verifyFormatDB).execWithResettableTimeout
        	github.com/cockroachdb/cockroach/pkg/sql/tests_test/pkg/sql/tests/rsg_test.go:172 +0x19d
        
    rsg_test.go:252: SELECT 1 executed successfully: probably a slow statement
    rsg_test.go:854: Crash detected: server panic: statement exec timeout
        SELECT crdb_internal.reset_activity_tables() ;
        
        Stack trace:
        timeout: "SELECT crdb_internal.reset_activity_tables() ". currently executing: map[SELECT crdb_internal.gen_rand_ident('kw':::STRING, 3163918212908756977:::INT8) :1 SELECT crdb_internal.gen_rand_ident(e'V\bx\x1e[\x07\x1f4j':::STRING, 6799294187757613353:::INT8) :1 SELECT crdb_internal.reset_activity_tables() :1 SELECT st_mpointfromwkb('\x42d5e08dec4dc0':::BYTES, (-8247629302832732689):::INT8) :1 SELECT st_mpolyfromtext('':::STRING, (-3623053846072590258):::INT8) :1 SELECT st_mpolyfromwkb('\x':::BYTES) :1 SELECT st_mpolyfromwkb('\xee035896aae1ea7e':::BYTES, 393795857011715116:::INT8) :1 SELECT st_multi('010700008007000000010500008003000000010200008003000000280BE8221864EDC13897967CFA9DF6C184978167DBDDF4414ACD58730A6AF841964BD2FA7F2CF64100F2533F1760C1C1409BFD27291FAE41F02B0702ED4AC04160E93AA9F95BB74101020000800A00000000641C7E154E93C1C0BF8AB2AA56CAC1649DA2217AB4FF41D0EFCA3354ABDB419A1524081BF6E6C11072E64DE281FF41587E248EE195DE41247BFC9BAA52EFC1E4193E1B088AF0413ECCDD9EC443024226B43D9710C5E3C170F40FEA0EC0F0C10060D1CB539BF0417870A621C5DAE041D8B276F8C95301425C673D700332EF41B40BA5392220E341849B6508596A0242649AD9CD2A0BF641D8B8BEE11A6502421A50FA348553F441BC3BD111001AEC41203A8AC9985DFB414A61C20499DDFCC114225B77C894E04126B9572FDF620142F84048D94062EA414C6E3E7B6E18F5C13024693439B6E541E4CE05CD3207E141010200008005000000EC68EAD6E4FFFD41B8605F143BEBE3C17EADE1DD1AEFECC18655CB5183E2F741FCEAF3C3A97DEC41508987A828C4FB4178BE89D292BAD8C1E655891B8BAE0042761D16694294F541472DF09BB6FDF3C1A043F330FD4AE541084A6ADC981ED34176554565FD1BEDC140E34EE0E3B2DA41C0E642913D3CCEC10105000080060000000102000080030000001029D8057B1DFA412893D77AE65BE3C1C0149B4EF686C741D246158DC0A3F441A0575882243CE4C15E474B6325F2EFC19C2685503108E6C1F6C1457C467DE9C1169D212074A2F54101020000800200000004E8B993F321FD4186412B14778DE1C1FB8A5277EFD0FDC160C3008BD625D0C1A4DACBB84C70F541DC31AA79DA73FF4101020000800200000044704B20DC47E741AE1718421D1100C2727FBCEA6BD2F9C1048F57A39872F2417ADC11F35DA700C2C055E48D5826B241010200008002000000EC137F1EFEDDE541D4D66EE9FD70FE41F0FADC57A47CF841D4833B7170CFE041363B078EA8F2014208FCC6C2FFBAFC4101020000800800000088C3991FA899E5C17AA58798F38EF8C1682040368441D8418F0F792FD54002C217C021A23ECFFFC104734C3EE53400426481727AA069E9C11C841E86E62701C25CDDE4243490F0414CA87528B9E4E341D2688152181F01C20E85A32EB6A8F6C1CCFE342443AB0042D0D21537311102C23C3D7B22D8ECF041801177876207E2417A308CBDFB1DF8C13ED398A286F7F9C1D8E62545C929FE41B86A1A673418D2412095D3345578F4C1A9C8D92016EDFDC182B99F17877CE9C1B81326506A0EFC41010200008002000000A77CB5D528C9F3C18AF6245F1911F641B07E5B77A71DE8419F873938BE0DF5C14CA399EFB181F0414C361097FB96FEC10102000080070000009C1BAB779A31F6C19C3012B45975FFC1954A12ECDE0401C29CA44F615284E341C1BAEF7B30C8FEC126EDE9A9ED2EECC1B0D18B85FB5EFD419B6E5306347DF0C130E3DEC7BA9DE44108533B221F44E64144473EC91151E2C1D02DB5778ED4CDC1445A4F6FB2C7E3410A2A49503A87F0418FA5E26B862DF9C180D8E03003EFD0C11C1CFEE2FB07E141574442DFC799FCC12C1567370876F6C19CF58ADB4765F74196C86E844664F6410106000080040000000103000080010000000400000014BD7F6E2319FCC1BC596B2C48A1EAC1F87986C6A33DFDC10E0D2030ED02E6C1A48D01164F21E641B0038FCE7A52ECC1B4BF0629559BF5C1FC043C45524CE6417003F2383E7DFC4114BD7F6E2319FCC1BC596B2C48A1EAC1F87986C6A33DFDC101030000800100000006000000B01CD427CCFBD041B4E4249E8D79E141782F03A737D3F2C14024D776B7DAD941005759563E41B041AC7A918A65BAF14180D3D8C5C746FC41B4E65E193CDFD9C1A6D4DA31C8B4F14134716A29E16CF44130E6D8E4A01EFD4190C5BA407196F041B544D1ABF8EBFCC1C86049E6E5B9F1419717CAD6F35AF0C1B01CD427CCFBD041B4E4249E8D79E141782F03A737D3F2C101030000800100000008000000F4B7F9647C9BF5C1087462C64C32F8C104DFF2B07283FD414C49969624DDF4C1C2BCB278524901C2D27ACDA9E205F541E0CD0DD7A8C6E34190A108F6C7B6C9C17DC45F6C582CFAC1C2917F6652C0F94124806139FDF2E8411097013BEB5FF7C1004D17376706AAC17C3635EB049BF641E0F45C2D2825C4C18CA60ACB4619E4C11872DF81ADB4E94146EA1F9E894BF4C1F9490070D282F6C10827EDF9979CD941960A218FD75DF341F4B7F9647C9BF5C1087462C64C32F8C104DFF2B07283FD410103000080010000000B00000002FCB029BA4DE5C1FE7AC8993054F7C10EA45EA1164B0142C07071F859ADDC41B0381DCEF3DDFFC1B03F313B7E5CC74110C3ADB20C77EB418CDC7956969AFBC1702859BDDF8AE441622EAB5557F5F5413766AE8002D4F5C1EB39D6DECFD8FFC1D0C6904BEDB9FD41F8C2E23D915BD4C1AC780DE57EBDE441BCCC1C8E23F0FE4100FF0F5FF582B241A0427E642E63D141767829D63E1FFA4160EF3ADEECCEE541707661291030FCC1EC799AC22F80EA419A1B7D52F50D024280A668C36875D741685282DE839DD341F23C41BD4A99F341D2E261C5F09EF1C14A43BAAD421FEFC118A729924D21FE41DC61EE9C01EEFD4102FCB029BA4DE5C1FE7AC8993054F7C10EA45EA1164B01420106000080050000000103000080010000000D00000000D0B88AC9528DC1E830015145ABE4C13C574BD64438FCC12AD8979FD51A02C279E1939292DA01C2725C0B7FBFE8FEC1F0B05C24C65DD7411C093792A37CE4C198EC3A23E11CF6414C31F9582BABEA41B8047B2C7E5CD7C1CA0675489D12EAC1108E7E52A9A8FA418E3C5F7C0471EAC18AE964AFC5D0EAC10496F51894D1F641B8BD0381A758E1C1E054A66401940242C2D3479AE353F141008A602F73E0D4C1381B62ED9274FCC1103555E334140142DC5AEE8AD170ED411E62FA09094B00C226CBD9EF6C6B0242849069D773A1F1417466D254641AE8414CE48854BBC1F14138FF150A48E0FA4138FA0720DB99DA41E0F6C228F7B1E24100182AE19E0EA9C1364BA3AAE33AF54196BA71CF8D1CF0C12061238A8B17C1C1F29E0FA75B0BF24100D0B88AC9528DC1E830015145ABE4C13C574BD64438FCC10103000080010000000600000028636F02E30DF6C11105F8380C03F5C1D24695D784340142CBF828C729C3F6C17D3A7106CB3A02C28C59DEC1CA6CEAC10CFC86416B750242B7FCCBBA7AA0FCC1083D04DDFA2CF0C1A80C0BB52D6501425049A6C52793FE41E0AA58C779A5C6C16CF11BAA78C5EF419C909D4B50E30042004BD21E3B60AF4128636F02E30DF6C11105F8380C03F5C1D24695D7843401420103000080010000000B00000050028B75DD84C8C19D1A217E580DF2C1AD5AC4EBC1B4F2C1AC6DE9C6C491EC41DA72D35D69B001C2A026478588B3C441B08D31FB3DC0D641C7BC14774F0DF2C1FC4752A9B1A3E9C1EC6259DD74F3F3412ACF14CAED9DFCC1A0DD67BD476201C214996EB815E1F74116FE0ED46B0AF741A0CA3DB84943C041F64C60C55E88FDC1CAB97444D8530142942E7A0E09A0F2C1AA4B2AF4B4B7FAC1A06BCB0ABE0FFB4114859F1B59B900429E04BD5EC69DEEC1840ACF392CD2E741C28262378E88E8C18C602D021F1D01C25C9BEFD5F1EEF24126ED5186967BECC167CA9FEE14C9F7C1D0A76FEDE9F8D44126665E12C6E700C250028B75DD84C8C19D1A217E580DF2C1AD5AC4EBC1B4F2C10103000080010000000A000000CE57CA27BCB5FBC18666201FD073E6C10AEED32C0866FCC1639D399FCAC7F0C1BFFA922FE7A301C21D9598355D7CF8C18867A2C94C97E0410C206A866583FFC180349218A7C0AEC100C0E4A8CB856A41501D27C08BC9EF41B455BD694E5F00C27401B822F7F0D6C1D83339F9CA7AF54178259AE2359DFDC1E4024B428585E4C1F6F1BA1D400FF1417782769529A4F6C1A47969358ACCEBC1788E2F29EC43DD4178A9E3DA2F54D841C32B6EA36077F2C10061A9814BCBF2418AF8A4FDFCAAF24141B134CE0FDF00C290EB8533235CF441D07663A38D1AD5C1CE57CA27BCB5FBC18666201FD073E6C10AEED32C0866FCC10103000080010000000400000098249D485414E4415C05B0151E9CF8C1285A1CFBCD86D041B05A2AB0BCA7EF41E4C528447779D6C144798DA980B5FC413825FD820B6CDD41CC1C68E87292F241D8C0AD2364C2D34198249D485414E4415C05B0151E9CF8C1285A1CFBCD86D04101060000800100000001030000800100000009000000C6C03ADC1DFBE6C1DEC269AAFF78FAC1A053C78030D4B141881E33408855E3C1CEE65863208AE7C10039682B47BAC2C10694B6EE5C44F94137E3A81B90FA01C2AE898C981A520142ECA4F5A4AA29F7419F0DECDFCA57F0C13E0F6654803FEFC1FAA39B45CA74F4C1B86BFFB6C9CBE841500A843E4426E541D20D57E1B90502C2A8A8F39C6ED8EE411E6B1B57DB11F5C120B6A9F7D75BF4C1B00585A0271FD9C1FC77B559EB6FF541B2D381A8D2FAFEC1901653D99039DAC1401602143349B641C6C03ADC1DFBE6C1DEC269AAFF78FAC1A053C78030D4B141010200008005000000202DC8795414F2C1EAE30DF59205E9C1B62B3714D1B9F641C8F8D462CFC5EA41DAEB07FBD58901C2C46956FD6F96FE416EDA71FF1EDCF141F21B44031E1B00C2F2622B77410DEBC1462A3119802DF9413C517FA5F11CF7C15EBACEC02F89F041B0DBF80A5450FF41EE0C28F5A75CF7C1E0843E1A200AE541':::GEOMETRY) :1]
    rsg_test.go:827: 4m45s of 5m0s: 562275 executions, 349799 successful
    rsg_test.go:827: 4m50s of 5m0s: 572877 executions, 356384 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #106970 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95617 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
- #95616 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10983948?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/10983948?buildTab=artifacts#/) on release-23.1 @ [d62a2f3ab1dff86bd80cac037b215121537a2a9a](https://github.com/cockroachdb/cockroach/commits/d62a2f3ab1dff86bd80cac037b215121537a2a9a):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 5885154 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
        github.com/cockroachdb/cockroach/pkg/server.startServeSQL.func1.1({0x7478e60?, 0xc004027aa0?}, {0x74c4210, 0xc03074ee00})
        	github.com/cockroachdb/cockroach/pkg/server/server_sql.go:1766 +0x24e
        github.com/cockroachdb/cockroach/pkg/util/netutil.(*TCPServer).ServeWith.func1({0x7478e60, 0xc004027aa0})
        	github.com/cockroachdb/cockroach/pkg/util/netutil/net.go:188 +0xc7
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func2()
        	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:470 +0x146
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx
        	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:461 +0x43b
        
    rsg_test.go:252: SELECT 1 executed successfully: probably a slow statement
    rsg_test.go:854: Crash detected: server panic: statement exec timeout
        SELECT crdb_internal.reset_activity_tables() ;
        
        Stack trace:
        timeout: "SELECT crdb_internal.reset_activity_tables() ". currently executing: map[SELECT crdb_internal.gen_rand_ident(e'\'':::STRING, 4603276908638047356:::INT8) :1 SELECT crdb_internal.gen_rand_ident(e'\x15h[':::STRING, 175505127776447869:::INT8, 'null':::JSONB) :1 SELECT crdb_internal.gen_rand_ident(e'o\x1ax?':::STRING, 7799035776469511505:::INT8) :1 SELECT crdb_internal.reset_activity_tables() :2 SELECT crdb_internal.show_create_all_tables('':::STRING) :1 SELECT has_function_privilege('NR|':::STRING, 2682103681:::OID, e'z\n\x01<Is\x1c\x0f':::STRING) :1 SELECT has_function_privilege(e'^FS?S;\x11\b':::STRING, e'j#B\x02':::STRING, e'2 \\p':::STRING) :1]
    rsg_test.go:827: 2m0s of 5m0s: 238999 executions, 148681 successful
    rsg_test.go:827: 2m5s of 5m0s: 246891 executions, 153620 successful
    rsg_test.go:827: 2m10s of 5m0s: 260550 executions, 162099 successful
    rsg_test.go:827: 2m15s of 5m0s: 275035 executions, 171091 successful
    rsg_test.go:827: 2m20s of 5m0s: 285224 executions, 177483 successful
    rsg_test.go:827: 2m25s of 5m0s: 293753 executions, 182851 successful
    rsg_test.go:827: 2m30s of 5m0s: 301454 executions, 187659 successful
    rsg_test.go:827: 2m35s of 5m0s: 313369 executions, 195027 successful
    rsg_test.go:827: 2m40s of 5m0s: 324630 executions, 202118 successful
    rsg_test.go:827: 2m45s of 5m0s: 336167 executions, 209257 successful
    rsg_test.go:827: 2m50s of 5m0s: 348253 executions, 216753 successful
    rsg_test.go:827: 2m55s of 5m0s: 360363 executions, 224283 successful
    rsg_test.go:827: 3m0s of 5m0s: 370977 executions, 230894 successful
    rsg_test.go:827: 3m5s of 5m0s: 379854 executions, 236539 successful
    rsg_test.go:827: 3m10s of 5m0s: 391337 executions, 243682 successful
    rsg_test.go:827: 3m15s of 5m0s: 403493 executions, 251147 successful
    rsg_test.go:827: 3m20s of 5m0s: 417398 executions, 259832 successful
    rsg_test.go:827: 3m25s of 5m0s: 430006 executions, 267700 successful
    rsg_test.go:827: 3m30s of 5m0s: 442697 executions, 275480 successful
    rsg_test.go:827: 3m35s of 5m0s: 453397 executions, 282104 successful
    rsg_test.go:827: 3m40s of 5m0s: 464630 executions, 289184 successful
    rsg_test.go:827: 3m45s of 5m0s: 472140 executions, 293860 successful
    rsg_test.go:827: 3m50s of 5m0s: 483634 executions, 300937 successful
    rsg_test.go:827: 3m55s of 5m0s: 494744 executions, 307788 successful
    rsg_test.go:827: 4m0s of 5m0s: 509187 executions, 316725 successful
    rsg_test.go:827: 4m5s of 5m0s: 522566 executions, 325031 successful
    rsg_test.go:827: 4m10s of 5m0s: 533463 executions, 331924 successful
    rsg_test.go:827: 4m15s of 5m0s: 544398 executions, 338636 successful
    rsg_test.go:827: 4m20s of 5m0s: 554967 executions, 345164 successful
    rsg_test.go:827: 4m25s of 5m0s: 566628 executions, 352468 successful
    rsg_test.go:827: 4m30s of 5m0s: 576147 executions, 358398 successful
    rsg_test.go:827: 4m35s of 5m0s: 584455 executions, 363542 successful
    rsg_test.go:827: 4m40s of 5m0s: 594042 executions, 369540 successful
    rsg_test.go:827: 4m45s of 5m0s: 601144 executions, 373949 successful
    rsg_test.go:827: 4m50s of 5m0s: 608364 executions, 378525 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/11002190?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/11002190?buildTab=artifacts#/) on release-23.1 @ [043024b889af86e0b8f23f09619d8d3c2c9acd13](https://github.com/cockroachdb/cockroach/commits/043024b889af86e0b8f23f09619d8d3c2c9acd13):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 5717808 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
        	GOROOT/src/bufio/bufio.go:237 +0x1bb
        io.ReadAtLeast({0x742aa60, 0xc01c28cae0}, {0xc0045e4920, 0x5, 0x200}, 0x5)
        	GOROOT/src/io/io.go:332 +0x9a
        io.ReadFull(...)
        	GOROOT/src/io/io.go:351
        github.com/lib/pq.(*conn).recvMessage(0xc0045e4900, 0xc008209958)
        	github.com/lib/pq/external/com_github_lib_pq/conn.go:1004 +0xca
        github.com/lib/pq.(*conn).recv1Buf(0xc0045e4900, 0xc0128b2cc0?)
        	github.com/lib/pq/external/com_github_lib_pq/conn.go:1059 +0x2e
        github.com/lib/pq.(*conn).recv1(...)
        	github.com/lib/pq/external/com_github_lib_pq/conn.go:1086
        github.com/lib/pq.(*conn).simpleExec(0xc0045e4900, {0xc0075f84d0?, 0x6e})
        	github.com/lib/pq/external/com_github_lib_pq/conn.go:664 +0x1ff
        github.com/lib/pq.(*conn).Exec(0xc0045e4900, {0xc0075f84d0, 0x6e}, {0xacbb438, 0x0, 0x0})
        	github.com/lib/pq/external/com_github_lib_pq/conn.go:921 +0x229
        github.com/lib/pq.(*conn).ExecContext(0x0?, {0x7479978, 0xc02861e300}, {0xc0075f84d0, 0x6e}, {0xacbb438, 0x0, 0x469ce5?})
        	github.com/lib/pq/external/com_github_lib_pq/conn_go18.go:46 +0x109
        database/sql.ctxDriverExec({0x7479978?, 0xc02861e300?}, {0x7f64567e5618?, 0xc0045e4900?}, {0x0?, 0x0?}, {0xc0075f84d0?, 0x0?}, {0xacbb438, 0x0, ...})
        	GOROOT/src/database/sql/ctxutil.go:31 +0xe5
        database/sql.(*DB).execDC.func2()
        	GOROOT/src/database/sql/sql.go:1679 +0x175
        database/sql.withLock({0x7449de0, 0xc02c6d79e0}, 0xc005cfddd8)
        	GOROOT/src/database/sql/sql.go:3439 +0x8c
        database/sql.(*DB).execDC(0x74ad101?, {0x7479978, 0xc02861e300}, 0xc02c6d79e0, 0x1773cac52ab0cdb3?, {0xc0075f84d0, 0x6e}, {0x0, 0x0, 0x0})
        	GOROOT/src/database/sql/sql.go:1674 +0x266
        database/sql.(*DB).exec(0xc000180000?, {0x7479978, 0xc02861e300}, {0xc0075f84d0, 0x6e}, {0x0, 0x0, 0x0}, 0xf8?)
        	GOROOT/src/database/sql/sql.go:1659 +0xe8
        database/sql.(*DB).ExecContext(0x744b758?, {0x7479978, 0xc02861e300}, {0xc0075f84d0, 0x6e}, {0x0, 0x0, 0x0})
        	GOROOT/src/database/sql/sql.go:1633 +0xe5
        github.com/cockroachdb/cockroach/pkg/sql/tests_test.(*verifyFormatDB).execWithResettableTimeout.func2()
        	github.com/cockroachdb/cockroach/pkg/sql/tests_test/pkg/sql/tests/rsg_test.go:173 +0x48
        created by github.com/cockroachdb/cockroach/pkg/sql/tests_test.(*verifyFormatDB).execWithResettableTimeout
        	github.com/cockroachdb/cockroach/pkg/sql/tests_test/pkg/sql/tests/rsg_test.go:172 +0x19d
        
    rsg_test.go:252: SELECT 1 executed successfully: probably a slow statement
    rsg_test.go:854: Crash detected: server panic: statement exec timeout
        SELECT crdb_internal.reset_activity_tables() ;
        
        Stack trace:
        timeout: "SELECT crdb_internal.reset_activity_tables() ". currently executing: map[SELECT box2d_send('BOX(0.761450393963791 -0.3319501372923972,1.3596064213509762 0.8062944585759774)':::BOX2D) :1 SELECT bpcharin(ARRAY[(-31039):::INT8,6529:::INT8]) :1 SELECT bpcharout('X':::STRING) :1 SELECT crdb_internal.gen_rand_ident('Q':::STRING, 8108908307947576502:::INT8) :1 SELECT crdb_internal.gen_rand_ident(e'\f':::STRING, 5159733409929104524:::INT8) :1 SELECT crdb_internal.gen_rand_ident(e'\x19':::STRING, 8396600530797976056:::INT8) :1 SELECT crdb_internal.reset_activity_tables() :1 SELECT crdb_internal.show_create_all_tables('':::STRING) :1]
    rsg_test.go:827: 4m5s of 5m0s: 519556 executions, 323020 successful
    rsg_test.go:827: 4m10s of 5m0s: 531711 executions, 330630 successful
    rsg_test.go:827: 4m15s of 5m0s: 543208 executions, 337776 successful
    rsg_test.go:827: 4m20s of 5m0s: 551470 executions, 342844 successful
    rsg_test.go:827: 4m25s of 5m0s: 562411 executions, 349681 successful
    rsg_test.go:827: 4m30s of 5m0s: 574499 executions, 357157 successful
    rsg_test.go:827: 4m35s of 5m0s: 585879 executions, 364232 successful
    rsg_test.go:827: 4m40s of 5m0s: 594721 executions, 369800 successful
    rsg_test.go:827: 4m45s of 5m0s: 605783 executions, 376590 successful
    rsg_test.go:827: 4m50s of 5m0s: 615366 executions, 382530 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #107237 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master]
- #95619 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/sql/sem/builtins/generator_builtins.go
+++ b/pkg/sql/sem/builtins/generator_builtins.go
@@ -3314,7 +3314,7 @@ func makeSpanStatsGenerator(
 	spans := make([]roachpb.Span, 0, argSpans.Len())
 	for _, span := range argSpans.Array {
 		s := tree.MustBeDTuple(span)
-		if s.D[0] == tree.DNull || s.D[1] == tree.DNull {
+		if len(s.D) != 2 || s.D[0] == tree.DNull || s.D[1] == tree.DNull {
 			continue
 		}
 		startKey := roachpb.Key(tree.MustBeDBytes(s.D[0]))

--- a/pkg/sql/tests/rsg_test.go
+++ b/pkg/sql/tests/rsg_test.go
@@ -352,7 +352,8 @@ func TestRandomSyntaxFunctions(t *testing.T) {
 					continue
 				case "crdb_internal.reset_sql_stats",
 					"crdb_internal.check_consistency",
-					"crdb_internal.request_statement_bundle":
+					"crdb_internal.request_statement_bundle",
+					"crdb_internal.reset_activity_tables":
 					// Skipped due to long execution time.
 					continue
 				}
@@ -414,7 +415,11 @@ func TestRandomSyntaxFunctions(t *testing.T) {
 			limit = " LIMIT 100"
 		}
 		s := fmt.Sprintf("SELECT %s(%s) %s", nb.name, strings.Join(args, ", "), limit)
-		return db.exec(t, ctx, s)
+		// Use a re-settable timeout since in concurrent scenario some operations may
+		// involve schema changes like truncates. In general this should make
+		// this test more resilient as the timeouts are reset as long progress
+		// is made on *some* connection.
+		return db.execWithResettableTimeout(t, ctx, s, *flagRSGExecTimeout, *flagRSGGoRoutines)
 	})
 }
 

